---
title: "Summary of available functionality in TCGAutils"
author:
  - name: Marcel Ramos
    affiliation: Roswell Park Comprehensive Cancer Center, Buffalo, NY
vignette: >
  %\VignetteIndexEntry{TCGAutils Cheatsheet}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
output:
  html_document:
    mathjax: null
---

```{r,echo=FALSE}
library(tibble)
rawtab <- tibble(
Section = c("MultiAssayExperiment helpers", NA, NA, NA, NA, NA, NA, NA,
"curatedTCGAData helpers", NA, NA, NA, NA, NA, NA, "TCGA Identifiers", NA, NA,
NA, NA, NA, NA, NA, "Flat to Bioconductor classes", NA, NA, NA, "Genome Builds",
NA, NA, NA, "Reference data", NA, NA, NA, NA, "Miscellaneous", NA),

`Category and Function` = c(NA, "generateMap", "imputeAssay", "mergeColData",
"qreduceTCGA", "symbolsToRanges", "mirToRanges", "simplifyTCGA", NA,
"getSubtypeMap", "getClinicalNames", "splitAssays", "sampleTables",
"TCGAprimaryTumors", "oncoPrintTCGA", NA, "TCGAbarcode", "TCGAbiospec",
"TCGAsampleSelect", "UUIDtoBarcode", "UUIDtoUUID", "barcodeToUUID",
"filenameToBarcode", NA, "makeGRangesListFromCopyNumber",
"makeGRangesListFromExonFiles", "makeSummarizedExperimentFromGISTIC", NA,
"translateBuild", "extractBuild", "uniformBuilds", NA, "diseaseCodes",
"sampleTypes", "clinicalNames", "getFilename", NA, "findGRangesCols"),

Description = c(NA, "Automatically generate a sampleMap structure from assays",
"Impute values in numerical assays based on KNN",
"Add additional data to the colData slot of MAE",
"Convert RaggedExperiment assays to RSE based on heuristics",
"Convert gene symbols to genomic ranges using org.db",
"Convert microRNA sequences to genomic ranges with mirbase.db",
"Use qreduceTCGA, symbolsToRanges, and mirToRanges in succession", NA,
"Obtain the available subtype information",
"Get a list of clinical variable names for all cancer types",
"Separate assays based on sample data found in barcodes",
"Get a list of samples in each assay in a MAE",
"Select only primary tumors from TCGA data barcodes",
"Produce an oncoPrint plot from TCGA mutation data",
NA,
"Chop TCGA barcode into sections",
"Get a table of information extracted from a vector of barcodes",
"Indicate which barcodes belong to a specific sample type",
"Translate universal identifiers to TCGA barcodes",
"Translate between case and file universal identifiers",
"Translate TCGA barcodes to universal identifiers",
"Obtain TCGA barcodes from a vector of TCGA file names",
NA,
"Create a GRangesList from a copy number data.frame",
"Obtain a GRangesList object from a list of in dividual exon files",
"Create a SummarizedExperiment object from a Firehose GISTIC RTCGAToolbox object",
NA,
"Translate build version name between UCSC and NCBI",
"Find build in string pattern such as a file name",
"Homogenize a vector of builds based on a threshold for the alternative build name",
NA,
"Get a table of TCGA cancer codes and subtype availability",
"Get a table of sample type codes and their definition",
"Obtain a CharacterList of common variable names for each TCGA disease code",
"Obtain a file name string for the relevant data query",
NA,
"Find the minimum necessary variable names for conversion to GRanges")
)
```

```{r,echo=FALSE,message=FALSE,warn=FALSE}
library(knitr)
library(kableExtra)

tabInfo <- rawtab[is.na(rawtab[[1]]), 2:3]
names(tabInfo) <- c("Category and Function", "Description")

groups <- unname(unlist(rawtab[!is.na(rawtab[[1]]), 1L]))

starts <- which(!is.na(rawtab[-nrow(rawtab), 1L]))+1
ends <- c(starts[-1]-1, nrow(rawtab)+1)
iwidth <- ends - starts
```

```{r,echo=FALSE}
knitr::kable(tabInfo, align = "l", escape = FALSE, format = "html") %>%
    kableExtra::kable_styling(
        bootstrap_options = c("hover", "striped", "responsive"),
        full_width = FALSE
    ) %>%
    kableExtra::group_rows(index = setNames(iwidth, groups))
```
