<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>cBioPortalData Studies • MultiAssayWorkshop</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="cBioPortalData Studies">
<meta property="og:description" content="MultiAssayWorkshop">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MultiAssayWorkshop</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">3.11.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/Ramos_MultiAssayExperiment.html">Workshop</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorial
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../tutorials/Exercise_MultiAssay.html">Build Your First MAE</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Reference
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/cBioPortal_Studies.html">cBioPortal Studies</a>
    </li>
    <li>
      <a href="../articles/curatedTCGAData_ref.html">curatedTCGAData Reference</a>
    </li>
    <li>
      <a href="../articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MultiAssayWorkshop">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="cBioPortal_Studies_files/accessible-code-block-0.0.1/empty-anchor.js"></script><script src="cBioPortal_Studies_files/kePrint-0.0.1/kePrint.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>cBioPortalData Studies</h1>
                        <h4 class="author">Marcel Ramos</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><small class="dont-index">Source: <a href="https://github.com/waldronlab/MultiAssayWorkshop/blob/master/vignettes/cBioPortal_Studies.Rmd"><code>vignettes/cBioPortal_Studies.Rmd</code></a></small>
      <div class="hidden name"><code>cBioPortal_Studies.Rmd</code></div>

    </address>
</div>

    
    
<div id="available-studies-by-function" class="section level1">
<h1 class="hasAnchor">
<a href="#available-studies-by-function" class="anchor"></a>Available Studies by function</h1>
<div id="cbioportaldata" class="section level2">
<h2 class="hasAnchor">
<a href="#cbioportaldata" class="anchor"></a>cBioPortalData()</h2>
<p>These studies are available through the cBioPortal API, i.e., <code>cBioPortalData</code> function.</p>
<div id="notes" class="section level3">
<h3 class="hasAnchor">
<a href="#notes" class="anchor"></a>NOTES</h3>
<ul>
<li><p>This list is different from those studies available via the <code>cBioDataPack</code> function.</p></li>
<li><p>Scroll to the right for more information (esp. for the <code>studyId</code> column)</p></li>
</ul>
<pre><code>## Warning: package 'knitr' was built under R version 4.0.2</code></pre>
<table class="table table table-hover table-striped table-responsive">
<caption>
List of available studies from cBioPortalData API
</caption>
<thead><tr>
<th style="text-align:left;">
name
</th>
<th style="text-align:left;">
shortName
</th>
<th style="text-align:left;">
description
</th>
<th style="text-align:left;">
publicStudy
</th>
<th style="text-align:left;">
pmid
</th>
<th style="text-align:left;">
citation
</th>
<th style="text-align:left;">
groups
</th>
<th style="text-align:left;">
status
</th>
<th style="text-align:left;">
importDate
</th>
<th style="text-align:left;">
allSampleCount
</th>
<th style="text-align:left;">
studyId
</th>
<th style="text-align:left;">
cancerTypeId
</th>
<th style="text-align:left;">
referenceGenome
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (National Cancer Centre of Singapore, Nat Genet 2013)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (NCCS)
</td>
<td style="text-align:left;">
Exome sequencing of 15 cholangiocarcinoma cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24185513
</td>
<td style="text-align:left;">
Chan-on et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-14 00:00:00
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
chol_nccs_2013
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Squamous Cell Carcinoma (UCLA, Nat Genet 2014)
</td>
<td style="text-align:left;">
ESCC (UCLA 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing (WXS) or targeted deep sequencing (TDS) of 139 paired esophageal squamous cell carcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24686850
</td>
<td style="text-align:left;">
Lin et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
139
</td>
<td style="text-align:left;">
escc_ucla_2014
</td>
<td style="text-align:left;">
escc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Oral Squamous Cell Carcinoma (MD Anderson, Cancer Discov 2013)
</td>
<td style="text-align:left;">
Head &amp; neck (MDA)
</td>
<td style="text-align:left;">
Comprehensive profiling of 40 oral squamous cell carcinoma tumor/normal sample pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23619168
</td>
<td style="text-align:left;">
Pickering et al. Cancer Discov 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
40
</td>
<td style="text-align:left;">
hnsc_mdanderson_2013
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hepatocellular Carcinomas (INSERM, Nat Genet 2015)
</td>
<td style="text-align:left;">
HCC (Inserm, 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 243 liver tumors and matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25822088
</td>
<td style="text-align:left;">
Schulze et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
243
</td>
<td style="text-align:left;">
hcc_inserm_fr_2015
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uveal Melanoma (QIMR, Oncotarget 2016)
</td>
<td style="text-align:left;">
UM (QIMR)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of tumor/normal pairs from 28 uveal melanoma samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26683228
</td>
<td style="text-align:left;">
Johansson et al. Oncotarget 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
28
</td>
<td style="text-align:left;">
um_qimr_2016
</td>
<td style="text-align:left;">
um
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroblastoma (AMC Amsterdam, Nature 2012)
</td>
<td style="text-align:left;">
NBL (AMC)
</td>
<td style="text-align:left;">
Whole genome sequencing of 87 primary neuroblastoma tumors and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22367537
</td>
<td style="text-align:left;">
Molenaar et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
87
</td>
<td style="text-align:left;">
nbl_amc_2012
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Nasopharyngeal Carcinoma (Singapore, Nat Genet 2014)
</td>
<td style="text-align:left;">
NPC (Singapore)
</td>
<td style="text-align:left;">
Whole exome sequencing of 56 NPC tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24952746
</td>
<td style="text-align:left;">
Lin et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
56
</td>
<td style="text-align:left;">
npc_nusingapore
</td>
<td style="text-align:left;">
npc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thymic Epithelial Tumors (NCI, Nat Genet 2014)
</td>
<td style="text-align:left;">
TET (NCI)
</td>
<td style="text-align:left;">
Whole exome sequencing of 28 thymic epithelial tumors (TETs) with matched normals
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24974848
</td>
<td style="text-align:left;">
Petrini at el. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
tet_nci_2014
</td>
<td style="text-align:left;">
tet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroblastoma (Broad, Nature 2015)
</td>
<td style="text-align:left;">
NBL (Cologne 2015)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 56 neuroblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26466568
</td>
<td style="text-align:left;">
Peifer et al. Nature 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
56
</td>
<td style="text-align:left;">
nbl_ucologne_2015
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Myelodysplasia (UTokyo, Nature 2011)
</td>
<td style="text-align:left;">
MDS (Tokyo)
</td>
<td style="text-align:left;">
Whole exome sequencing of 29 myelodysplasia tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21909114
</td>
<td style="text-align:left;">
Yoshida et al. Nature 2011
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
mds_tokyo_2011
</td>
<td style="text-align:left;">
mds
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Hodgkin Lymphoma (BCGSC, Nature 2011)
</td>
<td style="text-align:left;">
NHL (bcgsc 2011)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing of 14 NHL tumors and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21796119
</td>
<td style="text-align:left;">
Morin et al. Nature 2011
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
nhl_bcgsc_2011
</td>
<td style="text-align:left;">
nhl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-cell Lymphoma (BCGSC, Blood 2013)
</td>
<td style="text-align:left;">
NHL (BCGSC 2013)
</td>
<td style="text-align:left;">
Whole genome sequencing of 40 DLBCL tumor/normal pairs and 13 cell lines.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23699601
</td>
<td style="text-align:left;">
Morin et al. Blood 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
53
</td>
<td style="text-align:left;">
nhl_bcgsc_2013
</td>
<td style="text-align:left;">
nhl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Insulinoma (Shanghai, Nat Commun 2013)
</td>
<td style="text-align:left;">
Panet (Shanghai 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 10 benign insulinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24326773
</td>
<td style="text-align:left;">
Cao et al. Nat Commun 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-21 00:00:00
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
panet_shanghai_2013
</td>
<td style="text-align:left;">
panet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pleural Mesothelioma (NYU, Cancer Res 2015)
</td>
<td style="text-align:left;">
PLMESO (NYU 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 22 malignant pleural mesotheliomas (PLMESO) tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25488749
</td>
<td style="text-align:left;">
Guo et al. Cancer Res 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-21 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
plmeso_nyu_2015
</td>
<td style="text-align:left;">
plmeso
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pilocytic Astrocytoma (ICGC, Nature Genetics 2013)
</td>
<td style="text-align:left;">
PAST (Nature Genetics, 2013)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 96 pilocytic astrocytoma tumor/normal pairs. Conducted by the International Cancer Genome Consortium (ICGC) PedBrain Tumor Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23817572
</td>
<td style="text-align:left;">
Jones et al. Nature Genetics 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-21 00:00:00
</td>
<td style="text-align:left;">
96
</td>
<td style="text-align:left;">
past_dkfz_heidelberg_2013
</td>
<td style="text-align:left;">
past
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (RIKEN, Nat Genet 2012)
</td>
<td style="text-align:left;">
Liver (RIKEN)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 27 hepatocellular carcinoma tumor/norma pairs from RIKEN.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22634756
</td>
<td style="text-align:left;">
Fujimoto et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-22 00:00:00
</td>
<td style="text-align:left;">
27
</td>
<td style="text-align:left;">
lihc_riken
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (ICGC, Nature 2012)
</td>
<td style="text-align:left;">
MBL (ICGC)
</td>
<td style="text-align:left;">
Comprehensive profiling of 125 matched tumor and germline medulloblastoma samples
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22832583
</td>
<td style="text-align:left;">
Jones et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-22 00:00:00
</td>
<td style="text-align:left;">
125
</td>
<td style="text-align:left;">
mbl_icgc
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (Sickkids, Nature 2016)
</td>
<td style="text-align:left;">
MBL (Sickkids 2016)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 46 recurrent medulloblastoma samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26760213
</td>
<td style="text-align:left;">
Morrissy et al. Nature 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-22 00:00:00
</td>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
mbl_sickkids_2016
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (BGI, Nat Genet 2012)
</td>
<td style="text-align:left;">
ccRCC (BGI 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 10 ccRCC tumor/normal pairs and targeted sequencing of 88 clear cell carcinoma tumor/normal pairs. Generated by BGI-Shenzhen.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22138691
</td>
<td style="text-align:left;">
Guo et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-24 00:00:00
</td>
<td style="text-align:left;">
98
</td>
<td style="text-align:left;">
kirc_bgi
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (Johns Hopkins, Science 2011)
</td>
<td style="text-align:left;">
Head &amp; neck (JHU)
</td>
<td style="text-align:left;">
Exome sequencing of 32 head and neck squamous cell carcinoma tumor/normal sample pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21798897
</td>
<td style="text-align:left;">
Agrawal et al. Science 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-07 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
hnsc_jhu
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hepatocellular Adenoma (INSERM, Cancer Cell 2014)
</td>
<td style="text-align:left;">
Liad (Inserm 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 46 liver tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24735922
</td>
<td style="text-align:left;">
Pilati et al. Cancer Cell 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-07 00:00:00
</td>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
liad_inserm_fr_2014
</td>
<td style="text-align:left;">
liad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ampullary Carcinoma (Baylor College of Medicine, Cell Reports 2016)
</td>
<td style="text-align:left;">
Ampullary Carcinoma (BCM 2016)
</td>
<td style="text-align:left;">
Exome sequencing analysis of 160 cancers arising in the periampullary region, 62 of these clearly arising from either the bile duct (n = 44) or the duodenum (n = 18) and 98 for which the epithelium of origin could not be clearly defined morphologically (AMPCA).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26804919
</td>
<td style="text-align:left;">
Gingras et al. Cell Rep 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
160
</td>
<td style="text-align:left;">
ampca_bcm_2016
</td>
<td style="text-align:left;">
ampca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (DFCI/MSKCC, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Bladder (DFCI/MSKCC 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal samples from 50 patients treated with cisplatin-based chemotherapy for muscle invasive urothelial carcinoma.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25096233
</td>
<td style="text-align:left;">
Van Allen et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
50
</td>
<td style="text-align:left;">
blca_dfarber_mskcc_2014
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Renal Non-Clear Cell Carcinoma (Genentech, Nat Genet 2014)
</td>
<td style="text-align:left;">
nccRCC (Genentech 2014)
</td>
<td style="text-align:left;">
Exome sequencing of 139 samples with matched normals and RNA-sequencing of 7 samples (5 samples with matched normals) from 145 Renal Non-Clear Cell Carcinoma (nccRCC) patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25401301
</td>
<td style="text-align:left;">
Durinck et al. Nat Genet 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
146
</td>
<td style="text-align:left;">
nccrcc_genentech_2014
</td>
<td style="text-align:left;">
nccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (Pfizer and UHK, Nat Genet 2014)
</td>
<td style="text-align:left;">
Stomach (Pfizer UHK)
</td>
<td style="text-align:left;">
Whole genome sequencing of 100 gastric cancer tumor-normal pairs from the University of Hong Kong and Pfizer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24816253
</td>
<td style="text-align:left;">
Wang et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
100
</td>
<td style="text-align:left;">
stad_pfizer_uhongkong
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (Broad, PNAS 2012)
</td>
<td style="text-align:left;">
DLBCL (Broad 2012)
</td>
<td style="text-align:left;">
Whole exome sequencing of 58 primary DLBCL samples and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22343534
</td>
<td style="text-align:left;">
Lohr et al. PNAS 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
58
</td>
<td style="text-align:left;">
dlbc_broad_2012
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
MSS Mixed Solid Tumors (Broad/Dana-Farber, Nat Genet 2018)
</td>
<td style="text-align:left;">
Mixed Tumor (DFCI)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal pairs and corresponding clinical outcomes from patients treated with immune checkpoint inhibitors across multiple cancer types including melanoma, non-small cell lung cancer, head and neck cancer, and bladder cancer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30150660
</td>
<td style="text-align:left;">
Miao et al. Nat Genet 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
249
</td>
<td style="text-align:left;">
mixed_allen_2018
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Renal Clear Cell Carcinoma (UTokyo, Nat Genet 2013)
</td>
<td style="text-align:left;">
ccRCC (UTokyo)
</td>
<td style="text-align:left;">
Whole exome and/or whole genome sequencing of 106 clear cell renal cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23797736
</td>
<td style="text-align:left;">
Sato et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
106
</td>
<td style="text-align:left;">
ccrcc_utokyo_2013
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Intrahepatic Cholangiocarcinoma (JHU, Nat Genet 2013)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (JHU)
</td>
<td style="text-align:left;">
Exome sequencing of tumor/normal pairs from 32 intrahepatic cholangiocarcinoma cases and 8 gall bladder carcinoma cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24185509
</td>
<td style="text-align:left;">
Jiao et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
40
</td>
<td style="text-align:left;">
chol_jhu_2013
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Melanoma (Broad/Dana Farber, Nature 2012)
</td>
<td style="text-align:left;">
Melanoma (Broad/DFCI)
</td>
<td style="text-align:left;">
Whole Genome Sequencing of 25 metastatic melanoma samples with matched normals
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22622578
</td>
<td style="text-align:left;">
Berger et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
26
</td>
<td style="text-align:left;">
skcm_broad_dfarber
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Carcinosarcoma (Johns Hopkins, Nat Commun 2014)
</td>
<td style="text-align:left;">
ucs (Johns Hopkins 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing (WXS) of 22 uterine carcinosarcoma/uterine malignant mixed Mullerian (UCS) tumors with matched normals. <a href="http://www.nature.com/ncomms/2014/140919/ncomms6006/extref/ncomms6006-s1.pdf"> </a>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25233892
</td>
<td style="text-align:left;">
Jones et al. Nat Commun 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
ucs_jhu_2014
</td>
<td style="text-align:left;">
ucs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma(Broad, Cancer Discov 2014)
</td>
<td style="text-align:left;">
SKCM (BROAD 2014)
</td>
<td style="text-align:left;">
Exome Sequencing of tumors (with matched normals) from 45 patients with BRAFV600-mutant metastatic melanoma who received vemurafenib or dabrafenib monotherapy.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24265153
</td>
<td style="text-align:left;">
Van Allen et al. Cancer Discov. 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
skcm_broad_brafresist_2012
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (Sanger/MDA, JCI 2013)
</td>
<td style="text-align:left;">
ACyC (Sanger 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 24 ACCs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23778141
</td>
<td style="text-align:left;">
Stephens et al. JCI 2013
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
24
</td>
<td style="text-align:left;">
acyc_sanger_2013
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Squamous Cell Carcinoma of the Vulva (CUK, Exp Mol Med 2018)
</td>
<td style="text-align:left;">
VSC (CUK)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal pairs from 15 vulvar squamous cell carcinomas.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29422544
</td>
<td style="text-align:left;">
Han et al. Exp Mol Med 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
vsc_cuk_2018
</td>
<td style="text-align:left;">
vsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acinar Cell Carcinoma of the Pancreas (JHU, J Pathol 2014)
</td>
<td style="text-align:left;">
PAAC (JHU)
</td>
<td style="text-align:left;">
Whole exome sequencing of 23 surgically resected pancreatic carcinomas with acinar differentiation and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24293293
</td>
<td style="text-align:left;">
Jial et al. J Pathol 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
23
</td>
<td style="text-align:left;">
paac_jhu_2014
</td>
<td style="text-align:left;">
paac
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Rhabdomyosarcoma (NIH, Cancer Discov 2014)
</td>
<td style="text-align:left;">
RMS (NIH)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 43 rhabdomyosarcoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24436047
</td>
<td style="text-align:left;">
Shern et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
43
</td>
<td style="text-align:left;">
rms_nih_2014
</td>
<td style="text-align:left;">
rms
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Gastric Adenocarcinoma (TMUCIH, PNAS 2015)
</td>
<td style="text-align:left;">
Stomach (TMUCIH 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 78 Gastric Adenocarcinomas and their matched normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25583476
</td>
<td style="text-align:left;">
Chen et al. PNAS 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
egc_tmucih_2015
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (British Columbia, Nature 2012)
</td>
<td style="text-align:left;">
Breast (BCCRC 2012)
</td>
<td style="text-align:left;">
Whole genome/exome sequencing analysis of 65 breast cancer samples
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22495314
</td>
<td style="text-align:left;">
Shah et al. Nature
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
65
</td>
<td style="text-align:left;">
brca_bccrc
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Neuroendocrine Tumors (Multi-Institute, Nature 2017)
</td>
<td style="text-align:left;">
ACRNET (Nature, 2017)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 98 pancreatic neuroendocrine tumours (PanNETs) and matched normal DNA pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28199314
</td>
<td style="text-align:left;">
Scarpa et al. Nature 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
98
</td>
<td style="text-align:left;">
panet_arcnet_2017
</td>
<td style="text-align:left;">
panet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Primary Central Nervous System Lymphoma (Mayo Clinic, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
PCNSL (Mayo Clinic)
</td>
<td style="text-align:left;">
Exome sequencing of 19 immunocompetent primary central nervous system lymphoma (PCNSL) samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25991819
</td>
<td style="text-align:left;">
Braggio et al. Clin Cancer Res 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
pcnsl_mayo_2015
</td>
<td style="text-align:left;">
pcnsl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Multiple Myeloma (Broad, Cancer Cell 2014)
</td>
<td style="text-align:left;">
MM (Broad)
</td>
<td style="text-align:left;">
Whole-exome or whole-genome sequencing of 203 multiple myeloma paired tumor/normal sample pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24434212
</td>
<td style="text-align:left;">
Lohr et al. Cancer Cell 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-27 00:00:00
</td>
<td style="text-align:left;">
211
</td>
<td style="text-align:left;">
mm_broad
</td>
<td style="text-align:left;">
pcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroblastoma (Broad, Nat Genet 2013)
</td>
<td style="text-align:left;">
NBL (Broad)
</td>
<td style="text-align:left;">
Whole Genome and Whole Exome sequencing of 240 high-risk neuroblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23334666
</td>
<td style="text-align:left;">
Pugh et al. Nat Genet 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-27 00:00:00
</td>
<td style="text-align:left;">
240
</td>
<td style="text-align:left;">
nbl_broad_2013
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Rhabdoid Cancer (BCGSC, Cancer Cell 2016)
</td>
<td style="text-align:left;">
MRT (CELL 2016)
</td>
<td style="text-align:left;">
Whole-genome sequencing of tumor-normal sample pairs from 40 patients with Malignant Rhabdoid Cancer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26977886
</td>
<td style="text-align:left;">
E. Chun et al. Cancer Cell 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-27 00:00:00
</td>
<td style="text-align:left;">
40
</td>
<td style="text-align:left;">
mrt_bcgsc_2016
</td>
<td style="text-align:left;">
mrt
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Gallbladder Carcinoma (Shanghai, Nat Genet 2014)
</td>
<td style="text-align:left;">
GBC (Shanghai)
</td>
<td style="text-align:left;">
Whole exome sequencing of 32 gallbladder carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24997986
</td>
<td style="text-align:left;">
Maolan Li et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
gbc_shanghai_2014
</td>
<td style="text-align:left;">
gbc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (MSKCC, Science 2015)
</td>
<td style="text-align:left;">
Lung adeno (MSKCC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 34 non small cell lung cancer tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25765070
</td>
<td style="text-align:left;">
Rizvi et al. Science 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
35
</td>
<td style="text-align:left;">
luad_mskcc_2015
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (Broad, Nature 2012)
</td>
<td style="text-align:left;">
MBL (Broad)
</td>
<td style="text-align:left;">
Whole exome sequencing of 92 primary medulloblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22820256
</td>
<td style="text-align:left;">
Pugh et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
92
</td>
<td style="text-align:left;">
mbl_broad_2012
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Pan-cancer (Columbia U, Genome Med 2016)
</td>
<td style="text-align:left;">
pediatric-cancers
</td>
<td style="text-align:left;">
Whole exome sequencing of 103 samples (88 with matched normals) from 88 high-risk pediatric cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28007021
</td>
<td style="text-align:left;">
Oberg et al. Genome Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
mixed_pipseq_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (FMI, Am J Surg Pathl. 2014)
</td>
<td style="text-align:left;">
ACyC (FMI 2014)
</td>
<td style="text-align:left;">
Comprehensive genomic profiling of 28 metastatic ACCs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24418857
</td>
<td style="text-align:left;">
Ross et al. Am J Surg Pathl 2014
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
28
</td>
<td style="text-align:left;">
acyc_fmi_2014
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (JHU, Cancer Prev Res 2016)
</td>
<td style="text-align:left;">
ACyC (JHU 2016)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 25 adenoid cystic carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26862087
</td>
<td style="text-align:left;">
Rettig et al, Cancer Prev Res 2016
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
25
</td>
<td style="text-align:left;">
acyc_jhu_2016
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MDA, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
ACyC (MDA 2015)
</td>
<td style="text-align:left;">
WGS of 21 salivary ACCs and targeted molecular analyses of a validation set (81 patients).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26631609
</td>
<td style="text-align:left;">
Mitani et al. Clin Cancer Res 2015
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
102
</td>
<td style="text-align:left;">
acyc_mda_2015
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Lymphoblastic Leukemia (St Jude, Nat Genet 2016)
</td>
<td style="text-align:left;">
ALL (St. Jude, Nat Gen)
</td>
<td style="text-align:left;">
Whole-genome and/or whole-exome sequencing of ERG-altered B-ALL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27776115
</td>
<td style="text-align:left;">
Zhang et al. Nat Genet 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
73
</td>
<td style="text-align:left;">
all_stjude_2016
</td>
<td style="text-align:left;">
bll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Fibroepithelial Tumors (Duke-NUS, Nat Genet 2015)
</td>
<td style="text-align:left;">
BFN (Duke-NUS)
</td>
<td style="text-align:left;">
Whole exome sequencing of 22 phyllodes tumors
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26437033
</td>
<td style="text-align:left;">
Tan et al. Nat Genet 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
bfn_duke_nus_2015
</td>
<td style="text-align:left;">
bfn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (MSKCC, Eur Urol 2014)
</td>
<td style="text-align:left;">
Bladder (MSKCC 2014)
</td>
<td style="text-align:left;">
Genomic Predictors of Survival in Patients with High-grade Urothelial Carcinoma of the Bladder. Targeted sequencing of 109 tumor samples (MSKCC, European Urology 2014).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25092538
</td>
<td style="text-align:left;">
Kim et al. Eur Urol 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-27 00:00:00
</td>
<td style="text-align:left;">
109
</td>
<td style="text-align:left;">
blca_mskcc_solit_2014
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Nonmuscle Invasive Bladder Cancer (MSK Eur Urol 2017)
</td>
<td style="text-align:left;">
NMIBC (MSK Our Urol 2017)
</td>
<td style="text-align:left;">
Genomic Landscape and Predictors of Response for High Risk NMIBC
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28583311
</td>
<td style="text-align:left;">
Pietzak et al. Eur Urol 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-27 00:00:00
</td>
<td style="text-align:left;">
105
</td>
<td style="text-align:left;">
blca_nmibc_2017
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (Broad, Nature 2012)
</td>
<td style="text-align:left;">
Breast (Broad 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 103 breast cancer tumor/normal sample pairs. Generated by the Broad Institute.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722202
</td>
<td style="text-align:left;">
Banerji et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-30 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
brca_broad
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (IRC, Nat Genet 2014)
</td>
<td style="text-align:left;">
ccRCC (IRC)
</td>
<td style="text-align:left;">
Multi region exome sequencing of 10 clear cell renal cell carcinomas (ccRCCs)
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24487277
</td>
<td style="text-align:left;">
Gerlinger et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-01 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
ccrcc_irc_2014
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (IUOPA, Nature 2015)
</td>
<td style="text-align:left;">
CLL(IUOPA 2015)
</td>
<td style="text-align:left;">
Mutation data from whole-genome or whole-exome sequencing of 428 CLL, 54 MBL, and 24 SLL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26200345
</td>
<td style="text-align:left;">
Puente et al. Nature 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-01 00:00:00
</td>
<td style="text-align:left;">
506
</td>
<td style="text-align:left;">
cll_iuopa_2015
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic lymphocytic leukemia (ICGC, Nature Genetics 2011)
</td>
<td style="text-align:left;">
CLLSLL (Nature Genetics 2011)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 105 Chronic Lymphocytic Leukemia samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22158541
</td>
<td style="text-align:left;">
Quesada et al. Nature Genetics 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-01 00:00:00
</td>
<td style="text-align:left;">
105
</td>
<td style="text-align:left;">
cllsll_icgc_2011
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cutaneous Squamous Cell Carcinoma (DFCI, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
CSCC (DFCI 2015)
</td>
<td style="text-align:left;">
Targeted sequencing of 29 metastatic cutaneous squamous cell carcinoma samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25589618
</td>
<td style="text-align:left;">
Li et al. Clin Cancer Res 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
cscc_dfarber_2015
</td>
<td style="text-align:left;">
cscc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Desmoplastic Melanoma (Broad Institute, Nat Genet 2015)
</td>
<td style="text-align:left;">
DESM (Broad 2015)
</td>
<td style="text-align:left;">
Exome/Whole Genome sequencing of 20 desmoplastic melanoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26343386
</td>
<td style="text-align:left;">
Shain et al. Nat Genet 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
desm_broad_2015
</td>
<td style="text-align:left;">
desm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ewing Sarcoma (Institut Curie, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Ewing Sarcoma (Institut Curie)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 112 Ewing sarcoma samples and matched germ line DNA.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25223734
</td>
<td style="text-align:left;">
Tirode et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
112
</td>
<td style="text-align:left;">
es_iocurie_2014
</td>
<td style="text-align:left;">
es
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cutaneous Squamous Cell Carcinoma (MD Anderson, Clin Cancer Res 2014)
</td>
<td style="text-align:left;">
cSCC (MD ANDERSON 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/normal pairs from 39 aggressive cutaneous squamous cell carcinoma cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25303977
</td>
<td style="text-align:left;">
Pickering et al. Clin Cancer Res 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
39
</td>
<td style="text-align:left;">
cscc_hgsc_bcm_2014
</td>
<td style="text-align:left;">
cscc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B cell Lymphoma (DFCI, Nat Med 2018)
</td>
<td style="text-align:left;">
DLBCLNOS (DFCI)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/matching normal sample pairs from 135 DLBCL patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29713087
</td>
<td style="text-align:left;">
Chapuy et al. Nat Med 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
135
</td>
<td style="text-align:left;">
dlbcl_dfci_2018
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Adenoma and Carcinomas (MSK, PLOS One 2018)
</td>
<td style="text-align:left;">
HCC (MSK, PLOS One)
</td>
<td style="text-align:left;">
Targeted sequencing of tumor/normal pairs of 9 LIAD and 10 HCC patients via MSK-IMPACT platform.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30052636
</td>
<td style="text-align:left;">
Zheng et al. PLOS One 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-03 00:00:00
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
hcc_msk_venturaa_2018
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Gallbladder Cancer (MSK, Cancer 2018)
</td>
<td style="text-align:left;">
GBC (MSK CANCER 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of 103 samples with matched normals from 101 Gallbladder cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30427539
</td>
<td style="text-align:left;">
Raja et al, Cancer 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-03 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
gbc_msk_2018
</td>
<td style="text-align:left;">
gbc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Liver (TCGA)
</td>
<td style="text-align:left;">
TCGA Liver Hepatocellular Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LIHC/20160128/gdac.broadinstitute.org_LIHC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
442
</td>
<td style="text-align:left;">
lihc_tcga
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-Grade Gliomas (UCSF, Science 2014)
</td>
<td style="text-align:left;">
Glioma (UCSF)
</td>
<td style="text-align:left;">
Whole exome sequencing of 23 grade II glioma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24336570
</td>
<td style="text-align:left;">
Johnson et al. Science 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
61
</td>
<td style="text-align:left;">
lgg_ucsf_2014
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (AMC, Hepatology 2014)
</td>
<td style="text-align:left;">
Liver (AMC)
</td>
<td style="text-align:left;">
Whole exome sequencing of 231 hepatocellular carcinomas and matched normal tissue pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24798001
</td>
<td style="text-align:left;">
Ahn et al. Hepatology 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
231
</td>
<td style="text-align:left;">
lihc_amc_prv
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TSP, Nature 2008)
</td>
<td style="text-align:left;">
Lung adeno (TSP)
</td>
<td style="text-align:left;">
Targeted sequencing of 163 lung adenocarcinoma tumor/normal pairs. Generated by WashU/Broad.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
18948947
</td>
<td style="text-align:left;">
Ding et al. Nature 2008
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
163
</td>
<td style="text-align:left;">
luad_tsp
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (PCGP, Nature 2012)
</td>
<td style="text-align:left;">
MBL (PCGP)
</td>
<td style="text-align:left;">
Whole genome sequencing of 37 medulloblastoma tumors and their matched normals from the Pediatric Cancer Genome Project (PCGP), a joint project by St. Jude Children’s Research Hospital and Washington University.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722829
</td>
<td style="text-align:left;">
Robinson et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
37
</td>
<td style="text-align:left;">
mbl_pcgp
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thoracic PDX (MSK, Provisional)
</td>
<td style="text-align:left;">
MIXED (CMO CR_pdx IMPACT)
</td>
<td style="text-align:left;">
Targeted sequencing of thoracic PDX lines to characterize genetic alterations in 139 tumor/normal pairs via MSK-IMPACT. Data generated in Charles Rudin Lab.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
SCLCPORTAL;RUDINC;COMPONC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
139
</td>
<td style="text-align:left;">
lung_msk_pdx
</td>
<td style="text-align:left;">
lung
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mesothelioma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Mesothelioma (TCGA)
</td>
<td style="text-align:left;">
TCGA Mesothelioma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data at the <a href="https://portal.gdc.cancer.gov/legacy-archive/search/f?filters=%7B%22op%22:%22and%22,%22content%22:%5B%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.program.name%22,%22value%22:%5B%22TCGA%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.project_id%22,%22value%22:%5B%22TCGA-MESO%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22files.data_format%22,%22value%22:%5B%22MAF%22%5D%7D%7D%5D%7D">GDC</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-07 00:00:00
</td>
<td style="text-align:left;">
87
</td>
<td style="text-align:left;">
meso_tcga
</td>
<td style="text-align:left;">
plmeso
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (ICGC, Nature 2012)
</td>
<td style="text-align:left;">
Pancreas (ICGC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 99 pancreatic samples and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23103869
</td>
<td style="text-align:left;">
Biankin et al. Nature 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-07 00:00:00
</td>
<td style="text-align:left;">
99
</td>
<td style="text-align:left;">
paad_icgc
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
MSK-IMPACT Clinical Sequencing Cohort (MSKCC, Nat Med 2017)
</td>
<td style="text-align:left;">
MSK-IMPACT
</td>
<td style="text-align:left;">
Targeted sequencing of 10,000 clinical cases using the MSK-IMPACT assay
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28481359
</td>
<td style="text-align:left;">
Zehir et al. Nat Med 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-07 00:00:00
</td>
<td style="text-align:left;">
10945
</td>
<td style="text-align:left;">
msk_impact_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Pancreas (TCGA)
</td>
<td style="text-align:left;">
TCGA Pancreatic Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PAAD/20160128/gdac.broadinstitute.org_PAAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
186
</td>
<td style="text-align:left;">
paad_tcga
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
PCPG (TCGA)
</td>
<td style="text-align:left;">
TCGA Pheochromocytoma and Paraganglioma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PCPG/20160128/gdac.broadinstitute.org_PCPG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
184
</td>
<td style="text-align:left;">
pcpg_tcga
</td>
<td style="text-align:left;">
mnet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Cancer (UTSW, Nat Commun 2015)
</td>
<td style="text-align:left;">
Pancreas (UTSW)
</td>
<td style="text-align:left;">
Whole exome sequencing of 109 micro-dissected pancreatic cancer cases and normal control tissue.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25855536
</td>
<td style="text-align:left;">
Witkiewicz et al. Nat Commun 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
109
</td>
<td style="text-align:left;">
paad_utsw_2015
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Neuroendocrine Tumors (Johns Hopkins University, Science 2011)
</td>
<td style="text-align:left;">
PANET(Johns Hopkins 2011)
</td>
<td style="text-align:left;">
Whole exome sequencing of 10 pancreatic neuroendocrine tumor patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21252315
</td>
<td style="text-align:left;">
Jiao et al. Science 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
panet_jhu_2011
</td>
<td style="text-align:left;">
panet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Prostate Adenocarcinoma (MCTP, Nature 2012)
</td>
<td style="text-align:left;">
Prostate (MICH)
</td>
<td style="text-align:left;">
Exome sequencing of 61 prostate cancer tumor/normal pairs (50 metastatic CRPCs and 11 high-grade localized prostate cancers). Generated by Arul Chinnaiyan’s and Scott Tomlins’ labs at the University of Michigan.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722839
</td>
<td style="text-align:left;">
Grasso et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
61
</td>
<td style="text-align:left;">
prad_mich
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Sarcoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Sarcoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Sarcoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SARC/20160128/gdac.broadinstitute.org_SARC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
265
</td>
<td style="text-align:left;">
sarc_tcga
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Lung Cancer (CLCGP, Nat Genet 2012)
</td>
<td style="text-align:left;">
Small Cell Lung (CLCGP)
</td>
<td style="text-align:left;">
Comprehensive whole exam profiling of 29 small cell lung cancer samples. Generated by the Clinical Lung Cancer Genome Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22941188
</td>
<td style="text-align:left;">
Peifer et al. Nat Genet 2012
</td>
<td style="text-align:left;">
SCLCPORTAL;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
sclc_clcgp
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Carcinoma of the Ovary (MSKCC, Nat Genet 2014)
</td>
<td style="text-align:left;">
Ovary SC (MSKCC)
</td>
<td style="text-align:left;">
Targeted sequencing of 12 SCCO paired tumor and normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24658004
</td>
<td style="text-align:left;">
Jelinic et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
scco_mskcc
</td>
<td style="text-align:left;">
scco
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC, PNAS 2014)
</td>
<td style="text-align:left;">
Prostate (MSKCC 2014)
</td>
<td style="text-align:left;">
Genome-wide copy number profiling in 104 primary prostate cancer tumor/normal pairs. <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54691">Raw data via GEO (GSE54691)</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25024180
</td>
<td style="text-align:left;">
Hieronymus et al. PNAS 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
104
</td>
<td style="text-align:left;">
prad_mskcc_2014
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Cancer (MSKCC, JCO Precis Oncol 2017)
</td>
<td style="text-align:left;">
MSK-IMPACT Prostate
</td>
<td style="text-align:left;">
Targeted sequencing of 504 prostate cancer tumor/normal pairs via MSK-IMPACT.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28825054
</td>
<td style="text-align:left;">
Abida et al. JCO Precis Oncol 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
504
</td>
<td style="text-align:left;">
prad_mskcc_2017
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Testicular Germ Cell Cancer (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Testicular germ cell (TCGA)
</td>
<td style="text-align:left;">
TCGA Testicular Germ Cell Cancer; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/TGCT/20160128/gdac.broadinstitute.org_TGCT.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
156
</td>
<td style="text-align:left;">
tgct_tcga
</td>
<td style="text-align:left;">
nsgct
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thyroid Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Thyroid (TCGA)
</td>
<td style="text-align:left;">
TCGA Thyroid Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THCA/20160128/gdac.broadinstitute.org_THCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
516
</td>
<td style="text-align:left;">
thca_tcga
</td>
<td style="text-align:left;">
thpa
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (Broad, Cell 2012)
</td>
<td style="text-align:left;">
Melanoma (Broad 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 121 melanoma samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22817889
</td>
<td style="text-align:left;">
Hodis et al. Cell 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
121
</td>
<td style="text-align:left;">
skcm_broad
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (Yale, Nat Genet 2012)
</td>
<td style="text-align:left;">
Melanoma (Yale 2012)
</td>
<td style="text-align:left;">
Exome sequencing of 147 melanoma samples with 99 matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22842228
</td>
<td style="text-align:left;">
Krauthammer et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
147
</td>
<td style="text-align:left;">
skcm_yale
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (UHK, Nat Genet 2011)
</td>
<td style="text-align:left;">
Stomach (UHK)
</td>
<td style="text-align:left;">
Exome sequencing of 22 gastric cancer samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22037554
</td>
<td style="text-align:left;">
Wang et al. Nat Genet 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
stad_uhongkong
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thymoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Thymoma(TCGA)
</td>
<td style="text-align:left;">
TCGA Thymoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THYM/20160128/gdac.broadinstitute.org_THYM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
124
</td>
<td style="text-align:left;">
thym_tcga
</td>
<td style="text-align:left;">
thym
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Uterine (TCGA)
</td>
<td style="text-align:left;">
TCGA Uterine Corpus Endometrial Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCEC/20160128/gdac.broadinstitute.org_UCEC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
548
</td>
<td style="text-align:left;">
ucec_tcga
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Carcinosarcoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Uterine CS (TCGA)
</td>
<td style="text-align:left;">
TCGA Uterine Carcinosarcoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCS/20160128/gdac.broadinstitute.org_UCS.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
57
</td>
<td style="text-align:left;">
ucs_tcga
</td>
<td style="text-align:left;">
ucs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Poorly-Differentiated and Anaplastic Thyroid Cancers (MSKCC, JCI 2016)
</td>
<td style="text-align:left;">
Thyroid (MSKCC 2016)
</td>
<td style="text-align:left;">
Next-generation sequencing of 341 cancer genes from 117 patient-derived PDTCs and ATCs
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26878173
</td>
<td style="text-align:left;">
Landa et al. J Clin Invest 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
117
</td>
<td style="text-align:left;">
thyroid_mskcc_2016
</td>
<td style="text-align:left;">
thyroid
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pan-Lung Cancer (TCGA, Nat Genet 2016)
</td>
<td style="text-align:left;">
NSCLC (TCGA 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 660 lung ADC and 484 lung SqCC tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27158780
</td>
<td style="text-align:left;">
TCGA, Nat Genet 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
1144
</td>
<td style="text-align:left;">
nsclc_tcga_broad_2016
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Clear Cell Carcinoma (NIH, Cancer 2017)
</td>
<td style="text-align:left;">
UCCC (NIH)
</td>
<td style="text-align:left;">
Whole exome sequencing of paired tumor-normal DNAs from 16 cases of clear cell endometrial tumors.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28485815
</td>
<td style="text-align:left;">
Le Gallo et al. Cancer 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
uccc_nih_2017
</td>
<td style="text-align:left;">
uccc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Lung squ (TCGA)
</td>
<td style="text-align:left;">
TCGA Lung Squamous Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUSC/20160128/gdac.broadinstitute.org_LUSC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
511
</td>
<td style="text-align:left;">
lusc_tcga
</td>
<td style="text-align:left;">
lusc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Stomach (TCGA)
</td>
<td style="text-align:left;">
TCGA Stomach Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/STAD/20160128/gdac.broadinstitute.org_STAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
478
</td>
<td style="text-align:left;">
stad_tcga
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (Broad, Science 2011)
</td>
<td style="text-align:left;">
Head &amp; neck (Broad)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 74 head and neck squamous cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21798893
</td>
<td style="text-align:left;">
Stransky et al. Science 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
74
</td>
<td style="text-align:left;">
hnsc_broad
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (University of Turin, Lung Cancer 2017)
</td>
<td style="text-align:left;">
UNITO (Lung Cancer, 2016)
</td>
<td style="text-align:left;">
Targeted next-generation sequencing of 41 NSCLC samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27346245
</td>
<td style="text-align:left;">
VavalÃ  et al. Lung Cancer 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
41
</td>
<td style="text-align:left;">
nsclc_unito_2016
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Malignant Peripheral Nerve Sheath Tumor (MSKCC, Nat Genet 2014)
</td>
<td style="text-align:left;">
MPNST (MSKCC)
</td>
<td style="text-align:left;">
Whole exome sequencing of 15 malignant peripheral nerve sheath tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25240281
</td>
<td style="text-align:left;">
Lee et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
mpnst_mskcc
</td>
<td style="text-align:left;">
mpnst
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MSKCC, Nat Genet 2013)
</td>
<td style="text-align:left;">
ACyC (MSKCC, 2013)
</td>
<td style="text-align:left;">
Whole-exome or whole-genome sequencing analysis of 60 ACC tumor/normal pairs
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23685749
</td>
<td style="text-align:left;">
Ho et al. Nat Genet 2013
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
60
</td>
<td style="text-align:left;">
acyc_mskcc_2013
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
TMB and Immunotherapy (MSKCC, Nat Genet 2019)
</td>
<td style="text-align:left;">
TMB (MSKCC 2019)
</td>
<td style="text-align:left;">
Genomic and survival data from 1661 tumor-normal pairs from 1661 patients with various cancer types sequenced with the MSK-IMPACT assay.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30643254
</td>
<td style="text-align:left;">
Samstein et al. Nat Genet 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
1661
</td>
<td style="text-align:left;">
tmb_mskcc_2018
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uveal Melanoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Uveal melanoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Uveal Melanoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UVM/20160128/gdac.broadinstitute.org_UVM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-03 00:00:00
</td>
<td style="text-align:left;">
80
</td>
<td style="text-align:left;">
uvm_tcga
</td>
<td style="text-align:left;">
um
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma of the Breast (MSKCC, J Pathol. 2015)
</td>
<td style="text-align:left;">
ACBC (MSKCC/Breast 2015)
</td>
<td style="text-align:left;">
Whole exome sequencing of 12 breast AdCCs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26095796
</td>
<td style="text-align:left;">
Martelotto et al. J Pathol 2015
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-03 00:00:00
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
acbc_mskcc_2015
</td>
<td style="text-align:left;">
acbc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, Nature 2011)
</td>
<td style="text-align:left;">
Ovarian (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing (316 samples with matched normals), mRNA expression, miRNA expression, promoter methylation, and DNA copy number in 489 high-grade serous ovarian adenocarcinomas (HGS-OvCa). <a href="http://cancergenome.nih.gov/">The Cancer Genome Atlas (TCGA)</a> Serous Ovarian Cancer project. 489 cases. <br><a href="https://tcga-data.nci.nih.gov/docs/publications/ov_2011/">Raw data via the TCGA Data Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21720365
</td>
<td style="text-align:left;">
TCGA, Nature 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-04 00:00:00
</td>
<td style="text-align:left;">
489
</td>
<td style="text-align:left;">
ov_tcga_pub
</td>
<td style="text-align:left;">
hgsoc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (Broad/Cornell, Cell 2013)
</td>
<td style="text-align:left;">
Prostate (Broad/Cornell 2013)
</td>
<td style="text-align:left;">
Whole genome sequencing of 57 prostate tumor and matches normal tissues.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23622249
</td>
<td style="text-align:left;">
Baca et al. Cell 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-04 00:00:00
</td>
<td style="text-align:left;">
57
</td>
<td style="text-align:left;">
prad_broad_2013
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Urothelial Carcinoma (Cornell/Trento, Nat Gen 2016)
</td>
<td style="text-align:left;">
BLCA (Cornell 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 72 tumor samples (with 32 matched normals) in Urothelial Carcinoma.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27749842
</td>
<td style="text-align:left;">
Faltas and Prandi, Nature Genetics 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-07 00:00:00
</td>
<td style="text-align:left;">
72
</td>
<td style="text-align:left;">
blca_cornell_2016
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Merged Cohort of LGG and GBM (TCGA, Cell 2016)
</td>
<td style="text-align:left;">
LGG-GBM (TCGA 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing from 1,122 TCGA LGG and GBM cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26824661
</td>
<td style="text-align:left;">
TCGA, Cell 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-10 00:00:00
</td>
<td style="text-align:left;">
1102
</td>
<td style="text-align:left;">
lgggbm_tcga_pub
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytoma (MSKCC, Neuro Oncol 2017)
</td>
<td style="text-align:left;">
Glioma MSK (Neuro Oncol 2017)
</td>
<td style="text-align:left;">
Genomic characterization using next generation sequencing (NGS) in 22 Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytomas.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28472509
</td>
<td style="text-align:left;">
Thomas et al. Neuro Oncol 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-11 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
odg_msk_2017
</td>
<td style="text-align:left;">
odg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Germ Cell Tumors (MSKCC, J Clin Oncol 2016)
</td>
<td style="text-align:left;">
GCT (JCO, 2016)
</td>
<td style="text-align:left;">
Whole-exome and Targeted sequencing of 19 samples and Targeted sequencing of 161 samples from 180 Germ Cell Tumor patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27646943
</td>
<td style="text-align:left;">
Bagrodia et al. J Clin Oncol 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-11 00:00:00
</td>
<td style="text-align:left;">
180
</td>
<td style="text-align:left;">
gct_msk_2016
</td>
<td style="text-align:left;">
testis
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Solid Cancers (UMich, Nature 2017)
</td>
<td style="text-align:left;">
MET500 (Michigan U 2017)
</td>
<td style="text-align:left;">
Whole-exome and -transcriptome sequencing of 500 adult patients with metastatic solid tumor/primary normal pairs of diverse lineage and biopsy site.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28783718
</td>
<td style="text-align:left;">
Robinson et al. Nature 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-12 00:00:00
</td>
<td style="text-align:left;">
500
</td>
<td style="text-align:left;">
metastatic_solid_tumors_mich_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute myeloid leukemia or myelodysplastic syndromes (WashU, 2016)
</td>
<td style="text-align:left;">
AML/MDS (WashU, 2016)
</td>
<td style="text-align:left;">
Enhanced exome or targeted sequencing of 116 acute myeloid leukemia (AML) or myelodysplastic syndromes (MDS) patients’ tumor/normal pairs .
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27959731
</td>
<td style="text-align:left;">
Welch et al. N Engl J Med. 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-01 00:00:00
</td>
<td style="text-align:left;">
136
</td>
<td style="text-align:left;">
mnm_washu_2016
</td>
<td style="text-align:left;">
mnm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colon Adenocarcinoma (CaseCCC, PNAS 2015)
</td>
<td style="text-align:left;">
COAD (CaseCCC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/normal pairs from 29 African American Colon Cancers.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25583493
</td>
<td style="text-align:left;">
Guda et al. PNAS 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-02 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
coad_caseccc_2015
</td>
<td style="text-align:left;">
coad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Myeloproliferative Neoplasms (CIMR, NEJM 2013)
</td>
<td style="text-align:left;">
MPN (CIMR)
</td>
<td style="text-align:left;">
Whole exome sequencing of paired tumor/normal pairs from 151 cases of Myeloproliferative Neoplasms.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24325359
</td>
<td style="text-align:left;">
Nangalia et al. NEJM 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-24 00:00:00
</td>
<td style="text-align:left;">
151
</td>
<td style="text-align:left;">
mpn_cimr_2013
</td>
<td style="text-align:left;">
mpn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Clear Cell Renal Cell Carcinoma (DFCI, Science 2019)
</td>
<td style="text-align:left;">
ccRCC (DFCI 2019)
</td>
<td style="text-align:left;">
Whole-exome sequencing of metastatic clear cell renal carcinoma (ccRCC) from 35 tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29301960
</td>
<td style="text-align:left;">
Miao et al. Science 2019
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-26 00:00:00
</td>
<td style="text-align:left;">
35
</td>
<td style="text-align:left;">
ccrcc_dfci_2019
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-small cell lung cancer (MSK, Science 2015)
</td>
<td style="text-align:left;">
NSCLC (MSKCC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor-normal tissue pairs from 16 non-small cell lung cancer samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25765070
</td>
<td style="text-align:left;">
Rivzi et al. Science 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-08-09 00:00:00
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
nsclc_mskcc_2015
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Prostate (TCGA)
</td>
<td style="text-align:left;">
TCGA Prostate Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PRAD/20160128/gdac.broadinstitute.org_PRAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-08-23 00:00:00
</td>
<td style="text-align:left;">
499
</td>
<td style="text-align:left;">
prad_tcga
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Cancer (MSKCC, Cancer Discov 2017)
</td>
<td style="text-align:left;">
LUAD (MSK,2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 860 patients with metastatic lung adenocarcinoma
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28336552
</td>
<td style="text-align:left;">
Jordan et al. Cancer Discov 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-03 00:00:00
</td>
<td style="text-align:left;">
915
</td>
<td style="text-align:left;">
lung_msk_2017
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (MSKCC, J Clin Oncol 2018)
</td>
<td style="text-align:left;">
NSCLC (MSK PD1)
</td>
<td style="text-align:left;">
IMPACT sequencing of 240 NSCLC tumor/normal pairs treated at MSKCC with anti-PD-(L)1 based therapy.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29337640
</td>
<td style="text-align:left;">
Rizvi et al. J Clin Oncol 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-03 00:00:00
</td>
<td style="text-align:left;">
240
</td>
<td style="text-align:left;">
nsclc_pd1_msk_2018
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Endometrial Cancer (MSK, 2018)
</td>
<td style="text-align:left;">
UCEC (MSK 2018)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 197 advanced-staged endometrial cancer tumor samples with matched normals, from 189 patients treated at MSK.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30068706
</td>
<td style="text-align:left;">
Soumerai et al. Clin Cancer Res. 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-03 00:00:00
</td>
<td style="text-align:left;">
197
</td>
<td style="text-align:left;">
ucec_msk_2018
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Melanoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Skin Cutaneous Melanoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SKCM/20160128/gdac.broadinstitute.org_SKCM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
479
</td>
<td style="text-align:left;">
skcm_tcga
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Ovarian (TCGA)
</td>
<td style="text-align:left;">
TCGA Ovarian Serous Cystadenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/OV/20160128/gdac.broadinstitute.org_OV.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
606
</td>
<td style="text-align:left;">
ov_tcga
</td>
<td style="text-align:left;">
hgsoc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colon Cancer (CPTAC-2 Prospective, Cell 2019)
</td>
<td style="text-align:left;">
COAD (CPTAC-2 2019)
</td>
<td style="text-align:left;">
Proteogenomic profiling of 110 colon adenocarcinoma tumor samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31031003
</td>
<td style="text-align:left;">
Vasaikar et al. Cell 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
110
</td>
<td style="text-align:left;">
coad_cptac_2019
</td>
<td style="text-align:left;">
coad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Rectal Cancer (MSK,Nature Medicine 2019)
</td>
<td style="text-align:left;">
Rectal (MSK, 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT targeted sequencing of matched rectal cancer tumor and tumoroid samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
SELCUKLS;SAUVEC;ZHENGY1;SMITHJ5;PUBLIC;SANCHEZF
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-08 00:00:00
</td>
<td style="text-align:left;">
339
</td>
<td style="text-align:left;">
rectal_msk_2019
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Unclassified Renal Cell Carcinoma (MSK, Nature 2016)
</td>
<td style="text-align:left;">
Renal unclass (MSKCC)
</td>
<td style="text-align:left;">
Targeted sequencing of 62 high-grade primary Unclassified Renal Cell Carcinoma tumor samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27713405
</td>
<td style="text-align:left;">
Chen et al. Nat Commun 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-09 00:00:00
</td>
<td style="text-align:left;">
62
</td>
<td style="text-align:left;">
urcc_mskcc_2016
</td>
<td style="text-align:left;">
urcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Esophagogastric Cancer (MSKCC, Cancer Discovery 2017)
</td>
<td style="text-align:left;">
EGC (MSK 2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 341 samples from 305 metastatic esophagogastric cancer patients
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29122777
</td>
<td style="text-align:left;">
Janjigian et al. Cancer Discov 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-09 00:00:00
</td>
<td style="text-align:left;">
341
</td>
<td style="text-align:left;">
egc_msk_2017
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (MSK, Clin Cancer Res 2018)
</td>
<td style="text-align:left;">
CHOL (MSK 2018)
</td>
<td style="text-align:left;">
Targeted sequencing (MSK-IMPACT) of 195 tumor samples (194 with matched normals) from 192 cholangiocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29848569
</td>
<td style="text-align:left;">
Lowery et al. Clin Cancer Res 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-09 00:00:00
</td>
<td style="text-align:left;">
195
</td>
<td style="text-align:left;">
chol_msk_2018
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Intrahepatic Cholangiocarcinoma (Shanghai, Nat Commun 2014)
</td>
<td style="text-align:left;">
IHCH (Shanghai)
</td>
<td style="text-align:left;">
Whole exome sequencing of 103 tumor/normal sample pairs in 103 intrahepatic cholangiocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25526346
</td>
<td style="text-align:left;">
Shanshan et al. Nat Commun 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-30 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
ihch_smmu_2014
</td>
<td style="text-align:left;">
ihch
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
NSCLC (MSKCC 2018)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 75 tumor/normal NSCLC pairs treated with PD-1 plus CTLA-4 blockade..
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29657128
</td>
<td style="text-align:left;">
Hellmann et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-30 00:00:00
</td>
<td style="text-align:left;">
75
</td>
<td style="text-align:left;">
nsclc_mskcc_2018
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC/DFCI, Nature Genetics 2018)
</td>
<td style="text-align:left;">
PRAD (MSKCC/DFCI 2018)
</td>
<td style="text-align:left;">
Whole exome Sequencing of 1013 prostate cancer samples and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29610475
</td>
<td style="text-align:left;">
Armenia et al. Nat Genet 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-11-05 00:00:00
</td>
<td style="text-align:left;">
1013
</td>
<td style="text-align:left;">
prad_p1000
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (DFCI, Cell Reports 2016)
</td>
<td style="text-align:left;">
Colorectal (DFCI 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 619 colorectal cancers with clinicopathologic annotations.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27149842
</td>
<td style="text-align:left;">
Giannakis et al. Cell Reports 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-11-09 00:00:00
</td>
<td style="text-align:left;">
619
</td>
<td style="text-align:left;">
coadread_dfci_2016
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Cancer (MSK, Eur Urol 2015)
</td>
<td style="text-align:left;">
Upper tract (MSKCC)
</td>
<td style="text-align:left;">
Next-generation sequencing (MSK-IMPACT) on all protein-coding exons of 300 cancer associated genes of 85 Upper Tract Urothelial Carcinoma tumor samples and paired normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26278805
</td>
<td style="text-align:left;">
Sfakianos et al. Eur Urol 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-11-20 00:00:00
</td>
<td style="text-align:left;">
85
</td>
<td style="text-align:left;">
utuc_mskcc_2015
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Breast Cancer (INSERM, PLoS Med 2016)
</td>
<td style="text-align:left;">
BRCA (INSERM 2016)
</td>
<td style="text-align:left;">
Whole exome sequencing of 216 tumor/normal (blood) pairs from metastatic breast cancer patients who underwent a biopsy in the context of the SAFIR01/SAFIR02 (Unicancer, France), SHIVA (Institut Curie, France) or MOSCATO (Gustave Roussy, France) prospective trials.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28027327
</td>
<td style="text-align:left;">
Lefebvre et al. PLoS Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-11 00:00:00
</td>
<td style="text-align:left;">
216
</td>
<td style="text-align:left;">
brca_igr_2015
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Lung adeno (TCGA)
</td>
<td style="text-align:left;">
TCGA Lung Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUAD/20160128/gdac.broadinstitute.org_LUAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-24 00:00:00
</td>
<td style="text-align:left;">
586
</td>
<td style="text-align:left;">
luad_tcga
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
NCI-60 Cell Lines (NCI, Cancer Res 2012)
</td>
<td style="text-align:left;">
NCI-60
</td>
<td style="text-align:left;">
Whole-exome sequencing of 67 samples by NCI-60 cell line project; raw data at <a href="http://discover.nci.nih.gov/cellminer/loadDownload.do">CellMiner</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22802077
</td>
<td style="text-align:left;">
Reinhold et al. Cancer Res 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
67
</td>
<td style="text-align:left;">
cellline_nci60
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Melanoma (UCLA, Cell 2016)
</td>
<td style="text-align:left;">
Melanoma (Hugo, Cell)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 38 pretreated (pembrolizumab, nivolumab) melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26997480
</td>
<td style="text-align:left;">
Hugo et al. Cell 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
38
</td>
<td style="text-align:left;">
mel_ucla_2016
</td>
<td style="text-align:left;">
mel
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Lymphoblastic Leukemia (St Jude, Nat Genet 2015)
</td>
<td style="text-align:left;">
ALL (St. Jude 2015)
</td>
<td style="text-align:left;">
Comprehensive profiling of infant MLL-rearranged acute lymphoblastic leukemia (MLL-R ALL)
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25730765
</td>
<td style="text-align:left;">
Andersson et al. Nat Genet 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
93
</td>
<td style="text-align:left;">
all_stjude_2015
</td>
<td style="text-align:left;">
bll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer Xenografts (British Columbia, Nature 2015)
</td>
<td style="text-align:left;">
Breast (BCCRC Xenograft)
</td>
<td style="text-align:left;">
Whole genome/targeted sequencing to evaluate the clonal dynamics in 116 breast cancer patient xenografts.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25470049
</td>
<td style="text-align:left;">
Eirew et al. Nature 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
117
</td>
<td style="text-align:left;">
brca_bccrc_xenograft_2014
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (Genentech, Nature 2012)
</td>
<td style="text-align:left;">
Colorectal (Genentech)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 74 paired colon tumor-normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22895193
</td>
<td style="text-align:left;">
Seshagiri et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
74
</td>
<td style="text-align:left;">
coadread_genentech
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Lung Cancer (Johns Hopkins, Nat Genet 2012)
</td>
<td style="text-align:left;">
Small Cell Lung (JHU)
</td>
<td style="text-align:left;">
Whole exome-sequencing of 80 human SCLCs, including 36 primary SCLC human tumor and adjacent normal sample pairs and 17 paired SCLC cell lines and their patient-matched lymphoblastoid lines, as well as 4 primary SCLC tumors and 23 SCLC cell lines without matched normal controls.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22941189
</td>
<td style="text-align:left;">
Rudin et al. Nat Genet 2012
</td>
<td style="text-align:left;">
SCLCPORTAL;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
80
</td>
<td style="text-align:left;">
sclc_jhu
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MGH, Nat Gen 2016)
</td>
<td style="text-align:left;">
ACyC (MGH 2016)
</td>
<td style="text-align:left;">
Whole-genome/exome sequencing of 10 ACC PDX models.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26829750
</td>
<td style="text-align:left;">
Drier et al. Nature Genetics 2016
</td>
<td style="text-align:left;">
ACYC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
acyc_mgh_2016
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (CPC-GENE, Nature 2017)
</td>
<td style="text-align:left;">
PRAD Hallmarks (CPCG-GENE, 2017)
</td>
<td style="text-align:left;">
Whole-genome/exome sequencing of 477 prostate samples from CPC-GENE and public data sets, including TCGA-PRAD.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28068672
</td>
<td style="text-align:left;">
Fraser et al. Nature 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
477
</td>
<td style="text-align:left;">
prad_cpcg_2017
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
SUMMIT - Neratinib Basket Study (Multi-Institute, Nature 2018)
</td>
<td style="text-align:left;">
Neratinib Basket Study
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 86 patients with HER2- and HER3-mutant cancers
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29420467
</td>
<td style="text-align:left;">
Hyman et al. Nature 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
141
</td>
<td style="text-align:left;">
summit_2018
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Acute Lymphoid Leukemia - Phase II (TARGET, 2018)
</td>
<td style="text-align:left;">
ALL-Phase II (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing on 298 Pediatric Acute Lymphoid Leukemia - Phase II samples. Comprehensive profiling of ALL Phase 2 samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
1978
</td>
<td style="text-align:left;">
all_phase2_target_2018_pub
</td>
<td style="text-align:left;">
bll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (Duke, Cell 2017)
</td>
<td style="text-align:left;">
DLBCLNOS (Duke 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 1,001 Diffuse Large B Cell Lymphoma (DLBCL) tumors (N = 1001) and paired-normal tissue (N = 400).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28985567
</td>
<td style="text-align:left;">
Reddy et al. Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
1001
</td>
<td style="text-align:left;">
dlbcl_duke_2017
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (MSKCC, Nat Genet 2016)
</td>
<td style="text-align:left;">
Bladder PV (MSKCC)
</td>
<td style="text-align:left;">
Whole exome and targeted sequencing of 34 plasmacytoid-variant (signet ring cell) tumors of the bladder
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26901067
</td>
<td style="text-align:left;">
Al-Ahmadie et al. Nat Genet 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
34
</td>
<td style="text-align:left;">
blca_plasmacytoid_mskcc_2016
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma Triplets (MSKCC, Genome Biol 2014)
</td>
<td style="text-align:left;">
Colorectal (MSKCC)
</td>
<td style="text-align:left;">
Targeted sequencing of primary, metastatic, and normal tissues from 69 colorectal adenocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25164765
</td>
<td style="text-align:left;">
Brannon et al. Genome Biol 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
138
</td>
<td style="text-align:left;">
coadread_mskcc
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hepatocellular Carcinoma (MSK, Clin Cancer Res 2018)
</td>
<td style="text-align:left;">
HCC (MSK 2018)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 127 Hepatocellular Carcinoma tumor samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30373752
</td>
<td style="text-align:left;">
Harding et al. Clin Cancer Res 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
127
</td>
<td style="text-align:left;">
hcc_mskimpact_2018
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Breast Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of tumor/normal sample pairs from 1918 Breast cancers.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30205045
</td>
<td style="text-align:left;">
Razavi et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
1918
</td>
<td style="text-align:left;">
breast_msk_2018
</td>
<td style="text-align:left;">
breast
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Histiocytosis Cobimetinib (MSK, Nature 2019)
</td>
<td style="text-align:left;">
HIST (COBI 2019)
</td>
<td style="text-align:left;">
Targeted and whole exome sequencing of 52 histiocytosis tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30867592
</td>
<td style="text-align:left;">
Eli et al. Nature 2019
</td>
<td style="text-align:left;">
DIAMONDHISTCOBI;PUBLIC;HYMANHISTCOBI;OMARHISTCOBI
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
52
</td>
<td style="text-align:left;">
histiocytosis_cobi_msk_2019
</td>
<td style="text-align:left;">
hdcn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (Broad, Cell 2013)
</td>
<td style="text-align:left;">
CLL (Broad 2013)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 160 CLL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23415222
</td>
<td style="text-align:left;">
Landau et al. Cell 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-01-28 00:00:00
</td>
<td style="text-align:left;">
160
</td>
<td style="text-align:left;">
lcll_broad_2013
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lymphoma Cell Lines (MSKCC, 2020)
</td>
<td style="text-align:left;">
Lymph Cellline (MSKCC, 2020)
</td>
<td style="text-align:left;">
Targeted sequencing of 34 lymphoma cell lines using HemePACT_v3
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-02-12 00:00:00
</td>
<td style="text-align:left;">
34
</td>
<td style="text-align:left;">
lymphoma_cellline_msk_2020
</td>
<td style="text-align:left;">
lymph
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (Broad, Cell 2012)
</td>
<td style="text-align:left;">
Lung adeno (Broad)
</td>
<td style="text-align:left;">
Whole-exome and/or whole-genome sequencing of 183 lung adenocarcinomas and matched normal tissue pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22980975
</td>
<td style="text-align:left;">
Imielinksi et al. Cell 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-02-26 00:00:00
</td>
<td style="text-align:left;">
183
</td>
<td style="text-align:left;">
luad_broad
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adrenocortical Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
ACC (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Adrenocortical Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
92
</td>
<td style="text-align:left;">
acc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
acc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Bladder (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
411
</td>
<td style="text-align:left;">
blca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma Breast (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
1084
</td>
<td style="text-align:left;">
brca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Cervical (TCGA PanCan)
</td>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
297
</td>
<td style="text-align:left;">
cesc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
cesc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Colorectal (TCGA PanCan)
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
594
</td>
<td style="text-align:left;">
coadread_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA PanCan)
</td>
<td style="text-align:left;">
Cholangiocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
36
</td>
<td style="text-align:left;">
chol_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
DLBC (TCGA PanCan)
</td>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
dlbc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Esophagus (TCGA PanCan)
</td>
<td style="text-align:left;">
Esophageal Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
182
</td>
<td style="text-align:left;">
esca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
esca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma Multiforme (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
GBM (TCGA PanCan)
</td>
<td style="text-align:left;">
Glioblastoma Multiforme TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
592
</td>
<td style="text-align:left;">
gbm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Head &amp; neck (TCGA PanCan)
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
523
</td>
<td style="text-align:left;">
hnsc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
chromophobe RCC (TCGA PanCan)
</td>
<td style="text-align:left;">
Kidney Chromophobe TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
65
</td>
<td style="text-align:left;">
kich_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
chrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
ccRCC (TCGA PanCan)
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
512
</td>
<td style="text-align:left;">
kirc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
pRCC (TCGA PanCan)
</td>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
283
</td>
<td style="text-align:left;">
kirp_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
prcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
AML (TCGA PanCan)
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
200
</td>
<td style="text-align:left;">
laml_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Brain Lower Grade Glioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
LGG (TCGA PanCan)
</td>
<td style="text-align:left;">
Brain Lower Grade Glioma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
514
</td>
<td style="text-align:left;">
lgg_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Liver (TCGA PanCan)
</td>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
372
</td>
<td style="text-align:left;">
lihc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Lung adeno (TCGA PanCan)
</td>
<td style="text-align:left;">
Lung Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
566
</td>
<td style="text-align:left;">
luad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mixed cfDNA (MSK, Nature Medicine 2019)
</td>
<td style="text-align:left;">
Mixed cfDNA (MSK 2019)
</td>
<td style="text-align:left;">
Targeted sequencing of 248 samples from 124 patients’ cfDNA/WBC and tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31768066
</td>
<td style="text-align:left;">
Razavi et al. Nature Medicine 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
248
</td>
<td style="text-align:left;">
cfdna_msk_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioma (MSKCC, Clin Cancer Res 2019)
</td>
<td style="text-align:left;">
Glioma (MSK 2019)
</td>
<td style="text-align:left;">
Targeted sequencing on MSK-IMPACT and FMI Panels of 1004 samples (837 with matched normals) from 923 glioma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31263031
</td>
<td style="text-align:left;">
Jonsson et al. Clin Cancer Res 2019
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
1004
</td>
<td style="text-align:left;">
glioma_mskcc_2019
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Lung squ (TCGA PanCan)
</td>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
487
</td>
<td style="text-align:left;">
lusc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
lusc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mesothelioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Mesothelioma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Mesothelioma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
87
</td>
<td style="text-align:left;">
meso_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
plmeso
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Ovarian (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
585
</td>
<td style="text-align:left;">
ov_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
hgsoc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Pancreas (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Pancreatic Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
184
</td>
<td style="text-align:left;">
paad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
PCPG (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
178
</td>
<td style="text-align:left;">
pcpg_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Prostate (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
494
</td>
<td style="text-align:left;">
prad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Sarcoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Sarcoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Sarcoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
255
</td>
<td style="text-align:left;">
sarc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Melanoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
448
</td>
<td style="text-align:left;">
skcm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Stomach (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
440
</td>
<td style="text-align:left;">
stad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Testicular Germ Cell Tumors (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Testicular germ cell (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Testicular Germ Cell Tumors TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
149
</td>
<td style="text-align:left;">
tgct_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
testis
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thyroid Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Thyroid (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Thyroid Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
500
</td>
<td style="text-align:left;">
thca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
thpa
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thymoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Thymoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Thymoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
123
</td>
<td style="text-align:left;">
thym_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
thym
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uterine (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
529
</td>
<td style="text-align:left;">
ucec_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Carcinosarcoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uterine CS (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Uterine Carcinosarcoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-27 00:00:00
</td>
<td style="text-align:left;">
57
</td>
<td style="text-align:left;">
ucs_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
ucs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uveal Melanoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uveal melanoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Uveal melanoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-27 00:00:00
</td>
<td style="text-align:left;">
80
</td>
<td style="text-align:left;">
uvm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
um
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adrenocortical Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
ACC (TCGA)
</td>
<td style="text-align:left;">
TCGA Adrenocortical Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ACC/20160128/gdac.broadinstitute.org_ACC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
92
</td>
<td style="text-align:left;">
acc_tcga
</td>
<td style="text-align:left;">
acc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Bladder (TCGA)
</td>
<td style="text-align:left;">
TCGA Bladder Urothelial Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BLCA/20160128/gdac.broadinstitute.org_BLCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
413
</td>
<td style="text-align:left;">
blca_tcga
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Breast (TCGA)
</td>
<td style="text-align:left;">
TCGA Breast Invasive Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BRCA/20160128/gdac.broadinstitute.org_BRCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
1108
</td>
<td style="text-align:left;">
brca_tcga
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Cervical (TCGA)
</td>
<td style="text-align:left;">
TCGA Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CESC/20160128/gdac.broadinstitute.org_CESC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
310
</td>
<td style="text-align:left;">
cesc_tcga
</td>
<td style="text-align:left;">
cesc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Colorectal (TCGA)
</td>
<td style="text-align:left;">
TCGA Colorectal Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/COADREAD/20160128/gdac.broadinstitute.org_COADREAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
640
</td>
<td style="text-align:left;">
coadread_tcga
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lymphoid Neoplasm Diffuse Large B-cell Lymphoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
DLBC (TCGA)
</td>
<td style="text-align:left;">
TCGA Lymphoid Neoplasm Diffuse Large B-cell Lymphoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/DLBC/20160128/gdac.broadinstitute.org_DLBC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
dlbc_tcga
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
AML (TCGA)
</td>
<td style="text-align:left;">
TCGA Acute Myeloid Leukemia; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LAML/20160128/gdac.broadinstitute.org_LAML.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
200
</td>
<td style="text-align:left;">
laml_tcga
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
ccRCC (TCGA)
</td>
<td style="text-align:left;">
TCGA Kidney Renal Clear Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRC/20160128/gdac.broadinstitute.org_KIRC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
538
</td>
<td style="text-align:left;">
kirc_tcga
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
chRCC (TCGA)
</td>
<td style="text-align:left;">
TCGA Kidney Chromophobe; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KICH/20160128/gdac.broadinstitute.org_KICH.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
66
</td>
<td style="text-align:left;">
kich_tcga
</td>
<td style="text-align:left;">
chrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Esophagus (TCGA)
</td>
<td style="text-align:left;">
TCGA Esophageal Carcinoma ; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ESCA/20160128/gdac.broadinstitute.org_ESCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
186
</td>
<td style="text-align:left;">
esca_tcga
</td>
<td style="text-align:left;">
esca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma Multiforme (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
GBM (TCGA)
</td>
<td style="text-align:left;">
TCGA Glioblastoma Multiforme; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/GBM/20160128/gdac.broadinstitute.org_GBM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
604
</td>
<td style="text-align:left;">
gbm_tcga
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Head &amp; neck (TCGA)
</td>
<td style="text-align:left;">
TCGA Head and Neck Squamous Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/HNSC/20160128/gdac.broadinstitute.org_HNSC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
530
</td>
<td style="text-align:left;">
hnsc_tcga
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
pRCC (TCGA)
</td>
<td style="text-align:left;">
TCGA Kidney Renal Papillary Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRP/20160128/gdac.broadinstitute.org_KIRP.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
293
</td>
<td style="text-align:left;">
kirp_tcga
</td>
<td style="text-align:left;">
prcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
The Metastatic Breast Cancer Project (Provisional, February 2020)
</td>
<td style="text-align:left;">
The MBC Project
</td>
<td style="text-align:left;">
The Metastatic Breast Cancer Project (www.mbcproject.org) is an ongoing patient-driven initiative. This dataset release is derived from 237 samples, including both primary and/or metastatic tumor specimens (FFPE) from 180 patients who developed metastatic breast cancer. 237 samples have undergone whole-exome sequencing (along with matched germline whole-exome sequencing) and 146 of those samples have also undergone RNA-sequencing. This study includes genomic data, patient-reported data (prepended as PRD), medical record data (MedR), and pathology report data (PATH). All annotations have been de-identified. As these data were generated in a research laboratory, not a clinical laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. For questions about these data, including how to cite, please visit www.mbcproject.org/data-release or direct questions to <a href="mailto:data@mbcproject.org" class="email">data@mbcproject.org</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-14 00:00:00
</td>
<td style="text-align:left;">
237
</td>
<td style="text-align:left;">
brca_mbcproject_wagle_2017
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (BGI, Nat Genet 2013)
</td>
<td style="text-align:left;">
Bladder (BGI 2013)
</td>
<td style="text-align:left;">
Whole-exome and whole-genome sequencing of 99 bladder cancer tumor/normal pairs. Generated by BGI.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24121792
</td>
<td style="text-align:left;">
Guo et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-25 00:00:00
</td>
<td style="text-align:left;">
99
</td>
<td style="text-align:left;">
blca_bgi
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (National University of Singapore, Nat Genet 2012)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (NUS)
</td>
<td style="text-align:left;">
Exome Sequencing for Opisthorchis viverrini-related cholangiocarcinoma in 8 O.viverrini-related tumors and matched normal tissue.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22561520
</td>
<td style="text-align:left;">
Ong et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-25 00:00:00
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
chol_nus_2012
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma (Columbia, Nat Med. 2019)
</td>
<td style="text-align:left;">
GBM (Columbia)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 32 out of 42 glioblastomas patients with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30742119
</td>
<td style="text-align:left;">
Zhao et al. Nat Med 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-08 00:00:00
</td>
<td style="text-align:left;">
42
</td>
<td style="text-align:left;">
gbm_columbia_2019
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Cancer (DKFZ, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Prostate (DKFZ)
</td>
<td style="text-align:left;">
Whole genome and transcriptome sequencing of tumor/normal pairs from 292 prostate cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30537516
</td>
<td style="text-align:left;">
Gerhauser et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-13 00:00:00
</td>
<td style="text-align:left;">
324
</td>
<td style="text-align:left;">
prostate_dkfz_2018
</td>
<td style="text-align:left;">
prostate
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma Organoids (MSKCC, Cell 2014)
</td>
<td style="text-align:left;">
Prostate Organoids
</td>
<td style="text-align:left;">
Exome sequencing of 12 advanced prostate cancer samples and matched organoids
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25201530
</td>
<td style="text-align:left;">
Gao et al. Cell 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-13 00:00:00
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
prad_mskcc_cheny1_organoids_2014
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Sarcoma (MSKCC/Broad, Nat Genet 2010)
</td>
<td style="text-align:left;">
Sarcoma (MSKCC/Broad 2010)
</td>
<td style="text-align:left;">
Sanger-sequenced 226 protein-coding genes (3,849 exons and adjacent intronic) on 207 samples (with 205 matched normals) encompassing 7 major sarcoma subtypes.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
20601955
</td>
<td style="text-align:left;">
Barretina et al. Nat Genet 2010
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
216
</td>
<td style="text-align:left;">
sarc_mskcc
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adult Soft Tissue Sarcomas (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
SARC (TCGA CELL)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 206 Adult Soft Tissue Sarcoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29100075
</td>
<td style="text-align:left;">
TCGA, Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
206
</td>
<td style="text-align:left;">
sarc_tcga_pub
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Melanoma (MSKCC, NEJM 2014)
</td>
<td style="text-align:left;">
Melanoma (Snyder, NEJM)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 64 pretreated (ipilimumab or tremelimumab) melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25409260
</td>
<td style="text-align:left;">
Snyder et al. NEJM 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
64
</td>
<td style="text-align:left;">
skcm_mskcc_2014
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Rhabdoid Tumor (TARGET, 2018)
</td>
<td style="text-align:left;">
RT (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole-genome sequencing 72 patients with Rhabdoid Tumor. Comprehensive profiling of 72 patients with Rhabdoid Tumor.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
72
</td>
<td style="text-align:left;">
rt_target_2018_pub
</td>
<td style="text-align:left;">
mrt
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Melanoma (DFCI, Science 2015)
</td>
<td style="text-align:left;">
Melanoma (Van Allen, Science)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 110 metastatic melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26359337
</td>
<td style="text-align:left;">
Van Allen et al. Science 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
110
</td>
<td style="text-align:left;">
skcm_dfci_2015
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic castration-sensitive prostate cancer (MSK, Clin Cancer Res 2020)
</td>
<td style="text-align:left;">
PRAD_MCSPC (MSK, CCR 2020)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 424 matched tumor-normal samples from patients with metastatic castration-sensitive prostate cancer
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32220891
</td>
<td style="text-align:left;">
Stopsack et al. Clin Cancer Res 2020
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-19 00:00:00
</td>
<td style="text-align:left;">
424
</td>
<td style="text-align:left;">
prad_mcspc_mskcc_2020
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Cholangiocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CHOL/20160128/gdac.broadinstitute.org_CHOL.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-29 00:00:00
</td>
<td style="text-align:left;">
51
</td>
<td style="text-align:left;">
chol_tcga
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (Sanger, Nature 2012)
</td>
<td style="text-align:left;">
Breast (Sanger)
</td>
<td style="text-align:left;">
Whole exome sequencing from 100 breast cancer tumor/normal sample pairs. Generated by the Sanger Institute.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722201
</td>
<td style="text-align:left;">
Stephens et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-29 00:00:00
</td>
<td style="text-align:left;">
100
</td>
<td style="text-align:left;">
brca_sanger
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Brain Lower Grade Glioma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Glioma (TCGA)
</td>
<td style="text-align:left;">
TCGA Brain Lower Grade Glioma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LGG/20160128/gdac.broadinstitute.org_LGG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
530
</td>
<td style="text-align:left;">
lgg_tcga
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC, Cancer Cell 2010)
</td>
<td style="text-align:left;">
Prostate (MSKCC 2010)
</td>
<td style="text-align:left;">
Comprehensive profiling of 218 prostate tumors (181 primaries, 37 metastases) and 14 prostate cancer cell lines and xenografts. MSKCC Prostate Oncogenome Project.<br><a href="http://cbio.mskcc.org/cancergenomics/prostate/data/">Normalized flat data files</a>. <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21032">Raw data via GEO (GSE21032)</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
20579941
</td>
<td style="text-align:left;">
Taylor et al. Cancer Cell 2010
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
240
</td>
<td style="text-align:left;">
prad_mskcc
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hypodiploid Acute Lymphoid Leukemia (St Jude, Nat Genet 2013)
</td>
<td style="text-align:left;">
ALL (St. Jude 2013)
</td>
<td style="text-align:left;">
Whole genome or exome sequencing of 44 (20 whole genome, 20 exome) ALL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23334668
</td>
<td style="text-align:left;">
Holmfeldt et al. Nat Genet 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
44
</td>
<td style="text-align:left;">
all_stjude_2013
</td>
<td style="text-align:left;">
myeloid
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Recurrent and Metastatic Head &amp; Neck Cancer (MSKCC, JAMA Oncol 2016)
</td>
<td style="text-align:left;">
HNC (MSKCC 2016)
</td>
<td style="text-align:left;">
Targeted sequencing of 151 patients with advanced, treatment resistant head and neck tumors
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27442865
</td>
<td style="text-align:left;">
Morris et al. JAMA Oncol 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
151
</td>
<td style="text-align:left;">
hnc_mskcc_2016
</td>
<td style="text-align:left;">
head_neck
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Lung Cancer (U Cologne, Nature 2015)
</td>
<td style="text-align:left;">
Small Cell Lung (UCOLOGNE 2015)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 120 small cell lung cancer(SCLC) tumour samples and matched normal material.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26168399
</td>
<td style="text-align:left;">
George et al. Nature 2015
</td>
<td style="text-align:left;">
SCLCPORTAL;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
120
</td>
<td style="text-align:left;">
sclc_ucologne_2015
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Gastric Cancer (OncoSG, 2018)
</td>
<td style="text-align:left;">
STAD (OncoSG)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 147 gastric cancer tumors with matched normals. Courtesy of <a href="https://src.gisapps.org/OncoSG/">OncoSG</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29670109
</td>
<td style="text-align:left;">
Guo et al. Nat Commun 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-10 00:00:00
</td>
<td style="text-align:left;">
147
</td>
<td style="text-align:left;">
stad_oncosg_2018
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (Alpelisib plus AI, Nature Cancer 2020)
</td>
<td style="text-align:left;">
BREAST (Alpelisib)
</td>
<td style="text-align:left;">
The pre- and post-treatment ctDNA sequencing of 51 Breast Cancer patients to understand the mechanisms of resistance to Alepelisib.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-11 00:00:00
</td>
<td style="text-align:left;">
141
</td>
<td style="text-align:left;">
breast_alpelisib_2020
</td>
<td style="text-align:left;">
breast
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Pan-Cancer (DKFZ, Nature 2017)
</td>
<td style="text-align:left;">
Pediatric Pan-Cancer (DKFZ-2017)
</td>
<td style="text-align:left;">
Whole-genome/exome sequencing of 961 Pediatric Cancer samples. Data generated from Pfister Lab &amp; DKFZ/German Cancer Consortium.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29489754
</td>
<td style="text-align:left;">
Grobner et al. Nature 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-13 00:00:00
</td>
<td style="text-align:left;">
961
</td>
<td style="text-align:left;">
pediatric_dkfz_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia (Novartis/Broad, Nature 2012)
</td>
<td style="text-align:left;">
CCLE (Novartis/Broad 2012)
</td>
<td style="text-align:left;">
Targeted sequencing of 1020 samples from Cancer Cell Line Encyclopedia from the Broad Institute and Novartis.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22460905
</td>
<td style="text-align:left;">
Barretina et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-16 00:00:00
</td>
<td style="text-align:left;">
1020
</td>
<td style="text-align:left;">
cellline_ccle_broad
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, Cancer Cell 2014)
</td>
<td style="text-align:left;">
chRCC (TCGA 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 66 chromophobe renal cell carcinoma (ChRCCs) tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25155756
</td>
<td style="text-align:left;">
TCGA, Cancer Cell 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-16 00:00:00
</td>
<td style="text-align:left;">
66
</td>
<td style="text-align:left;">
kich_tcga_pub
</td>
<td style="text-align:left;">
chrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Colorectal (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing in 224 of the 276 colorectal carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Colorectal Cancer project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22810696
</td>
<td style="text-align:left;">
TCGA, Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-16 00:00:00
</td>
<td style="text-align:left;">
276
</td>
<td style="text-align:left;">
coadread_tcga_pub
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma (TCGA, Nature 2008)
</td>
<td style="text-align:left;">
GBM (TCGA 2008)
</td>
<td style="text-align:left;">
Targeted sequencing in 91 of the 206 primary glioblastoma tumors (143 with matched normals) from the Cancer Genome Atlas (TCGA) Glioblastoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
18772890
</td>
<td style="text-align:left;">
TCGA, Nature 2008
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-16 00:00:00
</td>
<td style="text-align:left;">
206
</td>
<td style="text-align:left;">
gbm_tcga_pub
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (Broad, Nature 2015)
</td>
<td style="text-align:left;">
CLL (Broad 2015)
</td>
<td style="text-align:left;">
Whole Exome Sequencing of 537 chronic lymphocytic leukemia samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26466571
</td>
<td style="text-align:left;">
Landau et al. Nature 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-16 00:00:00
</td>
<td style="text-align:left;">
537
</td>
<td style="text-align:left;">
cll_broad_2015
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Ewing Sarcoma (DFCI, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Ewing Sarcoma (DFCI)
</td>
<td style="text-align:left;">
Whole exome sequencing of 96 pediatric Ewing Sarcoma tumors and 11 cell lines.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25186949
</td>
<td style="text-align:left;">
Crompton et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-17 00:00:00
</td>
<td style="text-align:left;">
107
</td>
<td style="text-align:left;">
es_dfarber_broad_2014
</td>
<td style="text-align:left;">
es
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small-Cell Lung Cancer (Multi-Institute, Cancer Cell 2017)
</td>
<td style="text-align:left;">
SCLC (GARDNER 2017)
</td>
<td style="text-align:left;">
Whole Exome Sequencing of 10 paired chemosensitive and chemoresistant SCLC PDX.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28196596
</td>
<td style="text-align:left;">
Gardner et al. Cancer Cell 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-17 00:00:00
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
sclc_cancercell_gardner_2017
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Breast (TCGA pub)
</td>
<td style="text-align:left;">
Whole-exome sequencing (510 samples with matched normals), genomic DNA copy number arrays, DNA methylation, messenger RNA arrays, microRNA sequencing and reverse-phase protein arrays analysis on 825 primary breast cancer samples. <a href="http://cancergenome.nih.gov/">The Cancer Genome Atlas (TCGA)</a> Breast Invasive Carcinoma project. 825 cases.<br><i>Nature 2012.</i> <a href="https://tcga-data.nci.nih.gov/docs/publications/brca_2012/">Raw data via the TCGA Data Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23000897
</td>
<td style="text-align:left;">
TCGA, Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-18 00:00:00
</td>
<td style="text-align:left;">
825
</td>
<td style="text-align:left;">
brca_tcga_pub
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (METABRIC, Nature 2012 &amp; Nat Commun 2016)
</td>
<td style="text-align:left;">
Breast (METABRIC 2016)
</td>
<td style="text-align:left;">
Targeted sequencing of 2509 primary breast tumors with 548 matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27161491
</td>
<td style="text-align:left;">
Pereira et al. Nat Commun 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-18 00:00:00
</td>
<td style="text-align:left;">
2509
</td>
<td style="text-align:left;">
brca_metabric
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Wilms’ Tumor (TARGET, 2018)
</td>
<td style="text-align:left;">
WT (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 657 Wilms’ tumor samples. Comprehensive profiling of 657 Wilms’ Tumor samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-18 00:00:00
</td>
<td style="text-align:left;">
657
</td>
<td style="text-align:left;">
wt_target_2018_pub
</td>
<td style="text-align:left;">
wt
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
BLCA (TCGA 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 412 muscle-invasive bladder tumors and matched normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28988769
</td>
<td style="text-align:left;">
Robertson et al. Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-18 00:00:00
</td>
<td style="text-align:left;">
413
</td>
<td style="text-align:left;">
blca_tcga_pub_2017
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Basal Cell Carcinoma (UNIGE, Nat Genet 2016)
</td>
<td style="text-align:left;">
BCC (UNIGE 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 126 basal cell carcinoma tumor/normal pairs; targeted sequencing of 163 sporadic samples (40 tumor/normal pairs) and 4 Gorlin symdrome basal cell carcinomas.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26950094
</td>
<td style="text-align:left;">
Bonilla et al. Nat Genet 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
293
</td>
<td style="text-align:left;">
bcc_unige_2016
</td>
<td style="text-align:left;">
bcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Squamous Cell Carcinoma (ICGC, Nature 2014)
</td>
<td style="text-align:left;">
Esophagus sq (ICGC)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 88 esophageal squamous cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24670651
</td>
<td style="text-align:left;">
Song et al. Nature 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
88
</td>
<td style="text-align:left;">
escc_icgc
</td>
<td style="text-align:left;">
escc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Melanoma (MSKCC, JCO Precis Oncol 2017)
</td>
<td style="text-align:left;">
SKCM (Vanderbilt/Mskcc)
</td>
<td style="text-align:left;">
Targeted sequencing of pre-treatment 66 Melanoma tumor samples. <a href="https://ascopubs.org/doi/full/10.1200/PO.16.00054">PubMed</a>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
Catalanotti et al. JCO Precis Oncol 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
66
</td>
<td style="text-align:left;">
skcm_vanderbilt_mskcc_2015
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioma (MSK, Nature 2019)
</td>
<td style="text-align:left;">
DIFG(MSK 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of CSF samples (91 samples with 86 matched normals) from 42 glioma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30675060
</td>
<td style="text-align:left;">
Miller et al. Nature 2019
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
91
</td>
<td style="text-align:left;">
glioma_msk_2018
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Cancer (MSK, Cell Metab 2020)
</td>
<td style="text-align:left;">
PROSTATE (MSK 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT Sequencing of 18 prostate cancer tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31564440
</td>
<td style="text-align:left;">
Granlund et al. Cell Metab 2020
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
prad_msk_2019
</td>
<td style="text-align:left;">
prostate
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma Project (J Clin Invest 2019)
</td>
<td style="text-align:left;">
(AACRF,2019)
</td>
<td style="text-align:left;">
Multi-Institute Cohort of 1045 Adenoid Cystic Carcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31483290
</td>
<td style="text-align:left;">
Allen et al. J Clin Invest 2019
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
1049
</td>
<td style="text-align:left;">
acc_2019
</td>
<td style="text-align:left;">
acc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (MSKCC, NPJ Breast Cancer 2019)
</td>
<td style="text-align:left;">
BRCA (MSKCC 2019)
</td>
<td style="text-align:left;">
Targeted Sequencing of buparlisib + letrozole and alpelisib + letrozole-treated metastatic ER+ unmatched breast tumors.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31552290
</td>
<td style="text-align:left;">
Nixon et al. NPJ Breast Cancer 2019
</td>
<td style="text-align:left;">
SOLITD;COMPONC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
70
</td>
<td style="text-align:left;">
brca_mskcc_2019
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Tumors with TRK fusions (MSK, Clin Cancer Res 2020)
</td>
<td style="text-align:left;">
NTRK (MSKCC 2019)
</td>
<td style="text-align:left;">
Tumors with TRK fusions based on MSK-IMPACT targeted sequencing.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31871300
</td>
<td style="text-align:left;">
Rosen et al. Clin Cancer Res 2020
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
106
</td>
<td style="text-align:left;">
ntrk_msk_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSK, Eur Urol 2020)
</td>
<td style="text-align:left;">
PRAD (MSKCC, 2020)
</td>
<td style="text-align:left;">
Targeted sequencing of 1465 prostate adenocarcinoma tumor/normal pairs via MSK-IMPACT.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32317181
</td>
<td style="text-align:left;">
Nguyen et al. Eur Urol 2020
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-22 00:00:00
</td>
<td style="text-align:left;">
1465
</td>
<td style="text-align:left;">
prad_cdk12_mskcc_2020
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (U Tokyo, Nat Genet 2014)
</td>
<td style="text-align:left;">
Stomach (UTokyo)
</td>
<td style="text-align:left;">
Whole exome sequencing of 30 diffuse-type gastric adenocarcinoma samples (with matched normals) from the University of Tokyo.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24816255
</td>
<td style="text-align:left;">
Kakiuchi et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
30
</td>
<td style="text-align:left;">
stad_utokyo
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
The Metastatic Prostate Cancer Project (Provisional, November 2019)
</td>
<td style="text-align:left;">
The MPC Project
</td>
<td style="text-align:left;">
The Metastatic Prostate Cancer Project is a patient-driven initiative. This study includes genomic data, cfDNA tumor fractions, patient-reported data (pre-pended as PRD), medical record data (MedR), and pathology report data (PATH). As these data were generated in a research, not a clinical, laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. All annotations have been de-identified. More information is available at www.mpcproject.org. For questions about these data, including how to cite, please visit www.mpcproject.org/data-release or direct questions to <a href="mailto:data@mpcproject.org" class="email">data@mpcproject.org</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
75
</td>
<td style="text-align:left;">
prad_mpcproject_2018
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Carcinoma (Cornell/Baylor/MDACC, Nat Commun 2019)
</td>
<td style="text-align:left;">
UTUC (Cornell/Baylor/MDACC 2019)
</td>
<td style="text-align:left;">
Whole exome sequencing (37 samples) and RNA-seq analysis (32 samples) of 47 upper tract urothelial carcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31278255
</td>
<td style="text-align:left;">
Robinson et al. Nat Commun 2019
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
47
</td>
<td style="text-align:left;">
utuc_cornell_baylor_mdacc_2019
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Preclinical Testing Consortium (CHOP, Cell Rep 2019)
</td>
<td style="text-align:left;">
PPTC (2019)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 261 patient derived xenografts (PDXs) samples from high-risk childhood cancers (with no matched normals). Website: <a href="http://www.ncipptc.org/">PPTC</a>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31693904
</td>
<td style="text-align:left;">
Rotika et al. Cell Rep 2019
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
261
</td>
<td style="text-align:left;">
pptc_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Carcinoma (MSK, Nat Commun 2020)
</td>
<td style="text-align:left;">
UTUC (MSK, 2019)
</td>
<td style="text-align:left;">
Targeted Sequencing of 119 upper tract urothelial carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32332851
</td>
<td style="text-align:left;">
Kwanghee et al. Nat Commun 2020
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
119
</td>
<td style="text-align:left;">
utuc_msk_2019
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Carcinoma PDX (MSK, Nat Commun 2020)
</td>
<td style="text-align:left;">
UTUC PDX (MSK, 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 34 upper tract urothelial patients and the matching PDX or PDC generated from 17 patient tumors.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32332851
</td>
<td style="text-align:left;">
Kwanghee et al. Nat Commun 2020
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
utuc_pdx_msk_2019
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Cancer-TRAP Project (MSK, Lancet Oncol 2020)
</td>
<td style="text-align:left;">
EGC-TRAP (MSK 2020)
</td>
<td style="text-align:left;">
Targeted and WES recapture sequencing of tumor/normal samples from 37 Esophageal Cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32437664
</td>
<td style="text-align:left;">
Janjigian et al. Lancet Oncol 2020
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
68
</td>
<td style="text-align:left;">
egc_trap_msk_2020
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Sarcoma/Mesenchymal (MSK, Clin Cancer Res 2020)
</td>
<td style="text-align:left;">
USARC (MSK, 2020)
</td>
<td style="text-align:left;">
Targeted Sequencing of 108 Uterine Sarcoma/Mesenchymal tumor/normal pairs from 107 patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32299819
</td>
<td style="text-align:left;">
Hensley et al. Clin Cancer Res 2020
</td>
<td style="text-align:left;">
DONOGHUM;CHAVANS;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-25 00:00:00
</td>
<td style="text-align:left;">
108
</td>
<td style="text-align:left;">
usarc_msk_2020
</td>
<td style="text-align:left;">
usarc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (Fred Hutchinson CRC, Nat Med 2016)
</td>
<td style="text-align:left;">
Prostate (FHCRC, 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing, array comparative genomic hybridization (CGH) and RNA transcript profiling of 176 tumors from 63 patients with disseminated prostate cancer with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26928463
</td>
<td style="text-align:left;">
Kumar et al. Nat Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
176
</td>
<td style="text-align:left;">
prad_fhcrc
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (Broad/Cornell, Nat Genet 2012)
</td>
<td style="text-align:left;">
Prostate (Broad/Cornell 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 112 prostate adenocarcinomas and matched normal samples. Generated by Levi Garraway’s lab at the Broad Institute and Mark Rubin’s lab at Cornell.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22610119
</td>
<td style="text-align:left;">
Barbieri et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
112
</td>
<td style="text-align:left;">
prad_broad
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Stomach (TCGA pub)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 295 primary gastric adenocarcinomas tumors with matched normals. TCGA Stomach Adenocarcinoma Project; raw data at the <a href="https://tcga-data.nci.nih.gov/docs/publications/stad_2014/">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25079317
</td>
<td style="text-align:left;">
TCGA, Nature 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
295
</td>
<td style="text-align:left;">
stad_tcga_pub
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Papillary Thyroid Carcinoma (TCGA, Cell 2014)
</td>
<td style="text-align:left;">
Thyroid (TCGA pub)
</td>
<td style="text-align:left;">
Exome and whole genome sequencing of 496 papillary thyroid carcinoma tumor/normal pairs
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25417114
</td>
<td style="text-align:left;">
TCGA, Cell 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
496
</td>
<td style="text-align:left;">
thca_tcga_pub
</td>
<td style="text-align:left;">
thpa
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, Nature 2013)
</td>
<td style="text-align:left;">
Uterine (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing of 373 endometrial carcinoma tumors with matched normals. TCGA Uterine Corpus Endometrial Carcinoma project; raw data at the <a href="https://tcga-data.nci.nih.gov/docs/publications/ucec_2013/">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23636398
</td>
<td style="text-align:left;">
TCGA, Nature 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
373
</td>
<td style="text-align:left;">
ucec_tcga_pub
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (QCMG, Nature 2016)
</td>
<td style="text-align:left;">
Pancreas (QCMG 2016)
</td>
<td style="text-align:left;">
Whole-genome and deep-exome sequencing analysis of 456 pancreatic ductal adenocarcinomas
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26909576
</td>
<td style="text-align:left;">
Bailey et al. Nature 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
456
</td>
<td style="text-align:left;">
paad_qcmg_uq_2016
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acral Melanoma (TGEN, Genome Res 2017)
</td>
<td style="text-align:left;">
TSAM
</td>
<td style="text-align:left;">
Whole exome sequencing and transcriptome analysis of 34 Acral Melanoma patients (33 with matched normals).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28373299
</td>
<td style="text-align:left;">
Liang et al. Genome Res 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
38
</td>
<td style="text-align:left;">
mel_tsam_liang_2017
</td>
<td style="text-align:left;">
mel
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Neuroblastoma (TARGET, 2018)
</td>
<td style="text-align:left;">
NBL (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing on 1089 NBL samples. Comprehensive profiling of 1089 NBL samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
1089
</td>
<td style="text-align:left;">
nbl_target_2018_pub
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mature B-cell malignancies (MD Anderson Cancer Center)
</td>
<td style="text-align:left;">
MBN (MDACC 2013)
</td>
<td style="text-align:left;">
We performed targeted deep sequencing of 380 recurrently mutated genes across a cohort of 755 mature B-cell malignancies to define conserved and disease-specific patterns of genomic alterations.<a href="https://www.biorxiv.org/content/10.1101/674259v1.full">PubMed</a>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-29 00:00:00
</td>
<td style="text-align:left;">
760
</td>
<td style="text-align:left;">
mbn_mdacc_2013
</td>
<td style="text-align:left;">
mbn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (SMMU, Eur Urol 2017)
</td>
<td style="text-align:left;">
Prostate (Eur Urol 2017)
</td>
<td style="text-align:left;">
Whole-genome and transcriptome sequencing of 65 Prostate Adenocarcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28927585
</td>
<td style="text-align:left;">
Shancheng et al. Eur Urol 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-30 00:00:00
</td>
<td style="text-align:left;">
65
</td>
<td style="text-align:left;">
prad_eururol_2017
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Brain Tumor PDXs (Mayo Clinic, 2019)
</td>
<td style="text-align:left;">
GBM (Mayo PDX)
</td>
<td style="text-align:left;">
WES on a total of 83 PDXs and 55 PDXs have the matched germline (blood) samples sequenced. A total of 66 PDXs have RNASeq available
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
SHULANTIANMAYO;PUBLIC;DECKERPAULMAYO
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-30 00:00:00
</td>
<td style="text-align:left;">
97
</td>
<td style="text-align:left;">
gbm_mayo_pdx_sarkaria_2019
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cutaneous T Cell Lymphoma (Columbia U, Nat Genet 2015)
</td>
<td style="text-align:left;">
CTCL (Columbia 2015)
</td>
<td style="text-align:left;">
Whole-Exome Sequencing (WXS) of tumor-normal sample pairs from 25 patients with Sezary Syndrome and 17 patients with other Cutaneous T Cell Lymphoma (CTCLs).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26551667
</td>
<td style="text-align:left;">
Da Silva Almeida et al. Nat Genet 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-01 00:00:00
</td>
<td style="text-align:left;">
43
</td>
<td style="text-align:left;">
ctcl_columbia_2015
</td>
<td style="text-align:left;">
nhl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Carcinoma (TCGA, Nature 2017)
</td>
<td style="text-align:left;">
Stomach/Esophageal (TCGA)
</td>
<td style="text-align:left;">
Whole exome sequencing of 559 esophageal and gastric carcinoma tumors with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28052061
</td>
<td style="text-align:left;">
TCGA, Nature 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-01 00:00:00
</td>
<td style="text-align:left;">
559
</td>
<td style="text-align:left;">
stes_tcga_pub
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (MSKCC, J Clin Onco 2013)
</td>
<td style="text-align:left;">
Bladder (MSKCC 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of 97 high-grade bladder tumors, including targeted sequencing of 15 cancer genes
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23897969
</td>
<td style="text-align:left;">
Iyer et al. J Clin Oncol 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
97
</td>
<td style="text-align:left;">
blca_mskcc_solit_2012
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Cell 2015)
</td>
<td style="text-align:left;">
Breast (TCGA 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 817 Breast Invasive Carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Breast Invasive Carcinoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26451490
</td>
<td style="text-align:left;">
TCGA, Cell 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
817
</td>
<td style="text-align:left;">
brca_tcga_pub2015
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma (TCGA, Cell 2013)
</td>
<td style="text-align:left;">
GBM (TCGA 2013)
</td>
<td style="text-align:left;">
Whole-exome and/or whole-genome sequencing of 257 of the 543 glioblastoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Glioblastoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24120142
</td>
<td style="text-align:left;">
TCGA, Cell 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
543
</td>
<td style="text-align:left;">
gbm_tcga_pub2013
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Prostate Cancer (SU2C/PCF Dream Team, Cell 2015)
</td>
<td style="text-align:left;">
Prostate (SU2C)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 150 metastatic prostate cancer tumor/normal pairs by the SU2C/PCF Dream Team.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26000489
</td>
<td style="text-align:left;">
Robinson et al. Cell 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
150
</td>
<td style="text-align:left;">
prad_su2c_2015
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Bladder (TCGA 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 131 high-grade muscle-invasive urothelial bladder carcinomas
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24476821
</td>
<td style="text-align:left;">
TCGA, Nature 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
131
</td>
<td style="text-align:left;">
blca_tcga_pub
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Acute Myeloid Leukemia (TARGET, 2018)
</td>
<td style="text-align:left;">
AML (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing on 333 TARGET-AML samples. Comprehensive profiling of TARGET-AML samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
1025
</td>
<td style="text-align:left;">
aml_target_2018_pub
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia (Broad, 2019)
</td>
<td style="text-align:left;">
CCLE (BROAD 2019)
</td>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia from the Broad Institute and Novartis, updated 2019. Data downloaded from the <a href="https://portals.broadinstitute.org/ccle/data">Broad CCLE Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31068700
</td>
<td style="text-align:left;">
Stransky et al. Nature. 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
1739
</td>
<td style="text-align:left;">
ccle_broad_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (OHSU, Nature 2018)
</td>
<td style="text-align:left;">
AML (OHSU)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 672 acute myeloid leukemia samples (with 454 matched normals) from the Beat AML program.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30333627
</td>
<td style="text-align:left;">
Tyner et al. Nature 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-02 00:00:00
</td>
<td style="text-align:left;">
672
</td>
<td style="text-align:left;">
aml_ohsu_2018
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, NEJM 2013)
</td>
<td style="text-align:left;">
AML (TCGA pub)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing analysis of 200 adult de novo AML tumor/normal pairs. TCGA Acute Myeloid Leukemia Project; raw data at the <a href="https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=LAML&amp;diseaseName=Acute%20Myeloid%20Leukemia">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23634996
</td>
<td style="text-align:left;">
TCGA, NEJM 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-04 00:00:00
</td>
<td style="text-align:left;">
200
</td>
<td style="text-align:left;">
laml_tcga_pub
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, Nature 2013)
</td>
<td style="text-align:left;">
ccRCC (TCGA pub)
</td>
<td style="text-align:left;">
Whole-exome sequencing in 417 of 446 clear cell renal cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23792563
</td>
<td style="text-align:left;">
TCGA, Nature 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-04 00:00:00
</td>
<td style="text-align:left;">
446
</td>
<td style="text-align:left;">
kirc_tcga_pub
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, Nature 2015)
</td>
<td style="text-align:left;">
Head &amp; neck (TCGA pub)
</td>
<td style="text-align:left;">
Whole-exome sequencing and/or whole-genome sequencing of 279 head and neck squamous cell carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) head and neck squamous cell carcinoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25631445
</td>
<td style="text-align:left;">
TCGA, Nature 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-04 00:00:00
</td>
<td style="text-align:left;">
279
</td>
<td style="text-align:left;">
hnsc_tcga_pub
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Lung adeno (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing of 230 lung adenocarcinoma tumor/normal pairs. <a href="https://tcga-data.nci.nih.gov/docs/publications/luad_2014/">Raw data via the TCGA Data Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25079552
</td>
<td style="text-align:left;">
TCGA, Nature 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-04 00:00:00
</td>
<td style="text-align:left;">
230
</td>
<td style="text-align:left;">
luad_tcga_pub
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Lung squ (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing of 178 lung squamous cell carcinoma tumor/normal pairs. <a href="http://cancergenome.nih.gov/">The Cancer Genome Atlas (TCGA)</a> Lung Squamous Cell Carcinoma project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22960745
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-04 00:00:00
</td>
<td style="text-align:left;">
178
</td>
<td style="text-align:left;">
lusc_tcga_pub
</td>
<td style="text-align:left;">
lusc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroendocrine Prostate Cancer (Multi-Institute, Nat Med 2016)
</td>
<td style="text-align:left;">
NEPC (Multi-Institute 2016)
</td>
<td style="text-align:left;">
Whole exome sequencing of 114 metastatic tumor/normal pairs to understand genomic overlap between castration resistant adenocarcinoma (CRPC-Adeno) and neuroendocrine histologies (CRPC-NE).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26855148
</td>
<td style="text-align:left;">
Beltran et al. Nat Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-04 00:00:00
</td>
<td style="text-align:left;">
114
</td>
<td style="text-align:left;">
nepc_wcm_2016
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
PCPG (TCGA)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 178 Pheochromocytoma or Paraganglioma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28162975
</td>
<td style="text-align:left;">
Fishbein et al. Cancer Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-04 00:00:00
</td>
<td style="text-align:left;">
178
</td>
<td style="text-align:left;">
pcpg_tcga_pub
</td>
<td style="text-align:left;">
mnet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mantle Cell Lymphoma (IDIBIPS, PNAS 2013)
</td>
<td style="text-align:left;">
MCL (IDIBIPS 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 29 primary mantle cell lymphomas and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24145436
</td>
<td style="text-align:left;">
BeÃ  et al. PNAS 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-07 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
mcl_idibips_2013
</td>
<td style="text-align:left;">
mcl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cystic Tumor of the Pancreas (Johns Hopkins, PNAS 2011)
</td>
<td style="text-align:left;">
Pancreas (JHU 2011)
</td>
<td style="text-align:left;">
Whole-exome sequencing of major neoplastic cyst types of the pancreas: 8 serous cystadenomas (SCAs), 8 intraductal papillary mucinous neoplasms (IPMNs), 8 mucinous cystic neoplasms (MCNs), and 8 solid pseudopapillary neoplasms (SPNs).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22158988
</td>
<td style="text-align:left;">
Wu et al. PNAS 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-07 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
pact_jhu_2011
</td>
<td style="text-align:left;">
pact
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Colorectal Cancer (MSKCC, Cancer Cell 2018)
</td>
<td style="text-align:left;">
CRC(MSK 2018)
</td>
<td style="text-align:left;">
Targeted sequencing of 1134 metastatic colorectal tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29316426
</td>
<td style="text-align:left;">
Yaeger et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-07 00:00:00
</td>
<td style="text-align:left;">
1134
</td>
<td style="text-align:left;">
crc_msk_2017
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
The Angiosarcoma Project - Count Me In (Nature Medicine, 2020)
</td>
<td style="text-align:left;">
ASC Project
</td>
<td style="text-align:left;">
The Angiosarcoma Project is an ongoing patient-driven initiative. This archived Angiosarcoma Project dataset was analyzed in the linked Nature Medicine 2020 manuscript, and is derived from 48 samples from 36 angiosarcoma patients. Angiosarcoma tumor specimens (FFPE) were subjected to whole-exome sequencing (along with matched germline whole-exome sequencing). In addition to genomic data, this study includes patient-reported data (pre-pended as PRD), medical record data (MedR), and pathology report data (PATH). All annotations have been de-identified. Questions about these data can be directed to <a href="mailto:data@ascproject.org" class="email">data@ascproject.org</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32042194
</td>
<td style="text-align:left;">
Painter et al. Nat Med 2020
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-10 00:00:00
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
angs_project_painter_2018
</td>
<td style="text-align:left;">
angs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Prostate Adenocarcinoma (SU2C/PCF Dream Team, PNAS 2019)
</td>
<td style="text-align:left;">
Prostate (SU2C 2019)
</td>
<td style="text-align:left;">
Whole exome sequencing of 444 castrate resistant prostate cancer tumor/normal pairs. SU2C-PCF Dream Team: Precision Therapy for Advanced Prostate Cancer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31061129
</td>
<td style="text-align:left;">
Abida et al. PNAS 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-11 00:00:00
</td>
<td style="text-align:left;">
444
</td>
<td style="text-align:left;">
prad_su2c_2019
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (TRACERx, NEJM &amp; Nature 2017)
</td>
<td style="text-align:left;">
NSCLC (TRACERx 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 327 primary tumors with matched germline samples, and phylogenetic ctDNA sequencing of 96 pre- and 24 postoperative samples from 100 NSCLC TRACERx patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28445112,28445469
</td>
<td style="text-align:left;">
Hanjani et al. NEJM 2017, Abbosh et al. Nature 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-15 00:00:00
</td>
<td style="text-align:left;">
447
</td>
<td style="text-align:left;">
nsclc_tracerx_2017
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Adenocarcinoma (DFCI, Nat Genet 2013)
</td>
<td style="text-align:left;">
Esophagus (Broad)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 149 tumor/normal pairs and whole-genome sequencing of 16 pairs from 149 esophageal adenocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23525077
</td>
<td style="text-align:left;">
Dulak et al. Nat Genet 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-18 00:00:00
</td>
<td style="text-align:left;">
151
</td>
<td style="text-align:left;">
esca_broad
</td>
<td style="text-align:left;">
esca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, Cell 2015)
</td>
<td style="text-align:left;">
Prostate (TCGA 2015)
</td>
<td style="text-align:left;">
Whole exome and/or whole genome sequencing of 333 primary prostate adenocarcinoma tumor/normal pairs; raw data at the <a href="https://tcga-data.nci.nih.gov/docs/publications/prad_2015/">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26544944
</td>
<td style="text-align:left;">
TCGA, Cell 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-18 00:00:00
</td>
<td style="text-align:left;">
333
</td>
<td style="text-align:left;">
prad_tcga_pub
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Myelodysplastic (MSKCC, 2020)
</td>
<td style="text-align:left;">
MDS (MSKCC)
</td>
<td style="text-align:left;">
Single-cell genomics reveals the genetic and molecular bases for escape from mutational epistasis in myeloid neoplasms
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27276561,30333627,24030381
</td>
<td style="text-align:left;">
Papaemmanuil et al. NEJM 2016; Tyner et al. Nature 2018; Papaemmanuil et al. Blood 2013
</td>
<td style="text-align:left;">
PENSONA;OMARHISTCOBI
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-07-18 00:00:00
</td>
<td style="text-align:left;">
4231
</td>
<td style="text-align:left;">
mds_mskcc_2020
</td>
<td style="text-align:left;">
mds
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cbiodatapack" class="section level2">
<h2 class="hasAnchor">
<a href="#cbiodatapack" class="anchor"></a>cBioDataPack()</h2>
<table class="table table table-hover table-striped table-responsive">
<caption>
List of available packaged studies in cBioDataPack
</caption>
<thead><tr>
<th style="text-align:left;">
cancer_study_id
</th>
<th style="text-align:left;">
study_name
</th>
<th style="text-align:left;">
description
</th>
<th style="text-align:left;">
URL
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
paac_jhu_2014
</td>
<td style="text-align:left;">
Acinar Cell Carcinoma of the Pancreas (JHU, J Pathol 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 23 surgically resected pancreatic carcinomas with acinar differentiation and their matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mel_tsam_liang_2017
</td>
<td style="text-align:left;">
Acral Melanoma (TGEN, Genome Res 2017)
</td>
<td style="text-align:left;">
Whole exome sequencing and transcriptome analysis of 34 Acral Melanoma patients (33 with matched normals).
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
all_stjude_2015
</td>
<td style="text-align:left;">
Acute Lymphoblastic Leukemia (St Jude, Nat Genet 2015)
</td>
<td style="text-align:left;">
Comprehensive profiling of infant MLL-rearranged acute lymphoblastic leukemia (MLL-R ALL)
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
all_stjude_2016
</td>
<td style="text-align:left;">
Acute Lymphoblastic Leukemia (St Jude, Nat Genet 2016)
</td>
<td style="text-align:left;">
Whole-genome and/or whole-exome sequencing of ERG-altered B-ALL tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
aml_ohsu_2018
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia (OHSU, Nature 2018)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 672 acute myeloid leukemia samples (with 454 matched normals) from the Beat AML program.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
laml_tcga_pub
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, NEJM 2013)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing analysis of 200 adult de novo AML tumor/normal pairs. TCGA Acute Myeloid Leukemia Project; raw data at the NCI.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=LAML&amp;diseaseName=Acute%20Myeloid%20Leukemia" class="uri">https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=LAML&amp;diseaseName=Acute%20Myeloid%20Leukemia</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
laml_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
laml_tcga
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Acute Myeloid Leukemia; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LAML/20160128/gdac.broadinstitute.org_LAML.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LAML/20160128/gdac.broadinstitute.org_LAML.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
mnm_washu_2016
</td>
<td style="text-align:left;">
Acute myeloid leukemia or myelodysplastic syndromes (WashU, 2016)
</td>
<td style="text-align:left;">
Enhanced exome or targeted sequencing of 116 acute myeloid leukemia (AML) or myelodysplastic syndromes (MDS) patients’ tumor/normal pairs .
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acyc_fmi_2014
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (FMI, Am J Surg Pathl. 2014)
</td>
<td style="text-align:left;">
Comprehensive genomic profiling of 28 metastatic ACCs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acyc_jhu_2016
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (JHU, Cancer Prev Res 2016)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 25 adenoid cystic carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acyc_mda_2015
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MDA, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
WGS of 21 salivary ACCs and targeted molecular analyses of a validation set (81 patients).
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acyc_mgh_2016
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MGH, Nat Gen 2016)
</td>
<td style="text-align:left;">
Genomic profiling of 10 ACC PDX models.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acyc_mskcc_2013
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MSKCC, Nat Genet 2013)
</td>
<td style="text-align:left;">
Whole-exome or whole-genome sequencing analysis of 60 ACC tumor/normal pairs
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acyc_sanger_2013
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (Sanger/MDA, JCI 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 24 ACCs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acc_2019
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma Project (2019)
</td>
<td style="text-align:left;">
Multi-Institute Cohort of 1045 Adenoid Cystic Carcinoma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acbc_mskcc_2015
</td>
<td style="text-align:left;">
Adenoid Cystic Carcinoma of the Breast (MSKCC, J Pathol. 2015)
</td>
<td style="text-align:left;">
Whole exome sequencing of 12 breast AdCCs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
acc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Adrenocortical Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Adrenocortical Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
acc_tcga
</td>
<td style="text-align:left;">
Adrenocortical Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Adrenocortical Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ACC/20160128/gdac.broadinstitute.org_ACC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ACC/20160128/gdac.broadinstitute.org_ACC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
sarc_tcga_pub
</td>
<td style="text-align:left;">
Adult Soft Tissue Sarcomas (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
Mutation data from whole exome sequencing of 206 Adult Soft Tissue Sarcomas with paired normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ampca_bcm_2016
</td>
<td style="text-align:left;">
Ampullary Carcinoma (Baylor College of Medicine, Cell Reports 2016)
</td>
<td style="text-align:left;">
Exome sequencing analysis of 160 cancers arising in the periampullary region, 62 of these clearly arising from either the bile duct (n = 44) or the duodenum (n = 18) and 98 for which the epithelium of origin could not be clearly defined morphologically (AMPCA).
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
odg_msk_2017
</td>
<td style="text-align:left;">
Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytoma (MSKCC, Neuro Oncol 2017)
</td>
<td style="text-align:left;">
Genomic characterization using next generation sequencing (NGS) in 22 Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytomas.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
bcc_unige_2016
</td>
<td style="text-align:left;">
Basal Cell Carcinoma (UNIGE, Nat Genet 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 126 basal cell carcinoma tumor/normal pairs; targeted sequencing of 163 sporadic samples (40 tumor/normal pairs) and 4 Gorlin symdrome basal cell carcinomas.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_mskcc_solit_2014
</td>
<td style="text-align:left;">
Bladder Cancer (MSKCC, Eur Urol 2014)
</td>
<td style="text-align:left;">
Genomic Predictors of Survival in Patients with High-grade Urothelial Carcinoma of the Bladder. Targeted sequencing of 109 tumor samples (MSKCC, European Urology 2014).
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_mskcc_solit_2012
</td>
<td style="text-align:left;">
Bladder Cancer (MSKCC, J Clin Onco 2013)
</td>
<td style="text-align:left;">
Comprehensive profiling of 97 high-grade bladder tumors, including targeted sequencing of 15 cancer genes
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_plasmacytoid_mskcc_2016
</td>
<td style="text-align:left;">
Bladder Cancer (MSKCC, Nat Genet 2016)
</td>
<td style="text-align:left;">
Whole exome and targeted sequencing of 34 plasmacytoid-variant (signet ring cell) tumors of the bladder
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_tcga_pub_2017
</td>
<td style="text-align:left;">
Bladder Cancer (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 412 muscle-invasive bladder tumors and matched normal samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_bgi
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (BGI, Nat Genet 2013)
</td>
<td style="text-align:left;">
Whole-exome and whole-genome sequencing of 99 bladder cancer tumor/normal pairs. Generated by BGI.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_dfarber_mskcc_2014
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (DFCI/MSKCC, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal samples from 50 patients treated with cisplatin-based chemotherapy for muscle invasive urothelial carcinoma.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_tcga_pub
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 131 high-grade muscle-invasive urothelial bladder carcinomas
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_tcga
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Bladder Urothelial Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BLCA/20160128/gdac.broadinstitute.org_BLCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BLCA/20160128/gdac.broadinstitute.org_BLCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
lgg_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Brain Lower Grade Glioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Brain Lower Grade Glioma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
lgg_tcga
</td>
<td style="text-align:left;">
Brain Lower Grade Glioma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Brain Lower Grade Glioma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LGG/20160128/gdac.broadinstitute.org_LGG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LGG/20160128/gdac.broadinstitute.org_LGG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
gbm_mayo_pdx_sarkaria_2019
</td>
<td style="text-align:left;">
Brain Tumor PDXs (Mayo Clinic, 2019)
</td>
<td style="text-align:left;">
WES on a total of 83 PDXs and 55 PDXs have the matched germline (blood) samples sequenced. A total of 66 PDXs have RNASeq available
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_metabric
</td>
<td style="text-align:left;">
Breast Cancer (METABRIC, Nature 2012 &amp; Nat Commun 2016)
</td>
<td style="text-align:left;">
Targeted sequencing of 2509 primary breast tumors with 548 matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
breast_msk_2018
</td>
<td style="text-align:left;">
Breast Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of tumor/normal sample pairs from 1918 Breast cancers.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_mskcc_2019
</td>
<td style="text-align:left;">
Breast Cancer (MSKCC, 2019)
</td>
<td style="text-align:left;">
Targeted Sequencing of buparlisib + letrozole and alpelisib + letrozole-treated metastatic ER+ unmatched breast tumors.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_bccrc_xenograft_2014
</td>
<td style="text-align:left;">
Breast Cancer Xenografts (British Columbia, Nature 2015)
</td>
<td style="text-align:left;">
Whole genome/targeted sequencing to evaluate the clonal dynamics in 116 breast cancer patient xenografts.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
bfn_duke_nus_2015
</td>
<td style="text-align:left;">
Breast Fibroepithelial Tumors (Duke-NUS, Nat Genet 2015)
</td>
<td style="text-align:left;">
Whole exome sequencing of 22 phyllodes tumors
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_bccrc
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma (British Columbia, Nature 2012)
</td>
<td style="text-align:left;">
Whole genome/exome sequencing analysis of 65 breast cancer samples
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_broad
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma (Broad, Nature 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 103 breast cancer tumor/normal sample pairs. Generated by the Broad Institute.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_sanger
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma (Sanger, Nature 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of protein coding and microRNA genes from 100 breast cancer tumor/normal sample pairs. Generated by the Sanger Institute.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_tcga_pub2015
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Cell 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 817 Breast Invasive Carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Breast Invasive Carcinoma Project.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_tcga_pub
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing (510 samples with matched normals), genomic DNA copy number arrays, DNA methylation, messenger RNA arrays, microRNA sequencing and reverse-phase protein arrays analysis on 825 primary breast cancer samples. The Cancer Genome Atlas (TCGA) Breast Invasive Carcinoma project. 825 cases.Nature 2012. Raw data via the TCGA Data Portal.
</td>
<td style="text-align:left;">
<a href="http://cancergenome.nih.gov/" class="uri">http://cancergenome.nih.gov/</a>, <a href="https://tcga-data.nci.nih.gov/docs/publications/brca_2012/" class="uri">https://tcga-data.nci.nih.gov/docs/publications/brca_2012/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_tcga
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Breast Invasive Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BRCA/20160128/gdac.broadinstitute.org_BRCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BRCA/20160128/gdac.broadinstitute.org_BRCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ccle_broad_2019
</td>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia (Broad, 2019)
</td>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia from the Broad Institute and Novartis, updated 2019. Data downloaded from the Broad CCLE Portal.
</td>
<td style="text-align:left;">
<a href="https://portals.broadinstitute.org/ccle/data" class="uri">https://portals.broadinstitute.org/ccle/data</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
cellline_ccle_broad
</td>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia (Novartis/Broad, Nature 2012)
</td>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia from the Broad Institute and Novartis, containing 1020 samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
cesc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
cesc_tcga
</td>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CESC/20160128/gdac.broadinstitute.org_CESC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CESC/20160128/gdac.broadinstitute.org_CESC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
chol_msk_2018
</td>
<td style="text-align:left;">
Cholangiocarcinoma (MSK, Clin Cancer Res 2018)
</td>
<td style="text-align:left;">
Targeted sequencing (MSK-IMPACT) of 195 tumor samples (194 with matched normals) from 192 cholangiocarcinoma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
chol_nccs_2013
</td>
<td style="text-align:left;">
Cholangiocarcinoma (National Cancer Centre of Singapore, Nat Genet 2013)
</td>
<td style="text-align:left;">
Exome sequencing of 15 cholangiocarcinoma cases.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
chol_nus_2012
</td>
<td style="text-align:left;">
Cholangiocarcinoma (National University of Singapore, Nat Genet 2012)
</td>
<td style="text-align:left;">
Exome Sequencing for Opisthorchis viverrini-related cholangiocarcinoma in 8 O.viverrini-related tumors and matched normal tissue.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
chol_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Cholangiocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
chol_tcga
</td>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Cholangiocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CHOL/20160128/gdac.broadinstitute.org_CHOL.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CHOL/20160128/gdac.broadinstitute.org_CHOL.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
lcll_broad_2013
</td>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (Broad, Cell 2013)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 160 CLL tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
cll_broad_2015
</td>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (Broad, Nature 2015)
</td>
<td style="text-align:left;">
Whole Exome Sequencing of 537 chronic lymphocytic leukemia samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
cll_iuopa_2015
</td>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (IUOPA, Nature 2015)
</td>
<td style="text-align:left;">
Mutation data from whole-genome or whole-exome sequencing of 428 CLL, 54 MBL, and 24 SLL tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
cllsll_icgc_2011
</td>
<td style="text-align:left;">
Chronic lymphocytic leukemia (ICGC, Nature Genetics 2011)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 105 Chronic Lymphocytic Leukemia samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ccrcc_dfci_2019
</td>
<td style="text-align:left;">
Clear Cell Renal Cell Carcinoma (DFCI, Science 2019)
</td>
<td style="text-align:left;">
Whole-exome sequencing of metastatic clear cell renal carcinoma (ccRCC) from 35 tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
coad_caseccc_2015
</td>
<td style="text-align:left;">
Colon Adenocarcinoma (CaseCCC, PNAS 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/normal pairs from 29 African American Colon Cancers.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
coad_cptac_2019
</td>
<td style="text-align:left;">
Colon Cancer (CPTAC-2 Prospective, Cell 2019)
</td>
<td style="text-align:left;">
Proteogenomic profiling of 110 colon adenocarcinoma tumor samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
coadread_dfci_2016
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma (DFCI, Cell Reports 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 619 colorectal cancers with clinicopathologic annotations.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
coadread_genentech
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma (Genentech, Nature 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of 74 paired colon tumor-normal samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
coadread_tcga_pub
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Whole exome sequencing in 224 of the 276 colorectal carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Colorectal Cancer project.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
coadread_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
coadread_tcga
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Colorectal Adenocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/COADREAD/20160128/gdac.broadinstitute.org_COADREAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/COADREAD/20160128/gdac.broadinstitute.org_COADREAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
coadread_mskcc
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma Triplets (MSKCC, Genome Biol 2014)
</td>
<td style="text-align:left;">
Targeted sequencing of primary, metastatic, and normal tissues from 69 colorectal adenocarcinoma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
cscc_dfarber_2015
</td>
<td style="text-align:left;">
Cutaneous Squamous Cell Carcinoma (DFCI, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
Targeted sequencing of 29 metastatic cutaneous squamous cell carcinoma samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
cscc_hgsc_bcm_2014
</td>
<td style="text-align:left;">
Cutaneous Squamous Cell Carcinoma (MD Anderson, Clin Cancer Res 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/normal pairs from 39 aggressive cutaneous squamous cell carcinoma cases.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ctcl_columbia_2015
</td>
<td style="text-align:left;">
Cutaneous T Cell Lymphoma (Columbia U, Nat Genet 2015)
</td>
<td style="text-align:left;">
Whole-Exome Sequencing (WXS) of tumor-normal sample pairs from 25 patients with Sezary Syndrome and 17 patients with other Cutaneous T Cell Lymphoma (CTCLs).
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
pact_jhu_2011
</td>
<td style="text-align:left;">
Cystic Tumor of the Pancreas (Johns Hopkins, PNAS 2011)
</td>
<td style="text-align:left;">
Whole-exome sequencing of major neoplastic cyst types of the pancreas: 8 serous cystadenomas (SCAs), 8 intraductal papillary mucinous neoplasms (IPMNs), 8 mucinous cystic neoplasms (MCNs), and 8 solid pseudopapillary neoplasms (SPNs).
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
desm_broad_2015
</td>
<td style="text-align:left;">
Desmoplastic Melanoma (Broad Institute, Nat Genet 2015)
</td>
<td style="text-align:left;">
Exome/Whole Genome sequencing of 20 desmoplastic melanoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
dlbcl_dfci_2018
</td>
<td style="text-align:left;">
Diffuse Large B cell Lymphoma (DFCI, Nat Med 2018)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/matching normal sample pairs from 135 DLBCL patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
dlbc_broad_2012
</td>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (Broad, PNAS 2012)
</td>
<td style="text-align:left;">
Whole exome sequencing of 58 primary DLBCL samples and their matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
dlbcl_duke_2017
</td>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (Duke, Cell 2017)
</td>
<td style="text-align:left;">
Genomic profiling of 1,001 DLBCL patients, mutations reported in only 150 driver genes.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
dlbc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
nhl_bcgsc_2013
</td>
<td style="text-align:left;">
Diffuse Large B-cell Lymphoma (BCGSC, Blood 2013)
</td>
<td style="text-align:left;">
Whole genome sequencing of 40 DLBCL tumor/normal pairs and 13 cell lines.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ucec_msk_2018
</td>
<td style="text-align:left;">
Endometrial Cancer (MSK, 2018)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 197 advanced-staged endometrial cancer tumor samples with matched normals, from 189 patients treated at MSK.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
esca_broad
</td>
<td style="text-align:left;">
Esophageal Adenocarcinoma (DFCI, Nat Genet 2013)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 149 tumor/normal pairs and whole-genome sequencing of 16 pairs from 149 esophageal adenocarcinoma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
esca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Esophageal Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Esophageal Adenocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
stes_tcga_pub
</td>
<td style="text-align:left;">
Esophageal Carcinoma (TCGA, Nature 2017)
</td>
<td style="text-align:left;">
Whole exome sequencing of 559 esophageal and gastric carcinoma tumors with matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
esca_tcga
</td>
<td style="text-align:left;">
Esophageal Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Esophageal Carcinoma ; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ESCA/20160128/gdac.broadinstitute.org_ESCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ESCA/20160128/gdac.broadinstitute.org_ESCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
escc_icgc
</td>
<td style="text-align:left;">
Esophageal Squamous Cell Carcinoma (ICGC, Nature 2014)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 88 esophageal squamous cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
escc_ucla_2014
</td>
<td style="text-align:left;">
Esophageal Squamous Cell Carcinoma (UCLA, Nat Genet 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing (WXS) or targeted deep sequencing (TDS) of 139 paired esophageal squamous cell carcinoma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
es_iocurie_2014
</td>
<td style="text-align:left;">
Ewing Sarcoma (Institut Curie, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 112 Ewing sarcoma samples and matched germ line DNA.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
gbc_msk_2018
</td>
<td style="text-align:left;">
Gallbladder Cancer (MSK, Cancer 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of 103 samples with matched normals from 101 Gallbladder cancer patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
gbc_shanghai_2014
</td>
<td style="text-align:left;">
Gallbladder Carcinoma (Shanghai, Nat Genet 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 32 gallbladder carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
egc_tmucih_2015
</td>
<td style="text-align:left;">
Gastric Adenocarcinoma (TMUCIH, PNAS 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 78 Gastric Adenocarcinomas and their matched normal samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
gct_msk_2016
</td>
<td style="text-align:left;">
Germ Cell Tumors (MSKCC, J Clin Oncol 2016)
</td>
<td style="text-align:left;">
Whole-exome and Targeted sequencing of 19 samples and Targeted sequencing of 161 samples from 180 Germ Cell Tumor patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
gbm_tcga_pub2013
</td>
<td style="text-align:left;">
Glioblastoma (TCGA, Cell 2013)
</td>
<td style="text-align:left;">
Whole-exome and/or whole-genome sequencing of 257 of the 543 glioblastoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Glioblastoma Project.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
gbm_tcga_pub
</td>
<td style="text-align:left;">
Glioblastoma (TCGA, Nature 2008)
</td>
<td style="text-align:left;">
Targeted sequencing in 91 of the 206 primary glioblastoma tumors (143 with matched normals) from the Cancer Genome Atlas (TCGA) Glioblastoma Project.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
gbm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Glioblastoma Multiforme (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Glioblastoma Multiforme TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
gbm_tcga
</td>
<td style="text-align:left;">
Glioblastoma Multiforme (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Glioblastoma Multiforme; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/GBM/20160128/gdac.broadinstitute.org_GBM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/GBM/20160128/gdac.broadinstitute.org_GBM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
glioma_msk_2018
</td>
<td style="text-align:left;">
Glioma (MSK, 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of CSF samples (91 samples with 86 matched normals) from 42 glioma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hnsc_broad
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (Broad, Science 2011)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 74 head and neck squamous cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hnsc_jhu
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (Johns Hopkins, Science 2011)
</td>
<td style="text-align:left;">
Exome sequencing of 32 head and neck squamous cell carcinoma tumor/normal sample pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hnsc_tcga_pub
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, Nature 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing and/or whole-genome sequencing of 279 head and neck squamous cell carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) head and neck squamous cell carcinoma Project.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hnsc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
hnsc_tcga
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Head and Neck Squamous Cell Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/HNSC/20160128/gdac.broadinstitute.org_HNSC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/HNSC/20160128/gdac.broadinstitute.org_HNSC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
liad_inserm_fr_2014
</td>
<td style="text-align:left;">
Hepatocellular Adenoma (INSERM, Cancer Cell 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 46 liver tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hcc_mskimpact_2018
</td>
<td style="text-align:left;">
Hepatocellular Carcinoma (MSK, Clin Cancer Res 2018)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 127 Hepatocellular Carcinoma tumor samples with matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hcc_inserm_fr_2015
</td>
<td style="text-align:left;">
Hepatocellular Carcinomas (INSERM, Nat Genet 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 243 liver tumors and matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
histiocytosis_cobi_msk_2019
</td>
<td style="text-align:left;">
Histiocytosis Cobimetinib (MSK, Nature 2019)
</td>
<td style="text-align:left;">
Targeted and whole exome sequencing of 52 histiocytosis tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
all_stjude_2013
</td>
<td style="text-align:left;">
Hypodiploid Acute Lymphoid Leukemia (St Jude, Nat Genet 2013)
</td>
<td style="text-align:left;">
Whole genome or exome sequencing of 44 (20 whole genome, 20 exome) ALL tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
panet_shanghai_2013
</td>
<td style="text-align:left;">
Insulinoma (Shanghai, Nat Commun 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 10 benign insulinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
chol_jhu_2013
</td>
<td style="text-align:left;">
Intrahepatic Cholangiocarcinoma (JHU, Nat Genet 2013)
</td>
<td style="text-align:left;">
Exome sequencing of tumor/normal pairs from 32 intrahepatic cholangiocarcinoma cases and 8 gall bladder carcinoma cases.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ihch_smmu_2014
</td>
<td style="text-align:left;">
Intrahepatic Cholangiocarcinoma (Shanghai, Nat Commun 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 103 tumor/normal sample pairs in 103 intrahepatic cholangiocarcinoma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
kich_tcga_pub
</td>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, Cancer Cell 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 66 chromophobe renal cell carcinoma (ChRCCs) tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
kich_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Kidney Chromophobe TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
kich_tcga
</td>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Kidney Chromophobe; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KICH/20160128/gdac.broadinstitute.org_KICH.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KICH/20160128/gdac.broadinstitute.org_KICH.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
kirc_bgi
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (BGI, Nat Genet 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 10 ccRCC tumor/normal pairs and targeted sequencing of 88 clear cell carcinoma tumor/normal pairs. Generated by BGI-Shenzhen.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ccrcc_irc_2014
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (IRC, Nat Genet 2014)
</td>
<td style="text-align:left;">
Multi region exome sequencing of 10 clear cell renal cell carcinomas (ccRCCs)
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
kirc_tcga_pub
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, Nature 2013)
</td>
<td style="text-align:left;">
Whole-exome sequencing in 417 of 446 clear cell renal cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
kirc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
kirc_tcga
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Kidney Renal Clear Cell Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRC/20160128/gdac.broadinstitute.org_KIRC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRC/20160128/gdac.broadinstitute.org_KIRC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
kirp_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
kirp_tcga
</td>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Kidney Renal Papillary Cell Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRP/20160128/gdac.broadinstitute.org_KIRP.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRP/20160128/gdac.broadinstitute.org_KIRP.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
hcc_msk_venturaa_2018
</td>
<td style="text-align:left;">
Liver Hepatocellular Adenoma and Carcinomas (MSK, PLOS One 2018)
</td>
<td style="text-align:left;">
Targeted sequencing of tumor/normal pairs of 9 LIAD and 10 HCC patients via MSK-IMPACT platform.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
lihc_amc_prv
</td>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (AMC, Hepatology 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 231 hepatocellular carcinomas and matched normal tissue pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
lihc_riken
</td>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (RIKEN, Nat Genet 2012)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 27 hepatocellular carcinoma tumor/norma pairs from RIKEN.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
lihc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
lihc_tcga
</td>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Liver Hepatocellular Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LIHC/20160128/gdac.broadinstitute.org_LIHC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LIHC/20160128/gdac.broadinstitute.org_LIHC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
lgg_ucsf_2014
</td>
<td style="text-align:left;">
Low-Grade Gliomas (UCSF, Science 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 23 grade II glioma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
luad_broad
</td>
<td style="text-align:left;">
Lung Adenocarcinoma (Broad, Cell 2012)
</td>
<td style="text-align:left;">
Whole-exome and/or whole-genome sequencing of 183 lung adenocarcinomas and matched normal tissue pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
luad_mskcc_2015
</td>
<td style="text-align:left;">
Lung Adenocarcinoma (MSKCC, Science 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 34 non small cell lung cancer tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
luad_tcga_pub
</td>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing, messenger RNA, microRNA, copy number, methylation and proteomic analysis of 230 lung adenocarcinoma tumor/normal pairs. Raw data via the TCGA Data Portal.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/docs/publications/luad_2014/" class="uri">https://tcga-data.nci.nih.gov/docs/publications/luad_2014/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
luad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Lung Adenocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
luad_tcga
</td>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Lung Adenocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUAD/20160128/gdac.broadinstitute.org_LUAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUAD/20160128/gdac.broadinstitute.org_LUAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
luad_tsp
</td>
<td style="text-align:left;">
Lung Adenocarcinoma (TSP, Nature 2008)
</td>
<td style="text-align:left;">
Targeted sequencing of 163 lung adenocarcinoma tumor/normal pairs. Generated by WashU/Broad.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
lusc_tcga_pub
</td>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Whole exome sequencing, DNA copy number, mRNA sequencing, mRNA expression and promoter methylation analysis of 178 lung squamous cell carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Lung Squamous Cell Carcinoma project. 178 cases.Nature 2012. Raw data via the TCGA Data Portal.
</td>
<td style="text-align:left;">
<a href="http://cancergenome.nih.gov/" class="uri">http://cancergenome.nih.gov/</a>, <a href="https://tcga-data.nci.nih.gov/docs/publications/lusc_2012/" class="uri">https://tcga-data.nci.nih.gov/docs/publications/lusc_2012/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
lusc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
lusc_tcga
</td>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Lung Squamous Cell Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUSC/20160128/gdac.broadinstitute.org_LUSC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUSC/20160128/gdac.broadinstitute.org_LUSC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
dlbc_tcga
</td>
<td style="text-align:left;">
Lymphoid Neoplasm Diffuse Large B-cell Lymphoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Lymphoid Neoplasm Diffuse Large B-cell Lymphoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/DLBC/20160128/gdac.broadinstitute.org_DLBC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/DLBC/20160128/gdac.broadinstitute.org_DLBC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
msk_impact_2017
</td>
<td style="text-align:left;">
MSK-IMPACT Clinical Sequencing Cohort (MSKCC, Nat Med 2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 10,000 clinical cases using the MSK-IMPACT assay
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mixed_allen_2018
</td>
<td style="text-align:left;">
MSS Mixed Solid Tumors (Broad/Dana-Farber, Nat Genet 2018)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal pairs and corresponding clinical outcomes from patients treated with immune checkpoint inhibitors across multiple cancer types including melanoma, non-small cell lung cancer, head and neck cancer, and bladder cancer.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mpnst_mskcc
</td>
<td style="text-align:left;">
Malignant Peripheral Nerve Sheath Tumor (MSKCC, Nat Genet 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 15 malignant peripheral nerve sheath tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mcl_idibips_2013
</td>
<td style="text-align:left;">
Mantle Cell Lymphoma (IDIBIPS, PNAS 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 29 primary mantle cell lymphomas and their matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mbn_mdacc_2013
</td>
<td style="text-align:left;">
Mature B-cell malignancies (MD Anderson Cancer Center)
</td>
<td style="text-align:left;">
We performed targeted deep sequencing of 380 recurrently mutated genes across a cohort of 755 mature B-cell malignancies to define conserved and disease-specific patterns of genomic alterations.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mbl_broad_2012
</td>
<td style="text-align:left;">
Medulloblastoma (Broad, Nature 2012)
</td>
<td style="text-align:left;">
Whole exome sequencing of 92 primary medulloblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mbl_icgc
</td>
<td style="text-align:left;">
Medulloblastoma (ICGC, Nature 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of 125 matched tumor and germline medulloblastoma samples
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mbl_pcgp
</td>
<td style="text-align:left;">
Medulloblastoma (PCGP, Nature 2012)
</td>
<td style="text-align:left;">
Whole genome sequencing of 37 medulloblastoma tumors and their matched normals from the Pediatric Cancer Genome Project (PCGP), a joint project by St. Jude Children’s Research Hospital and Washington University.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mbl_sickkids_2016
</td>
<td style="text-align:left;">
Medulloblastoma (Sickkids, Nature 2016)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 46 recurrent medulloblastoma samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_broad_dfarber
</td>
<td style="text-align:left;">
Melanoma (Broad/Dana Farber, Nature 2012)
</td>
<td style="text-align:left;">
Whole Genome Sequencing of 25 metastatic melanoma samples with matched normals
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_mskcc_2014
</td>
<td style="text-align:left;">
Melanoma (MSKCC, NEJM 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 64 pretreated (ipilimumab or tremelimumab) melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
lgggbm_tcga_pub
</td>
<td style="text-align:left;">
Merged Cohort of LGG and GBM (TCGA, Cell 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing from 1,122 TCGA LGG and GBM cases.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
meso_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Mesothelioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Mesothelioma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
meso_tcga
</td>
<td style="text-align:left;">
Mesothelioma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Mesothelioma; raw data at the NCI; source mutation data at the GDC.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="https://portal.gdc.cancer.gov/legacy-archive/search/f?filters=%7B%22op%22:%22and%22,%22content%22:%5B%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.program.name%22,%22value%22:%5B%22TCGA%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.project_id%22,%22value%22:%5B%22TCGA-MESO%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22files.data_format%22,%22value%22:%5B%22MAF%22%5D%7D%7D%5D%7D" class="uri">https://portal.gdc.cancer.gov/legacy-archive/search/f?filters=%7B%22op%22:%22and%22,%22content%22:%5B%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.program.name%22,%22value%22:%5B%22TCGA%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.project_id%22,%22value%22:%5B%22TCGA-MESO%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22files.data_format%22,%22value%22:%5B%22MAF%22%5D%7D%7D%5D%7D</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_igr_2015
</td>
<td style="text-align:left;">
Metastatic Breast Cancer (INSERM, PLoS Med 2016)
</td>
<td style="text-align:left;">
Whole exome sequencing of 216 tumor/normal (blood) pairs from metastatic breast cancer patients who underwent a biopsy in the context of the SAFIR01/SAFIR02 (Unicancer, France), SHIVA (Institut Curie, France) or MOSCATO (Gustave Roussy, France) prospective trials.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
crc_msk_2017
</td>
<td style="text-align:left;">
Metastatic Colorectal Cancer (MSKCC, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Targeted sequencing of 1134 metastatic colorectal tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
egc_msk_2017
</td>
<td style="text-align:left;">
Metastatic Esophagogastric Cancer (MSKCC, Cancer Discovery 2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 341 samples from 305 metastatic esophagogastric cancer patients
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_dfci_2015
</td>
<td style="text-align:left;">
Metastatic Melanoma (DFCI, Science 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 110 metastatic melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_vanderbilt_mskcc_2015
</td>
<td style="text-align:left;">
Metastatic Melanoma (MSKCC, JCO Precis Oncol 2017)
</td>
<td style="text-align:left;">
Next generation sequencing (NGS) of pre-treatment tumor to identify co-altered genes that predict for intrinsic resistance to BRAF inhibitor therapy in melanoma patients as a prelude to rational combination strategies.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mel_ucla_2016
</td>
<td style="text-align:left;">
Metastatic Melanoma (UCLA, Cell 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 38 pretreated (pembrolizumab, nivolumab) melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_mich
</td>
<td style="text-align:left;">
Metastatic Prostate Adenocarcinoma (MCTP, Nature 2012)
</td>
<td style="text-align:left;">
Exome sequencing of 61 prostate cancer tumor/normal pairs (50 metastatic CRPCs and 11 high-grade localized prostate cancers). Generated by Arul Chinnaiyan’s and Scott Tomlins’ labs at the University of Michigan.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_su2c_2019
</td>
<td style="text-align:left;">
Metastatic Prostate Adenocarcinoma (SU2C/PCF Dream Team, PNAS 2019)
</td>
<td style="text-align:left;">
Whole exome sequencing of 444 castrate resistant prostate cancer tumor/normal pairs. SU2C-PCF Dream Team: Precision Therapy for Advanced Prostate Cancer.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_su2c_2015
</td>
<td style="text-align:left;">
Metastatic Prostate Cancer (SU2C/PCF Dream Team, Cell 2015)
</td>
<td style="text-align:left;">
Comprehensive analysis of 150 metastatic prostate cancer tumor/normal pairs by the SU2C/PCF Dream Team
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
metastatic_solid_tumors_mich_2017
</td>
<td style="text-align:left;">
Metastatic Solid Cancers (UMich, Nature 2017)
</td>
<td style="text-align:left;">
Whole-exome and -transcriptome sequencing of 500 adult patients with metastatic solid tumor/primary normal pairs of diverse lineage and biopsy site.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mm_broad
</td>
<td style="text-align:left;">
Multiple Myeloma (Broad, Cancer Cell 2014)
</td>
<td style="text-align:left;">
Whole-exome or whole-genome sequencing of 203 multiple myeloma paired tumor/normal sample pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mds_tokyo_2011
</td>
<td style="text-align:left;">
Myelodysplasia (UTokyo, Nature 2011)
</td>
<td style="text-align:left;">
Whole exome sequencing of 29 myelodysplasia tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mpn_cimr_2013
</td>
<td style="text-align:left;">
Myeloproliferative Neoplasms (CIMR, NEJM 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of paired tumor/normal pairs from 151 cases of Myeloproliferative Neoplasms.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
cellline_nci60
</td>
<td style="text-align:left;">
NCI-60 Cell Lines (NCI, Cancer Res 2012)
</td>
<td style="text-align:left;">
NCI-60 cell line project; raw data at CellMiner.
</td>
<td style="text-align:left;">
<a href="http://discover.nci.nih.gov/cellminer/loadDownload.do" class="uri">http://discover.nci.nih.gov/cellminer/loadDownload.do</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
npc_nusingapore
</td>
<td style="text-align:left;">
Nasopharyngeal Carcinoma (Singapore, Nat Genet 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 56 NPC tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nbl_amc_2012
</td>
<td style="text-align:left;">
Neuroblastoma (AMC Amsterdam, Nature 2012)
</td>
<td style="text-align:left;">
Whole genome sequencing of 87 primary neuroblastoma tumors and their matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nbl_broad_2013
</td>
<td style="text-align:left;">
Neuroblastoma (Broad, Nat Genet 2013)
</td>
<td style="text-align:left;">
Whole Genome and Whole Exome sequencing of 240 high-risk neuroblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nbl_ucologne_2015
</td>
<td style="text-align:left;">
Neuroblastoma (Broad, Nature 2015)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 56 neuroblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nepc_wcm_2016
</td>
<td style="text-align:left;">
Neuroendocrine Prostate Cancer (Multi-Institute, Nat Med 2016)
</td>
<td style="text-align:left;">
Whole exome sequencing of 114 metastatic tumor/normal pairs to understand genomic overlap between castration resistant adenocarcinoma (CRPC-Adeno) and neuroendocrine histologies (CRPC-NE).
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nhl_bcgsc_2011
</td>
<td style="text-align:left;">
Non-Hodgkin Lymphoma (BCGSC, Nature 2011)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing of 14 NHL tumors and their matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
lung_msk_2017
</td>
<td style="text-align:left;">
Non-Small Cell Cancer (MSKCC, Cancer Discov 2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 860 patients with metastatic lung adenocarcinoma
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nsclc_mskcc_2018
</td>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 75 tumor/normal NSCLC pairs treated with PD-1 plus CTLA-4 blockade..
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nsclc_pd1_msk_2018
</td>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (MSKCC, J Clin Oncol 2018)
</td>
<td style="text-align:left;">
IMPACT sequencing of 240 NSCLC tumor/normal pairs treated at MSKCC with anti-PD-(L)1 based therapy.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nsclc_tracerx_2017
</td>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (TRACERx, NEJM 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 327 (with 100 matched germline samples) tumors from 100 early-stage NSCLC patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nsclc_unito_2016
</td>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (University of Turin, Lung Cancer 2017)
</td>
<td style="text-align:left;">
Targeted next-generation sequencing of 41 NSCLC samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nsclc_mskcc_2015
</td>
<td style="text-align:left;">
Non-small cell lung cancer (MSK, Science 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor-normal tissue pairs from 16 non-small cell lung cancer samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_nmibc_2017
</td>
<td style="text-align:left;">
Nonmuscle Invasive Bladder Cancer (MSK Eur Urol 2017)
</td>
<td style="text-align:left;">
Genomic Landscape and Predictors of Response for High Risk NMIBC
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hnsc_mdanderson_2013
</td>
<td style="text-align:left;">
Oral Squamous Cell Carcinoma (MD Anderson, Cancer Discov 2013)
</td>
<td style="text-align:left;">
Comprehensive profiling of 40 oral squamous cell carcinoma tumor/normal sample pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ov_tcga_pub
</td>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, Nature 2011)
</td>
<td style="text-align:left;">
Whole exome sequencing (316 samples with matched normals), mRNA expression, miRNA expression, promoter methylation, and DNA copy number in 489 high-grade serous ovarian adenocarcinomas (HGS-OvCa). The Cancer Genome Atlas (TCGA) Serous Ovarian Cancer project. 489 cases. Raw data via the TCGA Data Portal.
</td>
<td style="text-align:left;">
<a href="http://cancergenome.nih.gov/" class="uri">http://cancergenome.nih.gov/</a>, <a href="https://tcga-data.nci.nih.gov/docs/publications/ov_2011/" class="uri">https://tcga-data.nci.nih.gov/docs/publications/ov_2011/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ov_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ov_tcga
</td>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Ovarian Serous Cystadenocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/OV/20160128/gdac.broadinstitute.org_OV.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/OV/20160128/gdac.broadinstitute.org_OV.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
nsclc_tcga_broad_2016
</td>
<td style="text-align:left;">
Pan-Lung Cancer (TCGA, Nat Genet 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 660 lung ADC and 484 lung SqCC tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
paad_icgc
</td>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (ICGC, Nature 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 99 pancreatic samples and their matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
paad_qcmg_uq_2016
</td>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (QCMG, Nature 2016)
</td>
<td style="text-align:left;">
Whole-genome and deep-exome sequencing analysis of 456 pancreatic ductal adenocarcinomas
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
paad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Pancreatic Adenocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
paad_tcga
</td>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Pancreatic Adenocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PAAD/20160128/gdac.broadinstitute.org_PAAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PAAD/20160128/gdac.broadinstitute.org_PAAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
paad_utsw_2015
</td>
<td style="text-align:left;">
Pancreatic Cancer (UTSW, Nat Commun 2015)
</td>
<td style="text-align:left;">
Whole exome sequencing of 109 micro-dissected pancreatic cancer cases and normal control tissue.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
panet_jhu_2011
</td>
<td style="text-align:left;">
Pancreatic Neuroendocrine Tumors (Johns Hopkins University, Science 2011)
</td>
<td style="text-align:left;">
Whole exome sequencing of 10 pancreatic neuroendocrine tumor patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
panet_arcnet_2017
</td>
<td style="text-align:left;">
Pancreatic Neuroendocrine Tumors (Multi-Institute, Nature 2017)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 98 pancreatic neuroendocrine tumours (PanNETs) and matched normal DNA pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
thca_tcga_pub
</td>
<td style="text-align:left;">
Papillary Thyroid Carcinoma (TCGA, Cell 2014)
</td>
<td style="text-align:left;">
Exome and whole genome sequencing of 496 papillary thyroid carcinoma tumor/normal pairs
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
all_phase2_target_2018_pub
</td>
<td style="text-align:left;">
Pediatric Acute Lymphoid Leukemia - Phase II (TARGET, 2018)
</td>
<td style="text-align:left;">
Comprehensive profiling of ALL Phase 2 samples. TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see Using TARGET Data section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the TARGET Publication Guidelines.
</td>
<td style="text-align:left;">
<p https:>
</p>
</td>
</tr>
<tr>
<td style="text-align:left;">
aml_target_2018_pub
</td>
<td style="text-align:left;">
Pediatric Acute Myeloid Leukemia (TARGET, 2018)
</td>
<td style="text-align:left;">
Comprehensive profiling of TARGET-AML samples. TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see Using TARGET Data section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the TARGET Publication Guidelines.
</td>
<td style="text-align:left;">
<p https:>
</p>
</td>
</tr>
<tr>
<td style="text-align:left;">
es_dfarber_broad_2014
</td>
<td style="text-align:left;">
Pediatric Ewing Sarcoma (DFCI, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 96 pediatric Ewing Sarcoma tumors and 11 cell lines.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nbl_target_2018_pub
</td>
<td style="text-align:left;">
Pediatric Neuroblastoma (TARGET, 2018)
</td>
<td style="text-align:left;">
Comprehensive profiling of 1089 NBL samples. TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see Using TARGET Data section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the TARGET Publication Guidelines.
</td>
<td style="text-align:left;">
<p https:>
</p>
</td>
</tr>
<tr>
<td style="text-align:left;">
pediatric_dkfz_2017
</td>
<td style="text-align:left;">
Pediatric Pan-Cancer (DKFZ, Nature 2017)
</td>
<td style="text-align:left;">
Whole-genome/exome sequencing of 961 Pediatric Cancer samples. Data generated from Pfister Lab &amp; DKFZ/German Cancer Consortium.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mixed_pipseq_2017
</td>
<td style="text-align:left;">
Pediatric Pan-cancer (Columbia U, Genome Med 2016)
</td>
<td style="text-align:left;">
Whole exome sequencing of 103 samples (88 with matched normals) from 88 high-risk pediatric cancer patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
pptc_2019
</td>
<td style="text-align:left;">
Pediatric Preclinical Testing Consortium (Maris, 2019)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 261 patient derived xenografts (PDXs) samples from high-risk childhood cancers (with no matched normals). Website: PPTC
</td>
<td style="text-align:left;">
‘<a href="http://www.ncipptc.org/" class="uri">http://www.ncipptc.org/</a>’
</td>
</tr>
<tr>
<td style="text-align:left;">
rt_target_2018_pub
</td>
<td style="text-align:left;">
Pediatric Rhabdoid Tumor (TARGET, 2018)
</td>
<td style="text-align:left;">
Comprehensive profiling of 72 patients with Rhabdoid Tumor. TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see Using TARGET Data section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the TARGET Publication Guidelines.
</td>
<td style="text-align:left;">
<p https:>
</p>
</td>
</tr>
<tr>
<td style="text-align:left;">
wt_target_2018_pub
</td>
<td style="text-align:left;">
Pediatric Wilms’ Tumor (TARGET, 2018)
</td>
<td style="text-align:left;">
Comprehensive profiling of 657 Wilms’ Tumor samples. TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see Using TARGET Data section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the TARGET Publication Guidelines.
</td>
<td style="text-align:left;">
<p https:>
</p>
</td>
</tr>
<tr>
<td style="text-align:left;">
pcpg_tcga_pub
</td>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 178 Pheochromocytoma or Paraganglioma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
pcpg_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
pcpg_tcga
</td>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Pheochromocytoma and Paraganglioma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PCPG/20160128/gdac.broadinstitute.org_PCPG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PCPG/20160128/gdac.broadinstitute.org_PCPG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
past_dkfz_heidelberg_2013
</td>
<td style="text-align:left;">
Pilocytic Astrocytoma (ICGC, Nature Genetics 2013)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 96 pilocytic astrocytoma tumor/normal pairs. Conducted by the International Cancer Genome Consortium (ICGC) PedBrain Tumor Project.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
plmeso_nyu_2015
</td>
<td style="text-align:left;">
Pleural Mesothelioma (NYU, Cancer Res 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 22 malignant pleural mesotheliomas (PLMESO) tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
thyroid_mskcc_2016
</td>
<td style="text-align:left;">
Poorly-Differentiated and Anaplastic Thyroid Cancers (MSKCC, JCI 2016)
</td>
<td style="text-align:left;">
Next-generation sequencing of 341 cancer genes from 117 patient-derived PDTCs and ATCs
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
pcnsl_mayo_2015
</td>
<td style="text-align:left;">
Primary Central Nervous System Lymphoma (Mayo Clinic, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
Exome sequencing of 19 immunocompetent primary central nervous system lymphoma (PCNSL) samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_broad_2013
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (Broad/Cornell, Cell 2013)
</td>
<td style="text-align:left;">
Whole genome sequencing of 57 prostate tumor and matches normal tissues.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_broad
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (Broad/Cornell, Nat Genet 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of 112 prostate adenocarcinomas and matched normal samples. Generated by Levi Garraway’s lab at the Broad Institute and Mark Rubin’s lab at Cornell.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_cpcg_2017
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (CPC-GENE, Nature 2017)
</td>
<td style="text-align:left;">
Comprehensive genomic profiling of 477 Prostate Adenocarcinoma samples from CPC-GENE and public data sets, including TCGA-PRAD.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_fhcrc
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (Fred Hutchinson CRC, Nat Med 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing, array comparative genomic hybridization (CGH) and RNA transcript profiling of 176 tumors from 63 patients with disseminated prostate cancer with matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_mskcc
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC, Cancer Cell 2010)
</td>
<td style="text-align:left;">
Comprehensive profiling of 218 prostate tumors (181 primaries, 37 metastases) and 14 prostate cancer cell lines and xenografts. MSKCC Prostate Oncogenome Project.Normalized flat data files. Raw data via GEO (GSE21032).
</td>
<td style="text-align:left;">
<a href="http://cbio.mskcc.org/cancergenomics/prostate/data/" class="uri">http://cbio.mskcc.org/cancergenomics/prostate/data/</a>, <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21032" class="uri">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21032</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_mskcc_2014
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC, PNAS 2014)
</td>
<td style="text-align:left;">
Genome-wide copy number profiling in 104 primary prostate cancer tumor/normal pairs. Raw data via GEO (GSE54691).
</td>
<td style="text-align:left;">
<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54691" class="uri">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54691</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_p1000
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC/DFCI, Nature Genetics 2018)
</td>
<td style="text-align:left;">
Whole exome Sequencing of 1013 prostate cancer samples and their matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_eururol_2017
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (SMMU, Eur Urol 2017)
</td>
<td style="text-align:left;">
Whole-genome and transcriptome sequencing of 65 Prostate Adenocarcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_tcga_pub
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, Cell 2015)
</td>
<td style="text-align:left;">
Integrated profiling of 333 primary prostate adenocarcinoma samples; raw data at the NCI.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/docs/publications/prad_2015/" class="uri">https://tcga-data.nci.nih.gov/docs/publications/prad_2015/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_tcga
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Prostate Adenocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PRAD/20160128/gdac.broadinstitute.org_PRAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PRAD/20160128/gdac.broadinstitute.org_PRAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_mskcc_cheny1_organoids_2014
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma Organoids (MSKCC, Cell 2014)
</td>
<td style="text-align:left;">
Exome sequencing of 12 advanced prostate cancer samples and matched organoids
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prostate_dkfz_2018
</td>
<td style="text-align:left;">
Prostate Cancer (DKFZ, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Whole genome and transcriptome sequencing of tumor/normal pairs from 292 prostate cancer patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_msk_2019
</td>
<td style="text-align:left;">
Prostate Cancer (MSK, 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT Sequencing of 18 prostate cancer tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_mskcc_2017
</td>
<td style="text-align:left;">
Prostate Cancer (MSKCC, JCO Precis Oncol 2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 504 prostate cancer tumor/normal pairs via MSK-IMPACT.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
rectal_msk_2019
</td>
<td style="text-align:left;">
Rectal Cancer (MSK,Nature Medicine 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT targeted sequencing of matched rectal cancer tumor and tumoroid samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
hnc_mskcc_2016
</td>
<td style="text-align:left;">
Recurrent and Metastatic Head &amp; Neck Cancer (MSKCC, JAMA Oncol 2016)
</td>
<td style="text-align:left;">
Targeted sequencing of 151 patients with advanced, treatment resistant head and neck tumors
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ccrcc_utokyo_2013
</td>
<td style="text-align:left;">
Renal Clear Cell Carcinoma (UTokyo, Nat Genet 2013)
</td>
<td style="text-align:left;">
Whole exome and/or whole genome sequencing of 106 clear cell renal cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
nccrcc_genentech_2014
</td>
<td style="text-align:left;">
Renal Non-Clear Cell Carcinoma (Genentech, Nat Genet 2014)
</td>
<td style="text-align:left;">
Exome sequencing of 139 samples with matched normals and RNA-sequencing of 7 samples (5 samples with matched normals) from 145 Renal Non-Clear Cell Carcinoma (nccRCC) patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
mrt_bcgsc_2016
</td>
<td style="text-align:left;">
Rhabdoid Cancer (BCGSC, Cancer Cell 2016)
</td>
<td style="text-align:left;">
Whole-genome sequencing of tumor-normal sample pairs from 40 patients with Malignant Rhabdoid Cancer.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
rms_nih_2014
</td>
<td style="text-align:left;">
Rhabdomyosarcoma (NIH, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 43 rhabdomyosarcoma tumor/normal pairs.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
summit_2018
</td>
<td style="text-align:left;">
SUMMIT - Neratinib Basket Study (Multi-Institute, Nature 2018)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 86 patients with HER2- and HER3-mutant cancers
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
sarc_mskcc
</td>
<td style="text-align:left;">
Sarcoma (MSKCC/Broad, Nat Genet 2010)
</td>
<td style="text-align:left;">
Sanger-sequenced 226 protein-coding genes (3,849 exons and adjacent intronic) on 207 samples (with 205 matched normals) encompassing 7 major sarcoma subtypes.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
sarc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Sarcoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Sarcoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
sarc_tcga
</td>
<td style="text-align:left;">
Sarcoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Sarcoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SARC/20160128/gdac.broadinstitute.org_SARC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SARC/20160128/gdac.broadinstitute.org_SARC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_broad
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma (Broad, Cell 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 121 melanoma samples with matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_tcga
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Skin Cutaneous Melanoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SKCM/20160128/gdac.broadinstitute.org_SKCM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SKCM/20160128/gdac.broadinstitute.org_SKCM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_yale
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma (Yale, Nat Genet 2012)
</td>
<td style="text-align:left;">
Exome sequencing of 147 melanoma samples with 99 matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
skcm_broad_brafresist_2012
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma(Broad, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Exome Sequencing of tumors (with matched normals) from 45 patients with BRAFV600-mutant metastatic melanoma who received vemurafenib or dabrafenib monotherapy.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
scco_mskcc
</td>
<td style="text-align:left;">
Small Cell Carcinoma of the Ovary (MSKCC, Nat Genet 2014)
</td>
<td style="text-align:left;">
Targeted sequencing of 12 SCCO paired tumor and normal samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
sclc_clcgp
</td>
<td style="text-align:left;">
Small Cell Lung Cancer (CLCGP, Nat Genet 2012)
</td>
<td style="text-align:left;">
Comprehensive whole exam profiling of 29 small cell lung cancer samples. Generated by the Clinical Lung Cancer Genome Project.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
sclc_jhu
</td>
<td style="text-align:left;">
Small Cell Lung Cancer (Johns Hopkins, Nat Genet 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of 36 primary human SCLC tumor/normal pairs and 17 matched SCLC and lymphoblastoid cell lines.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
sclc_ucologne_2015
</td>
<td style="text-align:left;">
Small Cell Lung Cancer (U Cologne, Nature 2015)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 110 small cell lung cancer(SCLC) tumour samples and matched normal material.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
sclc_cancercell_gardner_2017
</td>
<td style="text-align:left;">
Small-Cell Lung Cancer (Multi-Institute, Cancer Cell 2017)
</td>
<td style="text-align:left;">
Whole Exome Sequencing of 10 paired chemosensitive and chemoresistant SCLC PDX.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
vsc_cuk_2018
</td>
<td style="text-align:left;">
Squamous Cell Carcinoma of the Vulva (CUK, Exp Mol Med 2018)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal pairs from 15 vulvar squamous cell carcinomas.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
stad_pfizer_uhongkong
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma (Pfizer and UHK, Nat Genet 2014)
</td>
<td style="text-align:left;">
Whole genome sequencing of 100 gastric cancer tumor-normal pairs from the University of Hong Kong and Pfizer.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
stad_tcga_pub
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Single nucleotide polymorphism array somatic copy-number analysis, whole-exome sequencing, mRNA sequencing, miRNA sequencing, array-based DNA methylation profiling and reverse-phase protein arrays of 295 primary gastric adenocarcinomas tumors with matched normals. TCGA Stomach Adenocarcinoma Project; raw data at the NCI.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/docs/publications/stad_2014/" class="uri">https://tcga-data.nci.nih.gov/docs/publications/stad_2014/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
stad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
stad_tcga
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Stomach Adenocarcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/STAD/20160128/gdac.broadinstitute.org_STAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/STAD/20160128/gdac.broadinstitute.org_STAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
stad_utokyo
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma (U Tokyo, Nat Genet 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 30 diffuse-type gastric adenocarcinoma samples (with matched normals) from the University of Tokyo.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
stad_uhongkong
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma (UHK, Nat Genet 2011)
</td>
<td style="text-align:left;">
Exome sequencing of 22 gastric cancer samples with matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
tmb_mskcc_2018
</td>
<td style="text-align:left;">
TMB and Immunotherapy (MSKCC, Nat Genet 2019)
</td>
<td style="text-align:left;">
Genomic and survival data from 1661 tumor-normal pairs from 1661 patients with various cancer types sequenced with the MSK-IMPACT assay.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
tgct_tcga
</td>
<td style="text-align:left;">
Testicular Germ Cell Cancer (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Testicular Germ Cell Cancer; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/TGCT/20160128/gdac.broadinstitute.org_TGCT.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/TGCT/20160128/gdac.broadinstitute.org_TGCT.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
tgct_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Testicular Germ Cell Tumors (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Testicular Germ Cell Tumors TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
angs_project_painter_2018
</td>
<td style="text-align:left;">
The Angiosarcoma Project - Count Me In (Provisional, September 2018)
</td>
<td style="text-align:left;">
The Angiosarcoma Project is a patient-driven initiative. This study includes genomic data, patient-reported data (pre-pended as PRD), medical record data (MedR), and pathology report data (PATH). As these data were generated in a research, not a clinical, laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. All annotations have been de-identified. More information is available at www.ascproject.org. Questions about these data can be directed to <a href="mailto:data@ascproject.org" class="email">data@ascproject.org</a>.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
brca_mbcproject_wagle_2017
</td>
<td style="text-align:left;">
The Metastatic Breast Cancer Project (Provisional, October 2018)
</td>
<td style="text-align:left;">
The Metastatic Breast Cancer Project is a patient-driven initiative. This study includes genomic data, patient-reported data (pre-pended as PRD), medical record data (MedR), and pathology report data (PATH). All of the titles and descriptive text for the clinical data elements have been finalized in partnership with numerous patients in the project. As these data were generated in a research, not a clinical, laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. All annotations have been de-identified. More information is available at www.mbcproject.org. For questions about these data, including how to cite, please visit www.mbcproject.org/data-release or direct questions to <a href="mailto:data@mbcproject.org" class="email">data@mbcproject.org</a>.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
prad_mpcproject_2018
</td>
<td style="text-align:left;">
The Metastatic Prostate Cancer Project (Provisional, December 2018)
</td>
<td style="text-align:left;">
The Metastatic Prostate Cancer Project is a patient-driven initiative. This study includes genomic data, patient-reported data (pre-pended as PRD), medical record data (MedR), and pathology report data (PATH). As these data were generated in a research, not a clinical, laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. All annotations have been de-identified. More information is available at www.mpcproject.org. For questions about these data, including how to cite, please visit www.mpcproject.org/data-release or direct questions to <a href="mailto:data@mpcproject.org" class="email">data@mpcproject.org</a>.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
lung_msk_pdx
</td>
<td style="text-align:left;">
Thoracic PDX (MSK, Provisional)
</td>
<td style="text-align:left;">
Targeted sequencing of thoracic PDX lines to characterize genetic alterations in 139 tumor/normal pairs via MSK-IMPACT. Data generated in Charles Rudin Lab.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
tet_nci_2014
</td>
<td style="text-align:left;">
Thymic Epithelial Tumors (NCI, Nat Genet 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of 28 thymic epithelial tumors (TETs) with matched normals
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
thym_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Thymoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Thymoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
thym_tcga
</td>
<td style="text-align:left;">
Thymoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Thymoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THYM/20160128/gdac.broadinstitute.org_THYM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THYM/20160128/gdac.broadinstitute.org_THYM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
thca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Thyroid Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Thyroid Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
thca_tcga
</td>
<td style="text-align:left;">
Thyroid Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Thyroid Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THCA/20160128/gdac.broadinstitute.org_THCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THCA/20160128/gdac.broadinstitute.org_THCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
urcc_mskcc_2016
</td>
<td style="text-align:left;">
Unclassified Renal Cell Carcinoma (MSK, Nature 2016)
</td>
<td style="text-align:left;">
Targeted sequencing of 62 high-grade primary Unclassified Renal Cell Carcinoma tumor samples with matched normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
utuc_mskcc_2015
</td>
<td style="text-align:left;">
Upper Tract Urothelial Cancer (MSK, Eur Urol 2015)
</td>
<td style="text-align:left;">
Next-generation sequencing (MSK-IMPACT) on all protein-coding exons of 300 cancer associated genes of 85 Upper Tract Urothelial Carcinoma tumor samples and paired normals.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
utuc_cornell_baylor_mdacc_2019
</td>
<td style="text-align:left;">
Upper Tract Urothelial Carcinoma (Cornell/Baylor/MDACC, Nat Comm 2019)
</td>
<td style="text-align:left;">
Whole exome sequencing (37 samples) and RNA-seq analysis (32 samples) of 47 upper tract urothelial carcinoma patients.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
blca_cornell_2016
</td>
<td style="text-align:left;">
Urothelial Carcinoma (Cornell/Trento, Nat Gen 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 72 tumor samples (with 32 matched normals) in Urothelial Carcinoma.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ucs_jhu_2014
</td>
<td style="text-align:left;">
Uterine Carcinosarcoma (Johns Hopkins, Nat Commun 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing (WXS) of 22 uterine carcinosarcoma/uterine malignant mixed Mullerian (UCS) tumors with matched normals.
</td>
<td style="text-align:left;">
<a href="http://www.nature.com/ncomms/2014/140919/ncomms6006/extref/ncomms6006-s1.pdf" class="uri">http://www.nature.com/ncomms/2014/140919/ncomms6006/extref/ncomms6006-s1.pdf</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ucs_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Uterine Carcinosarcoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uterine Carcinosarcoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ucs_tcga
</td>
<td style="text-align:left;">
Uterine Carcinosarcoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Uterine Carcinosarcoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCS/20160128/gdac.broadinstitute.org_UCS.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCS/20160128/gdac.broadinstitute.org_UCS.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
uccc_nih_2017
</td>
<td style="text-align:left;">
Uterine Clear Cell Carcinoma (NIH, Cancer 2017)
</td>
<td style="text-align:left;">
Whole exome sequencing of paired tumor-normal DNAs from 16 cases of clear cell endometrial tumors.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
ucec_tcga_pub
</td>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, Nature 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 373 endometrial carcinoma tumors with matched normals. TCGA Uterine Corpus Endometrial Carcinoma project; raw data at the NCI.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/docs/publications/ucec_2013/" class="uri">https://tcga-data.nci.nih.gov/docs/publications/ucec_2013/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ucec_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ucec_tcga
</td>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Uterine Corpus Endometrial Carcinoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCEC/20160128/gdac.broadinstitute.org_UCEC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCEC/20160128/gdac.broadinstitute.org_UCEC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
um_qimr_2016
</td>
<td style="text-align:left;">
Uveal Melanoma (QIMR, Oncotarget 2016)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of tumor/normal pairs from 28 uveal melanoma samples.
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
uvm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
Uveal Melanoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uveal melanoma TCGA PanCancer data. The original data is here. The publications are here.
</td>
<td style="text-align:left;">
<a href="https://gdc.cancer.gov/about-data/publications/pancanatlas" class="uri">https://gdc.cancer.gov/about-data/publications/pancanatlas</a>, <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html" class="uri">https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
uvm_tcga
</td>
<td style="text-align:left;">
Uveal Melanoma (TCGA, Provisional)
</td>
<td style="text-align:left;">
TCGA Uveal Melanoma; raw data at the NCI; source mutation data from GDAC Firehose.
</td>
<td style="text-align:left;">
<a href="https://tcga-data.nci.nih.gov/" class="uri">https://tcga-data.nci.nih.gov/</a>, <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UVM/20160128/gdac.broadinstitute.org_UVM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz" class="uri">http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UVM/20160128/gdac.broadinstitute.org_UVM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz</a>
</td>
</tr>
</tbody>
</table>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Marcel Ramos, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
