<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MultiAssayExperiment: Example multi-omic analyses with public data • MultiAssayWorkshop</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="MultiAssayExperiment: Example multi-omic analyses with public data">
<meta property="og:description" content="MultiAssayWorkshop">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MultiAssayWorkshop</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">3.13.4</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/Ramos_MultiAssayExperiment.html">Workshop</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorial
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../tutorials/Exercise_MultiAssay.html">Build Your First MAE</a>
    </li>
    <li>
      <a href="../tutorials/cBioPortalData_Intro.html">Download and Use cBioPortal Data</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Reference
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/cBioPortal_Studies.html">cBioPortal Studies</a>
    </li>
    <li>
      <a href="../articles/curatedTCGAData_ref.html">curatedTCGAData Reference</a>
    </li>
    <li>
      <a href="../articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MultiAssayWorkshop">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="MultiAssayExperiment_Analysis_files/header-attrs-2.9/header-attrs.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MultiAssayExperiment: Example multi-omic analyses with public data</h1>
                        <h4 class="author">Marcel Ramos</h4>
            <address class="author_afil">
      Roswell Park Comprehensive Cancer Center, Buffalo, NY<br><h4 class="author">Ludwig Geistlinger</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="author">Levi Waldron</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="date">July 16, 2021</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/MultiAssayWorkshop/blob/master/vignettes/MultiAssayExperiment_Analysis.Rmd"><code>vignettes/MultiAssayExperiment_Analysis.Rmd</code></a></small>
      <div class="hidden name"><code>MultiAssayExperiment_Analysis.Rmd</code></div>

    </address>
</address>
</address>
</div>

    
    
<div id="required-packages" class="section level1">
<h1 class="hasAnchor">
<a href="#required-packages" class="anchor"></a>Required packages</h1>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="http://waldronlab.io/MultiAssayExperiment/">MultiAssayExperiment</a></span><span class="op">)</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/therneau/survival">survival</a></span><span class="op">)</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://rpkgs.datanovia.com/survminer/index.html">survminer</a></span><span class="op">)</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">pheatmap</span><span class="op">)</span></code></pre></div>
</div>
<div id="k-m-plots-correl-pca-and-other-analyses" class="section level1">
<h1 class="hasAnchor">
<a href="#k-m-plots-correl-pca-and-other-analyses" class="anchor"></a>K-M plots, Correl, PCA, and other analyses</h1>
<p>These provide exercises and solutions using the <code>miniACC</code> dataset.</p>
<div id="how-many-miniacc-samples-have-data-for-each-combination-of-assays" class="section level2">
<h2 class="hasAnchor">
<a href="#how-many-miniacc-samples-have-data-for-each-combination-of-assays" class="anchor"></a>How many <code>miniACC</code> samples have data for each combination of assays?</h2>
<p><strong>Solution</strong></p>
<p>The built-in <code>upsetSamples</code> creates an “upset” Venn diagram to answer this question:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span><span class="op">(</span><span class="st">"miniACC"</span><span class="op">)</span>
<span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/upsetSamples.html">upsetSamples</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span></code></pre></div>
<p><img src="MultiAssayExperiment_Analysis_files/figure-html/unnamed-chunk-2-1.png" width="700"></p>
<p>In this dataset only 43 samples have all 5 assays, 32 are missing reverse-phase protein (RPPAArray), 2 are missing Mutations, 1 is missing gistict, 12 have only mutations and gistict, etc.</p>
</div>
<div id="kaplan-meier-plot-stratified-by-pathology_n_stage" class="section level2">
<h2 class="hasAnchor">
<a href="#kaplan-meier-plot-stratified-by-pathology_n_stage" class="anchor"></a>Kaplan-meier plot stratified by pathology_N_stage</h2>
<p>Create a Kaplan-meier plot, using pathology_N_stage as a stratifying variable.</p>
<p><strong>Solution</strong></p>
<p>The colData provides clinical data for things like a Kaplan-Meier plot for overall survival stratified by nodal stage.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">$</span><span class="va">days_to_death</span>, <span class="va">miniACC</span><span class="op">$</span><span class="va">vital_status</span><span class="op">)</span></code></pre></div>
<pre><code>##  [1] 1355  1677    NA+  423   365    NA+  490   579    NA+  922   551  1750 
## [13]   NA+ 2105    NA+  541    NA+   NA+  490    NA+   NA+  562    NA+   NA+
## [25]   NA+   NA+   NA+   NA+  289    NA+   NA+   NA+  552    NA+   NA+   NA+
## [37]  994    NA+   NA+  498    NA+   NA+  344    NA+   NA+   NA+   NA+   NA+
## [49]   NA+   NA+   NA+   NA+   NA+  391   125    NA+ 1852    NA+   NA+   NA+
## [61]   NA+   NA+   NA+   NA+ 1204   159  1197   662   445    NA+ 2385   436 
## [73] 1105    NA+ 1613    NA+   NA+ 2405    NA+   NA+   NA+   NA+   NA+  207 
## [85]    0    NA+   NA+   NA+   NA+   NA+   NA+  383</code></pre>
<p>And remove any patients missing overall survival information:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">miniACCsurv</span> <span class="op">&lt;-</span> <span class="va">miniACC</span><span class="op">[</span>, <span class="fu"><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">$</span><span class="va">days_to_death</span>, <span class="va">miniACC</span><span class="op">$</span><span class="va">vital_status</span><span class="op">)</span>, <span class="op">]</span>
<span class="va">miniACCsurv</span></code></pre></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 28 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 33 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 14 columns
##  [4] Mutations: matrix with 97 rows and 33 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 29 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">fit</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/survfit.html">survfit</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span><span class="op">(</span><span class="va">days_to_death</span>, <span class="va">vital_status</span><span class="op">)</span> <span class="op">~</span> <span class="va">pathology_N_stage</span>, data <span class="op">=</span> <span class="fu">colData</span><span class="op">(</span><span class="va">miniACCsurv</span><span class="op">)</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/pkg/survminer/man/ggsurvplot.html">ggsurvplot</a></span><span class="op">(</span><span class="va">fit</span>, data <span class="op">=</span> <span class="fu">colData</span><span class="op">(</span><span class="va">miniACCsurv</span><span class="op">)</span>, risk.table <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></code></pre></div>
<p><img src="MultiAssayExperiment_Analysis_files/figure-html/unnamed-chunk-5-1.png" width="700"></p>
</div>
<div id="multivariate-cox-regression-including-rna-seq-copy-number-and-pathology" class="section level2">
<h2 class="hasAnchor">
<a href="#multivariate-cox-regression-including-rna-seq-copy-number-and-pathology" class="anchor"></a>Multivariate Cox regression including RNA-seq, copy number, and pathology</h2>
<p>Choose the <em>EZH2</em> gene for demonstration. This subsetting will drop assays with no row named EZH2:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">wideacc</span> <span class="op">=</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html">wideFormat</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">[</span><span class="st">"EZH2"</span>, , <span class="op">]</span>,
    colDataCols<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>, <span class="st">"pathology_N_stage"</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<pre><code>## harmonizing input:
##   removing 216 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">wideacc</span><span class="op">$</span><span class="va">y</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span><span class="op">(</span><span class="va">wideacc</span><span class="op">$</span><span class="va">days_to_death</span>, <span class="va">wideacc</span><span class="op">$</span><span class="va">vital_status</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">wideacc</span><span class="op">)</span></code></pre></div>
<pre><code>## DataFrame with 6 rows and 7 columns
##        primary vital_status days_to_death pathology_N_stage
##    &lt;character&gt;    &lt;integer&gt;     &lt;integer&gt;       &lt;character&gt;
## 1 TCGA-OR-A5J1            1          1355                n0
## 2 TCGA-OR-A5J2            1          1677                n0
## 3 TCGA-OR-A5J3            0            NA                n0
## 4 TCGA-OR-A5J4            1           423                n1
## 5 TCGA-OR-A5J5            1           365                n0
## 6 TCGA-OR-A5J6            0            NA                n0
##   RNASeq2GeneNorm_EZH2 gistict_EZH2      y
##              &lt;numeric&gt;    &lt;numeric&gt; &lt;Surv&gt;
## 1              75.8886            0 1355:1
## 2             326.5332            1 1677:1
## 3             190.1940            1   NA:0
## 4                   NA           -2  423:1
## 5             366.3826            1  365:1
## 6              30.7371            1   NA:0</code></pre>
<p>Perform a multivariate Cox regression with <em>EZH2</em> copy number (gistict), log2-transformed <em>EZH2</em> expression (RNASeq2GeneNorm), and nodal status (pathology_N_stage) as predictors:</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span><span class="op">(</span>
    <span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span><span class="op">(</span><span class="va">days_to_death</span>, <span class="va">vital_status</span><span class="op">)</span> <span class="op">~</span>
        <span class="va">gistict_EZH2</span> <span class="op">+</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span><span class="op">(</span><span class="va">RNASeq2GeneNorm_EZH2</span><span class="op">)</span> <span class="op">+</span> <span class="va">pathology_N_stage</span>,
    data<span class="op">=</span><span class="va">wideacc</span>
<span class="op">)</span></code></pre></div>
<pre><code>## Call:
## coxph(formula = Surv(days_to_death, vital_status) ~ gistict_EZH2 + 
##     log2(RNASeq2GeneNorm_EZH2) + pathology_N_stage, data = wideacc)
## 
##                                coef exp(coef) se(coef)      z        p
## gistict_EZH2               -0.03723   0.96345  0.28205 -0.132 0.894986
## log2(RNASeq2GeneNorm_EZH2)  0.97731   2.65729  0.28105  3.477 0.000506
## pathology_N_stagen1         0.37749   1.45862  0.56992  0.662 0.507743
## 
## Likelihood ratio test=16.28  on 3 df, p=0.0009942
## n= 26, number of events= 26 
##    (66 observations deleted due to missingness)</code></pre>
<p>We see that <em>EZH2</em> expression is significantly associated with overal survival (p &lt; 0.001), but <em>EZH2</em> copy number and nodal status are not. This analysis could easily be extended to the whole genome for discovery of prognostic features by repeated univariate regressions over columns, penalized multivariate regression, etc.</p>
<p>For further detail, see the main <code>MultiAssayExperiment</code> vignette.</p>
</div>
<div id="correlation-between-rna-seq-and-copy-number" class="section level2">
<h2 class="hasAnchor">
<a href="#correlation-between-rna-seq-and-copy-number" class="anchor"></a>Correlation between RNA-seq and copy number</h2>
<p><strong>Part 1</strong></p>
<p>For all genes where there is both recurrent copy number (gistict assay) and RNA-seq, calculate the correlation between log2(RNAseq + 1) and copy number. Create a histogram of these correlations. Compare this with the histogram of correlations between all <em>unmatched</em> gene - copy number pairs.</p>
<p><strong>Solution</strong></p>
<p>First, narrow down <code>miniACC</code> to only the assays needed:</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">subacc</span> <span class="op">&lt;-</span> <span class="va">miniACC</span><span class="op">[</span>, , <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"gistict"</span><span class="op">)</span><span class="op">]</span></code></pre></div>
<pre><code>## Warning: 'experiments' dropped; see 'metadata'</code></pre>
<pre><code>## harmonizing input:
##   removing 216 sampleMap rows not in names(experiments)</code></pre>
<p>Align the rows and columns, keeping only samples with both assays available:</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">subacc</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html">intersectColumns</a></span><span class="op">(</span><span class="va">subacc</span><span class="op">)</span>
<span class="va">subacc</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html">intersectRows</a></span><span class="op">(</span><span class="va">subacc</span><span class="op">)</span></code></pre></div>
<p>Create a list of numeric matrices:</p>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">subacc.list</span> <span class="op">&lt;-</span> <span class="fu">assays</span><span class="op">(</span><span class="va">subacc</span><span class="op">)</span></code></pre></div>
<p>Log-transform the RNA-seq assay:</p>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">subacc.list</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span><span class="op">(</span><span class="va">subacc.list</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<p>Transpose both, so genes are in columns:</p>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">subacc.list</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html">lapply</a></span><span class="op">(</span><span class="va">subacc.list</span>, <span class="va">t</span><span class="op">)</span></code></pre></div>
<p>Calculate the correlation between columns in the first matrix and columns in the second matrix:</p>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">corres</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html">cor</a></span><span class="op">(</span><span class="va">subacc.list</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span>, <span class="va">subacc.list</span><span class="op">[[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span><span class="op">)</span></code></pre></div>
<p>And finally, create the histograms:</p>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html">hist</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span><span class="op">(</span><span class="va">corres</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<p><img src="MultiAssayExperiment_Analysis_files/figure-html/unnamed-chunk-14-1.png" width="700"></p>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html">hist</a></span><span class="op">(</span><span class="va">corres</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/lower.tri.html">upper.tri</a></span><span class="op">(</span><span class="va">corres</span><span class="op">)</span><span class="op">]</span><span class="op">)</span></code></pre></div>
<p><img src="MultiAssayExperiment_Analysis_files/figure-html/unnamed-chunk-14-2.png" width="700"></p>
<p><strong>Part 2</strong></p>
<p>For the gene with highest correlation to copy number, make a box plot of log2 expression against copy number.</p>
<p><strong>Solution</strong></p>
<p>First, identify the gene with highest correlation between expression and copy number:</p>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/which.min.html">which.max</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span><span class="op">(</span><span class="va">corres</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<pre><code>## EIF4E 
##    91</code></pre>
<p>You could now make the plot by taking the EIF4E columns from each element of the list subacc.list <em>list</em> that was extracted from the subacc <em>MultiAssayExperiment</em>, but let’s do it by subsetting and extracting from the <em>MultiAssayExperiment</em>:</p>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">df</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html">wideFormat</a></span><span class="op">(</span><span class="va">subacc</span><span class="op">[</span><span class="st">"EIF4E"</span>, , <span class="op">]</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">df</span><span class="op">)</span></code></pre></div>
<pre><code>## DataFrame with 6 rows and 3 columns
##        primary RNASeq2GeneNorm_EIF4E gistict_EIF4E
##    &lt;character&gt;             &lt;numeric&gt;     &lt;numeric&gt;
## 1 TCGA-OR-A5J1               460.615             0
## 2 TCGA-OR-A5J2               371.225             0
## 3 TCGA-OR-A5J3               516.072             0
## 4 TCGA-OR-A5J5              1129.357             1
## 5 TCGA-OR-A5J6               822.078             0
## 6 TCGA-OR-A5J7               344.565            -1</code></pre>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/boxplot.html">boxplot</a></span><span class="op">(</span><span class="va">RNASeq2GeneNorm_EIF4E</span> <span class="op">~</span> <span class="va">gistict_EIF4E</span>,
        data<span class="op">=</span><span class="va">df</span>, varwidth<span class="op">=</span><span class="cn">TRUE</span>,
        xlab<span class="op">=</span><span class="st">"GISTIC Relative Copy Number Call"</span>,
        ylab<span class="op">=</span><span class="st">"RNA-seq counts"</span><span class="op">)</span></code></pre></div>
<p><img src="MultiAssayExperiment_Analysis_files/figure-html/unnamed-chunk-17-1.png" width="700"></p>
</div>
<div id="identifying-correlated-principal-components" class="section level2">
<h2 class="hasAnchor">
<a href="#identifying-correlated-principal-components" class="anchor"></a>Identifying correlated principal components</h2>
<p>Perform Principal Components Analysis of each of the five assays, using samples available on each assay, log-transforming RNA-seq data first. Using the first 10 components, calculate Pearson correlation between all scores and plot these correlations as a heatmap to identify correlated components across assays.</p>
<p><strong>Solution</strong></p>
<p>Here’s a function to simplify doing the PCAs:</p>
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">getLoadings</span> <span class="op">&lt;-</span>
    <span class="kw">function</span><span class="op">(</span><span class="va">x</span>, <span class="va">ncomp</span><span class="op">=</span><span class="fl">10</span>, <span class="va">dolog</span><span class="op">=</span><span class="cn">FALSE</span>, <span class="va">center</span><span class="op">=</span><span class="cn">TRUE</span>, <span class="va">scale.</span><span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span> <span class="op">{</span>
        <span class="kw">if</span> <span class="op">(</span><span class="va">dolog</span><span class="op">)</span>
            <span class="va">x</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span><span class="op">(</span><span class="va">x</span> <span class="op">+</span> <span class="fl">1</span><span class="op">)</span>
        <span class="va">pc</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/prcomp.html">prcomp</a></span><span class="op">(</span><span class="va">x</span>, center<span class="op">=</span><span class="va">center</span>, scale.<span class="op">=</span><span class="va">scale.</span><span class="op">)</span>
        <span class="fu"><a href="https://rdrr.io/r/base/t.html">t</a></span><span class="op">(</span><span class="va">pc</span><span class="op">$</span><span class="va">rotation</span><span class="op">[</span>, <span class="fl">1</span><span class="op">:</span><span class="fl">10</span><span class="op">]</span><span class="op">)</span>
    <span class="op">}</span></code></pre></div>
<p>Although it would be possible to do the following with a loop, the different data types do require different options for PCA (e.g. mutations are a 0/1 matrix with 1 meaning there is a somatic mutation, and gistict varies between -2 for homozygous loss and 2 for a genome doubling, so neither make sense to scale and center). So it is just as well to do the following one by one, concatenating each PCA results to the MultiAssayExperiment:</p>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">miniACC2</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html">intersectColumns</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span>
<span class="va">miniACC2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="va">miniACC2</span>,
    rnaseqPCA <span class="op">=</span> <span class="fu">getLoadings</span><span class="op">(</span><span class="fu">assays</span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span>, dolog<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span>,
    gistictPCA <span class="op">=</span> <span class="fu">getLoadings</span><span class="op">(</span><span class="fu">assays</span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span><span class="op">[[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span>, center<span class="op">=</span><span class="cn">FALSE</span>, scale.<span class="op">=</span><span class="cn">FALSE</span><span class="op">)</span>,
    proteinPCA <span class="op">=</span> <span class="fu">getLoadings</span><span class="op">(</span><span class="fu">assays</span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span><span class="op">[[</span><span class="fl">3</span><span class="op">]</span><span class="op">]</span><span class="op">)</span>,
    mutationsPCA <span class="op">=</span> <span class="fu">getLoadings</span><span class="op">(</span><span class="fu">assays</span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span><span class="op">[[</span><span class="fl">4</span><span class="op">]</span><span class="op">]</span>, center<span class="op">=</span><span class="cn">FALSE</span>, scale.<span class="op">=</span><span class="cn">FALSE</span><span class="op">)</span>,
    miRNAPCA <span class="op">=</span> <span class="fu">getLoadings</span><span class="op">(</span><span class="fu">assays</span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span><span class="op">[[</span><span class="fl">5</span><span class="op">]</span><span class="op">]</span><span class="op">)</span>,
    mapFrom <span class="op">=</span> <span class="fl">1</span><span class="op">:</span><span class="fl">5</span>
<span class="op">)</span></code></pre></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<p>Now subset to keep <em>only</em> the PCA results:</p>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">miniACC2</span> <span class="op">&lt;-</span> <span class="va">miniACC2</span><span class="op">[</span>, , <span class="fl">6</span><span class="op">:</span><span class="fl">10</span><span class="op">]</span></code></pre></div>
<pre><code>## Warning: 'experiments' dropped; see 'metadata'</code></pre>
<pre><code>## harmonizing input:
##   removing 215 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb35"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html">experiments</a></span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span></code></pre></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] rnaseqPCA: matrix with 10 rows and 43 columns
##  [2] gistictPCA: matrix with 10 rows and 43 columns
##  [3] proteinPCA: matrix with 10 rows and 43 columns
##  [4] mutationsPCA: matrix with 10 rows and 43 columns
##  [5] miRNAPCA: matrix with 10 rows and 43 columns</code></pre>
<p>Note, it would be equally easy (and maybe better) to do PCA on all samples available for each assay, then do intersectColumns at this point instead.</p>
<p>Now, steps for calculating the correlations and plotting a heatmap:</p>
<ul>
<li>Use <em>wideFormat</em> to paste these together, which has the nice property of adding assay names to the column names.</li>
<li>The first column always contains the sample identifier, so remove it.</li>
<li>Coerce to a matrix</li>
<li>Calculate the correlations, and take the absolute value (since signs of principal components are arbitrary)</li>
<li>Set the diagonals to NA (each variable has a correlation of 1 to itself).</li>
</ul>
<div class="sourceCode" id="cb37"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">df</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html">wideFormat</a></span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span><span class="op">[</span>, <span class="op">-</span><span class="fl">1</span><span class="op">]</span>
<span class="va">mycors</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html">cor</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/matrix.html">as.matrix</a></span><span class="op">(</span><span class="va">df</span><span class="op">)</span><span class="op">)</span>
<span class="va">mycors</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">abs</a></span><span class="op">(</span><span class="va">mycors</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span><span class="op">(</span><span class="va">mycors</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="cn">NA</span></code></pre></div>
<p>To simplify the heatmap, show only components that have at least one correlation greater than 0.5.</p>
<div class="sourceCode" id="cb38"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">has.high.cor</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/apply.html">apply</a></span><span class="op">(</span><span class="va">mycors</span>, <span class="fl">2</span>, <span class="va">max</span>, na.rm<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span> <span class="op">&gt;</span> <span class="fl">0.5</span>
<span class="va">mycors</span> <span class="op">&lt;-</span> <span class="va">mycors</span><span class="op">[</span><span class="va">has.high.cor</span>, <span class="va">has.high.cor</span><span class="op">]</span>
<span class="fu"><a href="https://rdrr.io/pkg/pheatmap/man/pheatmap.html">pheatmap</a></span><span class="op">(</span><span class="va">mycors</span><span class="op">)</span></code></pre></div>
<p><img src="MultiAssayExperiment_Analysis_files/figure-html/unnamed-chunk-22-1.png" width="700"></p>
<p>The highest correlation present is between PC2 of the RNA-seq assay, and PC1 of the protein assay.</p>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Marcel Ramos, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
