<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MultiAssayExperiment: Integrative workflow for public multi-omics data • MultiAssayWorkshop</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="MultiAssayExperiment: Integrative workflow for public multi-omics data">
<meta property="og:description" content="MultiAssayWorkshop">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MultiAssayWorkshop</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">3.11.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/Ramos_MultiAssayExperiment.html">Workshop</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorial
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../tutorials/Exercise_MultiAssay.html">Build Your First MAE</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Reference
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/cBioPortal_Studies.html">cBioPortal Studies</a>
    </li>
    <li>
      <a href="../articles/curatedTCGAData_ref.html">curatedTCGAData Reference</a>
    </li>
    <li>
      <a href="../articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MultiAssayWorkshop">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="Ramos_MultiAssayExperiment_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MultiAssayExperiment: Integrative workflow for public multi-omics data</h1>
                        <h4 class="author">Marcel Ramos</h4>
            <address class="author_afil">
      Roswell Park Comprehensive Cancer Center, Buffalo, NY<br><h4 class="author">Ludwig Geistlinger</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="author">Levi Waldron</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="date">July 17, 2020</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/MultiAssayWorkshop/blob/master/vignettes/Ramos_MultiAssayExperiment.Rmd"><code>vignettes/Ramos_MultiAssayExperiment.Rmd</code></a></small>
      <div class="hidden name"><code>Ramos_MultiAssayExperiment.Rmd</code></div>

    </address>
</address>
</address>
</div>

    
    
<div id="multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment" class="section level1">
<h1 class="hasAnchor">
<a href="#multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment" class="anchor"></a>Multi-omic Integration and Analysis of cBioPortal and TCGA data with MultiAssayExperiment</h1>
<div id="instructor-names-and-contact-information" class="section level2">
<h2 class="hasAnchor">
<a href="#instructor-names-and-contact-information" class="anchor"></a>Instructor names and contact information</h2>
<ul>
<li>Marcel Ramos<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> <a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> (<a href="mailto:marcel.ramos@roswellpark.org" class="email">marcel.ramos@roswellpark.org</a>)</li>
<li>Ludwig Geistlinger<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>
</li>
<li>Levi Waldron<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>
</li>
</ul>
</div>
<div id="workshop-description" class="section level2">
<h2 class="hasAnchor">
<a href="#workshop-description" class="anchor"></a>Workshop Description</h2>
<p>This workshop demonstrates data management and analyses of multiple assays associated with a single set of biological specimens, using the <code>MultiAssayExperiment</code> data class and methods. It introduces the facilities made available by companion packages, <code>cBioPortalData</code>, <code>curatedTCGAData</code>, and <code>TCGAutils</code> to work with publicly available multi-omic data. Using the <code>MultiAssayExperiment</code> class, these packages make the management of multiple assays easier and more efficient. The workshop also covers, to a limited extent, relevant data classes such as <code>SummarizedExperiment</code> and <code>RangedSummarizedExperiment</code>, which provide efficient and powerful operations for representation of copy number, mutation, variant, and expression data that are represented by different genomic ranges for each specimen.</p>
<div id="pre-requisites" class="section level3">
<h3 class="hasAnchor">
<a href="#pre-requisites" class="anchor"></a>Pre-requisites</h3>
<ul>
<li>Basic knowledge of R</li>
<li>Familiarity with the GRanges and SummarizedExperiment classes</li>
<li>Familiarity with ’omics data types including copy number and gene expression</li>
</ul>
</div>
<div id="workshop-participation" class="section level3">
<h3 class="hasAnchor">
<a href="#workshop-participation" class="anchor"></a>Workshop Participation</h3>
<p>Participants will have a chance to build a <code>MultiAssayExperiment</code> object from scratch, and will also work with data and interfaces from the the <code>curatedTCGAData</code> and <code>cBioPortalData</code> packages.</p>
</div>
<div id="rbioconductor-packages-used" class="section level3">
<h3 class="hasAnchor">
<a href="#rbioconductor-packages-used" class="anchor"></a>R/Bioconductor packages used</h3>
<p>First, please make sure you have the latest Bioconductor release version 3.11. See <a href="https://www.bioconductor.org/install">Bioconductor installation</a> instructions:</p>
<div class="sourceCode" id="cb1"><html><body><pre class="r"><span class="kw">if</span> (!<span class="fu"><a href="https://rdrr.io/r/base/ns-load.html">requireNamespace</a></span>(<span class="st">"BiocManager"</span>, <span class="kw">quietly</span> <span class="kw">=</span> <span class="fl">TRUE</span>))
    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span>(<span class="st">"BiocManager"</span>)

<span class="co">## Bioconductor 3.11 Stable Release</span>
<span class="kw pkg">BiocManager</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/install.html">install</a></span>(<span class="kw">version</span> <span class="kw">=</span> <span class="st">"3.11"</span>)</pre></body></html></div>
<ul>
<li><a href="https://bioconductor.org/packages/MultiAssayExperiment">MultiAssayExperiment</a></li>
<li><a href="https://bioconductor.org/packages/GenomicRanges">GenomicRanges</a></li>
<li><a href="https://bioconductor.org/packages/RaggedExperiment">RaggedExperiment</a></li>
<li><a href="https://bioconductor.org/packages/curatedTCGAData">curatedTCGAData</a></li>
<li><a href="https://github.com/waldronlab/cBioPortalData">cBioPortalData</a></li>
<li><a href="https://bioconductor.org/packages/SummarizedExperiment">SummarizedExperiment</a></li>
<li><a href="https://bioconductor.org/packages/TCGAutils">TCGAutils</a></li>
<li><a href="https://bioconductor.org/packages/UpSetR">UpSetR</a></li>
<li><a href="https://bioconductor.org/packages/AnnotationFilter">AnnotationFilter</a></li>
<li><a href="https://bioconductor.org/packages/EnsDb.Hsapiens.v86">EnsDb.Hsapiens.v86</a></li>
<li><a href="https://cran.r-project.org/package=survival">survival</a></li>
<li><a href="https://cran.r-project.org/package=survminer">survminer</a></li>
<li><a href="https://cran.r-project.org/package=pheatmap">pheatmap</a></li>
</ul>
<div class="sourceCode" id="cb2"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">MultiAssayExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">GenomicRanges</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">RaggedExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">curatedTCGAData</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">cBioPortalData</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">GenomicDataCommons</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">SummarizedExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">SingleCellExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">TCGAutils</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">UpSetR</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">mirbase.db</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">AnnotationFilter</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">EnsDb.Hsapiens.v86</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">survival</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">survminer</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">pheatmap</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">readxl</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">readr</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">kableExtra</span>)</pre></body></html></div>
</div>
</div>
<div id="time-outline" class="section level2">
<h2 class="hasAnchor">
<a href="#time-outline" class="anchor"></a>Time outline</h2>
<p>55 minutes total</p>
<table class="table">
<thead><tr class="header">
<th>Activity</th>
<th>Time</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>Overview of key packages and data classes</td>
<td>15m</td>
</tr>
<tr class="even">
<td>Introduction to <code>curatedTCGAData</code>, <code>cBioPortalData</code>, and <code>TCGAutils</code>
</td>
<td>10m</td>
</tr>
<tr class="odd">
<td>Building a <code>MultiAssayExperiment</code> from scratch</td>
<td>10m</td>
</tr>
<tr class="even">
<td>Creating and importing TCGA multi-assay datasets</td>
<td>10m</td>
</tr>
<tr class="odd">
<td>Subsetting and reshaping multi-assay data</td>
<td>10m</td>
</tr>
</tbody>
</table>
</div>
<div id="workshop-goals-and-objectives" class="section level2">
<h2 class="hasAnchor">
<a href="#workshop-goals-and-objectives" class="anchor"></a>Workshop goals and objectives</h2>
<div id="learning-goals" class="section level3">
<h3 class="hasAnchor">
<a href="#learning-goals" class="anchor"></a>Learning goals</h3>
<ul>
<li>overview common data classes available in Bioconductor</li>
<li>understand the data structure for representing multi-omics data</li>
<li>explore available functionality in <code>TCGAutils</code> for the management and coordination of multi-omic data</li>
<li>gain familiarity with <code>cBioPortalData</code> and <code>curatedTCGAData</code>
</li>
<li>identify advantages of providing integrative data in an analysis-ready platform</li>
</ul>
</div>
<div id="learning-objectives" class="section level3">
<h3 class="hasAnchor">
<a href="#learning-objectives" class="anchor"></a>Learning objectives</h3>
<ul>
<li>use <code>curatedTCGAData</code> and <code>cBioPortalData</code> to create on-the-fly multi-omics data objects</li>
<li>apply knowlege of <code>MultiAssayExperiment</code> to represent other multi-omics data</li>
<li>learn about data management techniques such as subsetting, reshaping, growing, and extraction using MultAssayExperiment</li>
</ul>
</div>
</div>
<div id="overview-of-key-packages-and-data-classes" class="section level2">
<h2 class="hasAnchor">
<a href="#overview-of-key-packages-and-data-classes" class="anchor"></a>Overview of key packages and data classes</h2>
<div id="packages" class="section level3">
<h3 class="hasAnchor">
<a href="#packages" class="anchor"></a>Packages</h3>
<div id="multiassayexperiment" class="section level4">
<h4 class="hasAnchor">
<a href="#multiassayexperiment" class="anchor"></a><code>MultiAssayExperiment</code>
</h4>
<p>The <code>MultiAssayExperiment</code> package provides an integrative representation for multi-omics data on a set of samples. Modelled after the <code>SummarizedExperiment</code> representation for expression data, the <code>MultiAssayExperiment</code> data container provides efficient and easy-to-use operations for manipulating multiple sets of data such as copy number alterations, mutations, proteomics, methylation, and other types of data. Compatibility with the <code>MultiAssayExperiment</code> container hinges on a small set of requirements, making the data representation flexible to many of the currently available Bioconductor classes.</p>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic" width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic
</p>
</div>
</div>
<div id="cbioportaldata" class="section level4">
<h4 class="hasAnchor">
<a href="#cbioportaldata" class="anchor"></a><code>cBioPortalData</code>
</h4>
<p>The <code>cBioPortalData</code> package provides an R/Bioconductor interface to cBioPortal data. The <code>cBioPortalData</code> package makes use of the revamped API and provides an internal caching mechanism to avoid re-querying the API. It allows users to download data from <a href="https://cBioPortal.org" class="uri">https://cBioPortal.org</a> through customized queries that are handled for the user <em>in the background</em>. We provide an easy to use interface that requires minimal knowledge of the cBioPortal data model and API.</p>
</div>
<div id="curatedtcgadata" class="section level4">
<h4 class="hasAnchor">
<a href="#curatedtcgadata" class="anchor"></a><code>curatedTCGAData</code>
</h4>
<p>Many tools exist for accessing and downloading The Cancer Genome Atlas (TCGA) data. These include but are not limited to <code>RTCGAToolbox</code>, <code>GenomicDataCommons</code> (package and website), <code>TCGAbiolinks</code>, the cBioPortal website, and Broad GDAC Firehose. These tools encompass a spectrum of strengths in ease-of-use, integration, and completeness of data. Few tools provide an integrative and user-friendly representation of TCGA data in a widely used analysis platform like Bioconductor.</p>
<p>The <code>curatedTCGAData</code> experiment data package provides on-the-fly construction of TCGA datasets for 33 different cancer types from the <a href="https://gdac.broadinstitute.org">Broad GDAC Firehose</a> pipeline. Generally, it provides data from genome build <code>hg19</code>. <code>curatedTCGAData</code> facilitates access and integration of TCGA data by providing multi-’omics data objects using <code>MultiAssayExperiment</code>. Where other platforms provide fragmented datasets, <code>curatedTCGAData</code> ensures that all data provided is matched and accounted for within the phenotypic metadata.</p>
<p>Reference vignettes:</p>
<ul>
<li><p><code>Available Studies</code> – (curatedTCGAData section) A list of available cancer studies from <code><a href="https://rdrr.io/pkg/TCGAutils/man/diseaseCodes.html">TCGAutils::diseaseCodes</a></code>.</p></li>
<li><p><code>OmicsTypes</code> – A descriptive table of ’omics types in <code>curatedTCGAData</code> (thanks to Ludwig G. <code>@lgeistlinger</code>)</p></li>
</ul>
</div>
<div id="tcgautils" class="section level4">
<h4 class="hasAnchor">
<a href="#tcgautils" class="anchor"></a><code>TCGAutils</code>
</h4>
<p><code>TCGAutils</code> is a companion package that enhances <code>curatedTCGAData</code> by allowing additional exploration and manipulation of samples and metadata in rows and columns.</p>
<p>Available operations in <code>TCGAutils</code> and <code>MultiAssayExperiment</code> enable user-friendly operations for subsetting, separating, converting, and reshaping of sample and feature TCGA data.</p>
<p><code>TCGAutils</code> was developed specifically for TCGA data and for <code>curatedTCGAData</code> products. It provides convenience / helper functions in three major areas:</p>
<ol style="list-style-type: decimal">
<li>conversion / summarization of row annotations to genomic ranges</li>
<li>identification and separation of samples</li>
<li>translation and interpretation of TCGA identifiers</li>
</ol>
<p>along with additional reference data sets to explore TCGA data.</p>
<p>For the cheatsheet reference table, see the <code>TCGAutils Cheatsheet</code> vignette.</p>
<p>To better understand how it all fits together, this schematic shows the relationship among all as part of the curatedTCGAData pipeline.</p>
<div class="figure">
<img src="TCGAMAEPipeline20190204.png" alt="Schematic of curatedTCGAData Pipeline" width="100%"><p class="caption">
Schematic of curatedTCGAData Pipeline
</p>
</div>
</div>
</div>
<div id="data-classes" class="section level3">
<h3 class="hasAnchor">
<a href="#data-classes" class="anchor"></a>Data Classes</h3>
<p>This section summarizes three fundamental data classes for the representation of multi-omics experiments.</p>
</div>
<div id="rangedsummarizedexperiment" class="section level3">
<h3 class="hasAnchor">
<a href="#rangedsummarizedexperiment" class="anchor"></a><code>(Ranged)SummarizedExperiment</code>
</h3>
<div class="figure">
<img src="SummarizedExperiment.png" alt="A matrix-like container where rows represent features of interest and columns represent samples. The objects contain one or more assays, each represented by a matrix-like object of numeric or other mode." width="100%"><p class="caption">
A matrix-like container where rows represent features of interest and columns represent samples. The objects contain one or more assays, each represented by a matrix-like object of numeric or other mode.
</p>
</div>
<p><code>SummarizedExperiment</code> is the most important Bioconductor class for matrix-like experimental data, including from RNA sequencing and microarray experiments. It can store multiple experimental data matrices of identical dimensions, with associated metadata on the rows/genes/transcripts/other measurements (<code>rowData</code>), column/sample phenotype or clinical data (<code>colData</code>), and the overall experiment (<code>metadata</code>). The derivative class <code>RangedSummarizedExperiment</code> associates a <code>GRanges</code> or <code>GRangesList</code> vector with the rows. These classes supersede the use of <code>ExpressionSet</code>. Note that many other classes for experimental data are actually derived from <code>SummarizedExperiment</code>; for example, the <code>SingleCellExperiment</code> class for single-cell RNA sequencing experiments extends <code>RangedSummarizedExperiment</code>, which in turn extends <code>SummarizedExperiment</code>:</p>
<div class="sourceCode" id="cb3"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">SingleCellExperiment</span>)
<span class="fu">extends</span>(<span class="st">"SingleCellExperiment"</span>)</pre></body></html></div>
<pre><code>## [1] "SingleCellExperiment"       "RangedSummarizedExperiment"
## [3] "SummarizedExperiment"       "RectangularData"           
## [5] "Vector"                     "Annotated"                 
## [7] "vector_OR_Vector"</code></pre>
<p>Thus, although <code>SingleCellExperiment</code> provides additional methods over <code>RangedSummarizedExperiment</code>, it also inherits all the methods of <code>SummarizedExperiment</code> and <code>RangedSummarizedExperiment</code>, so everything you learn about <code>SummarizedExperiment</code> will be applicable to <code>SingleCellExperiment</code>.</p>
</div>
<div id="raggedexperiment" class="section level3">
<h3 class="hasAnchor">
<a href="#raggedexperiment" class="anchor"></a><code>RaggedExperiment</code>
</h3>
<p><code>RaggedExperiment</code> is a flexible data representation for segmented copy number, somatic mutations such as represented in <code>.vcf</code> files, and other ragged array schema for genomic location data. Like the <code>GRangesList</code> class from <code>GenomicRanges</code>, <code>RaggedExperiment</code> can be used to represent <em>differing</em> genomic ranges on each of a set of samples. In fact, <code>RaggedExperiment</code> contains an internal <code>GRangesList</code>:</p>
<div class="sourceCode" id="cb5"><html><body><pre class="r"><span class="fu">showClass</span>(<span class="st">"RaggedExperiment"</span>)</pre></body></html></div>
<pre><code>## Class "RaggedExperiment" [package "RaggedExperiment"]
## 
## Slots:
##                                                       
## Name:       assays      rowidx      colidx    metadata
## Class: GRangesList     integer     integer        list
## 
## Extends: "Annotated"</code></pre>
<p><code>RaggedExperiment</code> provides a flexible set of <em>Assay</em> methods to support transformation of data to matrix format.</p>
<div class="figure">
<img src="RaggedExperiment.png" alt="RaggedExperiment object schematic. Rows and columns represent genomic ranges and samples, respectively. Assay operations can be performed with (from left to right) compactAssay, qreduceAssay, and sparseAssay." width="\maxwidth"><p class="caption">
RaggedExperiment object schematic. Rows and columns represent genomic ranges and samples, respectively. Assay operations can be performed with (from left to right) compactAssay, qreduceAssay, and sparseAssay.
</p>
</div>
</div>
</div>
<div id="the-integrative-container" class="section level2">
<h2 class="hasAnchor">
<a href="#the-integrative-container" class="anchor"></a>The Integrative Container</h2>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic. colData provides data about the patients, cell lines, or other biological units, with one row per unit and one column per variable. The experiments are a list of assay datasets of arbitrary class.  The sampleMap relates each column (observation) in ExperimentList to exactly one row (biological unit) in colData; however, one row of colData may map to zero, one, or more columns per assay, allowing for missing and replicate assays. sampleMap allows for per-assay sample naming conventions. Metadata can be used to store information in arbitrary format about the MultiAssayExperiment. Green stripes indicate a mapping of one subject to multiple observations across experiments." width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic. colData provides data about the patients, cell lines, or other biological units, with one row per unit and one column per variable. The experiments are a list of assay datasets of arbitrary class. The sampleMap relates each column (observation) in ExperimentList to exactly one row (biological unit) in colData; however, one row of colData may map to zero, one, or more columns per assay, allowing for missing and replicate assays. sampleMap allows for per-assay sample naming conventions. Metadata can be used to store information in arbitrary format about the MultiAssayExperiment. Green stripes indicate a mapping of one subject to multiple observations across experiments.
</p>
</div>
<div id="multiassayexperiment-1" class="section level3">
<h3 class="hasAnchor">
<a href="#multiassayexperiment-1" class="anchor"></a><code>MultiAssayExperiment</code>
</h3>
<p><code>MultiAssayExperiment</code> is an integrative container for coordinating multi-omics experiment data on a set of biological specimens. As much as possible, its methods adopt the same vocabulary as <code>SummarizedExperiment</code>. A <code>MultiAssayExperiment</code> can contain any number of assays with different representations. Assays may be <em>ID-based</em>, where measurements are indexed identifiers of genes, microRNA, proteins, microbes, etc. Alternatively, assays may be <em>range-based</em>, where measurements correspond to genomic ranges that can be represented as <code>GRanges</code> objects, such as gene expression or copy number. For ID-based assays, there is no requirement that the same IDs be present for different experiments. For range-based assays, there is also no requirement that the same ranges be present for different experiments; furthermore, it is possible for different samples within an experiment to be represented by different ranges. The following data classes have been tested to work as elements of a <code>MultiAssayExperiment</code>:</p>
<ol style="list-style-type: decimal">
<li>
<code>matrix</code>: the most basic class for ID-based datasets, could be used for example for gene expression summarized per-gene, microRNA, metabolomics, or microbiome data.</li>
<li>
<code>SummarizedExperiment</code> and derived methods: described above, could be used for miRNA, gene expression, proteomics, or any matrix-like data where measurements are represented by IDs.</li>
<li>
<code>RangedSummarizedExperiment</code>: described above, could be used for gene expression, methylation, or other data types referring to genomic positions.</li>
<li>
<code>ExpressionSet</code>: Another rich representation for ID-based datasets, supported only for legacy reasons</li>
<li>
<code>RaggedExperiment</code>: described above, for non-rectangular (ragged) ranged-based datasets such as segmented copy number, where segmentation of copy number alterations occurs and different genomic locations in each sample.</li>
<li>
<code>RangedVcfStack</code>: For VCF archives broken up by chromosome (see <code>VcfStack</code> class defined in the <code>GenomicFiles</code> package)</li>
<li>
<code>DelayedMatrix</code>: An on-disk representation of matrix-like objects for large datasets. It reduces memory usage and optimizes performance with delayed operations. This class is part of the <code>DelayedArray</code> package.</li>
</ol>
<p>Note that any data class extending these classes, and in fact any data class supporting row and column names and subsetting can be used as an element of a <code>MultiAssayExperiment</code>.</p>
</div>
</div>
<div id="building-a-multiassayexperiment-from-scratch" class="section level2">
<h2 class="hasAnchor">
<a href="#building-a-multiassayexperiment-from-scratch" class="anchor"></a>Building a MultiAssayExperiment from scratch</h2>
<p>To start from scratch building your own <code>MultiAssayExperiment</code>, see the package vignette <a href="https://bioconductor.org/packages/release/bioc/vignettes/MultiAssayExperiment/inst/doc/MultiAssayExperiment.html">Coordinating Analysis of Multi-Assay Experiments vignette</a>. The package <a href="https://bioconductor.org/packages/release/bioc/vignettes/MultiAssayExperiment/inst/doc/MultiAssayExperiment_cheatsheet.pdf">cheat sheet</a> is also helpful.</p>
<p>If anything is unclear, please ask questions during the workshop.</p>
<p>For more involved / detailed questions and bug reports, resort to the Bioconductor <a href="https://support.bioconductor.org/">support site</a> or create an issue on the <a href="https://github.com/waldronlab/MultiAssayExperiment/issues">MultiAssayExperiment issue tracker</a> on Github.</p>
<div id="multiassayexperiment-construction-and-concatenation" class="section level3">
<h3 class="hasAnchor">
<a href="#multiassayexperiment-construction-and-concatenation" class="anchor"></a>MultiAssayExperiment construction and concatenation</h3>
<div id="constructor-function" class="section level4">
<h4 class="hasAnchor">
<a href="#constructor-function" class="anchor"></a>constructor function</h4>
<p>The <code>MultiAssayExperiment</code> constructor function can take three arguments:</p>
<ol style="list-style-type: decimal">
<li>
<code>experiments</code> - An <code>ExperimentList</code> or <code>list</code> of rectangular data</li>
<li>
<code>colData</code> - A <code>DataFrame</code> describing the patients (or cell lines, or other biological units)</li>
<li>
<code>sampleMap</code> - A <code>DataFrame</code> of <code>assay</code>, <code>primary</code>, and <code>colname</code> identifiers</li>
</ol>
<p>The miniACC object can be reconstructed as follows:</p>
<div class="sourceCode" id="cb7"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"miniACC"</span>)

<span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment.html">MultiAssayExperiment</a></span>(<span class="kw">experiments</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC</span>),
    <span class="kw">colData</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACC</span>),
    <span class="kw">sampleMap</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap</a></span>(<span class="no">miniACC</span>),
    <span class="kw">metadata</span><span class="kw">=</span><span class="fu">metadata</span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="prepmultiassay---constructor-function-helper" class="section level4">
<h4 class="hasAnchor">
<a href="#prepmultiassay---constructor-function-helper" class="anchor"></a><code>prepMultiAssay</code> - Constructor function helper</h4>
<p>The <code>prepMultiAssay</code> function allows the user to diagnose typical problems when creating a <code>MultiAssayExperiment</code> object. See <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/prepMultiAssay.html">?prepMultiAssay</a></code> for more details.</p>
</div>
<div id="c---concatenate-to-multiassayexperiment" class="section level4">
<h4 class="hasAnchor">
<a href="#c---concatenate-to-multiassayexperiment" class="anchor"></a><code>c</code> - concatenate to MultiAssayExperiment</h4>
<p>The <code>c</code> function allows the user to concatenate an additional experiment to an existing <code>MultiAssayExperiment</code>. The optional <code>sampleMap</code> argument allows concatenating an assay whose column names do not match the row names of <code>colData</code>. For convenience, the <em>mapFrom</em> argument allows the user to map from a particular experiment <strong>provided</strong> that the <strong>order</strong> of the colnames is in the <strong>same</strong>. A <code>warning</code> will be issued to make the user aware of this assumption. For example, to concatenate a matrix of log2-transformed RNA-seq results:</p>
<div class="sourceCode" id="cb9"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC</span>,
    <span class="kw">log2rnaseq</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC</span>)$<span class="no">RNASeq2GeneNorm</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">1L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb11"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC2</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 6:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
##  [6] log2rnaseq: matrix with 198 rows and 79 columns</code></pre>
</div>
</div>
</div>
<div id="working-with-multiassayexperiment" class="section level2">
<h2 class="hasAnchor">
<a href="#working-with-multiassayexperiment" class="anchor"></a>Working with MultiAssayExperiment</h2>
<div id="api-cheat-sheet" class="section level3">
<h3 class="hasAnchor">
<a href="#api-cheat-sheet" class="anchor"></a>API cheat sheet</h3>
<div class="figure">
<img src="MultiAssayExperiment_cheatsheet.png" alt="The MultiAssayExperiment API for construction, access, subsetting, management, and reshaping to formats for application of R/Bioconductor graphics and analysis packages." width="\maxwidth"><p class="caption">
The MultiAssayExperiment API for construction, access, subsetting, management, and reshaping to formats for application of R/Bioconductor graphics and analysis packages.
</p>
</div>
</div>
<div id="the-multiassayexperiment-miniacc-demo-object" class="section level3">
<h3 class="hasAnchor">
<a href="#the-multiassayexperiment-miniacc-demo-object" class="anchor"></a>The MultiAssayExperiment miniACC demo object</h3>
<p>Get started by trying out <code>MultiAssayExperiment</code> using a subset of the TCGA adrenocortical carcinoma (ACC) dataset provided with the package. This dataset provides five assays on 92 patients, although all five assays were not performed for every patient:</p>
<ol style="list-style-type: decimal">
<li>
<strong>RNASeq2GeneNorm</strong>: gene mRNA abundance by RNA-seq</li>
<li>
<strong>gistict</strong>: GISTIC genomic copy number by gene</li>
<li>
<strong>RPPAArray</strong>: protein abundance by Reverse Phase Protein Array</li>
<li>
<strong>Mutations</strong>: non-silent somatic mutations by gene</li>
<li>
<strong>miRNASeqGene</strong>: microRNA abundance by microRNA-seq.</li>
</ol>
<div class="sourceCode" id="cb13"><html><body><pre class="r"><span class="no">miniACC</span></pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="coldata---information-biological-units" class="section level3">
<h3 class="hasAnchor">
<a href="#coldata---information-biological-units" class="anchor"></a>colData - information biological units</h3>
<p>This slot is a <code>DataFrame</code> describing the characteristics of biological units, for example clinical data for patients. In the prepared datasets from [The Cancer Genome Atlas][], each row is one patient and each column is a clinical, pathological, subtype, or other variable. The <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> function provides a shortcut for accessing or setting <code>colData</code> columns.</p>
<div class="sourceCode" id="cb15"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACC</span>)[<span class="fl">1</span>:<span class="fl">4</span>, <span class="fl">1</span>:<span class="fl">4</span>]</pre></body></html></div>
<pre><code>## DataFrame with 4 rows and 4 columns
##                 patientID years_to_birth vital_status days_to_death
##               &lt;character&gt;      &lt;integer&gt;    &lt;integer&gt;     &lt;integer&gt;
## TCGA-OR-A5J1 TCGA-OR-A5J1             58            1          1355
## TCGA-OR-A5J2 TCGA-OR-A5J2             44            1          1677
## TCGA-OR-A5J3 TCGA-OR-A5J3             23            0            NA
## TCGA-OR-A5J4 TCGA-OR-A5J4             23            1           423</code></pre>
<div class="sourceCode" id="cb17"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/table.html">table</a></span>(<span class="no">miniACC</span>$<span class="no">race</span>)</pre></body></html></div>
<pre><code>## 
##                     asian black or african american                     white 
##                         2                         1                        78</code></pre>
<p><em>Key points about the colData:</em></p>
<ul>
<li>Each row maps to zero or more observations in each experiment in the <code>ExperimentList</code>, below.</li>
<li>One row per biological unit
<ul>
<li>
<code>MultiAssayExperiment</code> supports both missing observations and replicate observations, ie one row of <code>colData</code> can map to 0, 1, or more columns of any of the experimental data matrices.</li>
<li>therefore you could treat replicate observations as one or multiple rows of <code>colData</code>, and this will result in different behaviors of functions you will learn later like subsetting, <code><a href="https://rdrr.io/r/base/duplicated.html">duplicated()</a></code>, and <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat()</a></code>.</li>
<li>multiple time points, or distinct biological replicates, should probably be separate rows of the <code>colData</code>.</li>
</ul>
</li>
</ul>
</div>
<div id="experimentlist---experiment-data" class="section level3">
<h3 class="hasAnchor">
<a href="#experimentlist---experiment-data" class="anchor"></a>ExperimentList - experiment data</h3>
<p>A base <code>list</code> or <code>ExperimentList</code> object containing the experimental datasets for the set of samples collected. This gets converted into a class <code>ExperimentList</code> during construction.</p>
<div class="sourceCode" id="cb19"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns</code></pre>
<p><em>Key points:</em></p>
<ul>
<li>One matrix-like dataset per list element (although they do not even need to be matrix-like, see for example the <code>RaggedExperiment</code> package)</li>
<li>One matrix column per assayed specimen. Each matrix column must correspond to exactly one row of <code>colData</code>: in other words, you must know which patient or cell line the observation came from. However, multiple columns can come from the same patient, or there can be no data for that patient.</li>
<li>Matrix rows correspond to variables, e.g. genes or genomic ranges</li>
<li>
<code>ExperimentList</code> elements can be genomic range-based (e.g. <code>SummarizedExperiment::RangedSummarizedExperiment-class</code> or <code>RaggedExperiment::RaggedExperiment-class</code>) or ID-based data (e.g. <code>SummarizedExperiment::SummarizedExperiment-class</code>, <code>Biobase::eSet-class</code> <code>base::matrix-class</code>, <code>DelayedArray::DelayedArray-class</code>, and derived classes)</li>
<li>Any data class can be included in the <code>ExperimentList</code>, as long as it supports: single-bracket subsetting (<code><a href="https://rdrr.io/r/base/Extract.html">[</a></code>), <code>dimnames</code>, and <code>dim</code>. Most data classes defined in Bioconductor meet these requirements.</li>
</ul>
</div>
<div id="samplemap---relationship-graph" class="section level3">
<h3 class="hasAnchor">
<a href="#samplemap---relationship-graph" class="anchor"></a>sampleMap - relationship graph</h3>
<p><code>sampleMap</code> is a graph representation of the relationship between biological units and experimental results. In simple cases where the column names of <code>ExperimentList</code> data matrices match the row names of <code>colData</code>, the user won’t need to specify or think about a sample map, it can be created automatically by the <code>MultiAssayExperiment</code> constructor. <code>sampleMap</code> is a simple three-column <code>DataFrame</code>:</p>
<ol style="list-style-type: decimal">
<li>
<code>assay</code> column: the name of the assay, and found in the names of <code>ExperimentList</code> list names</li>
<li>
<code>primary</code> column: identifiers of patients or biological units, and found in the row names of <code>colData</code>
</li>
<li>
<code>colname</code> column: identifiers of assay results, and found in the column names of <code>ExperimentList</code> elements Helper functions are available for creating a map from a list. See <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/mapToList.html">?listToMap</a></code>
</li>
</ol>
<div class="sourceCode" id="cb21"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## DataFrame with 385 rows and 3 columns
##               assay      primary                colname
##            &lt;factor&gt;  &lt;character&gt;            &lt;character&gt;
## 1   RNASeq2GeneNorm TCGA-OR-A5J1 TCGA-OR-A5J1-01A-11R..
## 2   RNASeq2GeneNorm TCGA-OR-A5J2 TCGA-OR-A5J2-01A-11R..
## 3   RNASeq2GeneNorm TCGA-OR-A5J3 TCGA-OR-A5J3-01A-11R..
## 4   RNASeq2GeneNorm TCGA-OR-A5J5 TCGA-OR-A5J5-01A-11R..
## 5   RNASeq2GeneNorm TCGA-OR-A5J6 TCGA-OR-A5J6-01A-31R..
## ...             ...          ...                    ...
## 381    miRNASeqGene TCGA-PA-A5YG TCGA-PA-A5YG-01A-11R..
## 382    miRNASeqGene TCGA-PK-A5H8 TCGA-PK-A5H8-01A-11R..
## 383    miRNASeqGene TCGA-PK-A5H9 TCGA-PK-A5H9-01A-11R..
## 384    miRNASeqGene TCGA-PK-A5HA TCGA-PK-A5HA-01A-11R..
## 385    miRNASeqGene TCGA-PK-A5HB TCGA-PK-A5HB-01A-11R..</code></pre>
<p><em>Key points:</em></p>
<ul>
<li>relates experimental observations (<code>colnames</code>) to <code>colData</code>
</li>
<li>permits experiment-specific sample naming, missing, and replicate observations</li>
</ul>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="metadata" class="section level3">
<h3 class="hasAnchor">
<a href="#metadata" class="anchor"></a>metadata</h3>
<p>Metadata can be used to keep additional information about patients, assays performed on individuals or on the entire cohort, or features such as genes, proteins, and genomic ranges. There are many options available for storing metadata. First, <code>MultiAssayExperiment</code> has its own metadata for describing the entire experiment:</p>
<div class="sourceCode" id="cb23"><html><body><pre class="r"><span class="fu">metadata</span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## $title
## [1] "Comprehensive Pan-Genomic Characterization of Adrenocortical Carcinoma"
## 
## $PMID
## [1] "27165744"
## 
## $sourceURL
## [1] "http://s3.amazonaws.com/multiassayexperiments/accMAEO.rds"
## 
## $RPPAfeatureDataURL
## [1] "http://genomeportal.stanford.edu/pan-tcga/show_target_selection_file?filename=Allprotein.txt"
## 
## $colDataExtrasURL
## [1] "http://www.cell.com/cms/attachment/2062093088/2063584534/mmc3.xlsx"</code></pre>
<p>Additionally, the <code>DataFrame</code> class used by <code>sampleMap</code> and <code>colData</code>, as well as the <code>ExperimentList</code> class, similarly support metadata. Finally, many experimental data objects that can be used in the <code>ExperimentList</code> support metadata. These provide flexible options to users and to developers of derived classes.</p>
</div>
</div>
<div id="multiassayexperiment-subsetting" class="section level2">
<h2 class="hasAnchor">
<a href="#multiassayexperiment-subsetting" class="anchor"></a>MultiAssayExperiment Subsetting</h2>
<div id="single-bracket" class="section level3">
<h3 class="hasAnchor">
<a href="#single-bracket" class="anchor"></a>Single bracket <code>[</code>
</h3>
<p>In pseudo code below, the subsetting operations work on the rows of the following indices: 1. <em>i</em> experimental data rows 2. <em>j</em> the primary names or the column names (entered as a <code>list</code> or <code>List</code>) 3. <em>k</em> assay</p>
<pre><code>multiassayexperiment[i = rownames, j = primary or colnames, k = assay]</code></pre>
<p>Subsetting operations always return another <code>MultiAssayExperiment</code>. For example, the following will return any rows named “MAPK14” or “IGFBP2”, and remove any assays where no rows match:</p>
<div class="sourceCode" id="cb26"><html><body><pre class="r"><span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"MAPK14"</span>, <span class="st">"IGFBP2"</span>), , ]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 170 sampleMap rows not in names(experiments)</code></pre>
<p>The following will keep only patients of pathological stage iv, and all their associated assays:</p>
<div class="sourceCode" id="cb28"><html><body><pre class="r"><span class="no">miniACC</span>[, <span class="no">miniACC</span>$<span class="no">pathologic_stage</span> <span class="kw">==</span> <span class="st">"stage iv"</span>, ]</pre></body></html></div>
<p>And the following will keep only the RNA-seq dataset, and only patients for which this assay is available:</p>
<div class="sourceCode" id="cb29"><html><body><pre class="r"><span class="no">miniACC</span>[, , <span class="st">"RNASeq2GeneNorm"</span>]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 306 sampleMap rows not in names(experiments)
##   removing 13 colData rownames not in sampleMap 'primary'</code></pre>
</div>
<div id="subsetting-by-genomic-ranges" class="section level3">
<h3 class="hasAnchor">
<a href="#subsetting-by-genomic-ranges" class="anchor"></a>Subsetting by genomic ranges</h3>
<p>If any ExperimentList objects have features represented by genomic ranges (e.g. <code>RangedSummarizedExperiment</code>, <code>RaggedExperiment</code>), then a <code>GRanges</code> object in the first subsetting position will subset these objects as in <code><a href="https://rdrr.io/pkg/GenomicRanges/man/findOverlaps-methods.html">GenomicRanges::findOverlaps()</a></code>. Any non-ranged <code>ExperimentList</code> element will be subset to zero rows.</p>
</div>
<div id="double-bracket" class="section level3">
<h3 class="hasAnchor">
<a href="#double-bracket" class="anchor"></a>Double bracket <code>[[</code>
</h3>
<p>The “double bracket” method (<code><a href="https://rdrr.io/r/base/Extract.html">[[</a></code>) is a convenience function for extracting a single element of the <code>MultiAssayExperiment</code> <code>ExperimentList</code>. It avoids the use of <code>experiments(mae)[[1L]]</code>. For example, both of the following extract the <code>ExpressionSet</code> object containing RNA-seq data:</p>
<div class="sourceCode" id="cb31"><html><body><pre class="r"><span class="no">miniACC</span><span class="kw">[[</span><span class="fl">1L</span>]]</pre></body></html></div>
<pre><code>## class: SummarizedExperiment 
## dim: 198 79 
## metadata(3): experimentData annotation protocolData
## assays(1): exprs
## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13
## rowData names(0):
## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07
##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07
## colData names(0):</code></pre>
<div class="sourceCode" id="cb33"><html><body><pre class="r"><span class="co">## equivalently</span>
<span class="no">miniACC</span><span class="kw">[[</span><span class="st">"RNASeq2GeneNorm"</span>]]</pre></body></html></div>
<pre><code>## class: SummarizedExperiment 
## dim: 198 79 
## metadata(3): experimentData annotation protocolData
## assays(1): exprs
## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13
## rowData names(0):
## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07
##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07
## colData names(0):</code></pre>
</div>
<div id="complete-cases" class="section level3">
<h3 class="hasAnchor">
<a href="#complete-cases" class="anchor"></a>Complete cases</h3>
<p><code><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases()</a></code> shows which patients have complete data for all assays:</p>
<div class="sourceCode" id="cb35"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span>(<span class="fu"><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases</a></span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>##    Mode   FALSE    TRUE 
## logical      49      43</code></pre>
<p>The above logical vector could be used for patient subsetting. More simply, <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns()</a></code> will select complete cases and rearrange each <code>ExperimentList</code> element so its columns correspond exactly to rows of <code>colData</code> in the same order:</p>
<div class="sourceCode" id="cb37"><html><body><pre class="r"><span class="no">accmatched</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<p>Note, the column names of the assays in <code>accmatched</code> are not the same because of assay-specific identifiers, but they have been automatically re-arranged to correspond to the same patients. In these TCGA assays, the first three <code><a href="https://rdrr.io/r/base/Arithmetic.html">-</a></code> delimited positions correspond to patient, ie the first patient is <em>TCGA-OR-A5J2</em>:</p>
<div class="sourceCode" id="cb38"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="no">accmatched</span>)</pre></body></html></div>
<pre><code>## CharacterList of length 5
## [["RNASeq2GeneNorm"]] TCGA-OR-A5J2-01A-11R-A29S-07 ...
## [["gistict"]] TCGA-OR-A5J2-01A-11D-A29H-01 ... TCGA-PK-A5HA-01A-11D-A29H-01
## [["RPPAArray"]] TCGA-OR-A5J2-01A-21-A39K-20 ... TCGA-PK-A5HA-01A-21-A39K-20
## [["Mutations"]] TCGA-OR-A5J2-01A-11D-A29I-10 ... TCGA-PK-A5HA-01A-11D-A29I-10
## [["miRNASeqGene"]] TCGA-OR-A5J2-01A-11R-A29W-13 ...</code></pre>
</div>
<div id="row-names-that-are-common-across-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#row-names-that-are-common-across-assays" class="anchor"></a>Row names that are common across assays</h3>
<p><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows()</a></code> keeps only rows that are common to each assay, and aligns them in identical order. For example, to keep only genes where data are available for RNA-seq, GISTIC copy number, and somatic mutations:</p>
<div class="sourceCode" id="cb40"><html><body><pre class="r"><span class="no">accmatched2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows</a></span>(<span class="no">miniACC</span>[, ,
    <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"gistict"</span>, <span class="st">"Mutations"</span>)])</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb42"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(<span class="no">accmatched2</span>)</pre></body></html></div>
<pre><code>## CharacterList of length 3
## [["RNASeq2GeneNorm"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... RET CDKN2A MACC1 CHGA
## [["gistict"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA
## [["Mutations"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA</code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="extraction" class="section level3">
<h3 class="hasAnchor">
<a href="#extraction" class="anchor"></a>Extraction</h3>
</div>
<div id="assay-and-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#assay-and-assays" class="anchor"></a>assay and assays</h3>
<p>The <code>assay</code> and <code>assays</code> methods follow <code>SummarizedExperiment</code> convention. The <code>assay</code> (singular) method will extract the first element of the <code>ExperimentList</code> and will return a <code>matrix</code>.</p>
<div class="sourceCode" id="cb44"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/class.html">class</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assay</a></span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>## [1] "matrix" "array"</code></pre>
<p>The <code>assays</code> (plural) method will return a <code>SimpleList</code> of the data with each element being a <code>matrix</code>.</p>
<div class="sourceCode" id="cb46"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## List of length 5
## names(5): RNASeq2GeneNorm gistict RPPAArray Mutations miRNASeqGene</code></pre>
<p><em>Key point:</em></p>
<ul>
<li>Whereas the <code><a href="https://rdrr.io/r/base/Extract.html">[[</a></code> returned an assay as its original class, <code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assay()</a></code> and <code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays()</a></code> convert the assay data to matrix form.</li>
</ul>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="summary-of-slots-and-accessors" class="section level3">
<h3 class="hasAnchor">
<a href="#summary-of-slots-and-accessors" class="anchor"></a>Summary of slots and accessors</h3>
<p>Slot in the <code>MultiAssayExperiment</code> can be accessed or set using their accessor functions:</p>
<table class="table">
<thead><tr class="header">
<th>Slot</th>
<th>Accessor</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><code>ExperimentList</code></td>
<td><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments()</a></code></td>
</tr>
<tr class="even">
<td><code>colData</code></td>
<td>
<code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData()</a></code> and <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> *</td>
</tr>
<tr class="odd">
<td><code>sampleMap</code></td>
<td><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap()</a></code></td>
</tr>
<tr class="even">
<td><code>metadata</code></td>
<td><code>metadata()</code></td>
</tr>
</tbody>
</table>
<p>__*__ The <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> operator on a <code>MultiAssayExperiment</code> returns a single column of the <code>colData</code>.</p>
</div>
<div id="transformation-reshaping" class="section level3">
<h3 class="hasAnchor">
<a href="#transformation-reshaping" class="anchor"></a>Transformation / reshaping</h3>
<p>The <code>longFormat</code> or <code>wideFormat</code> functions will “reshape” and combine experiments with each other and with <code>colData</code> into one <code>DataFrame</code>. These functions provide compatibility with most of the common R/Bioconductor functions for regression, machine learning, and visualization.</p>
<div id="longformat" class="section level4">
<h4 class="hasAnchor">
<a href="#longformat" class="anchor"></a><code>longFormat</code>
</h4>
<p>In <em>long</em> format a single column provides all assay results, with additional optional <code>colData</code> columns whose values are repeated as necessary. Here <em>assay</em> is the name of the ExperimentList element, <em>primary</em> is the patient identifier (rowname of colData), <em>rowname</em> is the assay rowname (in this case genes), <em>colname</em> is the assay-specific identifier (column name), <em>value</em> is the numeric measurement (gene expression, copy number, presence of a non-silent mutation, etc), and following these are the <em>vital_status</em> and <em>days_to_death</em> colData columns that have been added:</p>
<div class="sourceCode" id="cb48"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">longFormat</a></span>(<span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span>), , ],
           <span class="kw">colDataCols</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## DataFrame with 518 rows and 7 columns
##               assay      primary     rowname                colname     value
##         &lt;character&gt;  &lt;character&gt; &lt;character&gt;            &lt;character&gt; &lt;numeric&gt;
## 1   RNASeq2GeneNorm TCGA-OR-A5J1        TP53 TCGA-OR-A5J1-01A-11R..   563.401
## 2   RNASeq2GeneNorm TCGA-OR-A5J1      CTNNB1 TCGA-OR-A5J1-01A-11R..  5634.467
## 3   RNASeq2GeneNorm TCGA-OR-A5J2        TP53 TCGA-OR-A5J2-01A-11R..   165.481
## 4   RNASeq2GeneNorm TCGA-OR-A5J2      CTNNB1 TCGA-OR-A5J2-01A-11R.. 62658.391
## 5   RNASeq2GeneNorm TCGA-OR-A5J3        TP53 TCGA-OR-A5J3-01A-11R..   956.303
## ...             ...          ...         ...                    ...       ...
## 514       Mutations TCGA-PK-A5HA      CTNNB1 TCGA-PK-A5HA-01A-11D..         0
## 515       Mutations TCGA-PK-A5HB        TP53 TCGA-PK-A5HB-01A-11D..         0
## 516       Mutations TCGA-PK-A5HB      CTNNB1 TCGA-PK-A5HB-01A-11D..         0
## 517       Mutations TCGA-PK-A5HC        TP53 TCGA-PK-A5HC-01A-11D..         0
## 518       Mutations TCGA-PK-A5HC      CTNNB1 TCGA-PK-A5HC-01A-11D..         0
##     vital_status days_to_death
##        &lt;integer&gt;     &lt;integer&gt;
## 1              1          1355
## 2              1          1355
## 3              1          1677
## 4              1          1677
## 5              0            NA
## ...          ...           ...
## 514            0            NA
## 515            0            NA
## 516            0            NA
## 517            0            NA
## 518            0            NA</code></pre>
</div>
<div id="wideformat" class="section level4">
<h4 class="hasAnchor">
<a href="#wideformat" class="anchor"></a><code>wideFormat</code>
</h4>
<p>In <em>wide</em> format, each feature from each assay goes in a separate column, with one row per primary identifier (patient). Here, each variable becomes a new column:</p>
<div class="sourceCode" id="cb51"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span>), , ],
           <span class="kw">colDataCols</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## DataFrame with 92 rows and 9 columns
##          primary vital_status days_to_death RNASeq2GeneNorm_CTNNB1
##      &lt;character&gt;    &lt;integer&gt;     &lt;integer&gt;              &lt;numeric&gt;
## 1   TCGA-OR-A5J1            1          1355                5634.47
## 2   TCGA-OR-A5J2            1          1677               62658.39
## 3   TCGA-OR-A5J3            0            NA                6337.43
## 4   TCGA-OR-A5J4            1           423                     NA
## 5   TCGA-OR-A5J5            1           365                5979.06
## ...          ...          ...           ...                    ...
## 88  TCGA-PK-A5H9            0            NA                5258.99
## 89  TCGA-PK-A5HA            0            NA                8120.17
## 90  TCGA-PK-A5HB            0            NA                5257.81
## 91  TCGA-PK-A5HC            0            NA                     NA
## 92  TCGA-P6-A5OG            1           383                6390.10
##     RNASeq2GeneNorm_TP53 gistict_CTNNB1 gistict_TP53 Mutations_CTNNB1
##                &lt;numeric&gt;      &lt;numeric&gt;    &lt;numeric&gt;        &lt;numeric&gt;
## 1                563.401              0            0                0
## 2                165.481              1            0                1
## 3                956.303              0            0                0
## 4                     NA              0            1                0
## 5               1169.636              0            0                0
## ...                  ...            ...          ...              ...
## 88               890.866              0            0                0
## 89               683.572              0           -1                0
## 90               237.370             -1           -1                0
## 91                    NA              1            1                0
## 92               815.345              1           -1               NA
##     Mutations_TP53
##          &lt;numeric&gt;
## 1                0
## 2                1
## 3                0
## 4                0
## 5                0
## ...            ...
## 88               0
## 89               0
## 90               0
## 91               0
## 92              NA</code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
</div>
</div>
<div id="the-cancer-genome-atlas-tcga-data-from-curatedtcgadata" class="section level2">
<h2 class="hasAnchor">
<a href="#the-cancer-genome-atlas-tcga-data-from-curatedtcgadata" class="anchor"></a>The Cancer Genome Atlas (TCGA) Data from <code>curatedTCGAData</code>
</h2>
<p>Most unrestricted TCGA data are available as MultiAssayExperiment objects from the <code>curatedTCGAData</code> package. This represents a lot of harmonization!</p>
<div class="sourceCode" id="cb54"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">curatedTCGAData</span>)
<span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="st">"ACC"</span>)</pre></body></html></div>
<pre><code>##                                    Title DispatchClass
## 1                    ACC_CNASNP-20160128           Rda
## 2                    ACC_CNVSNP-20160128           Rda
## 4          ACC_GISTIC_AllByGene-20160128           Rda
## 5              ACC_GISTIC_Peaks-20160128           Rda
## 6  ACC_GISTIC_ThresholdedByGene-20160128           Rda
## 8        ACC_Methylation-20160128_assays        H5File
## 9            ACC_Methylation-20160128_se           Rds
## 10             ACC_miRNASeqGene-20160128           Rda
## 11                 ACC_Mutation-20160128           Rda
## 12          ACC_RNASeq2GeneNorm-20160128           Rda
## 13                ACC_RPPAArray-20160128           Rda</code></pre>
<div class="sourceCode" id="cb56"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/message.html">suppressMessages</a></span>({
    <span class="no">acc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="st">"ACC"</span>,
        <span class="kw">assays</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"miRNASeqGene"</span>, <span class="st">"RPPAArray"</span>, <span class="st">"Mutation"</span>,
            <span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"CNVSNP"</span>),
        <span class="kw">dry.run</span> <span class="kw">=</span> <span class="fl">FALSE</span>)
})
<span class="no">acc</span></pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [4] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [5] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<p>These objects contain most unrestricted TCGA assay and clinical / pathological data, as well as material curated from the supplements of published TCGA primary papers at the end of the colData columns:</p>
<div class="sourceCode" id="cb58"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/dim.html">dim</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">acc</span>))</pre></body></html></div>
<pre><code>## [1]  92 822</code></pre>
<div class="sourceCode" id="cb60"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">tail</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">acc</span>)), <span class="fl">10</span>)</pre></body></html></div>
<pre><code>##  [1] "MethyLevel"       "miRNA.cluster"    "SCNA.cluster"     "protein.cluster" 
##  [5] "COC"              "OncoSign"         "purity"           "ploidy"          
##  [9] "genome_doublings" "ADS"</code></pre>
<p>The <code>TCGAutils</code> package provides additional helper functions.</p>
</div>
<div id="utilities-for-tcga" class="section level2">
<h2 class="hasAnchor">
<a href="#utilities-for-tcga" class="anchor"></a>Utilities for TCGA</h2>
<p>Aside from the available reshaping functions already included in the <code>MultiAssayExperiment</code> package, the <code>TCGAutils</code> package provides additional helper functions for working with TCGA data.</p>
<p>A number of helper functions are available for managing datasets from <code>curatedTCGAData</code>. These include:</p>
<ul>
<li>Conversions of <code>SummarizedExperiment</code> to <code>RangedSummarizedExperiment</code> based on <code>TxDb.Hsapiens.UCSC.hg19.knownGene</code> for:
<ul>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">mirToRanges()</a></code>: microRNA</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">symbolsToRanges()</a></code>: gene symbols</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">qreduceTCGA()</a></code>: convert <code>RaggedExperiment</code> objects to <code>RangedSummarizedExperiment</code> with one row per gene symbol, for:
<ul>
<li>segmented copy number datasets (“CNVSNP” and “CNASNP”)</li>
<li>somatic mutation datasets (“Mutation”), with a value of 1 for any non-silent mutation and a value of 0 for no mutation or silent mutation</li>
</ul>
</li>
</ul>
</li>
</ul>
<div id="conversion-of-row-metadata-for-curatedtcgadata-objects" class="section level3">
<h3 class="hasAnchor">
<a href="#conversion-of-row-metadata-for-curatedtcgadata-objects" class="anchor"></a>(1) Conversion of row metadata for <code>curatedTCGAData</code> objects</h3>
<div id="mirtoranges" class="section level4">
<h4 class="hasAnchor">
<a href="#mirtoranges" class="anchor"></a><code>mirToRanges</code>
</h4>
<p>microRNA assays obtained from <code>curatedTCGAData</code> have annotated sequences that can be converted to genomic ranges using the <code>mirbase.db</code> package. The function looks up all sequences and converts them to (‘hg19’) ranges. For those rows that cannot be found, an ‘unranged’ assay is introduced in the resulting MultiAssayExperiment object.</p>
<div class="sourceCode" id="cb62"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">mirToRanges</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 6 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 6:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [3] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [5] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns
##  [6] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="qreducetcga" class="section level4">
<h4 class="hasAnchor">
<a href="#qreducetcga" class="anchor"></a><code>qreduceTCGA</code>
</h4>
<p>The <code>qreduceTCGA</code> function converts <code>RaggedExperiment</code> mutation data objects to <code>RangedSummarizedExperiment</code> using <code>org.Hs.eg.db</code> and the <code>qreduceTCGA</code> utility function from <code>RaggedExperiment</code> to summarize ‘silent’ and ‘non-silent’ mutations based on a ‘Variant_Classification’ metadata column in the original object.</p>
<div class="sourceCode" id="cb65"><html><body><pre class="r"><span class="co">## Update build metadata to "hg19"</span>
<span class="fu">genome</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]]) <span class="kw">&lt;-</span>
    <span class="fu"><a href="https://rdrr.io/r/base/lapply.html">vapply</a></span>(<span class="fu">genome</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]]),
    <span class="no">translateBuild</span>, <span class="fu"><a href="https://rdrr.io/r/base/character.html">character</a></span>(<span class="fl">1L</span>))

<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">qreduceTCGA</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## </code></pre>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## Warning in .Seqinfo.mergexy(x, y): The 2 combined objects have no sequence levels in common. (Use
##   suppressWarnings() to suppress this warning.)</code></pre>
<pre><code>## Warning in .normarg_seqlevelsStyle(value): more than one seqlevels style
## supplied, using the 1st one only</code></pre>
<pre><code>## harmonizing input:
##   removing 270 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [2] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [3] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [4] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 90 columns
##  [5] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 180 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="symbolstoranges" class="section level4">
<h4 class="hasAnchor">
<a href="#symbolstoranges" class="anchor"></a><code>symbolsToRanges</code>
</h4>
<p>In the cases where row annotations indicate gene symbols, the <code>symbolsToRanges</code> utility function converts genes to genomic ranges and replaces existing assays with <code>RangedSummarizedExperiment</code> objects. Gene annotations are given as ‘hg19’ genomic regions.</p>
<div class="sourceCode" id="cb73"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">symbolsToRanges</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## harmonizing input:
##   removing 79 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 6 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 6:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [5] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 17296 rows and 79 columns
##  [6] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3205 rows and 79 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="simplifytcga" class="section level4">
<h4 class="hasAnchor">
<a href="#simplifytcga" class="anchor"></a><code>simplifyTCGA</code>
</h4>
<p>The <code>simplifyTCGA</code> function combines all of the above operations to create a more managable <code>MultiAssayExperiment</code> object and using <code>RangedSummarizedExperiment</code> assays where possible.</p>
<div class="sourceCode" id="cb78"><html><body><pre class="r"><span class="kw pkg">TCGAutils</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">simplifyTCGA</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## Warning in .Seqinfo.mergexy(x, y): The 2 combined objects have no sequence levels in common. (Use
##   suppressWarnings() to suppress this warning.)</code></pre>
<pre><code>## Warning in .normarg_seqlevelsStyle(value): more than one seqlevels style
## supplied, using the 1st one only</code></pre>
<pre><code>## harmonizing input:
##   removing 270 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## harmonizing input:
##   removing 79 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 7 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 7:
##  [1] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [2] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 90 columns
##  [3] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 180 columns
##  [4] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns
##  [5] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns
##  [6] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 17296 rows and 79 columns
##  [7] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3205 rows and 79 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
</div>
<div id="identification-and-separation-of-samples" class="section level3">
<h3 class="hasAnchor">
<a href="#identification-and-separation-of-samples" class="anchor"></a>(2) Identification and Separation of Samples</h3>
</div>
<div id="what-types-of-samples-are-in-the-data" class="section level3">
<h3 class="hasAnchor">
<a href="#what-types-of-samples-are-in-the-data" class="anchor"></a>What types of samples are in the data?</h3>
<p><strong>Solution</strong></p>
<p>The <code>sampleTables</code> function gives you an overview of samples in each assay:</p>
<div class="sourceCode" id="cb89"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">sampleTables</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## $`ACC_CNVSNP-20160128`
## 
## 01 10 11 
## 90 85  5 
## 
## $`ACC_miRNASeqGene-20160128`
## 
## 01 
## 80 
## 
## $`ACC_Mutation-20160128`
## 
## 01 
## 90 
## 
## $`ACC_RNASeq2GeneNorm-20160128`
## 
## 01 
## 79 
## 
## $`ACC_RPPAArray-20160128`
## 
## 01 
## 46</code></pre>
<p>Interpretation of sample codes:</p>
<div class="sourceCode" id="cb91"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"sampleTypes"</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">sampleTypes</span>)</pre></body></html></div>
<pre><code>##   Code                                      Definition Short.Letter.Code
## 1   01                             Primary Solid Tumor                TP
## 2   02                           Recurrent Solid Tumor                TR
## 3   03 Primary Blood Derived Cancer - Peripheral Blood                TB
## 4   04    Recurrent Blood Derived Cancer - Bone Marrow              TRBM
## 5   05                        Additional - New Primary               TAP
## 6   06                                      Metastatic                TM</code></pre>
</div>
<div id="splitassays-separate-the-data-from-different-tissue-types" class="section level3">
<h3 class="hasAnchor">
<a href="#splitassays-separate-the-data-from-different-tissue-types" class="anchor"></a><code>splitAssays</code>: separate the data from different tissue types</h3>
<p>TCGA datasets include multiple -omics for solid tumors, adjacent normal tissues, blood-derived cancers and normals, and other tissue types, which may be mixed together in a single dataset. The <code>MultiAssayExperiment</code> object generated here has one patient per row of its <code>colData</code>, but each patient may have two or more -omics profiles by any assay, whether due to assaying of different types of tissues or to technical replication. <code>splitAssays</code> separates profiles from different tissue types (such as tumor and adjacent normal) into different assays of the <code>MultiAssayExperiment</code> by taking a vector of sample codes, and partitioning the current assays into assays with an appended sample code:</p>
<div class="sourceCode" id="cb93"><html><body><pre class="r"><span class="no">split_acc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">splitAssays</a></span>(<span class="no">acc</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"01"</span>, <span class="st">"11"</span>))</pre></body></html></div>
<pre><code>## Warning: Some 'sampleCodes' not found in assays</code></pre>
<pre><code>## Warning in .checkBarcodes(barcodes): Inconsistent barcode lengths: 28, 27</code></pre>
<p>The <code>MultiAssayExperiment</code> package then provides functionality to merge replicate profiles for a single patient (<code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">mergeReplicates()</a></code>), which would now be appropriate but would <strong>not</strong> have been appropriate before splitting different tissue types into different assays, because that would average measurements from tumors and normal tissues.</p>
<p><code>MultiAssayExperiment</code> also defines the <code>MatchedAssayExperiment</code> class, which eliminates any profiles not present across all assays and ensures identical ordering of profiles (columns) in each assay. In this example, it will match tumors to adjacent normals in subsequent assays:</p>
<div class="sourceCode" id="cb96"><html><body><pre class="r">(<span class="no">matchmae</span> <span class="kw">&lt;-</span> <span class="fu">as</span>(<span class="no">split_acc</span>[, , <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="fl">1</span>, <span class="fl">3</span>, <span class="fl">4</span>, <span class="fl">5</span>, <span class="fl">6</span>)], <span class="st">"MatchedAssayExperiment"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 5 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MatchedAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] 01_ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 43 columns
##  [2] 01_ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 43 columns
##  [3] 01_ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 43 columns
##  [4] 01_ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 43 columns
##  [5] 01_ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 43 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<p>Only about 43 participants have data across all experiments.</p>
</div>
<div id="curated-molecular-subtypes" class="section level3">
<h3 class="hasAnchor">
<a href="#curated-molecular-subtypes" class="anchor"></a>Curated molecular subtypes</h3>
<p>Is there subtype data available in the <code>MultiAssayExperiment</code> obtained from <code>curatedTCGAData</code>?</p>
<p><strong>Solution</strong></p>
<p>The <code>getSubtypeMap</code> function will show actual variable names found in <code>colData</code> that contain subtype information. This can only be obtained from <code>MultiAssayExperiment</code> objects provided by <code>curatedTCGAData</code>.</p>
<div class="sourceCode" id="cb99"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">getSubtypeMap</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##          ACC_annotations     ACC_subtype
## 1             Patient_ID       patientID
## 2  histological_subtypes       Histology
## 3          mrna_subtypes         C1A/C1B
## 4          mrna_subtypes         mRNA_K4
## 5                   cimp      MethyLevel
## 6      microrna_subtypes   miRNA cluster
## 7          scna_subtypes    SCNA cluster
## 8       protein_subtypes protein cluster
## 9   integrative_subtypes             COC
## 10     mutation_subtypes        OncoSign</code></pre>
<div class="sourceCode" id="cb101"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">acc</span>)$<span class="no">Histology</span>)</pre></body></html></div>
<pre><code>## [1] "Usual Type" "Usual Type" "Usual Type" "Usual Type" "Usual Type"
## [6] "Usual Type"</code></pre>
</div>
<div id="translation-and-interpretation-of-tcga-identifiers" class="section level3">
<h3 class="hasAnchor">
<a href="#translation-and-interpretation-of-tcga-identifiers" class="anchor"></a>(3) Translation and Interpretation of TCGA identifiers</h3>
<p><code>TCGAutils</code> provides a number of ID translation functions. These allow the user to translate from either file or case UUIDs to TCGA barcodes and back. These functions work by querying the Genomic Data Commons API via the <code>GenomicDataCommons</code> package (thanks to Sean Davis). These include:</p>
<div id="uuidtobarcode---uuid-to-tcga-barcode" class="section level4">
<h4 class="hasAnchor">
<a href="#uuidtobarcode---uuid-to-tcga-barcode" class="anchor"></a><code>UUIDtoBarcode()</code> - UUID to TCGA barcode</h4>
<p>Here we have a known case UUID that we want to translate into a TCGA barcode.</p>
<div class="sourceCode" id="cb103"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoBarcode</a></span>(<span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, <span class="kw">from_type</span> <span class="kw">=</span> <span class="st">"case_id"</span>)</pre></body></html></div>
<pre><code>##                                case_id submitter_id
## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 TCGA-B0-5117</code></pre>
<p>In cases where we want to translate a known file UUID to the associated TCGA patient barcode, we can use <code>UUIDtoBarcode</code>.</p>
<div class="sourceCode" id="cb105"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoBarcode</a></span>(<span class="st">"0001801b-54b0-4551-8d7a-d66fb59429bf"</span>, <span class="kw">from_type</span> <span class="kw">=</span> <span class="st">"file_id"</span>)</pre></body></html></div>
<pre><code>##                                file_id associated_entities.entity_submitter_id
## 1 0001801b-54b0-4551-8d7a-d66fb59429bf            TCGA-B0-5094-11A-01D-1421-08</code></pre>
</div>
<div id="barcodetouuid---tcga-barcode-to-uuid" class="section level4">
<h4 class="hasAnchor">
<a href="#barcodetouuid---tcga-barcode-to-uuid" class="anchor"></a><code>barcodeToUUID()</code> - TCGA barcode to UUID</h4>
<p>Here we translate the first two TCGA barcodes of the previous copy-number alterations dataset to UUID:</p>
<div class="sourceCode" id="cb107"><html><body><pre class="r">(<span class="no">xbarcode</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="no">acc</span>)<span class="kw">[[</span><span class="st">"ACC_CNVSNP-20160128"</span>]], <span class="fl">4L</span>))</pre></body></html></div>
<pre><code>## [1] "TCGA-OR-A5J1-01A-11D-A29H-01" "TCGA-OR-A5J1-10A-01D-A29K-01"
## [3] "TCGA-OR-A5J2-01A-11D-A29H-01" "TCGA-OR-A5J2-10A-01D-A29K-01"</code></pre>
<div class="sourceCode" id="cb109"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">barcodeToUUID</a></span>(<span class="no">xbarcode</span>)</pre></body></html></div>
<pre><code>##           submitter_aliquot_ids                          aliquot_ids
## 6  TCGA-OR-A5J1-01A-11D-A29H-01 1387b6c7-48fe-4961-86a7-0bdcbd3fef92
## 9  TCGA-OR-A5J1-10A-01D-A29K-01 cb537629-6a01-4d67-84ea-dbf130bd59c7
## 21 TCGA-OR-A5J2-01A-11D-A29H-01 6f0290b0-4cb4-4f72-853e-9ac363bd2c3b
## 13 TCGA-OR-A5J2-10A-01D-A29K-01 4bf2e4ac-399f-4a00-854b-8e23b561bb4d</code></pre>
</div>
<div id="uuidtouuid---file-and-case-ids" class="section level4">
<h4 class="hasAnchor">
<a href="#uuidtouuid---file-and-case-ids" class="anchor"></a><code>UUIDtoUUID()</code> - file and case IDs</h4>
<p>We can also translate from file UUIDs to case UUIDs and vice versa as long as we know the input type. We can use the case UUID from the previous example to get the associated file UUIDs using <code>UUIDtoUUID</code>. Note that this translation is a one to many relationship, thus yielding a <code>data.frame</code> of file UUIDs for a single case UUID.</p>
<div class="sourceCode" id="cb111"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoUUID</a></span>(<span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, <span class="kw">to_type</span> <span class="kw">=</span> <span class="st">"file_id"</span>))</pre></body></html></div>
<pre><code>##                                case_id                        files.file_id
## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 48c342b0-e7a2-4a7b-8556-55bcd8ad9ea0
## 2 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 db8ba5d3-76be-4a67-a575-803ba483b6f9
## 3 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 f580489b-55ea-43c5-9489-b54c13146992
## 4 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 bf72ffef-d8c4-423d-9c5a-7bb5c23b2f31
## 5 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 b36f4e88-89ca-40bf-b543-d0e3c08ad342
## 6 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 4c3a899f-be0f-454f-b5dc-e30e29314c49</code></pre>
<p>One possible way to verify that file IDs are matching case UUIDS is to browse to the Genomic Data Commons webpage with the specific file UUID. Here we look at the first file UUID entry in the output <code>data.frame</code>:</p>
<p><a href="https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6" class="uri">https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6</a></p>
<p>In the page we check that the case UUID matches the input.</p>
</div>
<div id="filenametobarcode---using-file-names-as-input" class="section level4">
<h4 class="hasAnchor">
<a href="#filenametobarcode---using-file-names-as-input" class="anchor"></a><code>filenameToBarcode()</code> - Using file names as input</h4>
<div class="sourceCode" id="cb113"><html><body><pre class="r"><span class="no">fquery</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/query.html">files</a></span>() <span class="kw">%&gt;%</span>
    <span class="kw pkg">GenomicDataCommons</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/filtering.html">filter</a></span>(~ <span class="no">cases.project.project_id</span> <span class="kw">==</span> <span class="st">"TCGA-ACC"</span> <span class="kw">&amp;</span>
        <span class="no">data_category</span> <span class="kw">==</span> <span class="st">"Copy Number Variation"</span> <span class="kw">&amp;</span>
        <span class="no">data_type</span> <span class="kw">==</span> <span class="st">"Copy Number Segment"</span>)

<span class="no">fnames</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/results.html">results</a></span>(<span class="no">fquery</span>)$<span class="no">file_name</span>)

<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">filenameToBarcode</a></span>(<span class="no">fnames</span>)</pre></body></html></div>
<pre><code>##                                                                 file_name
## 1 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_D03_1348390.grch38.seg.v2.txt
## 2 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_B08_1348416.grch38.seg.v2.txt
## 3 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_D05_1348384.grch38.seg.v2.txt
## 4 BLAIN_p_TCGA_282_304_b2_N_GenomeWideSNP_6_E07_1348450.grch38.seg.v2.txt
## 5 BLAIN_p_TCGA_282_304_b2_N_GenomeWideSNP_6_F10_1348518.grch38.seg.v2.txt
## 6 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_F09_1348288.grch38.seg.v2.txt
##                                file_id        aliquots.submitter_id
## 1 c7c23df4-d1a8-47c1-9394-1948114dc756 TCGA-OR-A5KO-10A-01D-A29K-01
## 2 81754b77-e4b0-4123-baba-c2a8fbfaa487 TCGA-OR-A5KW-10A-01D-A29K-01
## 3 81380fa8-f63d-4b5d-9bb9-abab48ba3f1b TCGA-OR-A5LA-10A-01D-A29K-01
## 4 4b09eac7-8b71-4ff5-93d1-97dfcc342132 TCGA-OR-A5J9-01A-11D-A29H-01
## 5 e46a1773-92f0-4ed1-916d-d3493ba6bbe4 TCGA-OR-A5JZ-01A-11D-A29H-01
## 6 fb4acb71-b9c6-47ff-a585-5ddeff3d6039 TCGA-PK-A5HB-01A-11D-A29H-01</code></pre>
<p>See the <code>TCGAutils</code> vignette page for more details.</p>
</div>
</div>
</div>
<div id="bonus-multiassayexperiment---pan-cancer-example" class="section level2">
<h2 class="hasAnchor">
<a href="#bonus-multiassayexperiment---pan-cancer-example" class="anchor"></a>Bonus: <code>MultiAssayExperiment</code> - Pan-Cancer Example</h2>
<p>The <code>MultiAssayExperiment</code> class is most efficient with samples from a set of patients over multiple experiments but it also supports samples from disparate datasets where the patients are different in each dataset. The data container allows multiple sets of patients within the same object. This is nicely demonstrated using a PanCancer TCGA dataset where measurements are made on multiple groups of patients. In order to achieve this, <code>curatedTCGAData</code> allows the user to query for multiple cancer types.</p>
<p>In the example, we can focus on one type of data:</p>
<div class="sourceCode" id="cb115"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="st">"*"</span>, <span class="st">"miRNA*"</span>))</pre></body></html></div>
<pre><code>##                         Title DispatchClass
## 10  ACC_miRNASeqGene-20160128           Rda
## 25 BLCA_miRNASeqGene-20160128           Rda
## 43 BRCA_miRNASeqGene-20160128           Rda
## 60 CESC_miRNASeqGene-20160128           Rda
## 74 CHOL_miRNASeqGene-20160128           Rda
## 91 COAD_miRNASeqGene-20160128           Rda</code></pre>
<p>To obtain data for multiple cancers, we modify the first argument in <code>curatedTCGAData</code>:</p>
<div class="sourceCode" id="cb117"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"OV"</span>, <span class="st">"CESC"</span>, <span class="st">"BRCA"</span>, <span class="st">"UCEC"</span>, <span class="st">"UCS"</span>), <span class="st">"miRNA*"</span>)</pre></body></html></div>
<pre><code>##                          Title DispatchClass
## 43  BRCA_miRNASeqGene-20160128           Rda
## 60  CESC_miRNASeqGene-20160128           Rda
## 329     OV_miRNAArray-20160128           Rda
## 330   OV_miRNASeqGene-20160128           Rda
## 505 UCEC_miRNASeqGene-20160128           Rda
## 521  UCS_miRNASeqGene-20160128           Rda</code></pre>
<p>We then request the actual data by setting the <code>dry.run</code> argument to <code>FALSE</code>:</p>
<div class="sourceCode" id="cb119"><html><body><pre class="r"><span class="no">mirnapan</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(
    <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"OV"</span>, <span class="st">"CESC"</span>, <span class="st">"BRCA"</span>, <span class="st">"UCEC"</span>, <span class="st">"UCS"</span>), <span class="st">"miRNA*"</span>, <span class="fl">FALSE</span>)</pre></body></html></div>
<pre><code>## snapshotDate(): 2020-07-10</code></pre>
<pre><code>## Working on: BRCA_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_miRNAArray-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: BRCA_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: BRCA_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: BRCA_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## harmonizing input:
##   removing 32021 sampleMap rows not in names(experiments)
##   removing 503 colData rownames not in sampleMap 'primary'</code></pre>
<ul>
<li>We are currently working on helper functions to make Pan-Cancer analyses easier to achieve.</li>
</ul>
</div>
<div id="plotting-correlation-and-other-analyses" class="section level2">
<h2 class="hasAnchor">
<a href="#plotting-correlation-and-other-analyses" class="anchor"></a>Plotting, correlation, and other analyses</h2>
<p>These provide exercises and solutions using the <code>miniACC</code> dataset.</p>
<div id="how-many-miniacc-samples-have-data-for-each-combination-of-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#how-many-miniacc-samples-have-data-for-each-combination-of-assays" class="anchor"></a>How many <code>miniACC</code> samples have data for each combination of assays?</h3>
<p><strong>Solution</strong></p>
<p>The built-in <code>upsetSamples</code> creates an “upset” Venn diagram to answer this question:</p>
<div class="sourceCode" id="cb227"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/upsetSamples.html">upsetSamples</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-49-1.png" width="100%"></p>
<p>In this dataset only 43 samples have all 5 assays, 32 are missing reverse-phase protein (RPPAArray), 2 are missing Mutations, 1 is missing gistict, 12 have only mutations and gistict, etc.</p>
</div>
<div id="kaplan-meier-plot-stratified-by-pathology_n_stage" class="section level3">
<h3 class="hasAnchor">
<a href="#kaplan-meier-plot-stratified-by-pathology_n_stage" class="anchor"></a>Kaplan-meier plot stratified by pathology_N_stage</h3>
<p>Create a Kaplan-meier plot, using pathology_N_stage as a stratifying variable.</p>
<p><strong>Solution</strong></p>
<p>The colData provides clinical data for things like a Kaplan-Meier plot for overall survival stratified by nodal stage.</p>
<div class="sourceCode" id="cb228"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">miniACC</span>$<span class="no">days_to_death</span>, <span class="no">miniACC</span>$<span class="no">vital_status</span>)</pre></body></html></div>
<pre><code>##  [1] 1355  1677    NA+  423   365    NA+  490   579    NA+  922   551  1750 
## [13]   NA+ 2105    NA+  541    NA+   NA+  490    NA+   NA+  562    NA+   NA+
## [25]   NA+   NA+   NA+   NA+  289    NA+   NA+   NA+  552    NA+   NA+   NA+
## [37]  994    NA+   NA+  498    NA+   NA+  344    NA+   NA+   NA+   NA+   NA+
## [49]   NA+   NA+   NA+   NA+   NA+  391   125    NA+ 1852    NA+   NA+   NA+
## [61]   NA+   NA+   NA+   NA+ 1204   159  1197   662   445    NA+ 2385   436 
## [73] 1105    NA+ 1613    NA+   NA+ 2405    NA+   NA+   NA+   NA+   NA+  207 
## [85]    0    NA+   NA+   NA+   NA+   NA+   NA+  383</code></pre>
<p>And remove any patients missing overall survival information:</p>
<div class="sourceCode" id="cb230"><html><body><pre class="r"><span class="no">miniACCsurv</span> <span class="kw">&lt;-</span> <span class="no">miniACC</span>[,
    <span class="fu"><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases</a></span>(<span class="no">miniACC</span>$<span class="no">days_to_death</span>, <span class="no">miniACC</span>$<span class="no">vital_status</span>), ]</pre></body></html></div>
<div class="sourceCode" id="cb231"><html><body><pre class="r"><span class="no">fit</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/survfit.html">survfit</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">days_to_death</span>, <span class="no">vital_status</span>) ~ <span class="no">pathology_N_stage</span>,
    <span class="kw">data</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACCsurv</span>))
<span class="fu"><a href="https://rdrr.io/pkg/survminer/man/ggsurvplot.html">ggsurvplot</a></span>(<span class="no">fit</span>, <span class="kw">data</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACCsurv</span>), <span class="kw">risk.table</span> <span class="kw">=</span> <span class="fl">TRUE</span>)</pre></body></html></div>
<pre><code>## Warning: Vectorized input to `element_text()` is not officially supported.
## Results may be unexpected or may change in future versions of ggplot2.</code></pre>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-52-1.png" width="100%"></p>
</div>
<div id="multivariate-cox-regression-including-rna-seq-copy-number-and-pathology" class="section level3">
<h3 class="hasAnchor">
<a href="#multivariate-cox-regression-including-rna-seq-copy-number-and-pathology" class="anchor"></a>Multivariate Cox regression including RNA-seq, copy number, and pathology</h3>
<p>Choose the <em>EZH2</em> gene for demonstration. This subsetting will drop assays with no row named EZH2:</p>
<div class="sourceCode" id="cb233"><html><body><pre class="r"><span class="no">wideacc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">miniACC</span>[<span class="st">"EZH2"</span>, , ],
    <span class="kw">colDataCols</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>, <span class="st">"pathology_N_stage"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 216 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb235"><html><body><pre class="r"><span class="no">wideacc</span>$<span class="no">y</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">wideacc</span>$<span class="no">days_to_death</span>, <span class="no">wideacc</span>$<span class="no">vital_status</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">wideacc</span>)</pre></body></html></div>
<pre><code>## DataFrame with 6 rows and 7 columns
##        primary vital_status days_to_death pathology_N_stage
##    &lt;character&gt;    &lt;integer&gt;     &lt;integer&gt;       &lt;character&gt;
## 1 TCGA-OR-A5J1            1          1355                n0
## 2 TCGA-OR-A5J2            1          1677                n0
## 3 TCGA-OR-A5J3            0            NA                n0
## 4 TCGA-OR-A5J4            1           423                n1
## 5 TCGA-OR-A5J5            1           365                n0
## 6 TCGA-OR-A5J6            0            NA                n0
##   RNASeq2GeneNorm_EZH2 gistict_EZH2      y
##              &lt;numeric&gt;    &lt;numeric&gt; &lt;Surv&gt;
## 1              75.8886            0 1355:1
## 2             326.5332            1 1677:1
## 3             190.1940            1   NA:0
## 4                   NA           -2  423:1
## 5             366.3826            1  365:1
## 6              30.7371            1   NA:0</code></pre>
<p>Perform a multivariate Cox regression with <em>EZH2</em> copy number (gistict), log2-transformed <em>EZH2</em> expression (RNASeq2GeneNorm), and nodal status (pathology_N_stage) as predictors:</p>
<div class="sourceCode" id="cb237"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">days_to_death</span>, <span class="no">vital_status</span>) ~ <span class="no">gistict_EZH2</span> +
    <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="no">RNASeq2GeneNorm_EZH2</span>) + <span class="no">pathology_N_stage</span>, <span class="kw">data</span><span class="kw">=</span><span class="no">wideacc</span>)</pre></body></html></div>
<pre><code>## Call:
## coxph(formula = Surv(days_to_death, vital_status) ~ gistict_EZH2 + 
##     log2(RNASeq2GeneNorm_EZH2) + pathology_N_stage, data = wideacc)
## 
##                                coef exp(coef) se(coef)      z        p
## gistict_EZH2               -0.03723   0.96345  0.28205 -0.132 0.894986
## log2(RNASeq2GeneNorm_EZH2)  0.97731   2.65729  0.28105  3.477 0.000506
## pathology_N_stagen1         0.37749   1.45862  0.56992  0.662 0.507743
## 
## Likelihood ratio test=16.28  on 3 df, p=0.0009942
## n= 26, number of events= 26 
##    (66 observations deleted due to missingness)</code></pre>
<p>We see that <em>EZH2</em> expression is significantly associated with overal survival (p &lt; 0.001), but <em>EZH2</em> copy number and nodal status are not. This analysis could easily be extended to the whole genome for discovery of prognostic features by repeated univariate regressions over columns, penalized multivariate regression, etc.</p>
<p>For further detail, see the main MultiAssayExperiment vignette.</p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="correlation-between-rna-seq-and-copy-number" class="section level3">
<h3 class="hasAnchor">
<a href="#correlation-between-rna-seq-and-copy-number" class="anchor"></a>Correlation between RNA-seq and copy number</h3>
<p><strong>Part 1</strong></p>
<p>For all genes where there is both recurrent copy number (gistict assay) and RNA-seq, calculate the correlation between log2(RNAseq + 1) and copy number. Create a histogram of these correlations. Compare this with the histogram of correlations between all <em>unmatched</em> gene - copy number pairs.</p>
<p><strong>Solution</strong></p>
<p>First, narrow down <code>miniACC</code> to only the assays needed:</p>
<div class="sourceCode" id="cb239"><html><body><pre class="r"><span class="no">subacc</span> <span class="kw">&lt;-</span> <span class="no">miniACC</span>[, , <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"gistict"</span>)]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 216 sampleMap rows not in names(experiments)</code></pre>
<p>Align the rows and columns, keeping only samples with both assays available:</p>
<div class="sourceCode" id="cb241"><html><body><pre class="r"><span class="no">subacc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns</a></span>(<span class="no">subacc</span>)
<span class="no">subacc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows</a></span>(<span class="no">subacc</span>)</pre></body></html></div>
<p>Create a list of numeric matrices:</p>
<div class="sourceCode" id="cb242"><html><body><pre class="r"><span class="no">subacc.list</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">subacc</span>)</pre></body></html></div>
<p>Log-transform the RNA-seq assay:</p>
<div class="sourceCode" id="cb243"><html><body><pre class="r"><span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">1</span>]] <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">1</span>]] + <span class="fl">1</span>)</pre></body></html></div>
<p>Transpose both, so genes are in columns:</p>
<div class="sourceCode" id="cb244"><html><body><pre class="r"><span class="no">subacc.list</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html">lapply</a></span>(<span class="no">subacc.list</span>, <span class="no">t</span>)</pre></body></html></div>
<p>Calculate the correlation between columns in the first matrix and columns in the second matrix:</p>
<div class="sourceCode" id="cb245"><html><body><pre class="r"><span class="no">corres</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html">cor</a></span>(<span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">1</span>]], <span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">2</span>]])</pre></body></html></div>
<p>And finally, create the histograms:</p>
<div class="sourceCode" id="cb246"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html">hist</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span>(<span class="no">corres</span>))</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-61-1.png" width="100%"></p>
<div class="sourceCode" id="cb247"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html">hist</a></span>(<span class="no">corres</span>[<span class="fu"><a href="https://rdrr.io/r/base/lower.tri.html">upper.tri</a></span>(<span class="no">corres</span>)])</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-61-2.png" width="100%"></p>
<p><strong>Part 2</strong></p>
<p>For the gene with highest correlation to copy number, make a box plot of log2 expression against copy number.</p>
<p><strong>Solution</strong></p>
<p>First, identify the gene with highest correlation between expression and copy number:</p>
<div class="sourceCode" id="cb248"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/which.min.html">which.max</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span>(<span class="no">corres</span>))</pre></body></html></div>
<pre><code>## EIF4E 
##    91</code></pre>
<p>You could now make the plot by taking the EIF4E columns from each element of the list subacc.list <em>list</em> that was extracted from the subacc <em>MultiAssayExperiment</em>, but let’s do it by subsetting and extracting from the <em>MultiAssayExperiment</em>:</p>
<div class="sourceCode" id="cb250"><html><body><pre class="r"><span class="no">df</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">subacc</span>[<span class="st">"EIF4E"</span>, , ])
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">df</span>)</pre></body></html></div>
<pre><code>## DataFrame with 6 rows and 3 columns
##        primary RNASeq2GeneNorm_EIF4E gistict_EIF4E
##    &lt;character&gt;             &lt;numeric&gt;     &lt;numeric&gt;
## 1 TCGA-OR-A5J1               460.615             0
## 2 TCGA-OR-A5J2               371.225             0
## 3 TCGA-OR-A5J3               516.072             0
## 4 TCGA-OR-A5J5              1129.357             1
## 5 TCGA-OR-A5J6               822.078             0
## 6 TCGA-OR-A5J7               344.565            -1</code></pre>
<div class="sourceCode" id="cb252"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/graphics/boxplot.html">boxplot</a></span>(<span class="no">RNASeq2GeneNorm_EIF4E</span> ~ <span class="no">gistict_EIF4E</span>,
        <span class="kw">data</span><span class="kw">=</span><span class="no">df</span>, <span class="kw">varwidth</span><span class="kw">=</span><span class="fl">TRUE</span>,
        <span class="kw">xlab</span><span class="kw">=</span><span class="st">"GISTIC Relative Copy Number Call"</span>,
        <span class="kw">ylab</span><span class="kw">=</span><span class="st">"RNA-seq counts"</span>)</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-64-1.png" width="100%"></p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="identifying-correlated-principal-components" class="section level3">
<h3 class="hasAnchor">
<a href="#identifying-correlated-principal-components" class="anchor"></a>Identifying correlated principal components</h3>
<p>Perform Principal Components Analysis of each of the five assays, using samples available on each assay, log-transforming RNA-seq data first. Using the first 10 components, calculate Pearson correlation between all scores and plot these correlations as a heatmap to identify correlated components across assays.</p>
<p><strong>Solution</strong></p>
<p>Here’s a function to simplify doing the PCAs:</p>
<div class="sourceCode" id="cb253"><html><body><pre class="r"><span class="no">getLoadings</span> <span class="kw">&lt;-</span> <span class="kw">function</span>(<span class="no">x</span>, <span class="no">ncomp</span><span class="kw">=</span><span class="fl">10</span>, <span class="no">dolog</span><span class="kw">=</span><span class="fl">FALSE</span>, <span class="no">center</span><span class="kw">=</span><span class="fl">TRUE</span>, <span class="no">scale.</span><span class="kw">=</span><span class="fl">TRUE</span>){
  <span class="kw">if</span>(<span class="no">dolog</span>){
    <span class="no">x</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="no">x</span> + <span class="fl">1</span>)
  }
  <span class="no">pc</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/prcomp.html">prcomp</a></span>(<span class="no">x</span>, <span class="kw">center</span><span class="kw">=</span><span class="no">center</span>, <span class="kw">scale.</span><span class="kw">=</span><span class="no">scale.</span>)
  <span class="fu"><a href="https://rdrr.io/r/base/function.html">return</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/t.html">t</a></span>(<span class="no">pc</span>$<span class="no">rotation</span>[, <span class="fl">1</span>:<span class="fl">10</span>]))
}</pre></body></html></div>
<p>Although it would be possible to do the following with a loop, the different data types do require different options for PCA (e.g. mutations are a 0/1 matrix with 1 meaning there is a somatic mutation, and gistict varies between -2 for homozygous loss and 2 for a genome doubling, so neither make sense to scale and center). So it is just as well to do the following one by one, concatenating each PCA results to the MultiAssayExperiment:</p>
<div class="sourceCode" id="cb254"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns</a></span>(<span class="no">miniACC</span>)
<span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">rnaseqPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">1</span>]], <span class="kw">dolog</span><span class="kw">=</span><span class="fl">TRUE</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">1L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb256"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">gistictPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">2</span>]], <span class="kw">center</span><span class="kw">=</span><span class="fl">FALSE</span>, <span class="kw">scale.</span><span class="kw">=</span><span class="fl">FALSE</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">2L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb258"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">proteinPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">3</span>]]), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">3L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb260"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">mutationsPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">4</span>]], <span class="kw">center</span><span class="kw">=</span><span class="fl">FALSE</span>, <span class="kw">scale.</span><span class="kw">=</span><span class="fl">FALSE</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">4L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb262"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">miRNAPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">5</span>]]), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">5L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<p>Now subset to keep <em>only</em> the PCA results:</p>
<div class="sourceCode" id="cb264"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="no">miniACC2</span>[, , <span class="fl">6</span>:<span class="fl">10</span>]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 215 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb266"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC2</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] rnaseqPCA: matrix with 10 rows and 43 columns
##  [2] gistictPCA: matrix with 10 rows and 43 columns
##  [3] proteinPCA: matrix with 10 rows and 43 columns
##  [4] mutationsPCA: matrix with 10 rows and 43 columns
##  [5] miRNAPCA: matrix with 10 rows and 43 columns</code></pre>
<p>Note, it would be equally easy (and maybe better) to do PCA on all samples available for each assay, then do intersectColumns at this point instead.</p>
<p>Now, steps for calculating the correlations and plotting a heatmap: * Use <em>wideFormat</em> to paste these together, which has the nice property of adding assay names to the column names. * The first column always contains the sample identifier, so remove it. * Coerce to a matrix * Calculate the correlations, and take the absolute value (since signs of principal components are arbitrary) * Set the diagonals to NA (each variable has a correlation of 1 to itself).</p>
<div class="sourceCode" id="cb268"><html><body><pre class="r"><span class="no">df</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">miniACC2</span>)[, -<span class="fl">1</span>]
<span class="no">mycors</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html">cor</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/matrix.html">as.matrix</a></span>(<span class="no">df</span>))
<span class="no">mycors</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">abs</a></span>(<span class="no">mycors</span>)
<span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span>(<span class="no">mycors</span>) <span class="kw">&lt;-</span> <span class="fl">NA</span></pre></body></html></div>
<p>To simplify the heatmap, show only components that have at least one correlation greater than 0.5.</p>
<div class="sourceCode" id="cb269"><html><body><pre class="r"><span class="no">has.high.cor</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/apply.html">apply</a></span>(<span class="no">mycors</span>, <span class="fl">2</span>, <span class="no">max</span>, <span class="kw">na.rm</span><span class="kw">=</span><span class="fl">TRUE</span>) <span class="kw">&gt;</span> <span class="fl">0.5</span>
<span class="no">mycors</span> <span class="kw">&lt;-</span> <span class="no">mycors</span>[<span class="no">has.high.cor</span>, <span class="no">has.high.cor</span>]
<span class="kw pkg">pheatmap</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/pheatmap/man/pheatmap.html">pheatmap</a></span>(<span class="no">mycors</span>)</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-69-1.png" width="100%"></p>
<p>The highest correlation present is between PC2 of the RNA-seq assay, and PC1 of the protein assay.</p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="annotating-with-ranges" class="section level3">
<h3 class="hasAnchor">
<a href="#annotating-with-ranges" class="anchor"></a>Annotating with ranges</h3>
<p>This section doesn’t use the <code>TCGAutils</code> shortcuts, and is more generally applicable.</p>
<p>Convert all the <code>ExperimentList</code> elements in <code>miniACC</code> to <code>RangedSummarizedExperiment</code> objects. Then use <code>rowRanges</code> to annotate these objects with genomic ranges. For the microRNA assay, annotate instead with the genomic coordinates of predicted targets.</p>
<p><strong>Solution</strong></p>
<p>The following shortcut function takes a list of human gene symbols and uses <code>AnnotationFilter</code> and <code>EnsDb.Hsapiens.v86</code> to look up the ranges, and return these as a GRangesList which can be used to replace the rowRanges of the SummarizedExperiment objects:</p>
<div class="sourceCode" id="cb270"><html><body><pre class="r"><span class="no">getrr</span> <span class="kw">&lt;-</span> <span class="kw">function</span>(<span class="no">identifiers</span>, <span class="no">EnsDbFilterFunc</span><span class="kw">=</span><span class="kw pkg">AnnotationFilter</span><span class="kw ns">::</span><span class="no"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilter.html">SymbolFilter</a></span>) {
    <span class="no">edb</span> <span class="kw">&lt;-</span> <span class="kw pkg">EnsDb.Hsapiens.v86</span><span class="kw ns">::</span><span class="no"><a href="https://rdrr.io/pkg/EnsDb.Hsapiens.v86/man/package.html">EnsDb.Hsapiens.v86</a></span>
    <span class="no">afl</span> <span class="kw">&lt;-</span> <span class="kw pkg">AnnotationFilter</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilterList.html">AnnotationFilterList</a></span>(
        <span class="fu">EnsDbFilterFunc</span>(<span class="no">identifiers</span>),
        <span class="fu"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilter.html">SeqNameFilter</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="fl">1</span>:<span class="fl">21</span>, <span class="st">"X"</span>, <span class="st">"Y"</span>)),
        <span class="fu"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilter.html">TxBiotypeFilter</a></span>(<span class="st">"protein_coding"</span>))
    <span class="no">gr</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/query.html">genes</a></span>(<span class="no">edb</span>, <span class="kw">filter</span><span class="kw">=</span><span class="no">afl</span>)
    <span class="no">grl</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/split.html">split</a></span>(<span class="no">gr</span>, <span class="fu"><a href="https://rdrr.io/r/base/factor.html">factor</a></span>(<span class="no">identifiers</span>))
    <span class="no">grl</span> <span class="kw">&lt;-</span> <span class="no">grl</span>[<span class="fu"><a href="https://rdrr.io/r/base/match.html">match</a></span>(<span class="no">identifiers</span>, <span class="fu"><a href="https://rdrr.io/r/base/names.html">names</a></span>(<span class="no">grl</span>))]
    <span class="fu"><a href="https://rdrr.io/r/base/stopifnot.html">stopifnot</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/identical.html">identical</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/names.html">names</a></span>(<span class="no">grl</span>), <span class="no">identifiers</span>))
    <span class="fu"><a href="https://rdrr.io/r/base/function.html">return</a></span>(<span class="no">grl</span>)
}</pre></body></html></div>
<p>For example:</p>
<div class="sourceCode" id="cb271"><html><body><pre class="r"><span class="fu">getrr</span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(<span class="no">miniACC</span>)<span class="kw">[[</span><span class="fl">1</span>]])</pre></body></html></div>
<pre><code>## GRangesList object of length 198:
## $DIRAS3
## GRanges object with 1 range and 7 metadata columns:
##                   seqnames          ranges strand |         gene_id   gene_name
##                      &lt;Rle&gt;       &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt; &lt;character&gt;
##   ENSG00000116288        1 7954291-7985505      + | ENSG00000116288       PARK7
##                     gene_biotype seq_coord_system      symbol entrezid
##                      &lt;character&gt;      &lt;character&gt; &lt;character&gt;   &lt;list&gt;
##   ENSG00000116288 protein_coding       chromosome       PARK7    11315
##                       tx_biotype
##                      &lt;character&gt;
##   ENSG00000116288 protein_coding
##   -------
##   seqinfo: 22 sequences from GRCh38 genome
## 
## $MAPK14
## GRanges object with 1 range and 7 metadata columns:
##                   seqnames          ranges strand |         gene_id   gene_name
##                      &lt;Rle&gt;       &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt; &lt;character&gt;
##   ENSG00000116285        1 8004404-8026308      - | ENSG00000116285      ERRFI1
##                     gene_biotype seq_coord_system      symbol entrezid
##                      &lt;character&gt;      &lt;character&gt; &lt;character&gt;   &lt;list&gt;
##   ENSG00000116285 protein_coding       chromosome      ERRFI1    54206
##                       tx_biotype
##                      &lt;character&gt;
##   ENSG00000116285 protein_coding
##   -------
##   seqinfo: 22 sequences from GRCh38 genome
## 
## $YAP1
## GRanges object with 1 range and 7 metadata columns:
##                   seqnames            ranges strand |         gene_id
##                      &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt;
##   ENSG00000198793        1 11106535-11262507      - | ENSG00000198793
##                     gene_name   gene_biotype seq_coord_system      symbol
##                   &lt;character&gt;    &lt;character&gt;      &lt;character&gt; &lt;character&gt;
##   ENSG00000198793        MTOR protein_coding       chromosome        MTOR
##                   entrezid     tx_biotype
##                     &lt;list&gt;    &lt;character&gt;
##   ENSG00000198793     2475 protein_coding
##   -------
##   seqinfo: 22 sequences from GRCh38 genome
## 
## ...
## &lt;195 more elements&gt;</code></pre>
<p>Use this to set the rowRanges of experiments 1-4 with these GRangesList objects</p>
<div class="sourceCode" id="cb273"><html><body><pre class="r"><span class="no">rseACC</span> <span class="kw">&lt;-</span> <span class="no">miniACC</span>
<span class="no">withRSE</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="fl">1</span>:<span class="fl">3</span>, <span class="fl">5</span>)
<span class="kw">for</span> (<span class="no">i</span> <span class="kw">in</span> <span class="no">withRSE</span>){
  <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html">rowRanges</a></span>(<span class="no">rseACC</span><span class="kw">[[</span><span class="no">i</span>]]) <span class="kw">&lt;-</span> <span class="fu">getrr</span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(<span class="no">rseACC</span><span class="kw">[[</span><span class="no">i</span>]]))
}</pre></body></html></div>
<p>Note that the class of experiments 1-4 is now <code>RangedSummarizedExperiment</code>:</p>
<div class="sourceCode" id="cb274"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">rseACC</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: RangedSummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: RangedSummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: RangedSummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: RangedSummarizedExperiment with 471 rows and 80 columns</code></pre>
<p>With ranged objects in the MultiAssayExperiment, you can then do subsetting by ranges. For example, select all genes on chromosome 1 for the four <em>rangedSummarizedExperiment</em> objects:</p>
<div class="sourceCode" id="cb276"><html><body><pre class="r"><span class="no">rseACC</span>[<span class="fu"><a href="https://rdrr.io/pkg/GenomicRanges/man/GRanges-class.html">GRanges</a></span>(<span class="kw">seqnames</span><span class="kw">=</span><span class="st">"1:1-1e9"</span>), , <span class="no">withRSE</span>]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 90 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 3 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 3:
##  [1] RNASeq2GeneNorm: RangedSummarizedExperiment with 22 rows and 79 columns
##  [2] gistict: RangedSummarizedExperiment with 22 rows and 90 columns
##  [3] RPPAArray: RangedSummarizedExperiment with 3 rows and 46 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<p><em>Note about microRNA</em>: You can set ranges for the microRNA assay according to the genomic location of those microRNA, or the locations of their predicted targets, but we don’t do it here. Assigning genomic ranges of microRNA targets would be an easy way to subset them according to their targets.</p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
</div>
</div>
<div class="footnotes">
<hr>
<ol>
<li id="fn1"><p>City University of New York, New York, NY, USA<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Roswell Park Comprehensive Cancer Center, Buffalo, NY<a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>City University of New York, New York, NY, USA<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>City University of New York, New York, NY, USA<a href="#fnref4" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Marcel Ramos, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
