<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MultiAssayExperiment: Integrative workflow for public multi-omics data • MultiAssayWorkshop</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="MultiAssayExperiment: Integrative workflow for public multi-omics data">
<meta property="og:description" content="MultiAssayWorkshop">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MultiAssayWorkshop</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">3.11.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/Ramos_MultiAssayExperiment.html">Workshop</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorial
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../tutorials/Exercise_MultiAssay.html">Build Your First MAE</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Reference
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/cBioPortal_Studies.html">cBioPortal Studies</a>
    </li>
    <li>
      <a href="../articles/curatedTCGAData_ref.html">curatedTCGAData Reference</a>
    </li>
    <li>
      <a href="../articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MultiAssayWorkshop">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="Ramos_MultiAssayExperiment_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MultiAssayExperiment: Integrative workflow for public multi-omics data</h1>
                        <h4 class="author">Marcel Ramos</h4>
            <address class="author_afil">
      Roswell Park Comprehensive Cancer Center, Buffalo, NY<br><h4 class="author">Ludwig Geistlinger</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="author">Levi Waldron</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="date">July 27, 2020</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/MultiAssayWorkshop/blob/master/vignettes/Ramos_MultiAssayExperiment.Rmd"><code>vignettes/Ramos_MultiAssayExperiment.Rmd</code></a></small>
      <div class="hidden name"><code>Ramos_MultiAssayExperiment.Rmd</code></div>

    </address>
</address>
</address>
</div>

    
    
<style type="text/css">
.infobox {
  padding: 1em 1em 1em 4em;
  margin-bottom: 10px;
  border: 2px solid #008000;
  border-radius: 10px;
  background: #DCDCDC 5px center/3em no-repeat;
  color: #8B0000;
}

.center {
  text-align: center;
}
</style>
<div id="multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment" class="section level1">
<h1 class="hasAnchor">
<a href="#multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment" class="anchor"></a>Multi-omic Integration and Analysis of cBioPortal and TCGA data with MultiAssayExperiment</h1>
<div id="workshop-participation" class="section level2">
<h2 class="hasAnchor">
<a href="#workshop-participation" class="anchor"></a>Workshop participation</h2>
<ul>
<li>See the instruction <a href="https://tinyurl.com/MultiAssayWorkshop">Google slides</a>
</li>
<li>Go to the conference instance registration webpage and use these credentials:
<ul>
<li>username: <code>rstudio</code>
</li>
<li>password: <code>welcome-to-bioc2020</code>
</li>
</ul>
</li>
<li>Browse to the <a href="https://waldronlab.io/MultiAssayWorkshop/">workshop website</a>
</li>
<li>(Alternatively) Docker users can run this workshop locally via:</li>
</ul>
<pre><code>docker run -e PASSWORD=bioc -p 8787:8787 mr148/multiassayworkshop:latest</code></pre>
</div>
<div id="requirements-rbioconductor-packages" class="section level2">
<h2 class="hasAnchor">
<a href="#requirements-rbioconductor-packages" class="anchor"></a>Requirements: R/Bioconductor packages</h2>
<p>The workshop uses a Docker container with Bioconductor devel version <code>3.12</code>. If you would like to install Bioconductor on your computer <strong>at a later date</strong>, see the <a href="https://www.bioconductor.org/install">Bioconductor installation</a> instructions.</p>
<p>Here is a list of packages that we will be using:</p>
<div class="sourceCode" id="cb2"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">MultiAssayExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">curatedTCGAData</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">cBioPortalData</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">TCGAutils</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">GenomicDataCommons</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">UpSetR</span>)</pre></body></html></div>
<div id="citing-multiassayexperiment" class="section level3">
<h3 class="hasAnchor">
<a href="#citing-multiassayexperiment" class="anchor"></a>Citing MultiAssayExperiment</h3>
<p>Please use this citation <span class="citation">(Ramos et al. 2017)</span> when using <code>MultiAssayExperiment</code>, your citations are appreciated!</p>
</div>
<div id="key-packages" class="section level3">
<h3 class="hasAnchor">
<a href="#key-packages" class="anchor"></a>Key Packages</h3>
<div id="multiassayexperiment" class="section level4">
<h4 class="hasAnchor">
<a href="#multiassayexperiment" class="anchor"></a><code>MultiAssayExperiment</code>
</h4>
<ul>
<li>provides an integrative representation for multi-omics data</li>
<li>modelled after the <code>SummarizedExperiment</code> representation for expression data</li>
<li>easy-to-use operations for manipulating multiple sets of data such as copy number alterations, mutations, proteomics, methylation, and more</li>
</ul>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic" width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic
</p>
</div>
</div>
<div id="cbioportaldata" class="section level4">
<h4 class="hasAnchor">
<a href="#cbioportaldata" class="anchor"></a><code>cBioPortalData</code>
</h4>
<ul>
<li>R/Bioconductor interface to cBioPortal data</li>
<li>makes use of the revamped API with caching</li>
<li>queries are handled for the user <em>in the background</em>
</li>
<li>easy-to-use interface (no knowledge of the cBioPortal data model required)</li>
</ul>
</div>
<div id="curatedtcgadata" class="section level4">
<h4 class="hasAnchor">
<a href="#curatedtcgadata" class="anchor"></a><code>curatedTCGAData</code>
</h4>
<ul>
<li>Many tools exist for accessing and downloading <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga">The Cancer Genome Atlas</a> data: <code>RTCGAToolbox</code>, <code>GenomicDataCommons</code>, <code>TCGAbiolinks</code>, <code>cBioPortal</code> website, Broad GDAC Firehose, and more</li>
<li>makes it easy to obtain user-friendly and integrative data at very little cognitive overhead</li>
<li>conveniently places data in the analysis platform of choice, R/Bioconductor</li>
<li>provides 33 different cancer types from the <a href="https://gdac.broadinstitute.org">Broad GDAC Firehose</a>
<ul>
<li>On-the-fly construction from ‘flat’ files</li>
<li>
<code>hg19</code> data</li>
<li>
<code>MultiAssayExperiment</code> representations</li>
</ul>
</li>
</ul>
<p><a href="articles/curatedTCGAData_ref.html">Reference vignettes</a>:</p>
<ul>
<li><p><code>Available Studies</code> – (curatedTCGAData section) A list of available cancer studies from <code><a href="https://rdrr.io/pkg/TCGAutils/man/diseaseCodes.html">TCGAutils::diseaseCodes</a></code>.</p></li>
<li><p><code>OmicsTypes</code> – A descriptive table of ’omics types in <code>curatedTCGAData</code> (thanks to Ludwig G. <code>@lgeistlinger</code>)</p></li>
</ul>
</div>
<div id="tcgautils" class="section level4">
<h4 class="hasAnchor">
<a href="#tcgautils" class="anchor"></a><code>TCGAutils</code>
</h4>
<ul>
<li>allows additional exploration, and manipulation of samples and metadata</li>
<li>User-friendly operations for subsetting, separating, converting, and reshaping of sample and feature TCGA data</li>
<li>developed specifically for TCGA data and <code>curatedTCGAData</code> outputs</li>
</ul>
<p>It provides convenience / helper functions in three major areas:</p>
<ol style="list-style-type: decimal">
<li>conversion / summarization of row annotations to genomic ranges</li>
<li>identification and separation of samples</li>
<li>translation and interpretation of TCGA identifiers</li>
</ol>
<p>For the cheatsheet reference table, see the <a href="articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>.</p>
<p>To better understand how it all fits together, this schematic shows the relationship among all as part of the curatedTCGAData pipeline.</p>
<div class="figure">
<img src="TCGAMAEPipeline20190204.png" alt="Schematic of curatedTCGAData Pipeline" width="100%"><p class="caption">
Schematic of curatedTCGAData Pipeline
</p>
</div>
</div>
</div>
<div id="data-classes" class="section level3">
<h3 class="hasAnchor">
<a href="#data-classes" class="anchor"></a>Data Classes</h3>
<p>This section summarizes three fundamental data classes for the representation of multi-omics experiments.</p>
<div id="rangedsummarizedexperiment" class="section level4">
<h4 class="hasAnchor">
<a href="#rangedsummarizedexperiment" class="anchor"></a><code>(Ranged)SummarizedExperiment</code>
</h4>
<div class="figure">
<img src="SummarizedExperiment.png" alt="A matrix-like container where rows represent features of interest and columns represent samples. The objects contain one or more assays, each represented by a matrix-like object of numeric or other mode." width="100%"><p class="caption">
A matrix-like container where rows represent features of interest and columns represent samples. The objects contain one or more assays, each represented by a matrix-like object of numeric or other mode.
</p>
</div>
<ul>
<li>matrix-like representation of experimental data including RNA sequencing and microarray experiments.</li>
<li>stores multiple experimental data matrices of identical dimensions, with associated metadata on:
<ul>
<li>the rows/genes/transcripts/other measurements (<code>rowData</code>)</li>
<li>column/sample phenotype or clinical data (<code>colData</code>)</li>
<li>overall experiment (<code>metadata</code>).</li>
</ul>
</li>
<li>
<code>RangedSummarizedExperiment</code> associates a <code>GRanges</code> or <code>GRangesList</code> vector with the rows</li>
</ul>
<p><em>Note</em>. Many other classes for experimental data are actually derived from <code>SummarizedExperiment</code> (e.g., <code>SingleCellExperiment</code> for single-cell RNA sequencing experiments)</p>
<div class="sourceCode" id="cb3"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">SingleCellExperiment</span>)
<span class="fu">extends</span>(<span class="st">"SingleCellExperiment"</span>)</pre></body></html></div>
<pre><code>## [1] "SingleCellExperiment"       "RangedSummarizedExperiment"
## [3] "SummarizedExperiment"       "RectangularData"           
## [5] "Vector"                     "Annotated"                 
## [7] "vector_OR_Vector"</code></pre>
</div>
<div id="raggedexperiment" class="section level4">
<h4 class="hasAnchor">
<a href="#raggedexperiment" class="anchor"></a><code>RaggedExperiment</code>
</h4>
<ul>
<li>flexible representation for segmented copy number, somatic mutations such as represented in <code>.vcf</code> files, and other ragged array schema for genomic location data.</li>
<li>similar to the <code>GRangesList</code> class in <code>GenomicRanges</code>
</li>
<li>used to represent <em>differing</em> genomic ranges on each of a set of samples</li>
</ul>
<div class="sourceCode" id="cb5"><html><body><pre class="r"><span class="fu">showClass</span>(<span class="st">"RaggedExperiment"</span>)</pre></body></html></div>
<pre><code>## Class "RaggedExperiment" [package "RaggedExperiment"]
## 
## Slots:
##                                                       
## Name:       assays      rowidx      colidx    metadata
## Class: GRangesList     integer     integer        list
## 
## Extends: "Annotated"</code></pre>
<p><code>RaggedExperiment</code> provides a flexible set of _*Assay_ methods to support transformation of data to matrix format.</p>
<div class="figure">
<img src="RaggedExperiment.png" alt="RaggedExperiment object schematic. Rows and columns represent genomic ranges and samples, respectively. Assay operations can be performed with (from left to right) compactAssay, qreduceAssay, and sparseAssay." width="\maxwidth"><p class="caption">
RaggedExperiment object schematic. Rows and columns represent genomic ranges and samples, respectively. Assay operations can be performed with (from left to right) compactAssay, qreduceAssay, and sparseAssay.
</p>
</div>
</div>
</div>
</div>
<div id="the-integrative-container" class="section level2">
<h2 class="hasAnchor">
<a href="#the-integrative-container" class="anchor"></a>The Integrative Container</h2>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic. colData provides data about the patients, cell lines, or other biological units, with one row per unit and one column per variable. The experiments are a list of assay datasets of arbitrary class.  The sampleMap relates each column (observation) in ExperimentList to exactly one row (biological unit) in colData; however, one row of colData may map to zero, one, or more columns per assay, allowing for missing and replicate assays. sampleMap allows for per-assay sample naming conventions. Metadata can be used to store information in arbitrary format about the MultiAssayExperiment. Green stripes indicate a mapping of one subject to multiple observations across experiments." width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic. colData provides data about the patients, cell lines, or other biological units, with one row per unit and one column per variable. The experiments are a list of assay datasets of arbitrary class. The sampleMap relates each column (observation) in ExperimentList to exactly one row (biological unit) in colData; however, one row of colData may map to zero, one, or more columns per assay, allowing for missing and replicate assays. sampleMap allows for per-assay sample naming conventions. Metadata can be used to store information in arbitrary format about the MultiAssayExperiment. Green stripes indicate a mapping of one subject to multiple observations across experiments.
</p>
</div>
<div id="multiassayexperiment-1" class="section level3">
<h3 class="hasAnchor">
<a href="#multiassayexperiment-1" class="anchor"></a><code>MultiAssayExperiment</code>
</h3>
<ul>
<li>coordinates multi-omics experiment data on a set of biological specimens</li>
<li>can contain any number of assays with different representations and <em>dimensions</em>
</li>
<li>assays can be <em>ID-based</em>, where measurements are indexed identifiers of genes, microRNA, proteins, microbes, etc.</li>
<li>assays may be <em>range-based</em>, where measurements correspond to genomic ranges that can be represented as <code>GRanges</code> objects, such as gene expression or copy number.</li>
</ul>
<details><p><summary><strong>Click</strong> on the fold to see what data classes are supported! </summary></p>
<ol style="list-style-type: decimal">
<li>
<code>matrix</code>: the most basic class for ID-based datasets, could be used for example for gene expression summarized per-gene, microRNA, metabolomics, or microbiome data.</li>
<li>
<code>SummarizedExperiment</code> and derived methods: described above, could be used for miRNA, gene expression, proteomics, or any matrix-like data where measurements are represented by IDs.</li>
<li>
<code>RangedSummarizedExperiment</code>: described above, could be used for gene expression, methylation, or other data types referring to genomic positions.</li>
<li>
<code>ExpressionSet</code>: Another rich representation for ID-based datasets, supported only for legacy reasons</li>
<li>
<code>RaggedExperiment</code>: described above, for non-rectangular (ragged) ranged-based datasets such as segmented copy number, where segmentation of copy number alterations occurs and different genomic locations in each sample.</li>
<li>
<code>RangedVcfStack</code>: For VCF archives broken up by chromosome (see <code>VcfStack</code> class defined in the <code>GenomicFiles</code> package)</li>
<li>
<code>DelayedMatrix</code>: An on-disk representation of matrix-like objects for large datasets. It reduces memory usage and optimizes performance with delayed operations. This class is part of the <code>DelayedArray</code> package.</li>
</ol></details><p><em>Note</em>. many data classes that support row and column naming and subsetting can be used in a <code>MultiAssayExperiment</code>.</p>
</div>
<div id="matchedassayexperiment" class="section level3">
<h3 class="hasAnchor">
<a href="#matchedassayexperiment" class="anchor"></a><code>MatchedAssayExperiment</code>
</h3>
<ul>
<li>uniform subclass of <code>MultiAssayExperiment</code>
</li>
<li>“all patients have a sample in each assay”</li>
</ul>
<div class="sourceCode" id="cb7"><html><body><pre class="r"><span class="co"># coercion</span>
<span class="fu">as</span>(<span class="no">x</span>, <span class="st">"MatchedAssayExperiment"</span>)

<span class="co"># construction from MAE</span>
<span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MatchedAssayExperiment-class.html">MatchedAssayExperiment</a></span>(<span class="no">mae</span>)</pre></body></html></div>
<p><em>Note</em>. The <code>MultiAssayExperiment</code> package then provides functionality to merge replicate profiles for a single patient (<code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">mergeReplicates()</a></code>).</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>
<code>MultiAssayExperiment</code> coordinates different Bioconductor classes into one unified object</li>
<li>
<code>MultiAssayExperiment</code> is an infrastructure package while <code>curatedTCGAData</code> and <code>cBioPortalData</code> provide data on cancer studies including TCGA</li>
</ul>
</div>
</div>
</div>
<div id="building-from-scratch-multiassayexperiment" class="section level2">
<h2 class="hasAnchor">
<a href="#building-from-scratch-multiassayexperiment" class="anchor"></a>Building from Scratch: <code>MultiAssayExperiment</code>
</h2>
<div id="miniacc-demo" class="section level3">
<h3 class="hasAnchor">
<a href="#miniacc-demo" class="anchor"></a><code>miniACC</code> Demo</h3>
<p>Get started by trying out <code>MultiAssayExperiment</code> using a subset of the TCGA adrenocortical carcinoma (ACC) dataset provided with the package. This dataset provides five assays on 92 patients, although all five assays were not performed for every patient:</p>
<ol style="list-style-type: decimal">
<li>
<strong>RNASeq2GeneNorm</strong>: gene mRNA abundance by RNA-seq</li>
<li>
<strong>gistict</strong>: GISTIC genomic copy number by gene</li>
<li>
<strong>RPPAArray</strong>: protein abundance by Reverse Phase Protein Array</li>
<li>
<strong>Mutations</strong>: non-silent somatic mutations by gene</li>
<li>
<strong>miRNASeqGene</strong>: microRNA abundance by microRNA-seq.</li>
</ol>
<div class="sourceCode" id="cb8"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"miniACC"</span>)
<span class="no">miniACC</span></pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="shiny-demo" class="section level3">
<h3 class="hasAnchor">
<a href="#shiny-demo" class="anchor"></a><code>shiny</code> Demo</h3>
<p><a href="../tutorials/Exercise_MultiAssay.html">Click Here</a> to open the <code>shiny</code> tutorial.</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>Extractor functions allow users to take components from the <code>MultiAssayExperiment</code> object</li>
<li>They’re usually the same name as the component except for <code>experiments</code> which extracts the <code>ExperimentList</code>
</li>
</ul>
</div>
</div>
</div>
<div id="notes-on-working-with-multiassayexperiment" class="section level2">
<h2 class="hasAnchor">
<a href="#notes-on-working-with-multiassayexperiment" class="anchor"></a>Notes on Working with MultiAssayExperiment</h2>
<div id="api-cheat-sheet" class="section level3">
<h3 class="hasAnchor">
<a href="#api-cheat-sheet" class="anchor"></a>API cheat sheet</h3>
<div class="figure">
<img src="MultiAssayExperiment_cheatsheet.png" alt="The MultiAssayExperiment API for construction, access, subsetting, management, and reshaping to formats for application of R/Bioconductor graphics and analysis packages." width="\maxwidth"><p class="caption">
The MultiAssayExperiment API for construction, access, subsetting, management, and reshaping to formats for application of R/Bioconductor graphics and analysis packages.
</p>
</div>
</div>
<div id="multiassayexperiment-construction-and-concatenation" class="section level3">
<h3 class="hasAnchor">
<a href="#multiassayexperiment-construction-and-concatenation" class="anchor"></a>MultiAssayExperiment construction and concatenation</h3>
<div id="constructor-function" class="section level4">
<h4 class="hasAnchor">
<a href="#constructor-function" class="anchor"></a>constructor function</h4>
<p>The <code>MultiAssayExperiment</code> constructor function can take three arguments:</p>
<ol style="list-style-type: decimal">
<li>
<code>experiments</code> - An <code>ExperimentList</code> or <code>list</code> of rectangular data</li>
<li>
<code>colData</code> - A <code>DataFrame</code> describing the patients (or cell lines, or other biological units)</li>
<li>
<code>sampleMap</code> - A <code>DataFrame</code> of <code>assay</code>, <code>primary</code>, and <code>colname</code> identifiers</li>
</ol>
<p>The miniACC object can be reconstructed as follows:</p>
<div class="sourceCode" id="cb10"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment.html">MultiAssayExperiment</a></span>(
    <span class="kw">experiments</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC</span>),
    <span class="kw">colData</span> <span class="kw">=</span> <span class="fu">colData</span>(<span class="no">miniACC</span>),
    <span class="kw">sampleMap</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap</a></span>(<span class="no">miniACC</span>),
    <span class="kw">metadata</span> <span class="kw">=</span> <span class="fu">metadata</span>(<span class="no">miniACC</span>)
)</pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="prepmultiassay---constructor-function-helper" class="section level4">
<h4 class="hasAnchor">
<a href="#prepmultiassay---constructor-function-helper" class="anchor"></a><code>prepMultiAssay</code> - Constructor function helper</h4>
<p>The <code>prepMultiAssay</code> function allows the user to diagnose typical problems when creating a <code>MultiAssayExperiment</code> object. See <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/prepMultiAssay.html">?prepMultiAssay</a></code> for more details.</p>
</div>
<div id="c---concatenate-to-multiassayexperiment" class="section level4">
<h4 class="hasAnchor">
<a href="#c---concatenate-to-multiassayexperiment" class="anchor"></a><code>c</code> - concatenate to MultiAssayExperiment</h4>
<p>The <code>c</code> function allows the user to concatenate an additional experiment to an existing <code>MultiAssayExperiment</code>. The optional <code>sampleMap</code> argument allows concatenating an assay whose column names do not match the row names of <code>colData</code>. For convenience, the <em>mapFrom</em> argument allows the user to map from a particular experiment <strong>provided</strong> that the <strong>order</strong> of the colnames is in the <strong>same</strong>. A <code>warning</code> will be issued to make the user aware of this assumption. For example, to concatenate a matrix of log2-transformed RNA-seq results:</p>
<div class="sourceCode" id="cb12"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(
    <span class="no">miniACC</span>,
    <span class="kw">log2rnaseq</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="fu">assays</span>(<span class="no">miniACC</span>)$<span class="no">RNASeq2GeneNorm</span>),
    <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">1L</span>
)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb14"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC2</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 6:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
##  [6] log2rnaseq: matrix with 198 rows and 79 columns</code></pre>
</div>
</div>
<div id="coldata---information-biological-units" class="section level3">
<h3 class="hasAnchor">
<a href="#coldata---information-biological-units" class="anchor"></a>colData - information biological units</h3>
<p>This slot is a <code>DataFrame</code> describing the characteristics of biological units, for example clinical data for patients. In the prepared datasets from <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga">The Cancer Genome Atlas</a>, each row is one patient and each column is a clinical, pathological, subtype, or other variable. The <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> function provides a shortcut for accessing or setting <code>colData</code> columns.</p>
<div class="sourceCode" id="cb16"><html><body><pre class="r"><span class="fu">colData</span>(<span class="no">miniACC</span>)[<span class="fl">1</span>:<span class="fl">4</span>, <span class="fl">1</span>:<span class="fl">4</span>]</pre></body></html></div>
<pre><code>## DataFrame with 4 rows and 4 columns
##                 patientID years_to_birth vital_status days_to_death
##               &lt;character&gt;      &lt;integer&gt;    &lt;integer&gt;     &lt;integer&gt;
## TCGA-OR-A5J1 TCGA-OR-A5J1             58            1          1355
## TCGA-OR-A5J2 TCGA-OR-A5J2             44            1          1677
## TCGA-OR-A5J3 TCGA-OR-A5J3             23            0            NA
## TCGA-OR-A5J4 TCGA-OR-A5J4             23            1           423</code></pre>
<div class="sourceCode" id="cb18"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/table.html">table</a></span>(<span class="no">miniACC</span>$<span class="no">race</span>)</pre></body></html></div>
<pre><code>## 
##                     asian black or african american                     white 
##                         2                         1                        78</code></pre>
<p><em>Key points about the colData:</em></p>
<ul>
<li>Each row maps to zero or more observations in each experiment in the <code>ExperimentList</code>, below.</li>
<li>One row per biological unit
<ul>
<li>
<code>MultiAssayExperiment</code> supports both missing observations and replicate observations, ie one row of <code>colData</code> can map to 0, 1, or more columns of any of the experimental data matrices.</li>
<li>therefore you could treat replicate observations as one or multiple rows of <code>colData</code>, and this will result in different behaviors of functions you will learn later like subsetting, <code><a href="https://rdrr.io/r/base/duplicated.html">duplicated()</a></code>, and <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat()</a></code>.</li>
<li>multiple time points, or distinct biological replicates, should probably be separate rows of the <code>colData</code>.</li>
</ul>
</li>
</ul>
</div>
<div id="experimentlist---experiment-data" class="section level3">
<h3 class="hasAnchor">
<a href="#experimentlist---experiment-data" class="anchor"></a>ExperimentList - experiment data</h3>
<p>A base <code>list</code> or <code>ExperimentList</code> object containing the experimental datasets for the set of samples collected. This gets converted into a class <code>ExperimentList</code> during construction.</p>
<div class="sourceCode" id="cb20"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns</code></pre>
<p><em>Key points:</em></p>
<ul>
<li>One matrix-like dataset per list element (although they do not even need to be matrix-like, see for example the <code>RaggedExperiment</code> package)</li>
<li>One matrix column per assayed specimen. Each matrix column must correspond to exactly one row of <code>colData</code>: in other words, you must know which patient or cell line the observation came from. However, multiple columns can come from the same patient, or there can be no data for that patient.</li>
<li>Matrix rows correspond to variables, e.g. genes or genomic ranges</li>
<li>
<code>ExperimentList</code> elements can be genomic range-based (e.g. <code>SummarizedExperiment::RangedSummarizedExperiment-class</code> or <code>RaggedExperiment::RaggedExperiment-class</code>) or ID-based data (e.g. <code>SummarizedExperiment::SummarizedExperiment-class</code>, <code>Biobase::eSet-class</code> <code>base::matrix-class</code>, <code>DelayedArray::DelayedArray-class</code>, and derived classes)</li>
<li>Any data class can be included in the <code>ExperimentList</code>, as long as it supports: single-bracket subsetting (<code><a href="https://rdrr.io/r/base/Extract.html">[</a></code>), <code>dimnames</code>, and <code>dim</code>. Most data classes defined in Bioconductor meet these requirements.</li>
</ul>
</div>
<div id="samplemap---relationship-graph" class="section level3">
<h3 class="hasAnchor">
<a href="#samplemap---relationship-graph" class="anchor"></a>sampleMap - relationship graph</h3>
<p><code>sampleMap</code> is a graph representation of the relationship between biological units and experimental results. In simple cases where the column names of <code>ExperimentList</code> data matrices match the row names of <code>colData</code>, the user won’t need to specify or think about a sample map, it can be created automatically by the <code>MultiAssayExperiment</code> constructor. <code>sampleMap</code> is a simple three-column <code>DataFrame</code>:</p>
<ol style="list-style-type: decimal">
<li>
<code>assay</code> column: the name of the assay, and found in the names of <code>ExperimentList</code> list names</li>
<li>
<code>primary</code> column: identifiers of patients or biological units, and found in the row names of <code>colData</code>
</li>
<li>
<code>colname</code> column: identifiers of assay results, and found in the column names of <code>ExperimentList</code> elements Helper functions are available for creating a map from a list. See <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/mapToList.html">?listToMap</a></code>
</li>
</ol>
<div class="sourceCode" id="cb22"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## DataFrame with 385 rows and 3 columns
##               assay      primary                colname
##            &lt;factor&gt;  &lt;character&gt;            &lt;character&gt;
## 1   RNASeq2GeneNorm TCGA-OR-A5J1 TCGA-OR-A5J1-01A-11R..
## 2   RNASeq2GeneNorm TCGA-OR-A5J2 TCGA-OR-A5J2-01A-11R..
## 3   RNASeq2GeneNorm TCGA-OR-A5J3 TCGA-OR-A5J3-01A-11R..
## 4   RNASeq2GeneNorm TCGA-OR-A5J5 TCGA-OR-A5J5-01A-11R..
## 5   RNASeq2GeneNorm TCGA-OR-A5J6 TCGA-OR-A5J6-01A-31R..
## ...             ...          ...                    ...
## 381    miRNASeqGene TCGA-PA-A5YG TCGA-PA-A5YG-01A-11R..
## 382    miRNASeqGene TCGA-PK-A5H8 TCGA-PK-A5H8-01A-11R..
## 383    miRNASeqGene TCGA-PK-A5H9 TCGA-PK-A5H9-01A-11R..
## 384    miRNASeqGene TCGA-PK-A5HA TCGA-PK-A5HA-01A-11R..
## 385    miRNASeqGene TCGA-PK-A5HB TCGA-PK-A5HB-01A-11R..</code></pre>
<p><em>Key points:</em></p>
<ul>
<li>relates experimental observations (<code>colnames</code>) to <code>colData</code>
</li>
<li>permits experiment-specific sample naming, missing, and replicate observations</li>
</ul>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="metadata" class="section level3">
<h3 class="hasAnchor">
<a href="#metadata" class="anchor"></a>metadata</h3>
<p>Metadata can be used to keep additional information about patients, assays performed on individuals or on the entire cohort, or features such as genes, proteins, and genomic ranges. There are many options available for storing metadata. First, <code>MultiAssayExperiment</code> has its own metadata for describing the entire experiment:</p>
<div class="sourceCode" id="cb24"><html><body><pre class="r"><span class="fu">metadata</span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## $title
## [1] "Comprehensive Pan-Genomic Characterization of Adrenocortical Carcinoma"
## 
## $PMID
## [1] "27165744"
## 
## $sourceURL
## [1] "http://s3.amazonaws.com/multiassayexperiments/accMAEO.rds"
## 
## $RPPAfeatureDataURL
## [1] "http://genomeportal.stanford.edu/pan-tcga/show_target_selection_file?filename=Allprotein.txt"
## 
## $colDataExtrasURL
## [1] "http://www.cell.com/cms/attachment/2062093088/2063584534/mmc3.xlsx"</code></pre>
<p>Additionally, the <code>DataFrame</code> class used by <code>sampleMap</code> and <code>colData</code>, as well as the <code>ExperimentList</code> class, similarly support metadata. Finally, many experimental data objects that can be used in the <code>ExperimentList</code> support metadata. These provide flexible options to users and to developers of derived classes.</p>
</div>
</div>
<div id="the-cancer-genome-atlas-tcga-data-from-curatedtcgadata" class="section level2">
<h2 class="hasAnchor">
<a href="#the-cancer-genome-atlas-tcga-data-from-curatedtcgadata" class="anchor"></a>The Cancer Genome Atlas (TCGA) Data from <code>curatedTCGAData</code>
</h2>
<p>Most unrestricted TCGA data are available as <code>MultiAssayExperiment</code> objects from the <code>curatedTCGAData</code> package. This represents a lot of harmonization!</p>
<div class="sourceCode" id="cb26"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">curatedTCGAData</span>)
<span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="st">"ACC"</span>)</pre></body></html></div>
<pre><code>##                                    Title DispatchClass
## 1                    ACC_CNASNP-20160128           Rda
## 2                    ACC_CNVSNP-20160128           Rda
## 4          ACC_GISTIC_AllByGene-20160128           Rda
## 5              ACC_GISTIC_Peaks-20160128           Rda
## 6  ACC_GISTIC_ThresholdedByGene-20160128           Rda
## 8        ACC_Methylation-20160128_assays        H5File
## 9            ACC_Methylation-20160128_se           Rds
## 10             ACC_miRNASeqGene-20160128           Rda
## 11                 ACC_Mutation-20160128           Rda
## 12          ACC_RNASeq2GeneNorm-20160128           Rda
## 13                ACC_RPPAArray-20160128           Rda</code></pre>
<div class="sourceCode" id="cb28"><html><body><pre class="r"><span class="no">acc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(
    <span class="kw">diseaseCode</span> <span class="kw">=</span> <span class="st">"ACC"</span>,
    <span class="kw">assays</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(
        <span class="st">"miRNASeqGene"</span>, <span class="st">"RPPAArray"</span>, <span class="st">"Mutation"</span>, <span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"CNVSNP"</span>
    ),
    <span class="kw">dry.run</span> <span class="kw">=</span> <span class="fl">FALSE</span>
)
<span class="no">acc</span></pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [4] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [5] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<p>These objects contain most unrestricted TCGA assay and clinical / pathological data, as well as material curated from the supplements of published TCGA primary papers at the end of the colData columns:</p>
<div class="sourceCode" id="cb30"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/dim.html">dim</a></span>(<span class="fu">colData</span>(<span class="no">acc</span>))</pre></body></html></div>
<pre><code>## [1]  92 822</code></pre>
<div class="sourceCode" id="cb32"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">tail</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="fu">colData</span>(<span class="no">acc</span>)), <span class="fl">10</span>)</pre></body></html></div>
<pre><code>##  [1] "MethyLevel"       "miRNA.cluster"    "SCNA.cluster"     "protein.cluster" 
##  [5] "COC"              "OncoSign"         "purity"           "ploidy"          
##  [9] "genome_doublings" "ADS"</code></pre>
<p>The <code>TCGAutils</code> package provides additional helper functions.</p>
</div>
<div id="cbioportaldata-1" class="section level2">
<h2 class="hasAnchor">
<a href="#cbioportaldata-1" class="anchor"></a><code>cBioPortalData</code>
</h2>
<p>The <a href="https://www.cbioportal.org/">cBio Genomics Portal</a> provides access to more than 260 datasets collected and curated from different instutions.</p>
<p>There are two main ways of accessing this data:</p>
<ol style="list-style-type: decimal">
<li>
<code>cBioDataPack</code> - tarball (<code>.tar.gz</code>) data files</li>
<li>
<code>cBioPortalData</code> - data from the API</li>
</ol>
<p><em>Note</em>. <code>pkgdown</code> reference website here: <a href="https://waldronlab.io/cBioPortalData/" class="uri">https://waldronlab.io/cBioPortalData/</a></p>
<div id="cbiodatapack" class="section level3">
<h3 class="hasAnchor">
<a href="#cbiodatapack" class="anchor"></a>cBioDataPack</h3>
<div class="sourceCode" id="cb34"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">cBioPortalData</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"studiesTable"</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">studiesTable</span><span class="kw">[[</span><span class="st">"cancer_study_id"</span>]])</pre></body></html></div>
<pre><code>## [1] "paac_jhu_2014"       "mel_tsam_liang_2017" "all_stjude_2015"    
## [4] "all_stjude_2016"     "aml_ohsu_2018"       "laml_tcga_pub"</code></pre>
<div class="sourceCode" id="cb36"><html><body><pre class="r">(<span class="no">uvm</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioDataPack.html">cBioDataPack</a></span>(<span class="st">"uvm_tcga"</span>))</pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 9 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 9:
##  [1] CNA: SummarizedExperiment with 24776 rows and 80 columns
##  [2] RNA_Seq_v2_expression_median: SummarizedExperiment with 20531 rows and 80 columns
##  [3] RNA_Seq_v2_mRNA_median_Zscores: SummarizedExperiment with 20440 rows and 80 columns
##  [4] RNA_Seq_v2_mRNA_median_all_sample_Zscores: SummarizedExperiment with 20531 rows and 80 columns
##  [5] cna_hg19.seg: RaggedExperiment with 7618 rows and 80 columns
##  [6] linear_CNA: SummarizedExperiment with 24776 rows and 80 columns
##  [7] methylation_hm450: SummarizedExperiment with 15191 rows and 80 columns
##  [8] mutations_extended: RaggedExperiment with 2174 rows and 80 columns
##  [9] mutations_mskcc: RaggedExperiment with 2174 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="cbioportaldata-2" class="section level3">
<h3 class="hasAnchor">
<a href="#cbioportaldata-2" class="anchor"></a>cBioPortalData</h3>
<p>First, we create an API object using the <code>cBioPortal</code> function. This will allow us to subsequently generate queries for the service.</p>
<div class="sourceCode" id="cb38"><html><body><pre class="r"><span class="no">cbio</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html">cBioPortal</a></span>()
<span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html">getStudies</a></span>(<span class="no">cbio</span>)</pre></body></html></div>
<pre><code>## # A tibble: 287 x 13
##    name  shortName description publicStudy pmid  citation groups status
##    &lt;chr&gt; &lt;chr&gt;     &lt;chr&gt;       &lt;lgl&gt;       &lt;chr&gt; &lt;chr&gt;    &lt;chr&gt;   &lt;int&gt;
##  1 Chol… Cholangi… Exome sequ… TRUE        2418… Chan-on… "PUBL…      0
##  2 Esop… ESCC (UC… Whole exom… TRUE        2468… Lin et … "PUBL…      0
##  3 Oral… Head &amp; n… Comprehens… TRUE        2361… Pickeri… ""          0
##  4 Hepa… HCC (Ins… Whole-exom… TRUE        2582… Schulze… "PUBL…      0
##  5 Uvea… UM (QIMR) Whole-geno… TRUE        2668… Johanss… "PUBL…      0
##  6 Neur… NBL (AMC) Whole geno… TRUE        2236… Molenaa… "PUBL…      0
##  7 Naso… NPC (Sin… Whole exom… TRUE        2495… Lin et … "PUBL…      0
##  8 Thym… TET (NCI) Whole exom… TRUE        2497… Petrini… "PUBL…      0
##  9 Neur… NBL (Col… Whole-geno… TRUE        2646… Peifer … ""          0
## 10 Myel… MDS (Tok… Whole exom… TRUE        2190… Yoshida… ""          0
## # … with 277 more rows, and 5 more variables: importDate &lt;chr&gt;,
## #   allSampleCount &lt;int&gt;, studyId &lt;chr&gt;, cancerTypeId &lt;chr&gt;,
## #   referenceGenome &lt;chr&gt;</code></pre>
<div class="sourceCode" id="cb40"><html><body><pre class="r">(
    <span class="no">urcc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortalData.html">cBioPortalData</a></span>(
        <span class="no">cbio</span>, <span class="kw">studyId</span> <span class="kw">=</span> <span class="st">"urcc_mskcc_2016"</span>, <span class="kw">genePanelId</span> <span class="kw">=</span> <span class="st">"IMPACT341"</span>
    )
)</pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 2 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 2:
##  [1] urcc_mskcc_2016_cna: SummarizedExperiment with 213 rows and 62 columns
##  [2] urcc_mskcc_2016_mutations: RangedSummarizedExperiment with 147 rows and 53 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>
<code>curatedTCGAData</code> provides TCGA data with some curation including tumor subtype information</li>
<li>
<code>cBioPortalData</code> has two main functions, one for downloading pre-packaged data and another for sending queries through the cBioPortal API</li>
</ul>
</div>
</div>
</div>
<div id="utilities-for-tcga" class="section level2">
<h2 class="hasAnchor">
<a href="#utilities-for-tcga" class="anchor"></a>Utilities for TCGA</h2>
<p>Aside from the available reshaping functions already included in the <code>MultiAssayExperiment</code> package, the <code>TCGAutils</code> package provides additional helper functions for working with TCGA data.</p>
<p>A number of helper functions are available for managing datasets from <code>curatedTCGAData</code>. These include:</p>
<ul>
<li>Conversions of <code>SummarizedExperiment</code> to <code>RangedSummarizedExperiment</code> based on <code>TxDb.Hsapiens.UCSC.hg19.knownGene</code> for:
<ul>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">mirToRanges()</a></code>: microRNA</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">symbolsToRanges()</a></code>: gene symbols</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">qreduceTCGA()</a></code>: convert <code>RaggedExperiment</code> objects to <code>RangedSummarizedExperiment</code> with one row per gene symbol, for:
<ul>
<li>segmented copy number datasets (“CNVSNP” and “CNASNP”)</li>
<li>somatic mutation datasets (“Mutation”), with a value of 1 for any non-silent mutation and a value of 0 for no mutation or silent mutation</li>
</ul>
</li>
</ul>
</li>
</ul>
<div id="conversion-of-row-metadata-for-curatedtcgadata-objects" class="section level3">
<h3 class="hasAnchor">
<a href="#conversion-of-row-metadata-for-curatedtcgadata-objects" class="anchor"></a>(1) Conversion of row metadata for <code>curatedTCGAData</code> objects</h3>
<div id="mirtoranges" class="section level4">
<h4 class="hasAnchor">
<a href="#mirtoranges" class="anchor"></a><code>mirToRanges</code>
</h4>
<p>microRNA assays obtained from <code>curatedTCGAData</code> have annotated sequences that can be converted to genomic ranges using the <code>mirbase.db</code> package. The function looks up all sequences and converts them to (‘hg19’) ranges. For those rows that cannot be found, an ‘unranged’ assay is introduced in the resulting MultiAssayExperiment object.</p>
<div class="sourceCode" id="cb42"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">mirToRanges</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 6 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 6:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [3] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [5] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns
##  [6] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="qreducetcga" class="section level4">
<h4 class="hasAnchor">
<a href="#qreducetcga" class="anchor"></a><code>qreduceTCGA</code>
</h4>
<p>The <code>qreduceTCGA</code> function converts <code>RaggedExperiment</code> mutation data objects to <code>RangedSummarizedExperiment</code> using <code>org.Hs.eg.db</code> and the <code>qreduceTCGA</code> utility function from <code>RaggedExperiment</code> to summarize ‘silent’ and ‘non-silent’ mutations based on a ‘Variant_Classification’ metadata column in the original object.</p>
<div class="sourceCode" id="cb45"><html><body><pre class="r"><span class="co">## Update build metadata to "hg19"</span>
<span class="fu">genome</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]]) <span class="kw">&lt;-</span> <span class="st">"NCBI37"</span>
<span class="fu">seqlevelsStyle</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]]) <span class="kw">&lt;-</span> <span class="st">"UCSC"</span>

<span class="no">gnome</span> <span class="kw">&lt;-</span> <span class="fu">genome</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]])
<span class="no">gnome</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/builds.html">translateBuild</a></span>(<span class="no">gnome</span>)
<span class="fu">genome</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]]) <span class="kw">&lt;-</span> <span class="no">gnome</span>

<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">qreduceTCGA</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## </code></pre>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## Warning in .normarg_seqlevelsStyle(value): more than one seqlevels style
## supplied, using the 1st one only</code></pre>
<pre><code>## harmonizing input:
##   removing 270 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [2] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [3] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [4] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 90 columns
##  [5] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 180 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="symbolstoranges" class="section level4">
<h4 class="hasAnchor">
<a href="#symbolstoranges" class="anchor"></a><code>symbolsToRanges</code>
</h4>
<p>In the cases where row annotations indicate gene symbols, the <code>symbolsToRanges</code> utility function converts genes to genomic ranges and replaces existing assays with <code>RangedSummarizedExperiment</code> objects. Gene annotations are given as ‘hg19’ genomic regions.</p>
<div class="sourceCode" id="cb52"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">symbolsToRanges</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## harmonizing input:
##   removing 79 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 6 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 6:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [5] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 17296 rows and 79 columns
##  [6] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3205 rows and 79 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="simplifytcga" class="section level4">
<h4 class="hasAnchor">
<a href="#simplifytcga" class="anchor"></a><code>simplifyTCGA</code>
</h4>
<p>The <code>simplifyTCGA</code> function combines all of the above operations to create a more managable <code>MultiAssayExperiment</code> object and using <code>RangedSummarizedExperiment</code> assays where possible.</p>
<div class="sourceCode" id="cb57"><html><body><pre class="r"><span class="kw pkg">TCGAutils</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">simplifyTCGA</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## Warning in .normarg_seqlevelsStyle(value): more than one seqlevels style
## supplied, using the 1st one only</code></pre>
<pre><code>## harmonizing input:
##   removing 270 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## harmonizing input:
##   removing 79 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 7 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 7:
##  [1] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [2] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 90 columns
##  [3] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 180 columns
##  [4] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns
##  [5] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns
##  [6] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 17296 rows and 79 columns
##  [7] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3205 rows and 79 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
</div>
<div id="identification-and-separation-of-samples" class="section level3">
<h3 class="hasAnchor">
<a href="#identification-and-separation-of-samples" class="anchor"></a>(2) Identification and Separation of Samples</h3>
</div>
<div id="what-types-of-samples-are-in-the-data" class="section level3">
<h3 class="hasAnchor">
<a href="#what-types-of-samples-are-in-the-data" class="anchor"></a>What types of samples are in the data?</h3>
<p><strong>Solution</strong></p>
<p>The <code>sampleTables</code> function gives you an overview of samples in each assay:</p>
<div class="sourceCode" id="cb67"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">sampleTables</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## $`ACC_CNVSNP-20160128`
## 
## 01 10 11 
## 90 85  5 
## 
## $`ACC_miRNASeqGene-20160128`
## 
## 01 
## 80 
## 
## $`ACC_Mutation-20160128`
## 
## 01 
## 90 
## 
## $`ACC_RNASeq2GeneNorm-20160128`
## 
## 01 
## 79 
## 
## $`ACC_RPPAArray-20160128`
## 
## 01 
## 46</code></pre>
<p>Interpretation of sample codes:</p>
<div class="sourceCode" id="cb69"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"sampleTypes"</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">sampleTypes</span>)</pre></body></html></div>
<pre><code>##   Code                                      Definition Short.Letter.Code
## 1   01                             Primary Solid Tumor                TP
## 2   02                           Recurrent Solid Tumor                TR
## 3   03 Primary Blood Derived Cancer - Peripheral Blood                TB
## 4   04    Recurrent Blood Derived Cancer - Bone Marrow              TRBM
## 5   05                        Additional - New Primary               TAP
## 6   06                                      Metastatic                TM</code></pre>
</div>
<div id="splitassays-separate-the-data-from-different-tissue-types" class="section level3">
<h3 class="hasAnchor">
<a href="#splitassays-separate-the-data-from-different-tissue-types" class="anchor"></a><code>splitAssays</code>: separate the data from different tissue types</h3>
<p>TCGA datasets include multiple -omics for solid tumors, adjacent normal tissues, blood-derived cancers and normals, and other tissue types, which may be mixed together in a single dataset. The <code>MultiAssayExperiment</code> object generated here has one patient per row of its <code>colData</code>, but each patient may have two or more -omics profiles by any assay, whether due to assaying of different types of tissues or to technical replication. <code>splitAssays</code> separates profiles from different tissue types (such as tumor and adjacent normal) into different assays of the <code>MultiAssayExperiment</code> by taking a vector of sample codes, and partitioning the current assays into assays with an appended sample code:</p>
<div class="sourceCode" id="cb71"><html><body><pre class="r"><span class="no">split_acc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">splitAssays</a></span>(<span class="no">acc</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"01"</span>, <span class="st">"11"</span>))</pre></body></html></div>
<pre><code>## Warning: Some 'sampleCodes' not found in assays</code></pre>
<pre><code>## Warning in .checkBarcodes(barcodes): Inconsistent barcode lengths: 28, 27</code></pre>
<p>Only about 43 participants have data across all experiments.</p>
</div>
<div id="curated-molecular-subtypes" class="section level3">
<h3 class="hasAnchor">
<a href="#curated-molecular-subtypes" class="anchor"></a>Curated molecular subtypes</h3>
<p>Is there subtype data available in the <code>MultiAssayExperiment</code> obtained from <code>curatedTCGAData</code>?</p>
<p><strong>Solution</strong></p>
<p>The <code>getSubtypeMap</code> function will show actual variable names found in <code>colData</code> that contain subtype information. This can only be obtained from <code>MultiAssayExperiment</code> objects provided by <code>curatedTCGAData</code>.</p>
<div class="sourceCode" id="cb74"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">getSubtypeMap</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##          ACC_annotations     ACC_subtype
## 1             Patient_ID       patientID
## 2  histological_subtypes       Histology
## 3          mrna_subtypes         C1A/C1B
## 4          mrna_subtypes         mRNA_K4
## 5                   cimp      MethyLevel
## 6      microrna_subtypes   miRNA cluster
## 7          scna_subtypes    SCNA cluster
## 8       protein_subtypes protein cluster
## 9   integrative_subtypes             COC
## 10     mutation_subtypes        OncoSign</code></pre>
<div class="sourceCode" id="cb76"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu">colData</span>(<span class="no">acc</span>)$<span class="no">Histology</span>)</pre></body></html></div>
<pre><code>## [1] "Usual Type" "Usual Type" "Usual Type" "Usual Type" "Usual Type"
## [6] "Usual Type"</code></pre>
</div>
<div id="translation-and-interpretation-of-tcga-identifiers" class="section level3">
<h3 class="hasAnchor">
<a href="#translation-and-interpretation-of-tcga-identifiers" class="anchor"></a>(3) Translation and Interpretation of TCGA identifiers</h3>
<p><code>TCGAutils</code> provides a number of ID translation functions. These allow the user to translate from either file or case UUIDs to TCGA barcodes and back. These functions work by querying the Genomic Data Commons API via the <code>GenomicDataCommons</code> package (thanks to Sean Davis). These include:</p>
<div id="uuidtobarcode---uuid-to-tcga-barcode" class="section level4">
<h4 class="hasAnchor">
<a href="#uuidtobarcode---uuid-to-tcga-barcode" class="anchor"></a><code>UUIDtoBarcode()</code> - UUID to TCGA barcode</h4>
<p>Here we have a known case UUID that we want to translate into a TCGA barcode.</p>
<div class="sourceCode" id="cb78"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoBarcode</a></span>(<span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, <span class="kw">from_type</span> <span class="kw">=</span> <span class="st">"case_id"</span>)</pre></body></html></div>
<pre><code>##                                case_id submitter_id
## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 TCGA-B0-5117</code></pre>
<p>In cases where we want to translate a known file UUID to the associated TCGA patient barcode, we can use <code>UUIDtoBarcode</code>.</p>
<div class="sourceCode" id="cb80"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoBarcode</a></span>(<span class="st">"0001801b-54b0-4551-8d7a-d66fb59429bf"</span>, <span class="kw">from_type</span> <span class="kw">=</span> <span class="st">"file_id"</span>)</pre></body></html></div>
<pre><code>##                                file_id associated_entities.entity_submitter_id
## 1 0001801b-54b0-4551-8d7a-d66fb59429bf            TCGA-B0-5094-11A-01D-1421-08</code></pre>
</div>
<div id="barcodetouuid---tcga-barcode-to-uuid" class="section level4">
<h4 class="hasAnchor">
<a href="#barcodetouuid---tcga-barcode-to-uuid" class="anchor"></a><code>barcodeToUUID()</code> - TCGA barcode to UUID</h4>
<p>Here we translate the first two TCGA barcodes of the previous copy-number alterations dataset to UUID:</p>
<div class="sourceCode" id="cb82"><html><body><pre class="r">(<span class="no">xbarcode</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="no">acc</span>)<span class="kw">[[</span><span class="st">"ACC_CNVSNP-20160128"</span>]], <span class="fl">4L</span>))</pre></body></html></div>
<pre><code>## [1] "TCGA-OR-A5J1-01A-11D-A29H-01" "TCGA-OR-A5J1-10A-01D-A29K-01"
## [3] "TCGA-OR-A5J2-01A-11D-A29H-01" "TCGA-OR-A5J2-10A-01D-A29K-01"</code></pre>
<div class="sourceCode" id="cb84"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">barcodeToUUID</a></span>(<span class="no">xbarcode</span>)</pre></body></html></div>
<pre><code>##           submitter_aliquot_ids                          aliquot_ids
## 6  TCGA-OR-A5J1-01A-11D-A29H-01 1387b6c7-48fe-4961-86a7-0bdcbd3fef92
## 9  TCGA-OR-A5J1-10A-01D-A29K-01 cb537629-6a01-4d67-84ea-dbf130bd59c7
## 21 TCGA-OR-A5J2-01A-11D-A29H-01 6f0290b0-4cb4-4f72-853e-9ac363bd2c3b
## 13 TCGA-OR-A5J2-10A-01D-A29K-01 4bf2e4ac-399f-4a00-854b-8e23b561bb4d</code></pre>
</div>
<div id="uuidtouuid---file-and-case-ids" class="section level4">
<h4 class="hasAnchor">
<a href="#uuidtouuid---file-and-case-ids" class="anchor"></a><code>UUIDtoUUID()</code> - file and case IDs</h4>
<p>We can also translate from file UUIDs to case UUIDs and vice versa as long as we know the input type. We can use the case UUID from the previous example to get the associated file UUIDs using <code>UUIDtoUUID</code>. Note that this translation is a one to many relationship, thus yielding a <code>data.frame</code> of file UUIDs for a single case UUID.</p>
<div class="sourceCode" id="cb86"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoUUID</a></span>(<span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, <span class="kw">to_type</span> <span class="kw">=</span> <span class="st">"file_id"</span>))</pre></body></html></div>
<pre><code>##                                case_id                        files.file_id
## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 48c342b0-e7a2-4a7b-8556-55bcd8ad9ea0
## 2 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 db8ba5d3-76be-4a67-a575-803ba483b6f9
## 3 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 f580489b-55ea-43c5-9489-b54c13146992
## 4 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 bf72ffef-d8c4-423d-9c5a-7bb5c23b2f31
## 5 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 b36f4e88-89ca-40bf-b543-d0e3c08ad342
## 6 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 4c3a899f-be0f-454f-b5dc-e30e29314c49</code></pre>
<p>One possible way to verify that file IDs are matching case UUIDS is to browse to the Genomic Data Commons webpage with the specific file UUID. Here we look at the first file UUID entry in the output <code>data.frame</code>:</p>
<p><a href="https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6" class="uri">https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6</a></p>
<p>In the page we check that the case UUID matches the input.</p>
</div>
<div id="filenametobarcode---using-file-names-as-input" class="section level4">
<h4 class="hasAnchor">
<a href="#filenametobarcode---using-file-names-as-input" class="anchor"></a><code>filenameToBarcode()</code> - Using file names as input</h4>
<div class="sourceCode" id="cb88"><html><body><pre class="r"><span class="no">fquery</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/query.html">files</a></span>() <span class="kw">%&gt;%</span>
    <span class="kw pkg">GenomicDataCommons</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/filtering.html">filter</a></span>(~ <span class="no">cases.project.project_id</span> <span class="kw">==</span> <span class="st">"TCGA-ACC"</span> <span class="kw">&amp;</span>
        <span class="no">data_category</span> <span class="kw">==</span> <span class="st">"Copy Number Variation"</span> <span class="kw">&amp;</span>
        <span class="no">data_type</span> <span class="kw">==</span> <span class="st">"Copy Number Segment"</span>)

<span class="no">fnames</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/results.html">results</a></span>(<span class="no">fquery</span>)$<span class="no">file_name</span>)

<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">filenameToBarcode</a></span>(<span class="no">fnames</span>)</pre></body></html></div>
<pre><code>##                                                                 file_name
## 1 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_D03_1348390.grch38.seg.v2.txt
## 2 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_B08_1348416.grch38.seg.v2.txt
## 3 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_D05_1348384.grch38.seg.v2.txt
## 4 BLAIN_p_TCGA_282_304_b2_N_GenomeWideSNP_6_E07_1348450.grch38.seg.v2.txt
## 5 BLAIN_p_TCGA_282_304_b2_N_GenomeWideSNP_6_F10_1348518.grch38.seg.v2.txt
## 6 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_F09_1348288.grch38.seg.v2.txt
##                                file_id        aliquots.submitter_id
## 1 c7c23df4-d1a8-47c1-9394-1948114dc756 TCGA-OR-A5KO-10A-01D-A29K-01
## 2 81754b77-e4b0-4123-baba-c2a8fbfaa487 TCGA-OR-A5KW-10A-01D-A29K-01
## 3 81380fa8-f63d-4b5d-9bb9-abab48ba3f1b TCGA-OR-A5LA-10A-01D-A29K-01
## 4 4b09eac7-8b71-4ff5-93d1-97dfcc342132 TCGA-OR-A5J9-01A-11D-A29H-01
## 5 e46a1773-92f0-4ed1-916d-d3493ba6bbe4 TCGA-OR-A5JZ-01A-11D-A29H-01
## 6 fb4acb71-b9c6-47ff-a585-5ddeff3d6039 TCGA-PK-A5HB-01A-11D-A29H-01</code></pre>
<p>See the <code>TCGAutils</code> vignette page for more details.</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>
<code>TCGAutils</code> provides users additional tools for modifying row and column metadata</li>
<li>The package works mainly with TCGA data including barcode identifiers</li>
</ul>
</div>
</div>
</div>
</div>
<div id="multiassayexperiment-subsetting" class="section level2">
<h2 class="hasAnchor">
<a href="#multiassayexperiment-subsetting" class="anchor"></a>MultiAssayExperiment Subsetting</h2>
<div id="single-bracket" class="section level3">
<h3 class="hasAnchor">
<a href="#single-bracket" class="anchor"></a>Single bracket <code>[</code>
</h3>
<p>In pseudo code below, the subsetting operations work on the rows of the following indices: 1. <em>i</em> experimental data rows 2. <em>j</em> the primary names or the column names (entered as a <code>list</code> or <code>List</code>) 3. <em>k</em> assay</p>
<pre><code>multiassayexperiment[i = rownames, j = primary or colnames, k = assay]</code></pre>
<p>Subsetting operations always return another <code>MultiAssayExperiment</code>. For example, the following will return any rows named “MAPK14” or “IGFBP2”, and remove any assays where no rows match:</p>
<div class="sourceCode" id="cb91"><html><body><pre class="r"><span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"MAPK14"</span>, <span class="st">"IGFBP2"</span>), , ]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 170 sampleMap rows not in names(experiments)</code></pre>
<p>The following will keep only patients of pathological stage iv, and all their associated assays:</p>
<div class="sourceCode" id="cb93"><html><body><pre class="r"><span class="no">miniACC</span>[, <span class="no">miniACC</span>$<span class="no">pathologic_stage</span> <span class="kw">==</span> <span class="st">"stage iv"</span>, ]</pre></body></html></div>
<p>And the following will keep only the RNA-seq dataset, and only patients for which this assay is available:</p>
<div class="sourceCode" id="cb94"><html><body><pre class="r"><span class="no">miniACC</span>[, , <span class="st">"RNASeq2GeneNorm"</span>]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 306 sampleMap rows not in names(experiments)
##   removing 13 colData rownames not in sampleMap 'primary'</code></pre>
</div>
<div id="subsetting-by-genomic-ranges" class="section level3">
<h3 class="hasAnchor">
<a href="#subsetting-by-genomic-ranges" class="anchor"></a>Subsetting by genomic ranges</h3>
<p>If any ExperimentList objects have features represented by genomic ranges (e.g. <code>RangedSummarizedExperiment</code>, <code>RaggedExperiment</code>), then a <code>GRanges</code> object in the first subsetting position will subset these objects as in <code><a href="https://rdrr.io/pkg/GenomicRanges/man/findOverlaps-methods.html">GenomicRanges::findOverlaps()</a></code>. Any non-ranged <code>ExperimentList</code> element will be subset to zero rows.</p>
</div>
<div id="double-bracket" class="section level3">
<h3 class="hasAnchor">
<a href="#double-bracket" class="anchor"></a>Double bracket <code>[[</code>
</h3>
<p>The “double bracket” method (<code><a href="https://rdrr.io/r/base/Extract.html">[[</a></code>) is a convenience function for extracting a single element of the <code>MultiAssayExperiment</code> <code>ExperimentList</code>. It avoids the use of <code>experiments(mae)[[1L]]</code>. For example, both of the following extract the <code>ExpressionSet</code> object containing RNA-seq data:</p>
<div class="sourceCode" id="cb96"><html><body><pre class="r"><span class="no">miniACC</span><span class="kw">[[</span><span class="fl">1L</span>]]</pre></body></html></div>
<pre><code>## class: SummarizedExperiment 
## dim: 198 79 
## metadata(3): experimentData annotation protocolData
## assays(1): exprs
## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13
## rowData names(0):
## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07
##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07
## colData names(0):</code></pre>
<div class="sourceCode" id="cb98"><html><body><pre class="r"><span class="co">## equivalently</span>
<span class="no">miniACC</span><span class="kw">[[</span><span class="st">"RNASeq2GeneNorm"</span>]]</pre></body></html></div>
<pre><code>## class: SummarizedExperiment 
## dim: 198 79 
## metadata(3): experimentData annotation protocolData
## assays(1): exprs
## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13
## rowData names(0):
## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07
##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07
## colData names(0):</code></pre>
</div>
<div id="complete-cases" class="section level3">
<h3 class="hasAnchor">
<a href="#complete-cases" class="anchor"></a>Complete cases</h3>
<p><code><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases()</a></code> shows which patients have complete data for all assays:</p>
<div class="sourceCode" id="cb100"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span>(<span class="fu"><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases</a></span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>##    Mode   FALSE    TRUE 
## logical      49      43</code></pre>
<p>The above logical vector could be used for patient subsetting. More simply, <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns()</a></code> will select complete cases and rearrange each <code>ExperimentList</code> element so its columns correspond exactly to rows of <code>colData</code> in the same order:</p>
<div class="sourceCode" id="cb102"><html><body><pre class="r"><span class="no">accmatched</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<p>Note, the column names of the assays in <code>accmatched</code> are not the same because of assay-specific identifiers, but they have been automatically re-arranged to correspond to the same patients. In these TCGA assays, the first three <code><a href="https://rdrr.io/r/base/Arithmetic.html">-</a></code> delimited positions correspond to patient, ie the first patient is <em>TCGA-OR-A5J2</em>:</p>
<div class="sourceCode" id="cb103"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="no">accmatched</span>)</pre></body></html></div>
<pre><code>## CharacterList of length 5
## [["RNASeq2GeneNorm"]] TCGA-OR-A5J2-01A-11R-A29S-07 ...
## [["gistict"]] TCGA-OR-A5J2-01A-11D-A29H-01 ... TCGA-PK-A5HA-01A-11D-A29H-01
## [["RPPAArray"]] TCGA-OR-A5J2-01A-21-A39K-20 ... TCGA-PK-A5HA-01A-21-A39K-20
## [["Mutations"]] TCGA-OR-A5J2-01A-11D-A29I-10 ... TCGA-PK-A5HA-01A-11D-A29I-10
## [["miRNASeqGene"]] TCGA-OR-A5J2-01A-11R-A29W-13 ...</code></pre>
</div>
<div id="row-names-that-are-common-across-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#row-names-that-are-common-across-assays" class="anchor"></a>Row names that are common across assays</h3>
<p><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows()</a></code> keeps only rows that are common to each assay, and aligns them in identical order. For example, to keep only genes where data are available for RNA-seq, GISTIC copy number, and somatic mutations:</p>
<div class="sourceCode" id="cb105"><html><body><pre class="r"><span class="no">accmatched2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows</a></span>(<span class="no">miniACC</span>[, ,
    <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"gistict"</span>, <span class="st">"Mutations"</span>)])</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb107"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(<span class="no">accmatched2</span>)</pre></body></html></div>
<pre><code>## CharacterList of length 3
## [["RNASeq2GeneNorm"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... RET CDKN2A MACC1 CHGA
## [["gistict"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA
## [["Mutations"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA</code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="extraction" class="section level3">
<h3 class="hasAnchor">
<a href="#extraction" class="anchor"></a>Extraction</h3>
</div>
<div id="assay-and-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#assay-and-assays" class="anchor"></a>assay and assays</h3>
<p>The <code>assay</code> and <code>assays</code> methods follow <code>SummarizedExperiment</code> convention. The <code>assay</code> (singular) method will extract the first element of the <code>ExperimentList</code> and will return a <code>matrix</code>.</p>
<div class="sourceCode" id="cb109"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/class.html">class</a></span>(<span class="fu">assay</span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>## [1] "matrix" "array"</code></pre>
<p>The <code>assays</code> (plural) method will return a <code>SimpleList</code> of the data with each element being a <code>matrix</code>.</p>
<div class="sourceCode" id="cb111"><html><body><pre class="r"><span class="fu">assays</span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## List of length 5
## names(5): RNASeq2GeneNorm gistict RPPAArray Mutations miRNASeqGene</code></pre>
<p><em>Key point:</em></p>
<ul>
<li>Whereas the <code><a href="https://rdrr.io/r/base/Extract.html">[[</a></code> returned an assay as its original class, <code>assay()</code> and <code>assays()</code> convert the assay data to matrix form.</li>
</ul>
<p style="text-align: right;">
<a href="#overview-of-key-packages-and-data-classes">back to top</a>
</p>
</div>
<div id="summary-of-slots-and-accessors" class="section level3">
<h3 class="hasAnchor">
<a href="#summary-of-slots-and-accessors" class="anchor"></a>Summary of slots and accessors</h3>
<p>Slot in the <code>MultiAssayExperiment</code> can be accessed or set using their accessor functions:</p>
<table class="table">
<thead><tr class="header">
<th>Slot</th>
<th>Accessor</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><code>ExperimentList</code></td>
<td><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments()</a></code></td>
</tr>
<tr class="even">
<td><code>colData</code></td>
<td>
<code>colData()</code> and <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> *</td>
</tr>
<tr class="odd">
<td><code>sampleMap</code></td>
<td><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap()</a></code></td>
</tr>
<tr class="even">
<td><code>metadata</code></td>
<td><code>metadata()</code></td>
</tr>
</tbody>
</table>
<p>__*__ The <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> operator on a <code>MultiAssayExperiment</code> returns a single column of the <code>colData</code>.</p>
</div>
<div id="transformation-reshaping" class="section level3">
<h3 class="hasAnchor">
<a href="#transformation-reshaping" class="anchor"></a>Transformation / reshaping</h3>
<p>The <code>longFormat</code> or <code>wideFormat</code> functions will “reshape” and combine experiments with each other and with <code>colData</code> into one <code>DataFrame</code>. These functions provide compatibility with most of the common R/Bioconductor functions for regression, machine learning, and visualization.</p>
<div id="longformat" class="section level4">
<h4 class="hasAnchor">
<a href="#longformat" class="anchor"></a><code>longFormat</code>
</h4>
<p>In <em>long</em> format a single column provides all assay results, with additional optional <code>colData</code> columns whose values are repeated as necessary. Here <em>assay</em> is the name of the ExperimentList element, <em>primary</em> is the patient identifier (rowname of colData), <em>rowname</em> is the assay rowname (in this case genes), <em>colname</em> is the assay-specific identifier (column name), <em>value</em> is the numeric measurement (gene expression, copy number, presence of a non-silent mutation, etc), and following these are the <em>vital_status</em> and <em>days_to_death</em> colData columns that have been added:</p>
<div class="sourceCode" id="cb113"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">longFormat</a></span>(<span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span>), , ],
    <span class="kw">colDataCols</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## DataFrame with 518 rows and 7 columns
##               assay      primary     rowname                colname     value
##         &lt;character&gt;  &lt;character&gt; &lt;character&gt;            &lt;character&gt; &lt;numeric&gt;
## 1   RNASeq2GeneNorm TCGA-OR-A5J1        TP53 TCGA-OR-A5J1-01A-11R..   563.401
## 2   RNASeq2GeneNorm TCGA-OR-A5J1      CTNNB1 TCGA-OR-A5J1-01A-11R..  5634.467
## 3   RNASeq2GeneNorm TCGA-OR-A5J2        TP53 TCGA-OR-A5J2-01A-11R..   165.481
## 4   RNASeq2GeneNorm TCGA-OR-A5J2      CTNNB1 TCGA-OR-A5J2-01A-11R.. 62658.391
## 5   RNASeq2GeneNorm TCGA-OR-A5J3        TP53 TCGA-OR-A5J3-01A-11R..   956.303
## ...             ...          ...         ...                    ...       ...
## 514       Mutations TCGA-PK-A5HA      CTNNB1 TCGA-PK-A5HA-01A-11D..         0
## 515       Mutations TCGA-PK-A5HB        TP53 TCGA-PK-A5HB-01A-11D..         0
## 516       Mutations TCGA-PK-A5HB      CTNNB1 TCGA-PK-A5HB-01A-11D..         0
## 517       Mutations TCGA-PK-A5HC        TP53 TCGA-PK-A5HC-01A-11D..         0
## 518       Mutations TCGA-PK-A5HC      CTNNB1 TCGA-PK-A5HC-01A-11D..         0
##     vital_status days_to_death
##        &lt;integer&gt;     &lt;integer&gt;
## 1              1          1355
## 2              1          1355
## 3              1          1677
## 4              1          1677
## 5              0            NA
## ...          ...           ...
## 514            0            NA
## 515            0            NA
## 516            0            NA
## 517            0            NA
## 518            0            NA</code></pre>
</div>
<div id="wideformat" class="section level4">
<h4 class="hasAnchor">
<a href="#wideformat" class="anchor"></a><code>wideFormat</code>
</h4>
<p>In <em>wide</em> format, each feature from each assay goes in a separate column, with one row per primary identifier (patient). Here, each variable becomes a new column:</p>
<div class="sourceCode" id="cb116"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span>), , ],
    <span class="kw">colDataCols</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## DataFrame with 92 rows and 9 columns
##          primary vital_status days_to_death RNASeq2GeneNorm_CTNNB1
##      &lt;character&gt;    &lt;integer&gt;     &lt;integer&gt;              &lt;numeric&gt;
## 1   TCGA-OR-A5J1            1          1355                5634.47
## 2   TCGA-OR-A5J2            1          1677               62658.39
## 3   TCGA-OR-A5J3            0            NA                6337.43
## 4   TCGA-OR-A5J4            1           423                     NA
## 5   TCGA-OR-A5J5            1           365                5979.06
## ...          ...          ...           ...                    ...
## 88  TCGA-PK-A5H9            0            NA                5258.99
## 89  TCGA-PK-A5HA            0            NA                8120.17
## 90  TCGA-PK-A5HB            0            NA                5257.81
## 91  TCGA-PK-A5HC            0            NA                     NA
## 92  TCGA-P6-A5OG            1           383                6390.10
##     RNASeq2GeneNorm_TP53 gistict_CTNNB1 gistict_TP53 Mutations_CTNNB1
##                &lt;numeric&gt;      &lt;numeric&gt;    &lt;numeric&gt;        &lt;numeric&gt;
## 1                563.401              0            0                0
## 2                165.481              1            0                1
## 3                956.303              0            0                0
## 4                     NA              0            1                0
## 5               1169.636              0            0                0
## ...                  ...            ...          ...              ...
## 88               890.866              0            0                0
## 89               683.572              0           -1                0
## 90               237.370             -1           -1                0
## 91                    NA              1            1                0
## 92               815.345              1           -1               NA
##     Mutations_TP53
##          &lt;numeric&gt;
## 1                0
## 2                1
## 3                0
## 4                0
## 5                0
## ...            ...
## 88               0
## 89               0
## 90               0
## 91               0
## 92              NA</code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-packages-and-data-classes">back to top</a>
</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>Knowing how to subset a <code>MultiAssayExperiment</code> object is important to be able to restrict observations and measurements to particular phenotypes or sample types</li>
<li>Functions such as <code>longFormat</code> and <code>wideFormat</code> are helpful for downstream analysis functions that require a certain type of input format</li>
</ul>
</div>
</div>
</div>
<div id="bonus-visualization-of-sample-sets" class="section level3">
<h3 class="hasAnchor">
<a href="#bonus-visualization-of-sample-sets" class="anchor"></a>Bonus: Visualization of Sample Sets</h3>
<p>Here we use the built-in visualization functionality to what extent the samples in each assay overlap:</p>
<div class="sourceCode" id="cb119"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/upsetSamples.html">upsetSamples</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-49-1.png" width="100%"></p>
</div>
</div>
<div id="references" class="section level2 unnumbered">
<h2 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h2>
<div id="refs" class="references">
<div id="ref-Ramos2017-er">
<p>Ramos, Marcel, Lucas Schiffer, Angela Re, Rimsha Azhar, Azfar Basunia, Carmen Rodriguez, Tiffany Chan, et al. 2017. “Software for the Integration of Multiomics Experiments in Bioconductor.” <em>Cancer Res.</em> 77 (21): e39–e42.</p>
</div>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Marcel Ramos, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
