<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MultiAssayExperiment: Integrative workflow for public multi-omics data • MultiAssayWorkshop</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="MultiAssayExperiment: Integrative workflow for public multi-omics data">
<meta property="og:description" content="MultiAssayWorkshop">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MultiAssayWorkshop</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">3.18.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/Ramos_MultiAssayExperiment.html">Workshop</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Tutorial
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../tutorials/Exercise_MultiAssay.html">Build Your First MAE</a>
    </li>
    <li>
      <a href="../tutorials/cBioPortalData_Intro.html">Download and Use cBioPortal Data</a>
    </li>
    <li>
      <a href="../articles/SingleCellMultiModal.html">SingleCellMultiModal Intro</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Reference
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/cBioPortal_Studies.html">cBioPortal Studies</a>
    </li>
    <li>
      <a href="../articles/curatedTCGAData_ref.html">curatedTCGAData Reference</a>
    </li>
    <li>
      <a href="../articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MultiAssayWorkshop" class="external-link">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MultiAssayExperiment: Integrative workflow for
public multi-omics data</h1>
                        <h4 data-toc-skip class="author">Marcel
Ramos</h4>
            <address class="author_afil">
      Roswell Park Comprehensive Cancer Center, Buffalo,
NY<br><h4 data-toc-skip class="author">Ludwig
Geistlinger</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York,
NY<br><h4 data-toc-skip class="author">Levi
Waldron</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 data-toc-skip class="date">December 15, 2023</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/MultiAssayWorkshop/blob/HEAD/vignettes/Ramos_MultiAssayExperiment.Rmd" class="external-link"><code>vignettes/Ramos_MultiAssayExperiment.Rmd</code></a></small>
      <div class="hidden name"><code>Ramos_MultiAssayExperiment.Rmd</code></div>

    </address>
</address>
</address>
</div>

    
    
<style type="text/css">
.infobox {
  padding: 1em 1em 1em 4em;
  margin-bottom: 10px;
  border: 2px solid #008000;
  border-radius: 10px;
  background: #DCDCDC 5px center/3em no-repeat;
  color: #8B0000;
}

.center {
  text-align: center;
}
</style>
<div class="section level2">
<h2 id="multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment">Multi-omic Integration and Analysis of cBioPortal and TCGA data with
MultiAssayExperiment<a class="anchor" aria-label="anchor" href="#multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment"></a>
</h2>
<div class="section level3">
<h3 id="workshop-participation">Workshop participation<a class="anchor" aria-label="anchor" href="#workshop-participation"></a>
</h3>
<ul>
<li>See the instruction <a href="https://tinyurl.com/MultiAssayWorkshop" class="external-link">Google slides</a>
</li>
<li>Browse to the <a href="https://waldronlab.io/MultiAssayWorkshop/">workshop website</a>.
This page includes:
<ul>
<li><a href="https://waldronlab.io/MultiAssayWorkshop/articles/Ramos_MultiAssayExperiment.html">A
built version of this workshop</a></li>
<li><a href="https://waldronlab.io/MultiAssayWorkshop/articles/cBioPortal_Studies.html">List
of available cBioPortal studies</a></li>
<li><a href="https://waldronlab.io/MultiAssayWorkshop/articles/curatedTCGAData_ref.html">TCGA
terminology quick-reference</a></li>
<li><a href="https://waldronlab.io/MultiAssayWorkshop/articles/TCGAutilsCheatsheet.html">TCGAutils
package cheat-sheet</a></li>
<li><a href="https://waldronlab.io/MultiAssayWorkshop/tutorials/Exercise_MultiAssay.html">Build
your first MultiAssayExperiment interactive tutorial</a></li>
<li><a href="https://waldronlab.io/MultiAssayWorkshop/tutorials/cBioPortalData_Intro.html">Download
and use cBioPortal data interactive tutorial</a></li>
</ul>
</li>
<li>Run this workshop on the <a href="http://app.orchestra.cancerdatasci.org/" class="external-link">free Orchestra Cloud
platform</a> (search for title: “Multi-omic Integration of cBioPortal
and TCGA data with MultiAssayExperiment”)</li>
<li>Docker users can run this workshop locally via:</li>
</ul>
<pre><code>docker run -e PASSWORD=bioc -p 8787:8787 mr148/multiassayworkshop:latest</code></pre>
</div>
<div class="section level3">
<h3 id="requirements-rbioconductor-packages">Requirements: R/Bioconductor packages<a class="anchor" aria-label="anchor" href="#requirements-rbioconductor-packages"></a>
</h3>
<p>The workshop uses a Docker container with Bioconductor version 3.18.
If you would like to install Bioconductor on your computer <strong>at a
later date</strong>, see the <a href="https://www.bioconductor.org/install" class="external-link">Bioconductor
installation</a> instructions.</p>
<p>Here is a list of packages that we will be using:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://waldronlab.io/MultiAssayExperiment/" class="external-link">MultiAssayExperiment</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">RaggedExperiment</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">curatedTCGAData</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">cBioPortalData</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">TCGAutils</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">SingleCellMultiModal</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://github.com/hms-dbmi/UpSetR" class="external-link">UpSetR</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://bioconductor.org/packages/GenomicDataCommons" class="external-link">GenomicDataCommons</a></span><span class="op">)</span></span></code></pre></div>
<div class="section level4">
<h4 id="citing-multiassayexperiment">Citing MultiAssayExperiment<a class="anchor" aria-label="anchor" href="#citing-multiassayexperiment"></a>
</h4>
<p>Please use these citations <span class="citation">(Ramos et al.
2017)</span>, <span class="citation">(Ramos et al. 2020)</span>, and
<span class="citation">(Ramos et al. 2023)</span> when using
<code>MultiAssayExperiment</code>, <code>curatedTCGAData</code>,
<code>cBioPortalData</code>, or <code>RaggedExperiment</code>. Your
citations are very much appreciated!</p>
</div>
<div class="section level4">
<h4 id="overview-of-packages">Overview of Packages<a class="anchor" aria-label="anchor" href="#overview-of-packages"></a>
</h4>
<table class="table">
<colgroup>
<col width="29%">
<col width="70%">
</colgroup>
<thead><tr class="header">
<th>Package</th>
<th>Description</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>MultiAssayExperiment</td>
<td>Infrastructure package to represent multi-omics data</td>
</tr>
<tr class="even">
<td>curatedTCGAData</td>
<td>Downloads TCGA data from ExperimentHub in MultiAssayExperiment
form</td>
</tr>
<tr class="odd">
<td>terraTCGAdata</td>
<td>Obtains TCGA data within Terra cloud service as
MultiAssayExperiment</td>
</tr>
<tr class="even">
<td>cBioPortalData</td>
<td>Access over 300 study datasets from the cBio Genomics Portal</td>
</tr>
<tr class="odd">
<td>TCGAutils</td>
<td>Make use of utility functions for working with TCGA data</td>
</tr>
<tr class="even">
<td>SingleCellMultiModal</td>
<td>Obtain single cell data from various multi-modality studies</td>
</tr>
<tr class="odd">
<td>TENxIO</td>
<td>Import 10X files as Bioconductor data classes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="key-packages">Key Packages<a class="anchor" aria-label="anchor" href="#key-packages"></a>
</h4>
<div class="section level5">
<h5 id="multiassayexperiment">
<code>MultiAssayExperiment</code><a class="anchor" aria-label="anchor" href="#multiassayexperiment"></a>
</h5>
<ul>
<li>provides an integrative representation for multi-omics data</li>
<li>modelled after the <code>SummarizedExperiment</code> representation
for expression data</li>
<li>easy-to-use operations for manipulating multiple sets of data such
as copy number alterations, mutations, proteomics, methylation, and
more</li>
</ul>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic" width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic
</p>
</div>
</div>
<div class="section level5">
<h5 id="cbioportaldata">
<code>cBioPortalData</code><a class="anchor" aria-label="anchor" href="#cbioportaldata"></a>
</h5>
<ul>
<li>R/Bioconductor interface to cBioPortal data</li>
<li>makes use of the revamped API with caching</li>
<li>queries are handled for the user <em>in the background</em>
</li>
<li>easy-to-use interface (no knowledge of the cBioPortal data model
required)</li>
<li>see <code><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html" class="external-link">getStudies()</a></code> for a list of available studies</li>
</ul>
</div>
<div class="section level5">
<h5 id="curatedtcgadata">
<code>curatedTCGAData</code><a class="anchor" aria-label="anchor" href="#curatedtcgadata"></a>
</h5>
<ul>
<li>Many tools exist for accessing and downloading <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga" class="external-link">The
Cancer Genome Atlas</a> data: <code>RTCGAToolbox</code>,
<code>GenomicDataCommons</code>, <code>TCGAbiolinks</code>,
<code>cBioPortal</code> website, Broad GDAC Firehose, and more</li>
<li>makes it easy to obtain user-friendly and integrative data at very
little cognitive overhead</li>
<li>conveniently places data in the analysis platform of choice,
R/Bioconductor</li>
<li>provides 33 different cancer types from the <a href="https://gdac.broadinstitute.org" class="external-link">Broad GDAC Firehose</a>
<ul>
<li>On-the-fly construction from ‘flat’ files</li>
<li>
<code>hg19</code> data</li>
<li>
<code>MultiAssayExperiment</code> representations</li>
</ul>
</li>
</ul>
<p><a href="articles/curatedTCGAData_ref.html">Reference
vignettes</a>:</p>
<ul>
<li><p><code>Available Studies</code> – (curatedTCGAData section) A list
of available cancer studies from
<code><a href="https://rdrr.io/pkg/TCGAutils/man/diseaseCodes.html" class="external-link">TCGAutils::diseaseCodes</a></code>.</p></li>
<li><p><code>OmicsTypes</code> – A descriptive table of ’omics types in
<code>curatedTCGAData</code> (thanks to Ludwig G.
<code>@lgeistlinger</code>)</p></li>
</ul>
</div>
<div class="section level5">
<h5 id="terratcgadata">
<code>terraTCGAdata</code><a class="anchor" aria-label="anchor" href="#terratcgadata"></a>
</h5>
<ul>
<li>Tool for making use of TCGA data within the Terra platform.</li>
<li>Go to <a href="https://app.terra.bio" class="external-link uri">https://app.terra.bio</a> and sign in with your Google
account (credit card required)</li>
<li>Select the latest RStudio / Bioconductor instance and install the
package.</li>
<li>Make sure to authenticate Google Cloud and check
<code><a href="https://rdrr.io/pkg/AnVIL/man/gcloud.html" class="external-link">AnVIL::gcloud_exists()</a></code>
</li>
<li>
<code>findTCGAworkspaces</code> to list all the available workspaces
with TCGA data</li>
<li>
<code>terraTCGAworkspace</code> to set the TCGA dataset to
download</li>
<li>
<code>terraTCGAdata</code> - main function to download data</li>
</ul>
</div>
<div class="section level5">
<h5 id="singlecellmultimodal">
<code>SingleCellMultiModal</code><a class="anchor" aria-label="anchor" href="#singlecellmultimodal"></a>
</h5>
<ul>
<li>Serves multi-modal datasets from GEO and other sources as
<code>MultiAssayExperiment</code> data representations. Some
representations are out of memory using the <code>HDF5</code> format as
well as the <code>MTX</code> format.</li>
<li>Available technologies are scNMT, 10X Multiome, seqFISH,
(EC)CITEseq, SCoPE2 and others.</li>
</ul>
</div>
<div class="section level5">
<h5 id="tenxio">
<code>TENxIO</code><a class="anchor" aria-label="anchor" href="#tenxio"></a>
</h5>
<ul>
<li>Allows import of 10X files as Bioconductor objects, e.g.,
<code>TENxH5</code> will produce a <code>SingleCellExperiment</code>
from an <code>.h5</code> file.</li>
<li>Available for a number of formats such as <code>MTX</code>,
<code>TSV</code>, peaks, fragments, and <code>.tar.gz</code> files.</li>
</ul>
</div>
<div class="section level5">
<h5 id="tcgautils">
<code>TCGAutils</code><a class="anchor" aria-label="anchor" href="#tcgautils"></a>
</h5>
<ul>
<li>allows additional exploration, and manipulation of samples and
metadata</li>
<li>User-friendly operations for subsetting, separating, converting, and
reshaping of sample and feature TCGA data</li>
<li>developed specifically for TCGA data and
<code>curatedTCGAData</code> outputs</li>
</ul>
<p>It provides convenience / helper functions in three major areas:</p>
<ol style="list-style-type: decimal">
<li>conversion / summarization of row annotations to genomic ranges</li>
<li>identification and separation of samples</li>
<li>translation and interpretation of TCGA identifiers</li>
</ol>
<p>For the cheatsheet reference table, see the <a href="articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>.</p>
<p>To better understand how it all fits together, this schematic shows
the relationship among all as part of the curatedTCGAData pipeline.</p>
<div class="figure">
<img src="TCGAMAEPipeline20190204.png" alt="Schematic of curatedTCGAData Pipeline" width="100%"><p class="caption">
Schematic of curatedTCGAData Pipeline
</p>
</div>
</div>
</div>
<div class="section level4">
<h4 id="data-classes">Data Classes<a class="anchor" aria-label="anchor" href="#data-classes"></a>
</h4>
<p>This section summarizes three fundamental data classes for the
representation of multi-omics experiments.</p>
<div class="section level5">
<h5 id="rangedsummarizedexperiment">
<code>(Ranged)SummarizedExperiment</code><a class="anchor" aria-label="anchor" href="#rangedsummarizedexperiment"></a>
</h5>
<div class="figure">
<img src="SummarizedExperiment.png" alt="A matrix-like container where rows represent features of interest and columns represent samples. The objects contain one or more assays, each represented by a matrix-like object of numeric or other mode." width="100%"><p class="caption">
A matrix-like container where rows represent features of interest and
columns represent samples. The objects contain one or more assays, each
represented by a matrix-like object of numeric or other mode.
</p>
</div>
<ul>
<li>matrix-like representation of experimental data including RNA
sequencing and microarray experiments.</li>
<li>stores multiple experimental data matrices of identical dimensions,
with associated metadata on:
<ul>
<li>the rows/genes/transcripts/other measurements
(<code>rowData</code>)</li>
<li>column/sample phenotype or clinical data (<code>colData</code>)</li>
<li>overall experiment (<code>metadata</code>).</li>
</ul>
</li>
<li>
<code>RangedSummarizedExperiment</code> associates a
<code>GRanges</code> or <code>GRangesList</code> vector with the
rows</li>
</ul>
<p><em>Note</em>. Many other classes for experimental data are actually
derived from <code>SummarizedExperiment</code> (e.g.,
<code>SingleCellExperiment</code> for single-cell RNA sequencing
experiments)</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">SingleCellExperiment</span><span class="op">)</span></span>
<span><span class="fu">extends</span><span class="op">(</span><span class="st">"SingleCellExperiment"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "SingleCellExperiment"       "RangedSummarizedExperiment"</span></span>
<span><span class="co">## [3] "SummarizedExperiment"       "RectangularData"           </span></span>
<span><span class="co">## [5] "Vector"                     "Annotated"                 </span></span>
<span><span class="co">## [7] "vector_OR_Vector"</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="raggedexperiment">
<code>RaggedExperiment</code><a class="anchor" aria-label="anchor" href="#raggedexperiment"></a>
</h5>
<ul>
<li>flexible representation for segmented copy number, somatic mutations
such as represented in <code>.vcf</code> files, and other ragged array
schema for genomic location data.</li>
<li>similar to the <code>GRangesList</code> class in
<code>GenomicRanges</code>
</li>
<li>used to represent <em>differing</em> genomic ranges on each of a set
of samples</li>
</ul>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">showClass</span><span class="op">(</span><span class="st">"RaggedExperiment"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Class "RaggedExperiment" [package "RaggedExperiment"]</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Slots:</span></span>
<span><span class="co">##                                                       </span></span>
<span><span class="co">## Name:       assays      rowidx      colidx    metadata</span></span>
<span><span class="co">## Class: GRangesList     integer     integer        list</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Extends: "Annotated"</span></span></code></pre>
<p><code>RaggedExperiment</code> provides a flexible set of _*Assay_
methods to support transformation of ranged list data to matrix
format.</p>
<div class="figure">
<img src="RaggedExperiment_Schematic.png" alt="RaggedExperiment object schematic. Rows and columns represent genomic ranges and samples, respectively. Assay operations can be performed with (from left to right) compactAssay, qreduceAssay, and sparseAssay." width="\maxwidth"><p class="caption">
RaggedExperiment object schematic. Rows and columns represent genomic
ranges and samples, respectively. Assay operations can be performed with
(from left to right) compactAssay, qreduceAssay, and sparseAssay.
</p>
</div>
</div>
</div>
</div>
<div class="section level3">
<h3 id="the-integrative-container">The Integrative Container<a class="anchor" aria-label="anchor" href="#the-integrative-container"></a>
</h3>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic. colData provides data about the patients, cell lines, or other biological units, with one row per unit and one column per variable. The experiments are a list of assay datasets of arbitrary class.  The sampleMap relates each column (observation) in ExperimentList to exactly one row (biological unit) in colData; however, one row of colData may map to zero, one, or more columns per assay, allowing for missing and replicate assays. sampleMap allows for per-assay sample naming conventions. Metadata can be used to store information in arbitrary format about the MultiAssayExperiment. Green stripes indicate a mapping of one subject to multiple observations across experiments." width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic. colData provides data about the
patients, cell lines, or other biological units, with one row per unit
and one column per variable. The experiments are a list of assay
datasets of arbitrary class. The sampleMap relates each column
(observation) in ExperimentList to exactly one row (biological unit) in
colData; however, one row of colData may map to zero, one, or more
columns per assay, allowing for missing and replicate assays. sampleMap
allows for per-assay sample naming conventions. Metadata can be used to
store information in arbitrary format about the MultiAssayExperiment.
Green stripes indicate a mapping of one subject to multiple observations
across experiments.
</p>
</div>
<div class="section level4">
<h4 id="multiassayexperiment-1">
<code>MultiAssayExperiment</code><a class="anchor" aria-label="anchor" href="#multiassayexperiment-1"></a>
</h4>
<ul>
<li>coordinates multi-omics experiment data on a set of biological
specimens</li>
<li>can contain any number of assays with different representations and
<em>dimensions</em>
</li>
<li>assays can be <em>ID-based</em>, where measurements are indexed
identifiers of genes, microRNA, proteins, microbes, etc.</li>
<li>assays may be <em>range-based</em>, where measurements correspond to
genomic ranges that can be represented as <code>GRanges</code> objects,
such as gene expression or copy number.</li>
</ul>
<details><summary><strong>Click</strong> on the fold to see what data classes are
supported!
</summary><ol style="list-style-type: decimal">
<li>
<code>matrix</code>: the most basic class for ID-based datasets,
could be used for example for gene expression summarized per-gene,
microRNA, metabolomics, or microbiome data.</li>
<li>
<code>SummarizedExperiment</code> and derived methods: described
above, could be used for miRNA, gene expression, proteomics, or any
matrix-like data where measurements are represented by IDs.</li>
<li>
<code>RangedSummarizedExperiment</code>: described above, could be
used for gene expression, methylation, or other data types referring to
genomic positions.</li>
<li>
<code>ExpressionSet</code>: Another rich representation for ID-based
datasets, supported only for legacy reasons</li>
<li>
<code>RaggedExperiment</code>: described above, for non-rectangular
(ragged) ranged-based datasets such as segmented copy number, where
segmentation of copy number alterations occurs and different genomic
locations in each sample.</li>
<li>
<code>RangedVcfStack</code>: For VCF archives broken up by
chromosome (see <code>VcfStack</code> class defined in the
<code>GenomicFiles</code> package)</li>
<li>
<code>DelayedMatrix</code>: An on-disk representation of matrix-like
objects for large datasets. It reduces memory usage and optimizes
performance with delayed operations. This class is part of the
<code>DelayedArray</code> package.</li>
<li>
<code>SingleCellExperiment</code>, <code>SpatialExperiment</code>,
etc.</li>
</ol></details><p><em>Note</em>. Data classes that support row and column naming and
subsetting may be used in a <code>MultiAssayExperiment</code>.</p>
</div>
<div class="section level4">
<h4 id="matchedassayexperiment">
<code>MatchedAssayExperiment</code><a class="anchor" aria-label="anchor" href="#matchedassayexperiment"></a>
</h4>
<ul>
<li>uniform subclass of <code>MultiAssayExperiment</code>
</li>
<li>“all patients have a sample in each assay”</li>
</ul>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># coercion</span></span>
<span><span class="fu">as</span><span class="op">(</span><span class="va">x</span>, <span class="st">"MatchedAssayExperiment"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># construction from MAE</span></span>
<span><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MatchedAssayExperiment-class.html" class="external-link">MatchedAssayExperiment</a></span><span class="op">(</span><span class="va">mae</span><span class="op">)</span></span></code></pre></div>
<p><em>Note</em>. The <code>MultiAssayExperiment</code> package provides
functionality to merge replicate profiles for a single patient
(<code><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">mergeReplicates()</a></code>).</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>
<code>MultiAssayExperiment</code> coordinates different Bioconductor
classes into one unified object</li>
<li>
<code>MultiAssayExperiment</code> is an infrastructure package while
<code>curatedTCGAData</code> and <code>cBioPortalData</code> provide
data on cancer studies including TCGA</li>
</ul>
</div>
</div>
</div>
<div class="section level3">
<h3 id="building-from-scratch-multiassayexperiment">Building from Scratch: <code>MultiAssayExperiment</code><a class="anchor" aria-label="anchor" href="#building-from-scratch-multiassayexperiment"></a>
</h3>
<div class="section level4">
<h4 id="miniacc-demo">
<code>miniACC</code> Demo<a class="anchor" aria-label="anchor" href="#miniacc-demo"></a>
</h4>
<p>Get started by trying out <code>MultiAssayExperiment</code> using a
subset of the TCGA adrenocortical carcinoma (ACC) dataset provided with
the package. This dataset provides five assays on 92 patients, although
all five assays were not performed for every patient:</p>
<ol style="list-style-type: decimal">
<li>
<strong>RNASeq2GeneNorm</strong>: gene mRNA abundance by
RNA-seq</li>
<li>
<strong>gistict</strong>: GISTIC genomic copy number by gene</li>
<li>
<strong>RPPAArray</strong>: protein abundance by Reverse Phase
Protein Array</li>
<li>
<strong>Mutations</strong>: non-silent somatic mutations by
gene</li>
<li>
<strong>miRNASeqGene</strong>: microRNA abundance by
microRNA-seq.</li>
</ol>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"miniACC"</span><span class="op">)</span></span>
<span><span class="va">miniACC</span></span></code></pre></div>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 5 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 5:</span></span>
<span><span class="co">##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns</span></span>
<span><span class="co">##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns</span></span>
<span><span class="co">##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns</span></span>
<span><span class="co">##  [4] Mutations: matrix with 97 rows and 90 columns</span></span>
<span><span class="co">##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="shiny-demo">
<code>shiny</code> Demo<a class="anchor" aria-label="anchor" href="#shiny-demo"></a>
</h4>
<p><a href="../tutorials/Exercise_MultiAssay.html">Click Here</a> to
open the <code>shiny</code> tutorial.</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>Extractor functions allow users to take components from the
<code>MultiAssayExperiment</code> object</li>
<li>They’re usually the same name as the component except for
<code>experiments</code> which extracts the
<code>ExperimentList</code>
</li>
</ul>
</div>
</div>
</div>
<div class="section level3">
<h3 id="notes-on-working-with-multiassayexperiment">Notes on Working with MultiAssayExperiment<a class="anchor" aria-label="anchor" href="#notes-on-working-with-multiassayexperiment"></a>
</h3>
<div class="section level4">
<h4 id="api-cheat-sheet">API cheat sheet<a class="anchor" aria-label="anchor" href="#api-cheat-sheet"></a>
</h4>
<div class="figure">
<img src="MultiAssayExperiment_cheatsheet.png" alt="The MultiAssayExperiment API for construction, access, subsetting, management, and reshaping to formats for application of R/Bioconductor graphics and analysis packages." width="\maxwidth"><p class="caption">
The MultiAssayExperiment API for construction, access, subsetting,
management, and reshaping to formats for application of R/Bioconductor
graphics and analysis packages.
</p>
</div>
</div>
<div class="section level4">
<h4 id="multiassayexperiment-construction-and-concatenation">MultiAssayExperiment construction and concatenation<a class="anchor" aria-label="anchor" href="#multiassayexperiment-construction-and-concatenation"></a>
</h4>
<div class="section level5">
<h5 id="constructor-function">constructor function<a class="anchor" aria-label="anchor" href="#constructor-function"></a>
</h5>
<p>The <code>MultiAssayExperiment</code> constructor function accepts
three arguments:</p>
<ol style="list-style-type: decimal">
<li>
<code>experiments</code> - An <code>ExperimentList</code> or
<code>list</code> of rectangular data</li>
<li>
<code>colData</code> - A <code>DataFrame</code> describing the
patients (or cell lines, or other biological units)</li>
<li>
<code>sampleMap</code> - A <code>DataFrame</code> of
<code>assay</code>, <code>primary</code>, and <code>colname</code>
identifiers</li>
</ol>
<p>The miniACC object can be reconstructed as follows:</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment.html" class="external-link">MultiAssayExperiment</a></span><span class="op">(</span></span>
<span>    experiments <span class="op">=</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html" class="external-link">experiments</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span>,</span>
<span>    colData <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span>,</span>
<span>    sampleMap <span class="op">=</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html" class="external-link">sampleMap</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span>,</span>
<span>    metadata <span class="op">=</span> <span class="fu">metadata</span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 5 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 5:</span></span>
<span><span class="co">##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns</span></span>
<span><span class="co">##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns</span></span>
<span><span class="co">##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns</span></span>
<span><span class="co">##  [4] Mutations: matrix with 97 rows and 90 columns</span></span>
<span><span class="co">##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="getting-help-with-prepmultiassay">Getting help with <code>prepMultiAssay</code><a class="anchor" aria-label="anchor" href="#getting-help-with-prepmultiassay"></a>
</h5>
<p>The <code>prepMultiAssay</code> function allows the user to diagnose
typical problems when creating a <code>MultiAssayExperiment</code>
object. See <code><a href="http://waldronlab.io/MultiAssayExperiment/reference/prepMultiAssay.html" class="external-link">?prepMultiAssay</a></code> for more details.</p>
</div>
<div class="section level5">
<h5 id="combining-multiassayexperiment-object-with-the-c-function">Combining MultiAssayExperiment object with the <code>c</code>
function<a class="anchor" aria-label="anchor" href="#combining-multiassayexperiment-object-with-the-c-function"></a>
</h5>
<p>The <code>c</code> function allows the user to concatenate an
additional experiment to an existing <code>MultiAssayExperiment</code>.
The optional <code>sampleMap</code> argument allows concatenating an
assay whose column names do not match the row names of
<code>colData</code>. For convenience, the <em>mapFrom</em> argument
allows the user to map from a particular experiment
<strong>provided</strong> that the <strong>order</strong> of the
colnames is in the <strong>same</strong>. A <code>warning</code> will be
issued to make the user aware of this assumption. For example, to
concatenate a matrix of log2-transformed RNA-seq results:</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">miniACC2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="va">miniACC</span>,</span>
<span>    log2rnaseq <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html" class="external-link">log2</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assays</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span><span class="op">$</span><span class="va">RNASeq2GeneNorm</span><span class="op">)</span>,</span>
<span>    mapFrom<span class="op">=</span><span class="fl">1L</span></span>
<span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: Assuming column order in the data provided </span></span>
<span><span class="co">##  matches the order in 'mapFrom' experiment(s) colnames</span></span></code></pre>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html" class="external-link">experiments</a></span><span class="op">(</span><span class="va">miniACC2</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## ExperimentList class object of length 6:</span></span>
<span><span class="co">##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns</span></span>
<span><span class="co">##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns</span></span>
<span><span class="co">##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns</span></span>
<span><span class="co">##  [4] Mutations: matrix with 97 rows and 90 columns</span></span>
<span><span class="co">##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns</span></span>
<span><span class="co">##  [6] log2rnaseq: matrix with 198 rows and 79 columns</span></span></code></pre>
</div>
</div>
<div class="section level4">
<h4 id="coldata---information-biological-units">colData - information biological units<a class="anchor" aria-label="anchor" href="#coldata---information-biological-units"></a>
</h4>
<p>This slot is a <code>DataFrame</code> describing the characteristics
of biological units, for example clinical data for patients. In the
prepared datasets from <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga" class="external-link">The
Cancer Genome Atlas</a>, each row is one patient and each column is a
clinical, pathological, subtype, or other variable. The <code>$</code>
function provides a shortcut for accessing or setting
<code>colData</code> columns.</p>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span><span class="op">[</span><span class="fl">1</span><span class="op">:</span><span class="fl">4</span>, <span class="fl">1</span><span class="op">:</span><span class="fl">4</span><span class="op">]</span></span></code></pre></div>
<pre><code><span><span class="co">## DataFrame with 4 rows and 4 columns</span></span>
<span><span class="co">##                 patientID years_to_birth vital_status days_to_death</span></span>
<span><span class="co">##               &lt;character&gt;      &lt;integer&gt;    &lt;integer&gt;     &lt;integer&gt;</span></span>
<span><span class="co">## TCGA-OR-A5J1 TCGA-OR-A5J1             58            1          1355</span></span>
<span><span class="co">## TCGA-OR-A5J2 TCGA-OR-A5J2             44            1          1677</span></span>
<span><span class="co">## TCGA-OR-A5J3 TCGA-OR-A5J3             23            0            NA</span></span>
<span><span class="co">## TCGA-OR-A5J4 TCGA-OR-A5J4             23            1           423</span></span></code></pre>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">$</span><span class="va">race</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">##                     asian black or african american                     white </span></span>
<span><span class="co">##                         2                         1                        78</span></span></code></pre>
<p>Note. <code>MultiAssayExperiment</code> supports both missing
observations and replicate observations, i.e., one row of
<code>colData</code> can map to 0, 1, or more columns of any of the
experimental data matrices. One could therefore treat replicate
observations as one or multiple rows of <code>colData</code>. This can
result in different subsetting, <code><a href="https://rdrr.io/r/base/duplicated.html" class="external-link">duplicated()</a></code>, and
<code><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">wideFormat()</a></code> behaviors.</p>
<p>Note. Multiple time points, or distinct biological replicates, can be
separate rows of the <code>colData</code>.</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>Each row maps to zero or more observations in each experiment</li>
<li>Usually, organized as one row per biological unit</li>
</ul>
</div>
</div>
<div class="section level4">
<h4 id="experimentlist---experiment-data">ExperimentList - experiment data<a class="anchor" aria-label="anchor" href="#experimentlist---experiment-data"></a>
</h4>
<p>Experimental datasets can be input as either a base <code>list</code>
or <code>ExperimentList</code> object for the set of samples collected.
To see the experiments use the <code>experiments</code> getter
function.</p>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html" class="external-link">experiments</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## ExperimentList class object of length 5:</span></span>
<span><span class="co">##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns</span></span>
<span><span class="co">##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns</span></span>
<span><span class="co">##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns</span></span>
<span><span class="co">##  [4] Mutations: matrix with 97 rows and 90 columns</span></span>
<span><span class="co">##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns</span></span></code></pre>
<p>Note. Each matrix column must correspond to exactly one row in
<code>colData</code>. In other words, each patient or cell line must be
traceable. However, multiple columns can come from the same patient, or
there can be no data for that patient.</p>
<div class="infobox">
<p><em>Key points:</em></p>
<ul>
<li>One rectangular dataset per list element</li>
<li>One column per assayed specimen.</li>
<li>Matrix rows correspond to variables, e.g. genes or genomic
ranges</li>
<li>
<code>ExperimentList</code> elements can be genomic range-based
(e.g. <code>SummarizedExperiment</code> or
<code>RaggedExperiment</code>) or ID-based data</li>
<li>Most rectangular-type data classes are supported</li>
</ul>
</div>
<p>Note. Any data class can be included in the
<code>ExperimentList</code>, as long as it supports: single-bracket
subsetting (<code>[</code>), <code>dimnames</code>, and
<code>dim</code>. Most data classes defined in Bioconductor meet these
requirements.</p>
</div>
<div class="section level4">
<h4 id="samplemap---relationship-graph">sampleMap - relationship graph<a class="anchor" aria-label="anchor" href="#samplemap---relationship-graph"></a>
</h4>
<p><code>sampleMap</code> is a graph representation of the relationship
between biological units and experimental results. In simple cases where
the column names of <code>ExperimentList</code> data matrices match the
row names of <code>colData</code>, the user won’t need to specify or
think about a sample map, it can be created automatically by the
<code>MultiAssayExperiment</code> constructor. <code>sampleMap</code> is
a simple three-column <code>DataFrame</code>:</p>
<ol style="list-style-type: decimal">
<li>
<code>assay</code> column: the name of the assay, and found in the
names of <code>ExperimentList</code> list names</li>
<li>
<code>primary</code> column: identifiers of patients or biological
units, and found in the row names of <code>colData</code>
</li>
<li>
<code>colname</code> column: identifiers of assay results, and found
in the column names of <code>ExperimentList</code> elements Helper
functions are available for creating a map from a list. See
<code><a href="http://waldronlab.io/MultiAssayExperiment/reference/mapToList.html" class="external-link">?listToMap</a></code>
</li>
</ol>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html" class="external-link">sampleMap</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## DataFrame with 385 rows and 3 columns</span></span>
<span><span class="co">##               assay      primary                colname</span></span>
<span><span class="co">##            &lt;factor&gt;  &lt;character&gt;            &lt;character&gt;</span></span>
<span><span class="co">## 1   RNASeq2GeneNorm TCGA-OR-A5J1 TCGA-OR-A5J1-01A-11R..</span></span>
<span><span class="co">## 2   RNASeq2GeneNorm TCGA-OR-A5J2 TCGA-OR-A5J2-01A-11R..</span></span>
<span><span class="co">## 3   RNASeq2GeneNorm TCGA-OR-A5J3 TCGA-OR-A5J3-01A-11R..</span></span>
<span><span class="co">## 4   RNASeq2GeneNorm TCGA-OR-A5J5 TCGA-OR-A5J5-01A-11R..</span></span>
<span><span class="co">## 5   RNASeq2GeneNorm TCGA-OR-A5J6 TCGA-OR-A5J6-01A-31R..</span></span>
<span><span class="co">## ...             ...          ...                    ...</span></span>
<span><span class="co">## 381    miRNASeqGene TCGA-PA-A5YG TCGA-PA-A5YG-01A-11R..</span></span>
<span><span class="co">## 382    miRNASeqGene TCGA-PK-A5H8 TCGA-PK-A5H8-01A-11R..</span></span>
<span><span class="co">## 383    miRNASeqGene TCGA-PK-A5H9 TCGA-PK-A5H9-01A-11R..</span></span>
<span><span class="co">## 384    miRNASeqGene TCGA-PK-A5HA TCGA-PK-A5HA-01A-11R..</span></span>
<span><span class="co">## 385    miRNASeqGene TCGA-PK-A5HB TCGA-PK-A5HB-01A-11R..</span></span></code></pre>
<div class="infobox">
<p><em>Key points:</em></p>
<ul>
<li>relates experimental observations (<code>colnames</code>) to
<code>colData</code>
</li>
<li>permits experiment-specific sample naming, missing, and replicate
observations</li>
</ul>
</div>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div class="section level4">
<h4 id="metadata">metadata<a class="anchor" aria-label="anchor" href="#metadata"></a>
</h4>
<p>Metadata can be used to keep additional information about patients,
assays performed on individuals or on the entire cohort, or features
such as genes, proteins, and genomic ranges. There are many options
available for storing metadata. First, <code>MultiAssayExperiment</code>
has its own metadata for describing the entire experiment:</p>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">metadata</span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## $title</span></span>
<span><span class="co">## [1] "Comprehensive Pan-Genomic Characterization of Adrenocortical Carcinoma"</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $PMID</span></span>
<span><span class="co">## [1] "27165744"</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $sourceURL</span></span>
<span><span class="co">## [1] "http://s3.amazonaws.com/multiassayexperiments/accMAEO.rds"</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $RPPAfeatureDataURL</span></span>
<span><span class="co">## [1] "http://genomeportal.stanford.edu/pan-tcga/show_target_selection_file?filename=Allprotein.txt"</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $colDataExtrasURL</span></span>
<span><span class="co">## [1] "http://www.cell.com/cms/attachment/2062093088/2063584534/mmc3.xlsx"</span></span></code></pre>
<p>Additionally, the <code>DataFrame</code> class used by
<code>sampleMap</code> and <code>colData</code>, as well as the
<code>ExperimentList</code> class, similarly support metadata. Finally,
many experimental data objects that can be used in the
<code>ExperimentList</code> support metadata. These provide flexible
options to users and to developers of derived classes.</p>
</div>
</div>
<div class="section level3">
<h3 id="the-cancer-genome-atlas-tcga-data-from-curatedtcgadata">The Cancer Genome Atlas (TCGA) Data from
<code>curatedTCGAData</code><a class="anchor" aria-label="anchor" href="#the-cancer-genome-atlas-tcga-data-from-curatedtcgadata"></a>
</h3>
<p>Most unrestricted TCGA data (from 33 cancer types) are available as
<code>MultiAssayExperiment</code> objects from the
<code>curatedTCGAData</code> package. This represents a lot of
harmonization!</p>
<p>Here we list the available data for the Adrenocortical Carcinoma
(<code>"ACC"</code>) cancer type:</p>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">curatedTCGAData</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html" class="external-link">curatedTCGAData</a></span><span class="op">(</span><span class="st">"ACC"</span>, version <span class="op">=</span> <span class="st">"2.0.1"</span>, dry.run <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##     ah_id                                 title file_size</span></span>
<span><span class="co">## 1  EH4737                   ACC_CNASNP-20160128    0.8 Mb</span></span>
<span><span class="co">## 2  EH4738                   ACC_CNVSNP-20160128    0.2 Mb</span></span>
<span><span class="co">## 3  EH4740         ACC_GISTIC_AllByGene-20160128    0.2 Mb</span></span>
<span><span class="co">## 4  EH4741             ACC_GISTIC_Peaks-20160128      0 Mb</span></span>
<span><span class="co">## 5  EH4742 ACC_GISTIC_ThresholdedByGene-20160128    0.2 Mb</span></span>
<span><span class="co">## 6  EH4744       ACC_Methylation-20160128_assays  239.2 Mb</span></span>
<span><span class="co">## 7  EH4745           ACC_Methylation-20160128_se      6 Mb</span></span>
<span><span class="co">## 8  EH4746             ACC_miRNASeqGene-20160128    0.1 Mb</span></span>
<span><span class="co">## 9  EH4747                 ACC_Mutation-20160128    0.7 Mb</span></span>
<span><span class="co">## 10 EH4748              ACC_RNASeq2Gene-20160128    2.7 Mb</span></span>
<span><span class="co">## 11 EH4749          ACC_RNASeq2GeneNorm-20160128      4 Mb</span></span>
<span><span class="co">## 12 EH4750                ACC_RPPAArray-20160128    0.1 Mb</span></span>
<span><span class="co">##                    rdataclass rdatadateadded rdatadateremoved</span></span>
<span><span class="co">## 1            RaggedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 2            RaggedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 3        SummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 4  RangedSummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 5        SummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 6        SummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 7            RaggedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 8        SummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 9        SummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 10       SummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 11                     DFrame     2021-01-27             &lt;NA&gt;</span></span>
<span><span class="co">## 12       SummarizedExperiment     2021-01-27             &lt;NA&gt;</span></span></code></pre>
<p>We then download the data with <code>dry.run</code> set to
<code>FALSE</code>.</p>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">acc</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html" class="external-link">curatedTCGAData</a></span><span class="op">(</span></span>
<span>    diseaseCode <span class="op">=</span> <span class="st">"ACC"</span>,</span>
<span>    assays <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>        <span class="st">"miRNASeqGene"</span>, <span class="st">"RPPAArray"</span>, <span class="st">"Mutation"</span>, <span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"CNVSNP"</span></span>
<span>    <span class="op">)</span>,</span>
<span>    version <span class="op">=</span> <span class="st">"2.0.1"</span>,</span>
<span>    dry.run <span class="op">=</span> <span class="cn">FALSE</span></span>
<span><span class="op">)</span></span>
<span><span class="va">acc</span></span></code></pre></div>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 5 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 5:</span></span>
<span><span class="co">##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns</span></span>
<span><span class="co">##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns</span></span>
<span><span class="co">##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns</span></span>
<span><span class="co">##  [4] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns</span></span>
<span><span class="co">##  [5] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
<p>These objects contain most unrestricted TCGA assay and clinical /
pathological data, as well as some curated data from the supplements of
published TCGA primary papers at the end of the colData columns:</p>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1]  92 822</span></span></code></pre>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">tail</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span><span class="op">)</span>, <span class="fl">10</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "MethyLevel"       "miRNA.cluster"    "SCNA.cluster"     "protein.cluster" </span></span>
<span><span class="co">##  [5] "COC"              "OncoSign"         "purity"           "ploidy"          </span></span>
<span><span class="co">##  [9] "genome_doublings" "ADS"</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="cbioportaldata-1">
<code>cBioPortalData</code><a class="anchor" aria-label="anchor" href="#cbioportaldata-1"></a>
</h3>
<p>To date, the <a href="https://www.cbioportal.org/" class="external-link">cBio Genomics
Portal</a> provides access to more than 300 datasets collected and
curated from different instutions.</p>
<p>There are two main ways of accessing this data:</p>
<ol style="list-style-type: decimal">
<li>
<code>cBioDataPack</code> - tarball (<code>.tar.gz</code>) data
files</li>
<li>
<code>cBioPortalData</code> - data from the API</li>
</ol>
<p><em>Note</em>. <code>pkgdown</code> reference website here: <a href="https://waldronlab.io/cBioPortalData/" class="external-link uri">https://waldronlab.io/cBioPortalData/</a></p>
<div class="section level4">
<h4 id="listing-the-studies-available">Listing the studies available<a class="anchor" aria-label="anchor" href="#listing-the-studies-available"></a>
</h4>
<p>First, we create an API object using the <code>cBioPortal</code>
function. This will allow us to subsequently generate queries for the
service.</p>
<div class="sourceCode" id="cb34"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">cbio</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html" class="external-link">cBioPortal</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html" class="external-link">getStudies</a></span><span class="op">(</span><span class="va">cbio</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 395 × 13</span></span></span>
<span><span class="co">##    name          description publicStudy groups status importDate allSampleCount</span></span>
<span><span class="co">##    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>   <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>               <span style="color: #949494; font-style: italic;">&lt;int&gt;</span></span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 1</span> Adrenocortic… <span style="color: #949494;">"</span>TCGA Adre… TRUE        PUBLIC      0 2023-06-1…             92</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 2</span> Acute Myeloi… <span style="color: #949494;">"</span>TCGA Acut… TRUE        PUBLIC      0 2023-06-1…            200</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 3</span> Bladder Urot… <span style="color: #949494;">"</span>TCGA Blad… TRUE        PUBLIC      0 2023-06-1…            413</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 4</span> Kidney Renal… <span style="color: #949494;">"</span>TCGA Kidn… TRUE        PUBLIC      0 2023-06-1…            538</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 5</span> Cervical Squ… <span style="color: #949494;">"</span>TCGA Cerv… TRUE        PUBLIC      0 2023-06-1…            310</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 6</span> Cholangiocar… <span style="color: #949494;">"</span>TCGA Chol… TRUE        PUBLIC      0 2023-06-1…             51</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 7</span> Kidney Chrom… <span style="color: #949494;">"</span>TCGA Kidn… TRUE        PUBLIC      0 2023-06-1…            113</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 8</span> Colorectal A… <span style="color: #949494;">"</span>TCGA Colo… TRUE        PUBLIC      0 2023-06-1…            640</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 9</span> Lymphoid Neo… <span style="color: #949494;">"</span>TCGA Lymp… TRUE        PUBLIC      0 2023-06-1…             48</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">10</span> Esophageal C… <span style="color: #949494;">"</span>TCGA Esop… TRUE        PUBLIC      0 2023-06-1…            186</span></span>
<span><span class="co">## <span style="color: #949494;"># ℹ 385 more rows</span></span></span>
<span><span class="co">## <span style="color: #949494;"># ℹ 6 more variables: readPermission &lt;lgl&gt;, studyId &lt;chr&gt;, cancerTypeId &lt;chr&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   referenceGenome &lt;chr&gt;, pmid &lt;chr&gt;, citation &lt;chr&gt;</span></span></span></code></pre>
<p>We can also see the build status of the datasets using the
<code>buildReport</code> argument.</p>
<div class="sourceCode" id="cb36"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html" class="external-link">getStudies</a></span><span class="op">(</span><span class="va">cbio</span>, buildReport <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 395 × 15</span></span></span>
<span><span class="co">##    name          description publicStudy groups status importDate allSampleCount</span></span>
<span><span class="co">##    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>   <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>               <span style="color: #949494; font-style: italic;">&lt;int&gt;</span></span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 1</span> Adrenocortic… <span style="color: #949494;">"</span>TCGA Adre… TRUE        PUBLIC      0 2023-06-1…             92</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 2</span> Acute Myeloi… <span style="color: #949494;">"</span>TCGA Acut… TRUE        PUBLIC      0 2023-06-1…            200</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 3</span> Bladder Urot… <span style="color: #949494;">"</span>TCGA Blad… TRUE        PUBLIC      0 2023-06-1…            413</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 4</span> Kidney Renal… <span style="color: #949494;">"</span>TCGA Kidn… TRUE        PUBLIC      0 2023-06-1…            538</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 5</span> Cervical Squ… <span style="color: #949494;">"</span>TCGA Cerv… TRUE        PUBLIC      0 2023-06-1…            310</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 6</span> Cholangiocar… <span style="color: #949494;">"</span>TCGA Chol… TRUE        PUBLIC      0 2023-06-1…             51</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 7</span> Kidney Chrom… <span style="color: #949494;">"</span>TCGA Kidn… TRUE        PUBLIC      0 2023-06-1…            113</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 8</span> Colorectal A… <span style="color: #949494;">"</span>TCGA Colo… TRUE        PUBLIC      0 2023-06-1…            640</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 9</span> Lymphoid Neo… <span style="color: #949494;">"</span>TCGA Lymp… TRUE        PUBLIC      0 2023-06-1…             48</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">10</span> Esophageal C… <span style="color: #949494;">"</span>TCGA Esop… TRUE        PUBLIC      0 2023-06-1…            186</span></span>
<span><span class="co">## <span style="color: #949494;"># ℹ 385 more rows</span></span></span>
<span><span class="co">## <span style="color: #949494;"># ℹ 8 more variables: readPermission &lt;lgl&gt;, studyId &lt;chr&gt;, cancerTypeId &lt;chr&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   referenceGenome &lt;chr&gt;, pmid &lt;chr&gt;, citation &lt;chr&gt;, api_build &lt;lgl&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   pack_build &lt;lgl&gt;</span></span></span></code></pre>
<p>This adds two additional columns to the end of the dataset reporting
the status of the builds. Not all studies can be converted to
<code>MultiAssayExperiment</code>. Some studies require additonal
cleaning to be represented with <code>MultiAssayExperiment</code>.</p>
</div>
<div class="section level4">
<h4 id="cbiodatapack">cBioDataPack<a class="anchor" aria-label="anchor" href="#cbiodatapack"></a>
</h4>
<div class="sourceCode" id="cb38"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">cBioPortalData</span><span class="op">)</span></span>
<span></span>
<span><span class="op">(</span><span class="va">uvm</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioDataPack.html" class="external-link">cBioDataPack</a></span><span class="op">(</span><span class="st">"uvm_tcga"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 8 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 8:</span></span>
<span><span class="co">##  [1] cna_hg19.seg: RaggedExperiment with 7618 rows and 80 columns</span></span>
<span><span class="co">##  [2] cna: SummarizedExperiment with 24776 rows and 80 columns</span></span>
<span><span class="co">##  [3] linear_cna: SummarizedExperiment with 24776 rows and 80 columns</span></span>
<span><span class="co">##  [4] methylation_hm450: SummarizedExperiment with 15468 rows and 80 columns</span></span>
<span><span class="co">##  [5] mrna_seq_v2_rsem_zscores_ref_all_samples: SummarizedExperiment with 20531 rows and 80 columns</span></span>
<span><span class="co">##  [6] mrna_seq_v2_rsem_zscores_ref_diploid_samples: SummarizedExperiment with 20440 rows and 80 columns</span></span>
<span><span class="co">##  [7] mrna_seq_v2_rsem: SummarizedExperiment with 20531 rows and 80 columns</span></span>
<span><span class="co">##  [8] mutations: RaggedExperiment with 2174 rows and 80 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="cbioportaldata-2">cBioPortalData<a class="anchor" aria-label="anchor" href="#cbioportaldata-2"></a>
</h4>
<div class="sourceCode" id="cb40"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="op">(</span></span>
<span>    <span class="va">urcc</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortalData.html" class="external-link">cBioPortalData</a></span><span class="op">(</span></span>
<span>        <span class="va">cbio</span>, studyId <span class="op">=</span> <span class="st">"urcc_mskcc_2016"</span>, genePanelId <span class="op">=</span> <span class="st">"IMPACT341"</span></span>
<span>    <span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 2 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 2:</span></span>
<span><span class="co">##  [1] urcc_mskcc_2016_mutations: RangedSummarizedExperiment with 147 rows and 53 columns</span></span>
<span><span class="co">##  [2] urcc_mskcc_2016_cna: SummarizedExperiment with 214 rows and 62 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>
<code>curatedTCGAData</code> provides TCGA data with some curation
including tumor subtype information</li>
<li>
<code>cBioPortalData</code> has two main functions, one for
downloading pre-packaged data and another for sending queries through
the cBioPortal API</li>
</ul>
</div>
</div>
</div>
<div class="section level3">
<h3 id="utilities-for-tcga">Utilities for TCGA<a class="anchor" aria-label="anchor" href="#utilities-for-tcga"></a>
</h3>
<p>Aside from the available reshaping functions already included in the
<code>MultiAssayExperiment</code> package, the <code>TCGAutils</code>
package provides additional helper functions for working with TCGA
data.</p>
<p>A number of helper functions are available for managing datasets from
<code>curatedTCGAData</code>. These include:</p>
<ul>
<li>Conversions of <code>SummarizedExperiment</code> to
<code>RangedSummarizedExperiment</code> based on
<code>TxDb.Hsapiens.UCSC.hg19.knownGene</code> for:
<ul>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html" class="external-link">mirToRanges()</a></code>: microRNA</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html" class="external-link">symbolsToRanges()</a></code>: gene symbols</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html" class="external-link">qreduceTCGA()</a></code>: convert <code>RaggedExperiment</code>
objects to <code>RangedSummarizedExperiment</code> with one row per gene
symbol, for:
<ul>
<li>segmented copy number datasets (“CNVSNP” and “CNASNP”)</li>
<li>somatic mutation datasets (“Mutation”), with a value of 1 for any
non-silent mutation and a value of 0 for no mutation or silent
mutation</li>
</ul>
</li>
</ul>
</li>
</ul>
<div class="section level4">
<h4 id="conversion-of-row-metadata-for-curatedtcgadata-objects">(1) Conversion of row metadata for <code>curatedTCGAData</code>
objects<a class="anchor" aria-label="anchor" href="#conversion-of-row-metadata-for-curatedtcgadata-objects"></a>
</h4>
<div class="section level5">
<h5 id="mirtoranges">
<code>mirToRanges</code><a class="anchor" aria-label="anchor" href="#mirtoranges"></a>
</h5>
<p>microRNA assays obtained from <code>curatedTCGAData</code> have
annotated sequences that can be converted to genomic ranges using the
<code>mirbase.db</code> package. The function looks up all sequences and
converts them to (‘hg19’) ranges. For those rows that cannot be found,
an ‘unranged’ assay is introduced in the resulting MultiAssayExperiment
object.</p>
<div class="sourceCode" id="cb42"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html" class="external-link">mirToRanges</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning in (function (seqlevels, genome, new_style) : cannot switch some hg19's</span></span>
<span><span class="co">## seqlevels from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## Warning: 'experiments' dropped; see 'drops()'</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 80 sampleMap rows not in names(experiments)</span></span></code></pre>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 6 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 6:</span></span>
<span><span class="co">##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns</span></span>
<span><span class="co">##  [2] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns</span></span>
<span><span class="co">##  [3] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns</span></span>
<span><span class="co">##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns</span></span>
<span><span class="co">##  [5] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns</span></span>
<span><span class="co">##  [6] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="qreducetcga">
<code>qreduceTCGA</code><a class="anchor" aria-label="anchor" href="#qreducetcga"></a>
</h5>
<p>The <code>qreduceTCGA</code> function converts
<code>RaggedExperiment</code> mutation data objects to
<code>RangedSummarizedExperiment</code> using <code>org.Hs.eg.db</code>
and the <code>qreduceTCGA</code> utility function from
<code>RaggedExperiment</code> to summarize ‘silent’ and ‘non-silent’
mutations based on a ‘Variant_Classification’ metadata column in the
original object.</p>
<div class="sourceCode" id="cb47"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Update build metadata to "hg19"</span></span>
<span><span class="fu">genome</span><span class="op">(</span><span class="va">acc</span><span class="op">[[</span><span class="st">"ACC_Mutation-20160128"</span><span class="op">]</span><span class="op">]</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="st">"NCBI37"</span></span>
<span><span class="fu">seqlevelsStyle</span><span class="op">(</span><span class="va">acc</span><span class="op">[[</span><span class="st">"ACC_Mutation-20160128"</span><span class="op">]</span><span class="op">]</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="st">"UCSC"</span></span>
<span></span>
<span><span class="va">gnome</span> <span class="op">&lt;-</span> <span class="fu">genome</span><span class="op">(</span><span class="va">acc</span><span class="op">[[</span><span class="st">"ACC_Mutation-20160128"</span><span class="op">]</span><span class="op">]</span><span class="op">)</span></span>
<span><span class="va">gnome</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/builds.html" class="external-link">translateBuild</a></span><span class="op">(</span><span class="va">gnome</span><span class="op">)</span></span>
<span><span class="fu">genome</span><span class="op">(</span><span class="va">acc</span><span class="op">[[</span><span class="st">"ACC_Mutation-20160128"</span><span class="op">]</span><span class="op">]</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="va">gnome</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html" class="external-link">qreduceTCGA</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span></code></pre>
<pre><code><span><span class="co">##   403 genes were dropped because they have exons located on both strands</span></span>
<span><span class="co">##   of the same reference sequence or on more than one reference sequence,</span></span>
<span><span class="co">##   so cannot be represented by a single genomic range.</span></span>
<span><span class="co">##   Use 'single.strand.genes.only=FALSE' to get all the genes in a</span></span>
<span><span class="co">##   GRangesList object, or use suppressMessages() to suppress this message.</span></span></code></pre>
<pre><code><span><span class="co">## Warning in (function (seqlevels, genome, new_style) : cannot switch some hg19's</span></span>
<span><span class="co">## seqlevels from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## 'select()' returned 1:1 mapping between keys and columns</span></span></code></pre>
<pre><code><span><span class="co">## Warning in .normarg_seqlevelsStyle(value): more than one seqlevels style</span></span>
<span><span class="co">## supplied, using the 1st one only</span></span>
<span></span>
<span><span class="co">## Warning in .normarg_seqlevelsStyle(value): cannot switch some hg19's seqlevels</span></span>
<span><span class="co">## from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## Warning: 'experiments' dropped; see 'drops()'</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 270 sampleMap rows not in names(experiments)</span></span></code></pre>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 5 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 5:</span></span>
<span><span class="co">##  [1] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns</span></span>
<span><span class="co">##  [2] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns</span></span>
<span><span class="co">##  [3] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns</span></span>
<span><span class="co">##  [4] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22913 rows and 90 columns</span></span>
<span><span class="co">##  [5] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22913 rows and 180 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="symbolstoranges">
<code>symbolsToRanges</code><a class="anchor" aria-label="anchor" href="#symbolstoranges"></a>
</h5>
<p>In the cases where row annotations indicate gene symbols, the
<code>symbolsToRanges</code> utility function converts genes to genomic
ranges and replaces existing assays with
<code>RangedSummarizedExperiment</code> objects. Gene annotations are
given as ‘hg19’ genomic regions.</p>
<div class="sourceCode" id="cb56"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html" class="external-link">symbolsToRanges</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##   403 genes were dropped because they have exons located on both strands</span></span>
<span><span class="co">##   of the same reference sequence or on more than one reference sequence,</span></span>
<span><span class="co">##   so cannot be represented by a single genomic range.</span></span>
<span><span class="co">##   Use 'single.strand.genes.only=FALSE' to get all the genes in a</span></span>
<span><span class="co">##   GRangesList object, or use suppressMessages() to suppress this message.</span></span></code></pre>
<pre><code><span><span class="co">## Warning in (function (seqlevels, genome, new_style) : cannot switch some hg19's</span></span>
<span><span class="co">## seqlevels from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## 'select()' returned 1:1 mapping between keys and columns</span></span></code></pre>
<pre><code><span><span class="co">## Warning: 'experiments' dropped; see 'drops()'</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 79 sampleMap rows not in names(experiments)</span></span></code></pre>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 6 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 6:</span></span>
<span><span class="co">##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns</span></span>
<span><span class="co">##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns</span></span>
<span><span class="co">##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns</span></span>
<span><span class="co">##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns</span></span>
<span><span class="co">##  [5] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 16991 rows and 79 columns</span></span>
<span><span class="co">##  [6] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3510 rows and 79 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="simplifytcga">
<code>simplifyTCGA</code><a class="anchor" aria-label="anchor" href="#simplifytcga"></a>
</h5>
<p>The <code>simplifyTCGA</code> function combines all of the above
operations to create a more managable <code>MultiAssayExperiment</code>
object and using <code>RangedSummarizedExperiment</code> assays where
possible.</p>
<div class="sourceCode" id="cb63"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">TCGAutils</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html" class="external-link">simplifyTCGA</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##   403 genes were dropped because they have exons located on both strands</span></span>
<span><span class="co">##   of the same reference sequence or on more than one reference sequence,</span></span>
<span><span class="co">##   so cannot be represented by a single genomic range.</span></span>
<span><span class="co">##   Use 'single.strand.genes.only=FALSE' to get all the genes in a</span></span>
<span><span class="co">##   GRangesList object, or use suppressMessages() to suppress this message.</span></span></code></pre>
<pre><code><span><span class="co">## Warning in (function (seqlevels, genome, new_style) : cannot switch some hg19's</span></span>
<span><span class="co">## seqlevels from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## 'select()' returned 1:1 mapping between keys and columns</span></span></code></pre>
<pre><code><span><span class="co">## Warning in .normarg_seqlevelsStyle(value): more than one seqlevels style</span></span>
<span><span class="co">## supplied, using the 1st one only</span></span>
<span></span>
<span><span class="co">## Warning in .normarg_seqlevelsStyle(value): cannot switch some hg19's seqlevels</span></span>
<span><span class="co">## from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## Warning: 'experiments' dropped; see 'drops()'</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 270 sampleMap rows not in names(experiments)</span></span></code></pre>
<pre><code><span><span class="co">## Warning in (function (seqlevels, genome, new_style) : cannot switch some hg19's</span></span>
<span><span class="co">## seqlevels from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 80 sampleMap rows not in names(experiments)</span></span>
<span><span class="co">##   403 genes were dropped because they have exons located on both strands</span></span>
<span><span class="co">##   of the same reference sequence or on more than one reference sequence,</span></span>
<span><span class="co">##   so cannot be represented by a single genomic range.</span></span>
<span><span class="co">##   Use 'single.strand.genes.only=FALSE' to get all the genes in a</span></span>
<span><span class="co">##   GRangesList object, or use suppressMessages() to suppress this message.</span></span></code></pre>
<pre><code><span><span class="co">## Warning in (function (seqlevels, genome, new_style) : cannot switch some hg19's</span></span>
<span><span class="co">## seqlevels from UCSC to NCBI style</span></span></code></pre>
<pre><code><span><span class="co">## 'select()' returned 1:1 mapping between keys and columns</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 79 sampleMap rows not in names(experiments)</span></span></code></pre>
<pre><code><span><span class="co">## A MultiAssayExperiment object of 7 listed</span></span>
<span><span class="co">##  experiments with user-defined names and respective classes.</span></span>
<span><span class="co">##  Containing an ExperimentList class object of length 7:</span></span>
<span><span class="co">##  [1] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns</span></span>
<span><span class="co">##  [2] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22913 rows and 90 columns</span></span>
<span><span class="co">##  [3] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22913 rows and 180 columns</span></span>
<span><span class="co">##  [4] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns</span></span>
<span><span class="co">##  [5] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns</span></span>
<span><span class="co">##  [6] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 16991 rows and 79 columns</span></span>
<span><span class="co">##  [7] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3510 rows and 79 columns</span></span>
<span><span class="co">## Functionality:</span></span>
<span><span class="co">##  experiments() - obtain the ExperimentList instance</span></span>
<span><span class="co">##  colData() - the primary/phenotype DataFrame</span></span>
<span><span class="co">##  sampleMap() - the sample coordination DataFrame</span></span>
<span><span class="co">##  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span></span>
<span><span class="co">##  *Format() - convert into a long or wide DataFrame</span></span>
<span><span class="co">##  assays() - convert ExperimentList to a SimpleList of matrices</span></span>
<span><span class="co">##  exportClass() - save data to flat files</span></span></code></pre>
</div>
</div>
<div class="section level4">
<h4 id="identification-and-separation-of-samples">(2) Identification and Separation of Samples<a class="anchor" aria-label="anchor" href="#identification-and-separation-of-samples"></a>
</h4>
</div>
<div class="section level4">
<h4 id="what-types-of-samples-are-in-the-data">What types of samples are in the data?<a class="anchor" aria-label="anchor" href="#what-types-of-samples-are-in-the-data"></a>
</h4>
<p><strong>Solution</strong></p>
<p>The <code>sampleTables</code> function gives you an overview of
samples in each assay:</p>
<div class="sourceCode" id="cb76"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html" class="external-link">sampleTables</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## $`ACC_CNVSNP-20160128`</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## 01 10 11 </span></span>
<span><span class="co">## 90 85  5 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $`ACC_miRNASeqGene-20160128`</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## 01 </span></span>
<span><span class="co">## 80 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $`ACC_Mutation-20160128`</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## 01 </span></span>
<span><span class="co">## 90 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $`ACC_RNASeq2GeneNorm-20160128`</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## 01 </span></span>
<span><span class="co">## 79 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## $`ACC_RPPAArray-20160128`</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## 01 </span></span>
<span><span class="co">## 46</span></span></code></pre>
<p>Interpretation of sample codes:</p>
<div class="sourceCode" id="cb78"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"sampleTypes"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">sampleTypes</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##   Code                                      Definition Short.Letter.Code</span></span>
<span><span class="co">## 1   01                             Primary Solid Tumor                TP</span></span>
<span><span class="co">## 2   02                           Recurrent Solid Tumor                TR</span></span>
<span><span class="co">## 3   03 Primary Blood Derived Cancer - Peripheral Blood                TB</span></span>
<span><span class="co">## 4   04    Recurrent Blood Derived Cancer - Bone Marrow              TRBM</span></span>
<span><span class="co">## 5   05                        Additional - New Primary               TAP</span></span>
<span><span class="co">## 6   06                                      Metastatic                TM</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="tcgasplitassays-separate-the-data-from-different-tissue-types">
<code>TCGAsplitAssays</code>: separate the data from different
tissue types<a class="anchor" aria-label="anchor" href="#tcgasplitassays-separate-the-data-from-different-tissue-types"></a>
</h4>
<p>TCGA datasets include multiple -omics for solid tumors, adjacent
normal tissues, blood-derived cancers and normals, and other tissue
types, which may be mixed together in a single dataset. The
<code>MultiAssayExperiment</code> object generated here has one patient
per row of its <code>colData</code>, but each patient may have two or
more -omics profiles by any assay, whether due to assaying of different
types of tissues or to technical replication.
<code>TCGAsplitAssays</code> separates profiles from different tissue
types (such as tumor and adjacent normal) into different assays of the
<code>MultiAssayExperiment</code> by taking a vector of sample codes,
and partitioning the current assays into assays with an appended sample
code:</p>
<div class="sourceCode" id="cb80"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">split_acc</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html" class="external-link">TCGAsplitAssays</a></span><span class="op">(</span><span class="va">acc</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"01"</span>, <span class="st">"11"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: Some 'sampleCodes' not found in assays</span></span></code></pre>
<pre><code><span><span class="co">## Warning in .checkBarcodes(barcodes): Inconsistent barcode lengths: 28, 27</span></span></code></pre>
<p>Only about 43 participants have data across all experiments.</p>
</div>
<div class="section level4">
<h4 id="curated-molecular-subtypes">Curated molecular subtypes<a class="anchor" aria-label="anchor" href="#curated-molecular-subtypes"></a>
</h4>
<p>Is there subtype data available in the
<code>MultiAssayExperiment</code> obtained from
<code>curatedTCGAData</code>?</p>
<p><strong>Solution</strong></p>
<p>The <code>getSubtypeMap</code> function will show actual variable
names found in <code>colData</code> that contain subtype information.
This can only be obtained from <code>MultiAssayExperiment</code> objects
provided by <code>curatedTCGAData</code>.</p>
<div class="sourceCode" id="cb83"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html" class="external-link">getSubtypeMap</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##          ACC_annotations     ACC_subtype</span></span>
<span><span class="co">## 1             Patient_ID       patientID</span></span>
<span><span class="co">## 2  histological_subtypes       Histology</span></span>
<span><span class="co">## 3          mrna_subtypes         C1A/C1B</span></span>
<span><span class="co">## 4          mrna_subtypes         mRNA_K4</span></span>
<span><span class="co">## 5                   cimp      MethyLevel</span></span>
<span><span class="co">## 6      microrna_subtypes   miRNA cluster</span></span>
<span><span class="co">## 7          scna_subtypes    SCNA cluster</span></span>
<span><span class="co">## 8       protein_subtypes protein cluster</span></span>
<span><span class="co">## 9   integrative_subtypes             COC</span></span>
<span><span class="co">## 10     mutation_subtypes        OncoSign</span></span></code></pre>
<div class="sourceCode" id="cb85"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span><span class="op">$</span><span class="va">Histology</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "Usual Type" "Usual Type" "Usual Type" "Usual Type" "Usual Type"</span></span>
<span><span class="co">## [6] "Usual Type"</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="translation-and-interpretation-of-tcga-identifiers">(3) Translation and Interpretation of TCGA identifiers<a class="anchor" aria-label="anchor" href="#translation-and-interpretation-of-tcga-identifiers"></a>
</h4>
<p><code>TCGAutils</code> provides a number of ID translation functions.
These allow the user to translate from either file or case UUIDs to TCGA
barcodes and back. These functions work by querying the Genomic Data
Commons API via the <code>GenomicDataCommons</code> package (thanks to
Sean Davis). These include:</p>
<div class="section level5">
<h5 id="uuidtobarcode---uuid-to-tcga-barcode">
<code>UUIDtoBarcode()</code> - UUID to TCGA barcode<a class="anchor" aria-label="anchor" href="#uuidtobarcode---uuid-to-tcga-barcode"></a>
</h5>
<p>Here we have a known case UUID that we want to translate into a TCGA
barcode.</p>
<div class="sourceCode" id="cb87"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html" class="external-link">UUIDtoBarcode</a></span><span class="op">(</span><span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, from_type <span class="op">=</span> <span class="st">"case_id"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                                case_id submitter_id</span></span>
<span><span class="co">## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 TCGA-B0-5117</span></span></code></pre>
<p>In cases where we want to translate a known file UUID to the
associated TCGA patient barcode, we can use
<code>UUIDtoBarcode</code>.</p>
<div class="sourceCode" id="cb89"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html" class="external-link">UUIDtoBarcode</a></span><span class="op">(</span><span class="st">"b4bce3ff-7fdc-4849-880b-56f2b348ceac"</span>, from_type <span class="op">=</span> <span class="st">"file_id"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                                file_id associated_entities.entity_submitter_id</span></span>
<span><span class="co">## 1 b4bce3ff-7fdc-4849-880b-56f2b348ceac            TCGA-B0-5094-11A-01D-1421-08</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="barcodetouuid---tcga-barcode-to-uuid">
<code>barcodeToUUID()</code> - TCGA barcode to UUID<a class="anchor" aria-label="anchor" href="#barcodetouuid---tcga-barcode-to-uuid"></a>
</h5>
<p>Here we translate the first two TCGA barcodes of the previous
copy-number alterations dataset to UUID:</p>
<div class="sourceCode" id="cb91"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="op">(</span><span class="va">xbarcode</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">acc</span><span class="op">)</span><span class="op">[[</span><span class="st">"ACC_CNVSNP-20160128"</span><span class="op">]</span><span class="op">]</span>, <span class="fl">4L</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "TCGA-OR-A5J1-01A-11D-A29H-01" "TCGA-OR-A5J1-10A-01D-A29K-01"</span></span>
<span><span class="co">## [3] "TCGA-OR-A5J2-01A-11D-A29H-01" "TCGA-OR-A5J2-10A-01D-A29K-01"</span></span></code></pre>
<div class="sourceCode" id="cb93"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html" class="external-link">barcodeToUUID</a></span><span class="op">(</span><span class="va">xbarcode</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##           submitter_aliquot_ids                          aliquot_ids</span></span>
<span><span class="co">## 18 TCGA-OR-A5J1-01A-11D-A29H-01 1387b6c7-48fe-4961-86a7-0bdcbd3fef92</span></span>
<span><span class="co">## 16 TCGA-OR-A5J1-10A-01D-A29K-01 cb537629-6a01-4d67-84ea-dbf130bd59c7</span></span>
<span><span class="co">## 8  TCGA-OR-A5J2-01A-11D-A29H-01 6f0290b0-4cb4-4f72-853e-9ac363bd2c3b</span></span>
<span><span class="co">## 11 TCGA-OR-A5J2-10A-01D-A29K-01 4bf2e4ac-399f-4a00-854b-8e23b561bb4d</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="uuidtouuid---file-and-case-ids">
<code>UUIDtoUUID()</code> - file and case IDs<a class="anchor" aria-label="anchor" href="#uuidtouuid---file-and-case-ids"></a>
</h5>
<p>We can also translate from file UUIDs to case UUIDs and vice versa as
long as we know the input type. We can use the case UUID from the
previous example to get the associated file UUIDs using
<code>UUIDtoUUID</code>. Note that this translation is a one to many
relationship, thus yielding a <code>data.frame</code> of file UUIDs for
a single case UUID.</p>
<div class="sourceCode" id="cb95"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html" class="external-link">UUIDtoUUID</a></span><span class="op">(</span><span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, to_type <span class="op">=</span> <span class="st">"file_id"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                                case_id                        files.file_id</span></span>
<span><span class="co">## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 e6265f05-3e0d-408a-a7bd-484f083724ba</span></span>
<span><span class="co">## 2 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 f65f8685-b253-4935-a12d-2aab2682b3ac</span></span>
<span><span class="co">## 3 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 9428be7f-3873-45e6-a883-84e63f92103c</span></span>
<span><span class="co">## 4 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 dce252bc-5930-4dc4-aa5b-4912c0aa7b81</span></span>
<span><span class="co">## 5 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 052db59c-a4b7-42c9-ac6b-bd611c0d6800</span></span>
<span><span class="co">## 6 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 f249a5b5-4172-4250-a5f7-002165e45a53</span></span></code></pre>
<p>One possible way to verify that file IDs are matching case UUIDS is
to browse to the Genomic Data Commons webpage with the specific file
UUID. Here we look at the first file UUID entry in the output
<code>data.frame</code>:</p>
<p><a href="https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6" class="external-link uri">https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6</a></p>
<p>In the page we check that the case UUID matches the input.</p>
</div>
<div class="section level5">
<h5 id="filenametobarcode---using-file-names-as-input">
<code>filenameToBarcode()</code> - Using file names as input<a class="anchor" aria-label="anchor" href="#filenametobarcode---using-file-names-as-input"></a>
</h5>
<div class="sourceCode" id="cb97"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fquery</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://bioconductor.github.io/GenomicDataCommons/reference/query.html" class="external-link">files</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu">GenomicDataCommons</span><span class="fu">::</span><span class="fu"><a href="http://bioconductor.github.io/GenomicDataCommons/reference/filtering.html" class="external-link">filter</a></span><span class="op">(</span><span class="op">~</span> <span class="va">cases.project.project_id</span> <span class="op">==</span> <span class="st">"TCGA-ACC"</span> <span class="op">&amp;</span></span>
<span>        <span class="va">data_category</span> <span class="op">==</span> <span class="st">"Copy Number Variation"</span> <span class="op">&amp;</span></span>
<span>        <span class="va">data_type</span> <span class="op">==</span> <span class="st">"Copy Number Segment"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fnames</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fu"><a href="http://bioconductor.github.io/GenomicDataCommons/reference/results.html" class="external-link">results</a></span><span class="op">(</span><span class="va">fquery</span><span class="op">)</span><span class="op">$</span><span class="va">file_name</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html" class="external-link">filenameToBarcode</a></span><span class="op">(</span><span class="va">fnames</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                                                                      file_name</span></span>
<span><span class="co">## 1      AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_F10_1348394.grch38.seg.v2.txt</span></span>
<span><span class="co">## 2      AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_B08_1348416.grch38.seg.v2.txt</span></span>
<span><span class="co">## 3      AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_E09_1348242.grch38.seg.v2.txt</span></span>
<span><span class="co">## 4      AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_A07_1348264.grch38.seg.v2.txt</span></span>
<span><span class="co">## 5      BLAIN_p_TCGA_282_304_b2_N_GenomeWideSNP_6_F01_1348434.grch38.seg.v2.txt</span></span>
<span><span class="co">## 6 GAMED_p_TCGA_B_312_313_314_NSP_GenomeWideSNP_6_E06_1361784.grch38.seg.v2.txt</span></span>
<span><span class="co">##                                file_id</span></span>
<span><span class="co">## 1 4ea747a7-890d-4f0a-a7e3-ede6e6315594</span></span>
<span><span class="co">## 2 81754b77-e4b0-4123-baba-c2a8fbfaa487</span></span>
<span><span class="co">## 3 4b488f8d-ddab-4fc9-a82e-0b29b0e1c173</span></span>
<span><span class="co">## 4 536609b5-6b6a-4f15-b266-b0a0634d2361</span></span>
<span><span class="co">## 5 18bfa23c-e013-4650-9878-5a1b15aacc45</span></span>
<span><span class="co">## 6 5dfae645-caa6-4021-8f29-54e3cf043d33</span></span>
<span><span class="co">##   cases.samples.portions.analytes.aliquots.submitter_id</span></span>
<span><span class="co">## 1                          TCGA-OR-A5KW-01A-11D-A29H-01</span></span>
<span><span class="co">## 2                          TCGA-OR-A5KW-10A-01D-A29K-01</span></span>
<span><span class="co">## 3                          TCGA-PK-A5H9-01A-11D-A29H-01</span></span>
<span><span class="co">## 4                          TCGA-PK-A5H9-10A-01D-A29K-01</span></span>
<span><span class="co">## 5                          TCGA-OR-A5JA-01A-11D-A29H-01</span></span>
<span><span class="co">## 6                          TCGA-OR-A5L1-01A-11D-A309-01</span></span></code></pre>
<p>See the <code>TCGAutils</code> vignette page for more details.</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>
<code>TCGAutils</code> provides users additional tools for modifying
row and column metadata</li>
<li>The package works mainly with TCGA data including barcode
identifiers</li>
</ul>
</div>
</div>
</div>
</div>
<div class="section level3">
<h3 id="data-management-with-multiassayexperiment">Data Management with MultiAssayExperiment<a class="anchor" aria-label="anchor" href="#data-management-with-multiassayexperiment"></a>
</h3>
<div class="section level4">
<h4 id="single-bracket-subsetting">Single bracket <code>[</code> subsetting<a class="anchor" aria-label="anchor" href="#single-bracket-subsetting"></a>
</h4>
<p>In the pseudo code below, the subsetting operations work on the rows
of the following indices:</p>
<ol style="list-style-type: decimal">
<li>
<em>i</em> experimental data rows</li>
<li>
<em>j</em> the primary names (vector input) or the column names
(<code>list</code> or <code>List</code> inputs)</li>
<li>
<em>k</em> assay</li>
</ol>
<pre><code>multiassayexperiment[i = rownames, j = primary or colnames, k = assay]</code></pre>
<p>Subsetting operations always return another
<code>MultiAssayExperiment</code>. For example, the following will
return any rows named “MAPK14” or “IGFBP2”, and remove any assays where
no rows match:</p>
<div class="sourceCode" id="cb100"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">miniACC</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"MAPK14"</span>, <span class="st">"IGFBP2"</span><span class="op">)</span>, , <span class="op">]</span></span></code></pre></div>
<p>The following will keep only patients of pathological stage IV, and
all their associated assays:</p>
<div class="sourceCode" id="cb101"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">stg4</span> <span class="op">&lt;-</span> <span class="va">miniACC</span><span class="op">$</span><span class="va">pathologic_stage</span> <span class="op">==</span> <span class="st">"stage iv"</span></span>
<span><span class="co"># remove NA values from vector</span></span>
<span><span class="va">miniACC</span><span class="op">[</span>, <span class="va">stg4</span> <span class="op">&amp;</span> <span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">stg4</span><span class="op">)</span>, <span class="op">]</span></span></code></pre></div>
<p>And the following will keep only the RNA-seq dataset, and only
patients for which this assay is available:</p>
<div class="sourceCode" id="cb102"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">miniACC</span><span class="op">[</span>, , <span class="st">"RNASeq2GeneNorm"</span><span class="op">]</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: 'experiments' dropped; see 'drops()'</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 306 sampleMap rows not in names(experiments)</span></span>
<span><span class="co">##   removing 13 colData rownames not in sampleMap 'primary'</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="subsetting-by-genomic-ranges">Subsetting by genomic ranges<a class="anchor" aria-label="anchor" href="#subsetting-by-genomic-ranges"></a>
</h4>
<p>If any ExperimentList objects have features represented by genomic
ranges (e.g. <code>RangedSummarizedExperiment</code>,
<code>RaggedExperiment</code>), then a <code>GRanges</code> object in
the first subsetting position will subset these objects as in
<code><a href="https://rdrr.io/pkg/IRanges/man/findOverlaps-methods.html" class="external-link">GenomicRanges::findOverlaps()</a></code>. Any non-ranged
<code>ExperimentList</code> element will be subset to zero rows.</p>
</div>
<div class="section level4">
<h4 id="double-bracket-subsetting">Double bracket <code>[[</code> subsetting<a class="anchor" aria-label="anchor" href="#double-bracket-subsetting"></a>
</h4>
<p>The “double bracket” method (<code>[[</code>) is a convenience
function for extracting a single element of the
<code>MultiAssayExperiment</code> <code>ExperimentList</code>. It avoids
the use of <code>experiments(mae)[[1L]]</code>. For example, both of the
following extract the <code>ExpressionSet</code> object containing
RNA-seq data:</p>
<div class="sourceCode" id="cb105"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">miniACC</span><span class="op">[[</span><span class="fl">1L</span><span class="op">]</span><span class="op">]</span></span></code></pre></div>
<pre><code><span><span class="co">## class: SummarizedExperiment </span></span>
<span><span class="co">## dim: 198 79 </span></span>
<span><span class="co">## metadata(3): experimentData annotation protocolData</span></span>
<span><span class="co">## assays(1): exprs</span></span>
<span><span class="co">## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13</span></span>
<span><span class="co">## rowData names(0):</span></span>
<span><span class="co">## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07</span></span>
<span><span class="co">##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07</span></span>
<span><span class="co">## colData names(0):</span></span></code></pre>
<div class="sourceCode" id="cb107"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## equivalently</span></span>
<span><span class="va">miniACC</span><span class="op">[[</span><span class="st">"RNASeq2GeneNorm"</span><span class="op">]</span><span class="op">]</span></span></code></pre></div>
<pre><code><span><span class="co">## class: SummarizedExperiment </span></span>
<span><span class="co">## dim: 198 79 </span></span>
<span><span class="co">## metadata(3): experimentData annotation protocolData</span></span>
<span><span class="co">## assays(1): exprs</span></span>
<span><span class="co">## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13</span></span>
<span><span class="co">## rowData names(0):</span></span>
<span><span class="co">## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07</span></span>
<span><span class="co">##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07</span></span>
<span><span class="co">## colData names(0):</span></span></code></pre>
<p>To preserve the <code>colData</code> during the extraction, see
<code><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">?getWithColData</a></code>.</p>
</div>
<div class="section level4">
<h4 id="complete-cases">Complete cases<a class="anchor" aria-label="anchor" href="#complete-cases"></a>
</h4>
<p><code><a href="https://rdrr.io/r/stats/complete.cases.html" class="external-link">complete.cases()</a></code> shows which patients have complete data
for all assays:</p>
<div class="sourceCode" id="cb109"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/stats/complete.cases.html" class="external-link">complete.cases</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##    Mode   FALSE    TRUE </span></span>
<span><span class="co">## logical      49      43</span></span></code></pre>
<p>The above logical vector could be used for patient subsetting. More
simply, <code><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">intersectColumns()</a></code> will select complete cases and
rearrange each <code>ExperimentList</code> element so its columns
correspond exactly to rows of <code>colData</code> in the same
order:</p>
<div class="sourceCode" id="cb111"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">accmatched</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">intersectColumns</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span></span></code></pre></div>
<p>Note, the column names of the assays in <code>accmatched</code> are
not the same because of assay-specific identifiers, but they have been
automatically re-arranged to correspond to the same patients. In these
TCGA assays, the first three <code>-</code> delimited positions
correspond to patient, ie the first patient is
<em>TCGA-OR-A5J2</em>:</p>
<div class="sourceCode" id="cb112"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">accmatched</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## CharacterList of length 5</span></span>
<span><span class="co">## [["RNASeq2GeneNorm"]] TCGA-OR-A5J2-01A-11R-A29S-07 ...</span></span>
<span><span class="co">## [["gistict"]] TCGA-OR-A5J2-01A-11D-A29H-01 ... TCGA-PK-A5HA-01A-11D-A29H-01</span></span>
<span><span class="co">## [["RPPAArray"]] TCGA-OR-A5J2-01A-21-A39K-20 ... TCGA-PK-A5HA-01A-21-A39K-20</span></span>
<span><span class="co">## [["Mutations"]] TCGA-OR-A5J2-01A-11D-A29I-10 ... TCGA-PK-A5HA-01A-11D-A29I-10</span></span>
<span><span class="co">## [["miRNASeqGene"]] TCGA-OR-A5J2-01A-11R-A29W-13 ...</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="row-names-that-are-common-across-assays">Row names that are common across assays<a class="anchor" aria-label="anchor" href="#row-names-that-are-common-across-assays"></a>
</h4>
<p><code><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">intersectRows()</a></code> keeps only rows that are common to each
assay, and aligns them in identical order. For example, to keep only
genes where data are available for RNA-seq, GISTIC copy number, and
somatic mutations:</p>
<div class="sourceCode" id="cb114"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">accmatched2</span> <span class="op">&lt;-</span> <span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">intersectRows</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">[</span>, ,</span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"gistict"</span>, <span class="st">"Mutations"</span><span class="op">)</span><span class="op">]</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: 'experiments' dropped; see 'drops()'</span></span></code></pre>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 126 sampleMap rows not in names(experiments)</span></span></code></pre>
<div class="sourceCode" id="cb117"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">rownames</a></span><span class="op">(</span><span class="va">accmatched2</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## CharacterList of length 3</span></span>
<span><span class="co">## [["RNASeq2GeneNorm"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... RET CDKN2A MACC1 CHGA</span></span>
<span><span class="co">## [["gistict"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA</span></span>
<span><span class="co">## [["Mutations"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA</span></span></code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div class="section level4">
<h4 id="extraction">Extraction<a class="anchor" aria-label="anchor" href="#extraction"></a>
</h4>
</div>
<div class="section level4">
<h4 id="assay-and-assays">assay and assays<a class="anchor" aria-label="anchor" href="#assay-and-assays"></a>
</h4>
<p>The <code>assay</code> and <code>assays</code> methods follow
<code>SummarizedExperiment</code> convention. The <code>assay</code>
(singular) method will extract the first element of the
<code>ExperimentList</code> and will return a <code>matrix</code>.</p>
<div class="sourceCode" id="cb119"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/class.html" class="external-link">class</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "matrix" "array"</span></span></code></pre>
<p>The <code>assays</code> (plural) method will return a
<code>SimpleList</code> of the data with each element being a
<code>matrix</code>.</p>
<div class="sourceCode" id="cb121"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assays</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## List of length 5</span></span>
<span><span class="co">## names(5): RNASeq2GeneNorm gistict RPPAArray Mutations miRNASeqGene</span></span></code></pre>
<div class="infobox">
<p><em>Key point:</em></p>
<ul>
<li>Whereas the <code>[[</code> returned an experiment in its original
class, <code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay()</a></code> and <code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assays()</a></code> convert the assay
data to matrix format.</li>
</ul>
</div>
<p style="text-align: right;">
<a href="#overview-of-key-packages-and-data-classes">back to top</a>
</p>
</div>
<div class="section level4">
<h4 id="summary-of-slots-and-accessors">Summary of slots and accessors<a class="anchor" aria-label="anchor" href="#summary-of-slots-and-accessors"></a>
</h4>
<p>Slot in the <code>MultiAssayExperiment</code> can be accessed or set
using the respective accessor functions:</p>
<table class="table">
<thead><tr class="header">
<th>Slot</th>
<th>Accessor</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><code>ExperimentList</code></td>
<td><code><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html" class="external-link">experiments()</a></code></td>
</tr>
<tr class="even">
<td><code>colData</code></td>
<td>
<code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData()</a></code> and <code>$</code> *</td>
</tr>
<tr class="odd">
<td><code>sampleMap</code></td>
<td><code><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-methods.html" class="external-link">sampleMap()</a></code></td>
</tr>
<tr class="even">
<td><code>metadata</code></td>
<td><code>metadata()</code></td>
</tr>
</tbody>
</table>
<p>__*__ The <code>$</code> operator on a
<code>MultiAssayExperiment</code> returns a single column of the
<code>colData</code>.</p>
</div>
<div class="section level4">
<h4 id="transformation-reshaping">Transformation / reshaping<a class="anchor" aria-label="anchor" href="#transformation-reshaping"></a>
</h4>
<p>The <code>longFormat</code> or <code>wideFormat</code> functions will
“reshape” and combine experiments with each other and with
<code>colData</code> into one <code>DataFrame</code>. These functions
provide compatibility with most of the common R/Bioconductor functions
for regression, machine learning, and visualization.</p>
<p><em>Note</em>. It is best to remove technical replicates before
performing these operations to minimize the number of <code>NA</code>
values, esp. to <code>wideFormat</code>.</p>
<div class="section level5">
<h5 id="longformat">
<code>longFormat</code><a class="anchor" aria-label="anchor" href="#longformat"></a>
</h5>
<p>In <em>long</em> format a single column provides all assay results,
with additional optional <code>colData</code> columns whose values are
repeated as necessary. Here <em>assay</em> is the name of the
ExperimentList element, <em>primary</em> is the patient identifier
(rowname of colData), <em>rowname</em> is the assay rowname (in this
case genes), <em>colname</em> is the assay-specific identifier (column
name), <em>value</em> is the numeric measurement (gene expression, copy
number, presence of a non-silent mutation, etc), and following these are
the <em>vital_status</em> and <em>days_to_death</em> colData columns
that have been added:</p>
<div class="sourceCode" id="cb123"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">longFormat</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span><span class="op">)</span>, , <span class="op">]</span>,</span>
<span>    colDataCols <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 126 sampleMap rows not in names(experiments)</span></span></code></pre>
<pre><code><span><span class="co">## DataFrame with 518 rows and 7 columns</span></span>
<span><span class="co">##               assay      primary     rowname                colname     value</span></span>
<span><span class="co">##         &lt;character&gt;  &lt;character&gt; &lt;character&gt;            &lt;character&gt; &lt;numeric&gt;</span></span>
<span><span class="co">## 1   RNASeq2GeneNorm TCGA-OR-A5J1        TP53 TCGA-OR-A5J1-01A-11R..   563.401</span></span>
<span><span class="co">## 2   RNASeq2GeneNorm TCGA-OR-A5J1      CTNNB1 TCGA-OR-A5J1-01A-11R..  5634.467</span></span>
<span><span class="co">## 3   RNASeq2GeneNorm TCGA-OR-A5J2        TP53 TCGA-OR-A5J2-01A-11R..   165.481</span></span>
<span><span class="co">## 4   RNASeq2GeneNorm TCGA-OR-A5J2      CTNNB1 TCGA-OR-A5J2-01A-11R.. 62658.391</span></span>
<span><span class="co">## 5   RNASeq2GeneNorm TCGA-OR-A5J3        TP53 TCGA-OR-A5J3-01A-11R..   956.303</span></span>
<span><span class="co">## ...             ...          ...         ...                    ...       ...</span></span>
<span><span class="co">## 514       Mutations TCGA-PK-A5HA      CTNNB1 TCGA-PK-A5HA-01A-11D..         0</span></span>
<span><span class="co">## 515       Mutations TCGA-PK-A5HB        TP53 TCGA-PK-A5HB-01A-11D..         0</span></span>
<span><span class="co">## 516       Mutations TCGA-PK-A5HB      CTNNB1 TCGA-PK-A5HB-01A-11D..         0</span></span>
<span><span class="co">## 517       Mutations TCGA-PK-A5HC        TP53 TCGA-PK-A5HC-01A-11D..         0</span></span>
<span><span class="co">## 518       Mutations TCGA-PK-A5HC      CTNNB1 TCGA-PK-A5HC-01A-11D..         0</span></span>
<span><span class="co">##     vital_status days_to_death</span></span>
<span><span class="co">##        &lt;integer&gt;     &lt;integer&gt;</span></span>
<span><span class="co">## 1              1          1355</span></span>
<span><span class="co">## 2              1          1355</span></span>
<span><span class="co">## 3              1          1677</span></span>
<span><span class="co">## 4              1          1677</span></span>
<span><span class="co">## 5              0            NA</span></span>
<span><span class="co">## ...          ...           ...</span></span>
<span><span class="co">## 514            0            NA</span></span>
<span><span class="co">## 515            0            NA</span></span>
<span><span class="co">## 516            0            NA</span></span>
<span><span class="co">## 517            0            NA</span></span>
<span><span class="co">## 518            0            NA</span></span></code></pre>
</div>
<div class="section level5">
<h5 id="wideformat">
<code>wideFormat</code><a class="anchor" aria-label="anchor" href="#wideformat"></a>
</h5>
<p>In <em>wide</em> format, each feature from each assay goes in a
separate column, with one row per primary identifier (patient). Here,
each variable becomes a new column:</p>
<div class="sourceCode" id="cb126"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="http://waldronlab.io/MultiAssayExperiment/reference/MultiAssayExperiment-helpers.html" class="external-link">wideFormat</a></span><span class="op">(</span><span class="va">miniACC</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span><span class="op">)</span>, , <span class="op">]</span>,</span>
<span>    colDataCols <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## harmonizing input:</span></span>
<span><span class="co">##   removing 126 sampleMap rows not in names(experiments)</span></span></code></pre>
<pre><code><span><span class="co">## DataFrame with 92 rows and 9 columns</span></span>
<span><span class="co">##          primary vital_status days_to_death RNASeq2GeneNorm_TP53</span></span>
<span><span class="co">##      &lt;character&gt;    &lt;integer&gt;     &lt;integer&gt;            &lt;numeric&gt;</span></span>
<span><span class="co">## 1   TCGA-OR-A5J1            1          1355              563.401</span></span>
<span><span class="co">## 2   TCGA-OR-A5J2            1          1677              165.481</span></span>
<span><span class="co">## 3   TCGA-OR-A5J3            0            NA              956.303</span></span>
<span><span class="co">## 4   TCGA-OR-A5J4            1           423                   NA</span></span>
<span><span class="co">## 5   TCGA-OR-A5J5            1           365             1169.636</span></span>
<span><span class="co">## ...          ...          ...           ...                  ...</span></span>
<span><span class="co">## 88  TCGA-PK-A5H9            0            NA              890.866</span></span>
<span><span class="co">## 89  TCGA-PK-A5HA            0            NA              683.572</span></span>
<span><span class="co">## 90  TCGA-PK-A5HB            0            NA              237.370</span></span>
<span><span class="co">## 91  TCGA-PK-A5HC            0            NA                   NA</span></span>
<span><span class="co">## 92  TCGA-P6-A5OG            1           383              815.345</span></span>
<span><span class="co">##     RNASeq2GeneNorm_CTNNB1 gistict_TP53 gistict_CTNNB1 Mutations_TP53</span></span>
<span><span class="co">##                  &lt;numeric&gt;    &lt;numeric&gt;      &lt;numeric&gt;      &lt;numeric&gt;</span></span>
<span><span class="co">## 1                  5634.47            0              0              0</span></span>
<span><span class="co">## 2                 62658.39            0              1              1</span></span>
<span><span class="co">## 3                  6337.43            0              0              0</span></span>
<span><span class="co">## 4                       NA            1              0              0</span></span>
<span><span class="co">## 5                  5979.06            0              0              0</span></span>
<span><span class="co">## ...                    ...          ...            ...            ...</span></span>
<span><span class="co">## 88                 5258.99            0              0              0</span></span>
<span><span class="co">## 89                 8120.17           -1              0              0</span></span>
<span><span class="co">## 90                 5257.81           -1             -1              0</span></span>
<span><span class="co">## 91                      NA            1              1              0</span></span>
<span><span class="co">## 92                 6390.10           -1              1             NA</span></span>
<span><span class="co">##     Mutations_CTNNB1</span></span>
<span><span class="co">##            &lt;numeric&gt;</span></span>
<span><span class="co">## 1                  0</span></span>
<span><span class="co">## 2                  1</span></span>
<span><span class="co">## 3                  0</span></span>
<span><span class="co">## 4                  0</span></span>
<span><span class="co">## 5                  0</span></span>
<span><span class="co">## ...              ...</span></span>
<span><span class="co">## 88                 0</span></span>
<span><span class="co">## 89                 0</span></span>
<span><span class="co">## 90                 0</span></span>
<span><span class="co">## 91                 0</span></span>
<span><span class="co">## 92                NA</span></span></code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-packages-and-data-classes">back to top</a>
</p>
<div class="infobox">
<p><strong>Key points</strong></p>
<ul>
<li>Knowing how to subset a <code>MultiAssayExperiment</code> object is
important to be able to restrict observations and measurements to
particular phenotypes or sample types</li>
<li>Functions such as <code>longFormat</code> and
<code>wideFormat</code> are helpful for downstream analysis functions
that require a certain type of input format</li>
</ul>
</div>
</div>
</div>
</div>
<div class="section level3">
<h3 id="singlecellmultimodal-1">SingleCellMultiModal<a class="anchor" aria-label="anchor" href="#singlecellmultimodal-1"></a>
</h3>
<p><code>SingleCellMultiModal</code> is an <code>ExperimentHub</code>
package that serves multiple datasets obtained from GEO and other
sources and represents them as <code>MultiAssayExperiment</code>
objects. We provide several multi-modal datasets including scNMT, 10X
Multiome, seqFISH, CITEseq, SCoPE2, and others. The scope of the package
is is to provide data for benchmarking and analysis.</p>
<p>Users can access the data for a particular technology with the
appropriate function. For example, to obtain a small data.frame of what
data is available for <code>scNMT</code>, the user would enter:</p>
<div class="sourceCode" id="cb129"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/SingleCellMultiModal/man/scNMT.html" class="external-link">scNMT</a></span><span class="op">(</span><span class="st">"mouse_gastrulation"</span>, dry.run <span class="op">=</span> <span class="cn">TRUE</span>, version <span class="op">=</span> <span class="st">'2.0.0'</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##     ah_id         mode file_size rdataclass rdatadateadded rdatadateremoved</span></span>
<span><span class="co">## 1  EH3753      acc_cgi   21.1 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 2  EH3754     acc_CTCF    1.2 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 3  EH3755      acc_DHS   16.2 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 4  EH3756 acc_genebody   60.1 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 5  EH3757     acc_p300    0.2 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 6  EH3758 acc_promoter   33.8 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 7  EH3760      met_cgi   12.1 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 8  EH3761     met_CTCF    0.1 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 9  EH3762      met_DHS    3.9 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 10 EH3763 met_genebody   33.9 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 11 EH3764     met_p300    0.1 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 12 EH3765 met_promoter   18.7 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span>
<span><span class="co">## 13 EH3766          rna   43.5 Mb     matrix     2020-09-03             &lt;NA&gt;</span></span></code></pre>
<p>We can see that there are a few assays for each modality.</p>
<p>See the <code>SingleCellMultiModal</code> tutorial above for more
information.</p>
</div>
<div class="section level3">
<h3 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Ramos2020-ya" class="csl-entry">
Ramos, Marcel, Ludwig Geistlinger, Sehyun Oh, Lucas Schiffer, Rimsha
Azhar, Hanish Kodali, Ino de Bruijn, et al. 2020. <span>“Multiomic
Integration of Public Oncology Databases in Bioconductor.”</span>
<em>JCO Clin Cancer Inform</em> 4 (October): 958–71.
</div>
<div id="ref-Ramos2023-vs" class="csl-entry">
Ramos, Marcel, Martin Morgan, Ludwig Geistlinger, Vincent J Carey, and
Levi Waldron. 2023. <span>“<span>RaggedExperiment</span>: The Missing
Link Between Genomic Ranges and Matrices in Bioconductor.”</span>
<em>Bioinformatics</em>, May.
</div>
<div id="ref-Ramos2017-er" class="csl-entry">
Ramos, Marcel, Lucas Schiffer, Angela Re, Rimsha Azhar, Azfar Basunia,
Carmen Rodriguez, Tiffany Chan, et al. 2017. <span>“Software for the
Integration of Multiomics Experiments in Bioconductor.”</span>
<em>Cancer Res.</em> 77 (21): e39–42.
</div>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Marcel Ramos, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
