<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MultiAssayExperiment: Integrative workflow for public multi-omics data • MultiAssayWorkshop</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="MultiAssayExperiment: Integrative workflow for public multi-omics data">
<meta property="og:description" content="MultiAssayWorkshop">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MultiAssayWorkshop</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">3.11.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/Ramos_MultiAssayExperiment.html">Workshop</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Reference
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/OmicsTypes.html">Omics Types</a>
    </li>
    <li>
      <a href="../articles/TCGAutilsCheatsheet.html">TCGAutils Cheatsheet</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MultiAssayWorkshop">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="Ramos_MultiAssayExperiment_files/kePrint-0.0.1/kePrint.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MultiAssayExperiment: Integrative workflow for public multi-omics data</h1>
                        <h4 class="author">Marcel Ramos</h4>
            <address class="author_afil">
      Roswell Park Comprehensive Cancer Center, Buffalo, NY<br><h4 class="author">Ludwig Geistlinger</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="author">Levi Waldron</h4>
            <address class="author_afil">
      CUNY School of Public Health, New York, NY<br><h4 class="date">June 05, 2020</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/MultiAssayWorkshop/blob/master/vignettes/Ramos_MultiAssayExperiment.Rmd"><code>vignettes/Ramos_MultiAssayExperiment.Rmd</code></a></small>
      <div class="hidden name"><code>Ramos_MultiAssayExperiment.Rmd</code></div>

    </address>
</address>
</address>
</div>

    
    
<div id="multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment" class="section level1">
<h1 class="hasAnchor">
<a href="#multi-omic-integration-and-analysis-of-cbioportal-and-tcga-data-with-multiassayexperiment" class="anchor"></a>Multi-omic Integration and Analysis of cBioPortal and TCGA data with MultiAssayExperiment</h1>
<div id="instructor-names-and-contact-information" class="section level2">
<h2 class="hasAnchor">
<a href="#instructor-names-and-contact-information" class="anchor"></a>Instructor names and contact information</h2>
<ul>
<li>Marcel Ramos<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> <a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> (<a href="mailto:marcel.ramos@roswellpark.org" class="email">marcel.ramos@roswellpark.org</a>)</li>
<li>Ludwig Geistlinger<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>
</li>
<li>Levi Waldron<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>
</li>
</ul>
</div>
<div id="workshop-description" class="section level2">
<h2 class="hasAnchor">
<a href="#workshop-description" class="anchor"></a>Workshop Description</h2>
<p>This workshop demonstrates data management and analyses of multiple assays associated with a single set of biological specimens, using the <code>MultiAssayExperiment</code> data class and methods. It introduces the facilities made available by companion packages, <code>cBioPortalData</code>, <code>curatedTCGAData</code>, and <code>TCGAutils</code> to work with publicly available multi-omic data. Using the <code>MultiAssayExperiment</code> class, these packages make the management of multiple assays easier and more efficient. The workshop also covers, to a limited extent, relevant data classes such as <code>SummarizedExperiment</code> and <code>RangedSummarizedExperiment</code>, which provide efficient and powerful operations for representation of copy number, mutation, variant, and expression data that are represented by different genomic ranges for each specimen.</p>
<div id="pre-requisites" class="section level3">
<h3 class="hasAnchor">
<a href="#pre-requisites" class="anchor"></a>Pre-requisites</h3>
<ul>
<li>Basic knowledge of R</li>
<li>Familiarity with the GRanges and SummarizedExperiment classes</li>
<li>Familiarity with ’omics data types including copy number and gene expression</li>
</ul>
</div>
<div id="workshop-participation" class="section level3">
<h3 class="hasAnchor">
<a href="#workshop-participation" class="anchor"></a>Workshop Participation</h3>
<p>Participants will have a chance to build a <code>MultiAssayExperiment</code> object from scratch, and will also work with data and interfaces from the the <code>curatedTCGAData</code> and <code>cBioPortalData</code> packages.</p>
</div>
<div id="rbioconductor-packages-used" class="section level3">
<h3 class="hasAnchor">
<a href="#rbioconductor-packages-used" class="anchor"></a>R/Bioconductor packages used</h3>
<p>First, please make sure you have the latest Bioconductor release version 3.11. See <a href="https://www.bioconductor.org/install">Bioconductor installation</a> instructions:</p>
<div class="sourceCode" id="cb1"><html><body><pre class="r"><span class="kw">if</span> (!<span class="fu"><a href="https://rdrr.io/r/base/ns-load.html">requireNamespace</a></span>(<span class="st">"BiocManager"</span>, <span class="kw">quietly</span> <span class="kw">=</span> <span class="fl">TRUE</span>))
    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span>(<span class="st">"BiocManager"</span>)

<span class="co">## Bioconductor 3.11 Stable Release</span>
<span class="kw pkg">BiocManager</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/install.html">install</a></span>(<span class="kw">version</span> <span class="kw">=</span> <span class="st">"3.11"</span>)</pre></body></html></div>
<ul>
<li><a href="https://bioconductor.org/packages/MultiAssayExperiment">MultiAssayExperiment</a></li>
<li><a href="https://bioconductor.org/packages/GenomicRanges">GenomicRanges</a></li>
<li><a href="https://bioconductor.org/packages/RaggedExperiment">RaggedExperiment</a></li>
<li><a href="https://bioconductor.org/packages/curatedTCGAData">curatedTCGAData</a></li>
<li><a href="https://github.com/waldronlab/cBioPortalData">cBioPortalData</a></li>
<li><a href="https://bioconductor.org/packages/SummarizedExperiment">SummarizedExperiment</a></li>
<li><a href="https://bioconductor.org/packages/TCGAutils">TCGAutils</a></li>
<li><a href="https://bioconductor.org/packages/UpSetR">UpSetR</a></li>
<li><a href="https://bioconductor.org/packages/AnnotationFilter">AnnotationFilter</a></li>
<li><a href="https://bioconductor.org/packages/EnsDb.Hsapiens.v86">EnsDb.Hsapiens.v86</a></li>
<li><a href="https://cran.r-project.org/package=survival">survival</a></li>
<li><a href="https://cran.r-project.org/package=survminer">survminer</a></li>
<li><a href="https://cran.r-project.org/package=pheatmap">pheatmap</a></li>
</ul>
<div class="sourceCode" id="cb2"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">MultiAssayExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">GenomicRanges</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">RaggedExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">curatedTCGAData</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">cBioPortalData</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">GenomicDataCommons</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">SummarizedExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">SingleCellExperiment</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">TCGAutils</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">UpSetR</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">mirbase.db</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">AnnotationFilter</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">EnsDb.Hsapiens.v86</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">survival</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">survminer</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">pheatmap</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">readxl</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">readr</span>)
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">kableExtra</span>)</pre></body></html></div>
</div>
</div>
<div id="time-outline" class="section level2">
<h2 class="hasAnchor">
<a href="#time-outline" class="anchor"></a>Time outline</h2>
<p>1h 45m total</p>
<table class="table">
<thead><tr class="header">
<th>Activity</th>
<th>Time</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>Overview of key data classes</td>
<td>25m</td>
</tr>
<tr class="even">
<td>Building a <code>MultiAssayExperiment</code> from scratch</td>
<td>10m</td>
</tr>
<tr class="odd">
<td>Creating and importing TCGA multi-assay datasets</td>
<td>10m</td>
</tr>
<tr class="even">
<td>Subsetting and reshaping multi-assay data</td>
<td>20m</td>
</tr>
<tr class="odd">
<td>Plotting, correlation, and other analyses</td>
<td>20m</td>
</tr>
<tr class="even">
<td>Introduction to <code>curatedTCGAData</code>, <code>cBioPortalData</code>, and <code>TCGAutils</code>
</td>
<td>20m</td>
</tr>
</tbody>
</table>
</div>
<div id="workshop-goals-and-objectives" class="section level2">
<h2 class="hasAnchor">
<a href="#workshop-goals-and-objectives" class="anchor"></a>Workshop goals and objectives</h2>
<div id="learning-goals" class="section level3">
<h3 class="hasAnchor">
<a href="#learning-goals" class="anchor"></a>Learning goals</h3>
<ul>
<li>overview common data classes available in Bioconductor</li>
<li>understand appropriate data structures for different ’omics data types</li>
<li>identify advantages of providing integrative data in an analysis-ready platform</li>
<li>gain familiarity with <code>cBioPortalData</code> and <code>curatedTCGAData</code>
</li>
<li>explore available functionality in <code>TCGAutils</code> for the management and coordination of multi-omic data</li>
</ul>
</div>
<div id="learning-objectives" class="section level3">
<h3 class="hasAnchor">
<a href="#learning-objectives" class="anchor"></a>Learning objectives</h3>
<ul>
<li>use <code>curatedTCGAData</code> and <code>cBioPortalData</code> to create on-the-fly multi-omics data objects</li>
<li>apply knowlege of <code>MultiAssayExperiment</code> to represent other multi’omics data</li>
<li>understand basic data management techniques with MultiAssayExperiment such as subsetting, reshaping, growing, and extraction</li>
<li>adapt <code>MultiAssayExperiment</code> data with packages for differential expression, machine learning, and plotting</li>
</ul>
</div>
</div>
<div id="overview-of-key-packages-and-data-classes" class="section level2">
<h2 class="hasAnchor">
<a href="#overview-of-key-packages-and-data-classes" class="anchor"></a>Overview of key packages and data classes</h2>
<div id="packages" class="section level3">
<h3 class="hasAnchor">
<a href="#packages" class="anchor"></a>Packages</h3>
<div id="multiassayexperiment" class="section level4">
<h4 class="hasAnchor">
<a href="#multiassayexperiment" class="anchor"></a><code>MultiAssayExperiment</code>
</h4>
<p>The <code>MultiAssayExperiment</code> package provides an integrative representation for multi-omics data on a set of samples. Modelled after the <code>SummarizedExperiment</code> representation for expression data, the <code>MultiAssayExperiment</code> data container provides efficient and easy-to-use operations for manipulating multiple sets of data such as copy number alterations, mutations, proteomics, methylation, and other types of data. Compatibility with the <code>MultiAssayExperiment</code> container hinges on a small set of requirements making the data representation flexible to many of the currently available Bioconductor classes.</p>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic" width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic
</p>
</div>
</div>
<div id="cbioportaldata" class="section level4">
<h4 class="hasAnchor">
<a href="#cbioportaldata" class="anchor"></a><code>cBioPortalData</code>
</h4>
<p>The <code>cBioPortalData</code> package provides an R/Bioconductor interface to cBioPortal data. The <code>cBioPortalData</code> package makes use of the revamped API and provides an internal caching mechanism to avoid re-querying the API. It allows users to download data from <a href="https://cBioPortal.org" class="uri">https://cBioPortal.org</a> through customized queries that are handled for the user <em>in the background</em>. We provide an easy to use interface that requires minimal knowledge of the cBioPortal data model and API.</p>
<p>First create an API object to be able to query the website:</p>
<div class="sourceCode" id="cb3"><html><body><pre class="r"><span class="no">cbio</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html">cBioPortal</a></span>()
<span class="no">cbio</span></pre></body></html></div>
<pre><code>## service: cBioPortal
## tags(); use cbioportal$&lt;tab completion&gt;:
## # A tibble: 70 x 3
##    tag          operation               summary                                 
##    &lt;chr&gt;        &lt;chr&gt;                   &lt;chr&gt;                                   
##  1 Cancer Types getAllCancerTypesUsing… Get all cancer types                    
##  2 Cancer Types getCancerTypeUsingGET   Get a cancer type                       
##  3 Clinical At… fetchClinicalAttribute… Fetch clinical attributes               
##  4 Clinical At… getAllClinicalAttribut… Get all clinical attributes in the spec…
##  5 Clinical At… getAllClinicalAttribut… Get all clinical attributes             
##  6 Clinical At… getClinicalAttributeCo… Get counts for clinical attributes acco…
##  7 Clinical At… getClinicalAttributeIn… Get specified clinical attribute        
##  8 Clinical Da… fetchAllClinicalDataIn… Fetch clinical data by patient IDs or s…
##  9 Clinical Da… fetchClinicalDataUsing… Fetch clinical data by patient IDs or s…
## 10 Clinical Da… getAllClinicalDataInSt… Get all clinical data in a study        
## # … with 60 more rows
## tag values:
##   Cancer Types, Clinical Attributes, Clinical Data, Clinical Events,
##   Copy Number Segments, Discrete Copy Number Alterations, Gene Panels,
##   Generic Assays, Genes, Molecular Data, Molecular Profiles, Mutations,
##   Patients, Reference Genome Genes, Resource Data, Resource
##   Definitions, Sample Lists, Samples, Structural Variants, Studies
## schemas():
##   AlleleSpecificCopyNumber, CancerStudy, CancerStudyTags,
##   ClinicalAttribute, ClinicalAttributeCount
##   # ... with 51 more elements</code></pre>
<p>To see what studies are available from the <code>cBioPortal</code>, use <code>getStudies</code> on the <code>cbio</code> API object:</p>
<div class="sourceCode" id="cb5"><html><body><pre class="r"><span class="kw pkg">knitr</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/cBioPortalData/man/cBioPortal.html">getStudies</a></span>(<span class="no">cbio</span>), <span class="kw">align</span> <span class="kw">=</span> <span class="st">"l"</span>, <span class="kw">escape</span> <span class="kw">=</span> <span class="fl">FALSE</span>,
    <span class="kw">caption</span> <span class="kw">=</span> <span class="st">"List of available studies from the cBioPortal"</span>) <span class="kw">%&gt;%</span>
    <span class="kw pkg">kableExtra</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html">kable_styling</a></span>(
        <span class="kw">bootstrap_options</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"hover"</span>, <span class="st">"striped"</span>, <span class="st">"responsive"</span>),
        <span class="kw">full_width</span> <span class="kw">=</span> <span class="fl">FALSE</span>
    )</pre></body></html></div>
<table class="table table table-hover table-striped table-responsive">
<caption>
List of available studies from the cBioPortal
</caption>
<thead><tr>
<th style="text-align:left;">
name
</th>
<th style="text-align:left;">
shortName
</th>
<th style="text-align:left;">
description
</th>
<th style="text-align:left;">
publicStudy
</th>
<th style="text-align:left;">
pmid
</th>
<th style="text-align:left;">
citation
</th>
<th style="text-align:left;">
groups
</th>
<th style="text-align:left;">
status
</th>
<th style="text-align:left;">
importDate
</th>
<th style="text-align:left;">
allSampleCount
</th>
<th style="text-align:left;">
studyId
</th>
<th style="text-align:left;">
cancerTypeId
</th>
<th style="text-align:left;">
referenceGenome
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (National Cancer Centre of Singapore, Nat Genet 2013)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (NCCS)
</td>
<td style="text-align:left;">
Exome sequencing of 15 cholangiocarcinoma cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24185513
</td>
<td style="text-align:left;">
Chan-on et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-14 00:00:00
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
chol_nccs_2013
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cutaneous T Cell Lymphoma (Columbia U, Nat Genet 2015)
</td>
<td style="text-align:left;">
CTCL (Columbia 2015)
</td>
<td style="text-align:left;">
Whole-Exome Sequencing (WXS) of tumor-normal sample pairs from 25 patients with Sezary Syndrome and 17 patients with other Cutaneous T Cell Lymphoma (CTCLs).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26551667
</td>
<td style="text-align:left;">
Da Silva Almeida et al. Nat Genet 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-15 00:00:00
</td>
<td style="text-align:left;">
43
</td>
<td style="text-align:left;">
ctcl_columbia_2015
</td>
<td style="text-align:left;">
nhl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Squamous Cell Carcinoma (UCLA, Nat Genet 2014)
</td>
<td style="text-align:left;">
ESCC (UCLA 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing (WXS) or targeted deep sequencing (TDS) of 139 paired esophageal squamous cell carcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24686850
</td>
<td style="text-align:left;">
Lin et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
139
</td>
<td style="text-align:left;">
escc_ucla_2014
</td>
<td style="text-align:left;">
escc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Oral Squamous Cell Carcinoma (MD Anderson, Cancer Discov 2013)
</td>
<td style="text-align:left;">
Head &amp; neck (MDA)
</td>
<td style="text-align:left;">
Comprehensive profiling of 40 oral squamous cell carcinoma tumor/normal sample pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23619168
</td>
<td style="text-align:left;">
Pickering et al. Cancer Discov 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
40
</td>
<td style="text-align:left;">
hnsc_mdanderson_2013
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hepatocellular Carcinomas (INSERM, Nat Genet 2015)
</td>
<td style="text-align:left;">
HCC (Inserm, 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 243 liver tumors and matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25822088
</td>
<td style="text-align:left;">
Schulze et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
243
</td>
<td style="text-align:left;">
hcc_inserm_fr_2015
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uveal Melanoma (QIMR, Oncotarget 2016)
</td>
<td style="text-align:left;">
UM (QIMR)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of tumor/normal pairs from 28 uveal melanoma samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26683228
</td>
<td style="text-align:left;">
Johansson et al. Oncotarget 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-19 00:00:00
</td>
<td style="text-align:left;">
28
</td>
<td style="text-align:left;">
um_qimr_2016
</td>
<td style="text-align:left;">
um
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroblastoma (AMC Amsterdam, Nature 2012)
</td>
<td style="text-align:left;">
NBL (AMC)
</td>
<td style="text-align:left;">
Whole genome sequencing of 87 primary neuroblastoma tumors and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22367537
</td>
<td style="text-align:left;">
Molenaar et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
87
</td>
<td style="text-align:left;">
nbl_amc_2012
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Nasopharyngeal Carcinoma (Singapore, Nat Genet 2014)
</td>
<td style="text-align:left;">
NPC (Singapore)
</td>
<td style="text-align:left;">
Whole exome sequencing of 56 NPC tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24952746
</td>
<td style="text-align:left;">
Lin et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
56
</td>
<td style="text-align:left;">
npc_nusingapore
</td>
<td style="text-align:left;">
npc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thymic Epithelial Tumors (NCI, Nat Genet 2014)
</td>
<td style="text-align:left;">
TET (NCI)
</td>
<td style="text-align:left;">
Whole exome sequencing of 28 thymic epithelial tumors (TETs) with matched normals
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24974848
</td>
<td style="text-align:left;">
Petrini at el. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
tet_nci_2014
</td>
<td style="text-align:left;">
tet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroblastoma (Broad, Nature 2015)
</td>
<td style="text-align:left;">
NBL (Cologne 2015)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 56 neuroblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26466568
</td>
<td style="text-align:left;">
Peifer et al. Nature 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
56
</td>
<td style="text-align:left;">
nbl_ucologne_2015
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Myelodysplasia (UTokyo, Nature 2011)
</td>
<td style="text-align:left;">
MDS (Tokyo)
</td>
<td style="text-align:left;">
Whole exome sequencing of 29 myelodysplasia tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21909114
</td>
<td style="text-align:left;">
Yoshida et al. Nature 2011
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
mds_tokyo_2011
</td>
<td style="text-align:left;">
mds
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Hodgkin Lymphoma (BCGSC, Nature 2011)
</td>
<td style="text-align:left;">
NHL (bcgsc 2011)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing of 14 NHL tumors and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21796119
</td>
<td style="text-align:left;">
Morin et al. Nature 2011
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
nhl_bcgsc_2011
</td>
<td style="text-align:left;">
nhl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-cell Lymphoma (BCGSC, Blood 2013)
</td>
<td style="text-align:left;">
NHL (BCGSC 2013)
</td>
<td style="text-align:left;">
Whole genome sequencing of 40 DLBCL tumor/normal pairs and 13 cell lines.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23699601
</td>
<td style="text-align:left;">
Morin et al. Blood 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-20 00:00:00
</td>
<td style="text-align:left;">
53
</td>
<td style="text-align:left;">
nhl_bcgsc_2013
</td>
<td style="text-align:left;">
nhl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Insulinoma (Shanghai, Nat Commun 2013)
</td>
<td style="text-align:left;">
Panet (Shanghai 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 10 benign insulinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24326773
</td>
<td style="text-align:left;">
Cao et al. Nat Commun 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-21 00:00:00
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
panet_shanghai_2013
</td>
<td style="text-align:left;">
panet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pleural Mesothelioma (NYU, Cancer Res 2015)
</td>
<td style="text-align:left;">
PLMESO (NYU 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 22 malignant pleural mesotheliomas (PLMESO) tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25488749
</td>
<td style="text-align:left;">
Guo et al. Cancer Res 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-21 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
plmeso_nyu_2015
</td>
<td style="text-align:left;">
plmeso
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cystic Tumor of the Pancreas (Johns Hopkins, PNAS 2011)
</td>
<td style="text-align:left;">
Pancreas (JHU 2011)
</td>
<td style="text-align:left;">
Whole-exome sequencing of major neoplastic cyst types of the pancreas: 8 serous cystadenomas (SCAs), 8 intraductal papillary mucinous neoplasms (IPMNs), 8 mucinous cystic neoplasms (MCNs), and 8 solid pseudopapillary neoplasms (SPNs).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22158988
</td>
<td style="text-align:left;">
Wu et al. PNAS 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-21 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
pact_jhu_2011
</td>
<td style="text-align:left;">
pact
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pilocytic Astrocytoma (ICGC, Nature Genetics 2013)
</td>
<td style="text-align:left;">
PAST (Nature Genetics, 2013)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 96 pilocytic astrocytoma tumor/normal pairs. Conducted by the International Cancer Genome Consortium (ICGC) PedBrain Tumor Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23817572
</td>
<td style="text-align:left;">
Jones et al. Nature Genetics 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-21 00:00:00
</td>
<td style="text-align:left;">
96
</td>
<td style="text-align:left;">
past_dkfz_heidelberg_2013
</td>
<td style="text-align:left;">
past
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (RIKEN, Nat Genet 2012)
</td>
<td style="text-align:left;">
Liver (RIKEN)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 27 hepatocellular carcinoma tumor/norma pairs from RIKEN.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22634756
</td>
<td style="text-align:left;">
Fujimoto et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-22 00:00:00
</td>
<td style="text-align:left;">
27
</td>
<td style="text-align:left;">
lihc_riken
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (ICGC, Nature 2012)
</td>
<td style="text-align:left;">
MBL (ICGC)
</td>
<td style="text-align:left;">
Comprehensive profiling of 125 matched tumor and germline medulloblastoma samples
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22832583
</td>
<td style="text-align:left;">
Jones et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-22 00:00:00
</td>
<td style="text-align:left;">
125
</td>
<td style="text-align:left;">
mbl_icgc
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (Sickkids, Nature 2016)
</td>
<td style="text-align:left;">
MBL (Sickkids 2016)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 46 recurrent medulloblastoma samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26760213
</td>
<td style="text-align:left;">
Morrissy et al. Nature 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-22 00:00:00
</td>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
mbl_sickkids_2016
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (BGI, Nat Genet 2012)
</td>
<td style="text-align:left;">
ccRCC (BGI 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 10 ccRCC tumor/normal pairs and targeted sequencing of 88 clear cell carcinoma tumor/normal pairs. Generated by BGI-Shenzhen.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22138691
</td>
<td style="text-align:left;">
Guo et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-02-24 00:00:00
</td>
<td style="text-align:left;">
98
</td>
<td style="text-align:left;">
kirc_bgi
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (Johns Hopkins, Science 2011)
</td>
<td style="text-align:left;">
Head &amp; neck (JHU)
</td>
<td style="text-align:left;">
Exome sequencing of 32 head and neck squamous cell carcinoma tumor/normal sample pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21798897
</td>
<td style="text-align:left;">
Agrawal et al. Science 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-07 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
hnsc_jhu
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hepatocellular Adenoma (INSERM, Cancer Cell 2014)
</td>
<td style="text-align:left;">
Liad (Inserm 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 46 liver tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24735922
</td>
<td style="text-align:left;">
Pilati et al. Cancer Cell 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-07 00:00:00
</td>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
liad_inserm_fr_2014
</td>
<td style="text-align:left;">
liad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ampullary Carcinoma (Baylor College of Medicine, Cell Reports 2016)
</td>
<td style="text-align:left;">
Ampullary Carcinoma (BCM 2016)
</td>
<td style="text-align:left;">
Exome sequencing analysis of 160 cancers arising in the periampullary region, 62 of these clearly arising from either the bile duct (n = 44) or the duodenum (n = 18) and 98 for which the epithelium of origin could not be clearly defined morphologically (AMPCA).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26804919
</td>
<td style="text-align:left;">
Gingras et al. Cell Rep 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
160
</td>
<td style="text-align:left;">
ampca_bcm_2016
</td>
<td style="text-align:left;">
ampca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (DFCI/MSKCC, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Bladder (DFCI/MSKCC 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal samples from 50 patients treated with cisplatin-based chemotherapy for muscle invasive urothelial carcinoma.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25096233
</td>
<td style="text-align:left;">
Van Allen et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
50
</td>
<td style="text-align:left;">
blca_dfarber_mskcc_2014
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Ewing Sarcoma (DFCI, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Ewing Sarcoma (DFCI)
</td>
<td style="text-align:left;">
Whole exome sequencing of 96 pediatric Ewing Sarcoma tumors and 11 cell lines.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25186949
</td>
<td style="text-align:left;">
Crompton et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
107
</td>
<td style="text-align:left;">
es_dfarber_broad_2014
</td>
<td style="text-align:left;">
es
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Renal Non-Clear Cell Carcinoma (Genentech, Nat Genet 2014)
</td>
<td style="text-align:left;">
nccRCC (Genentech 2014)
</td>
<td style="text-align:left;">
Exome sequencing of 139 samples with matched normals and RNA-sequencing of 7 samples (5 samples with matched normals) from 145 Renal Non-Clear Cell Carcinoma (nccRCC) patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25401301
</td>
<td style="text-align:left;">
Durinck et al. Nat Genet 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
146
</td>
<td style="text-align:left;">
nccrcc_genentech_2014
</td>
<td style="text-align:left;">
nccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (Pfizer and UHK, Nat Genet 2014)
</td>
<td style="text-align:left;">
Stomach (Pfizer UHK)
</td>
<td style="text-align:left;">
Whole genome sequencing of 100 gastric cancer tumor-normal pairs from the University of Hong Kong and Pfizer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24816253
</td>
<td style="text-align:left;">
Wang et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
100
</td>
<td style="text-align:left;">
stad_pfizer_uhongkong
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (Broad, PNAS 2012)
</td>
<td style="text-align:left;">
DLBCL (Broad 2012)
</td>
<td style="text-align:left;">
Whole exome sequencing of 58 primary DLBCL samples and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22343534
</td>
<td style="text-align:left;">
Lohr et al. PNAS 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
58
</td>
<td style="text-align:left;">
dlbc_broad_2012
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
MSS Mixed Solid Tumors (Broad/Dana-Farber, Nat Genet 2018)
</td>
<td style="text-align:left;">
Mixed Tumor (DFCI)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal pairs and corresponding clinical outcomes from patients treated with immune checkpoint inhibitors across multiple cancer types including melanoma, non-small cell lung cancer, head and neck cancer, and bladder cancer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30150660
</td>
<td style="text-align:left;">
Miao et al. Nat Genet 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-21 00:00:00
</td>
<td style="text-align:left;">
249
</td>
<td style="text-align:left;">
mixed_allen_2018
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Renal Clear Cell Carcinoma (UTokyo, Nat Genet 2013)
</td>
<td style="text-align:left;">
ccRCC (UTokyo)
</td>
<td style="text-align:left;">
Whole exome and/or whole genome sequencing of 106 clear cell renal cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23797736
</td>
<td style="text-align:left;">
Sato et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
106
</td>
<td style="text-align:left;">
ccrcc_utokyo_2013
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Intrahepatic Cholangiocarcinoma (JHU, Nat Genet 2013)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (JHU)
</td>
<td style="text-align:left;">
Exome sequencing of tumor/normal pairs from 32 intrahepatic cholangiocarcinoma cases and 8 gall bladder carcinoma cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24185509
</td>
<td style="text-align:left;">
Jiao et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
40
</td>
<td style="text-align:left;">
chol_jhu_2013
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Melanoma (Broad/Dana Farber, Nature 2012)
</td>
<td style="text-align:left;">
Melanoma (Broad/DFCI)
</td>
<td style="text-align:left;">
Whole Genome Sequencing of 25 metastatic melanoma samples with matched normals
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22622578
</td>
<td style="text-align:left;">
Berger et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
26
</td>
<td style="text-align:left;">
skcm_broad_dfarber
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Carcinosarcoma (Johns Hopkins, Nat Commun 2014)
</td>
<td style="text-align:left;">
ucs (Johns Hopkins 2014)
</td>
<td style="text-align:left;">
Whole exome sequencing (WXS) of 22 uterine carcinosarcoma/uterine malignant mixed Mullerian (UCS) tumors with matched normals. <a href="http://www.nature.com/ncomms/2014/140919/ncomms6006/extref/ncomms6006-s1.pdf"> </a>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25233892
</td>
<td style="text-align:left;">
Jones et al. Nat Commun 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
ucs_jhu_2014
</td>
<td style="text-align:left;">
ucs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma(Broad, Cancer Discov 2014)
</td>
<td style="text-align:left;">
SKCM (BROAD 2014)
</td>
<td style="text-align:left;">
Exome Sequencing of tumors (with matched normals) from 45 patients with BRAFV600-mutant metastatic melanoma who received vemurafenib or dabrafenib monotherapy.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24265153
</td>
<td style="text-align:left;">
Van Allen et al. Cancer Discov. 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
skcm_broad_brafresist_2012
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (Sanger/MDA, JCI 2013)
</td>
<td style="text-align:left;">
ACyC (Sanger 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 24 ACCs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23778141
</td>
<td style="text-align:left;">
Stephens et al. JCI 2013
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
24
</td>
<td style="text-align:left;">
acyc_sanger_2013
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Squamous Cell Carcinoma of the Vulva (CUK, Exp Mol Med 2018)
</td>
<td style="text-align:left;">
VSC (CUK)
</td>
<td style="text-align:left;">
Whole exome sequencing of tumor/normal pairs from 15 vulvar squamous cell carcinomas.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29422544
</td>
<td style="text-align:left;">
Han et al. Exp Mol Med 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-23 00:00:00
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
vsc_cuk_2018
</td>
<td style="text-align:left;">
vsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Adenocarcinoma (DFCI, Nat Genet 2013)
</td>
<td style="text-align:left;">
Esophagus (Broad)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 149 tumor/normal pairs and whole-genome sequencing of 16 pairs from 149 esophageal adenocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23525077
</td>
<td style="text-align:left;">
Dulak et al. Nat Genet 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
151
</td>
<td style="text-align:left;">
esca_broad
</td>
<td style="text-align:left;">
esca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acinar Cell Carcinoma of the Pancreas (JHU, J Pathol 2014)
</td>
<td style="text-align:left;">
PAAC (JHU)
</td>
<td style="text-align:left;">
Whole exome sequencing of 23 surgically resected pancreatic carcinomas with acinar differentiation and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24293293
</td>
<td style="text-align:left;">
Jial et al. J Pathol 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
23
</td>
<td style="text-align:left;">
paac_jhu_2014
</td>
<td style="text-align:left;">
paac
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Rhabdomyosarcoma (NIH, Cancer Discov 2014)
</td>
<td style="text-align:left;">
RMS (NIH)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 43 rhabdomyosarcoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24436047
</td>
<td style="text-align:left;">
Shern et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
43
</td>
<td style="text-align:left;">
rms_nih_2014
</td>
<td style="text-align:left;">
rms
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Gastric Adenocarcinoma (TMUCIH, PNAS 2015)
</td>
<td style="text-align:left;">
Stomach (TMUCIH 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 78 Gastric Adenocarcinomas and their matched normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25583476
</td>
<td style="text-align:left;">
Chen et al. PNAS 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
egc_tmucih_2015
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (British Columbia, Nature 2012)
</td>
<td style="text-align:left;">
Breast (BCCRC 2012)
</td>
<td style="text-align:left;">
Whole genome/exome sequencing analysis of 65 breast cancer samples
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22495314
</td>
<td style="text-align:left;">
Shah et al. Nature
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
65
</td>
<td style="text-align:left;">
brca_bccrc
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Neuroendocrine Tumors (Multi-Institute, Nature 2017)
</td>
<td style="text-align:left;">
ACRNET (Nature, 2017)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 98 pancreatic neuroendocrine tumours (PanNETs) and matched normal DNA pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28199314
</td>
<td style="text-align:left;">
Scarpa et al. Nature 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
98
</td>
<td style="text-align:left;">
panet_arcnet_2017
</td>
<td style="text-align:left;">
panet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Primary Central Nervous System Lymphoma (Mayo Clinic, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
PCNSL (Mayo Clinic)
</td>
<td style="text-align:left;">
Exome sequencing of 19 immunocompetent primary central nervous system lymphoma (PCNSL) samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25991819
</td>
<td style="text-align:left;">
Braggio et al. Clin Cancer Res 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-26 00:00:00
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
pcnsl_mayo_2015
</td>
<td style="text-align:left;">
pcnsl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Multiple Myeloma (Broad, Cancer Cell 2014)
</td>
<td style="text-align:left;">
MM (Broad)
</td>
<td style="text-align:left;">
Whole-exome or whole-genome sequencing of 203 multiple myeloma paired tumor/normal sample pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24434212
</td>
<td style="text-align:left;">
Lohr et al. Cancer Cell 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-27 00:00:00
</td>
<td style="text-align:left;">
211
</td>
<td style="text-align:left;">
mm_broad
</td>
<td style="text-align:left;">
pcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroblastoma (Broad, Nat Genet 2013)
</td>
<td style="text-align:left;">
NBL (Broad)
</td>
<td style="text-align:left;">
Whole Genome and Whole Exome sequencing of 240 high-risk neuroblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23334666
</td>
<td style="text-align:left;">
Pugh et al. Nat Genet 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-27 00:00:00
</td>
<td style="text-align:left;">
240
</td>
<td style="text-align:left;">
nbl_broad_2013
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Rhabdoid Cancer (BCGSC, Cancer Cell 2016)
</td>
<td style="text-align:left;">
MRT (CELL 2016)
</td>
<td style="text-align:left;">
Whole-genome sequencing of tumor-normal sample pairs from 40 patients with Malignant Rhabdoid Cancer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26977886
</td>
<td style="text-align:left;">
E. Chun et al. Cancer Cell 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-03-27 00:00:00
</td>
<td style="text-align:left;">
40
</td>
<td style="text-align:left;">
mrt_bcgsc_2016
</td>
<td style="text-align:left;">
mrt
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Gallbladder Carcinoma (Shanghai, Nat Genet 2014)
</td>
<td style="text-align:left;">
GBC (Shanghai)
</td>
<td style="text-align:left;">
Whole exome sequencing of 32 gallbladder carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24997986
</td>
<td style="text-align:left;">
Maolan Li et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
gbc_shanghai_2014
</td>
<td style="text-align:left;">
gbc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (MSKCC, Science 2015)
</td>
<td style="text-align:left;">
Lung adeno (MSKCC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 34 non small cell lung cancer tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25765070
</td>
<td style="text-align:left;">
Rizvi et al. Science 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
35
</td>
<td style="text-align:left;">
luad_mskcc_2015
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (Broad, Nature 2012)
</td>
<td style="text-align:left;">
MBL (Broad)
</td>
<td style="text-align:left;">
Whole exome sequencing of 92 primary medulloblastoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22820256
</td>
<td style="text-align:left;">
Pugh et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
92
</td>
<td style="text-align:left;">
mbl_broad_2012
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Stomach (TCGA pub)
</td>
<td style="text-align:left;">
Single nucleotide polymorphism array somatic copy-number analysis, whole-exome sequencing, mRNA sequencing, miRNA sequencing, array-based DNA methylation profiling and reverse-phase protein arrays of 295 primary gastric adenocarcinomas tumors with matched normals. TCGA Stomach Adenocarcinoma Project; raw data at the <a href="https://tcga-data.nci.nih.gov/docs/publications/stad_2014/">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25079317
</td>
<td style="text-align:left;">
TCGA, Nature 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
295
</td>
<td style="text-align:left;">
stad_tcga_pub
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Pan-cancer (Columbia U, Genome Med 2016)
</td>
<td style="text-align:left;">
pediatric-cancers
</td>
<td style="text-align:left;">
Whole exome sequencing of 103 samples (88 with matched normals) from 88 high-risk pediatric cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28007021
</td>
<td style="text-align:left;">
Oberg et al. Genome Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-02 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
mixed_pipseq_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (FMI, Am J Surg Pathl. 2014)
</td>
<td style="text-align:left;">
ACyC (FMI 2014)
</td>
<td style="text-align:left;">
Comprehensive genomic profiling of 28 metastatic ACCs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24418857
</td>
<td style="text-align:left;">
Ross et al. Am J Surg Pathl 2014
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
28
</td>
<td style="text-align:left;">
acyc_fmi_2014
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (JHU, Cancer Prev Res 2016)
</td>
<td style="text-align:left;">
ACyC (JHU 2016)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 25 adenoid cystic carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26862087
</td>
<td style="text-align:left;">
Rettig et al, Cancer Prev Res 2016
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
25
</td>
<td style="text-align:left;">
acyc_jhu_2016
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MDA, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
ACyC (MDA 2015)
</td>
<td style="text-align:left;">
WGS of 21 salivary ACCs and targeted molecular analyses of a validation set (81 patients).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26631609
</td>
<td style="text-align:left;">
Mitani et al. Clin Cancer Res 2015
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
102
</td>
<td style="text-align:left;">
acyc_mda_2015
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Lymphoblastic Leukemia (St Jude, Nat Genet 2016)
</td>
<td style="text-align:left;">
ALL (St. Jude, Nat Gen)
</td>
<td style="text-align:left;">
Whole-genome and/or whole-exome sequencing of ERG-altered B-ALL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27776115
</td>
<td style="text-align:left;">
Zhang et al. Nat Genet 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
73
</td>
<td style="text-align:left;">
all_stjude_2016
</td>
<td style="text-align:left;">
bll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
The Angiosarcoma Project - Count Me In (Provisional, September 2018)
</td>
<td style="text-align:left;">
(ASC Project)
</td>
<td style="text-align:left;">
The Angiosarcoma Project is a patient-driven initiative. This study includes genomic data, patient-reported data (pre-pended as PRD), medical record data (MedR), and pathology report data (PATH). As these data were generated in a research, not a clinical, laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. All annotations have been de-identified. More information is available at www.ascproject.org. Questions about these data can be directed to <a href="mailto:data@ascproject.org" class="email">data@ascproject.org</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
angs_project_painter_2018
</td>
<td style="text-align:left;">
angs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Fibroepithelial Tumors (Duke-NUS, Nat Genet 2015)
</td>
<td style="text-align:left;">
BFN (Duke-NUS)
</td>
<td style="text-align:left;">
Whole exome sequencing of 22 phyllodes tumors
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26437033
</td>
<td style="text-align:left;">
Tan et al. Nat Genet 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
bfn_duke_nus_2015
</td>
<td style="text-align:left;">
bfn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Carcinoma (Cornell/Baylor/MDACC, Nat Comm 2019)
</td>
<td style="text-align:left;">
UTUC (Cornell/Baylor/MDACC 2019)
</td>
<td style="text-align:left;">
Whole exome sequencing (37 samples) and RNA-seq analysis (32 samples) of 47 upper tract urothelial carcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-26 00:00:00
</td>
<td style="text-align:left;">
47
</td>
<td style="text-align:left;">
utuc_cornell_baylor_mdacc_2019
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (MSKCC, Eur Urol 2014)
</td>
<td style="text-align:left;">
Bladder (MSKCC 2014)
</td>
<td style="text-align:left;">
Genomic Predictors of Survival in Patients with High-grade Urothelial Carcinoma of the Bladder. Targeted sequencing of 109 tumor samples (MSKCC, European Urology 2014).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25092538
</td>
<td style="text-align:left;">
Kim et al. Eur Urol 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-27 00:00:00
</td>
<td style="text-align:left;">
109
</td>
<td style="text-align:left;">
blca_mskcc_solit_2014
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Nonmuscle Invasive Bladder Cancer (MSK Eur Urol 2017)
</td>
<td style="text-align:left;">
NMIBC (MSK Our Urol 2017)
</td>
<td style="text-align:left;">
Genomic Landscape and Predictors of Response for High Risk NMIBC
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28583311
</td>
<td style="text-align:left;">
Pietzak et al. Eur Urol 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-27 00:00:00
</td>
<td style="text-align:left;">
105
</td>
<td style="text-align:left;">
blca_nmibc_2017
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (Broad, Nature 2012)
</td>
<td style="text-align:left;">
Breast (Broad 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 103 breast cancer tumor/normal sample pairs. Generated by the Broad Institute.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722202
</td>
<td style="text-align:left;">
Banerji et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-04-30 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
brca_broad
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (IRC, Nat Genet 2014)
</td>
<td style="text-align:left;">
ccRCC (IRC)
</td>
<td style="text-align:left;">
Multi region exome sequencing of 10 clear cell renal cell carcinomas (ccRCCs)
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24487277
</td>
<td style="text-align:left;">
Gerlinger et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-01 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
ccrcc_irc_2014
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (IUOPA, Nature 2015)
</td>
<td style="text-align:left;">
CLL(IUOPA 2015)
</td>
<td style="text-align:left;">
Mutation data from whole-genome or whole-exome sequencing of 428 CLL, 54 MBL, and 24 SLL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26200345
</td>
<td style="text-align:left;">
Puente et al. Nature 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-01 00:00:00
</td>
<td style="text-align:left;">
506
</td>
<td style="text-align:left;">
cll_iuopa_2015
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic lymphocytic leukemia (ICGC, Nature Genetics 2011)
</td>
<td style="text-align:left;">
CLLSLL (Nature Genetics 2011)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 105 Chronic Lymphocytic Leukemia samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22158541
</td>
<td style="text-align:left;">
Quesada et al. Nature Genetics 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-01 00:00:00
</td>
<td style="text-align:left;">
105
</td>
<td style="text-align:left;">
cllsll_icgc_2011
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cutaneous Squamous Cell Carcinoma (DFCI, Clin Cancer Res 2015)
</td>
<td style="text-align:left;">
CSCC (DFCI 2015)
</td>
<td style="text-align:left;">
Targeted sequencing of 29 metastatic cutaneous squamous cell carcinoma samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25589618
</td>
<td style="text-align:left;">
Li et al. Clin Cancer Res 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
cscc_dfarber_2015
</td>
<td style="text-align:left;">
cscc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Desmoplastic Melanoma (Broad Institute, Nat Genet 2015)
</td>
<td style="text-align:left;">
DESM (Broad 2015)
</td>
<td style="text-align:left;">
Exome/Whole Genome sequencing of 20 desmoplastic melanoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26343386
</td>
<td style="text-align:left;">
Shain et al. Nat Genet 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
desm_broad_2015
</td>
<td style="text-align:left;">
desm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ewing Sarcoma (Institut Curie, Cancer Discov 2014)
</td>
<td style="text-align:left;">
Ewing Sarcoma (Institut Curie)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 112 Ewing sarcoma samples and matched germ line DNA.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25223734
</td>
<td style="text-align:left;">
Tirode et al. Cancer Discov 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
112
</td>
<td style="text-align:left;">
es_iocurie_2014
</td>
<td style="text-align:left;">
es
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cutaneous Squamous Cell Carcinoma (MD Anderson, Clin Cancer Res 2014)
</td>
<td style="text-align:left;">
cSCC (MD ANDERSON 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/normal pairs from 39 aggressive cutaneous squamous cell carcinoma cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25303977
</td>
<td style="text-align:left;">
Pickering et al. Clin Cancer Res 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
39
</td>
<td style="text-align:left;">
cscc_hgsc_bcm_2014
</td>
<td style="text-align:left;">
cscc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B cell Lymphoma (DFCI, Nat Med 2018)
</td>
<td style="text-align:left;">
DLBCLNOS (DFCI)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/matching normal sample pairs from 135 DLBCL patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29713087
</td>
<td style="text-align:left;">
Chapuy et al. Nat Med 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-02 00:00:00
</td>
<td style="text-align:left;">
135
</td>
<td style="text-align:left;">
dlbcl_dfci_2018
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Squamous Cell Carcinoma (ICGC, Nature 2014)
</td>
<td style="text-align:left;">
Esophagus sq (ICGC)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing of 88 esophageal squamous cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24670651
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-03 00:00:00
</td>
<td style="text-align:left;">
88
</td>
<td style="text-align:left;">
escc_icgc
</td>
<td style="text-align:left;">
escc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Adenoma and Carcinomas (MSK, PLOS One 2018)
</td>
<td style="text-align:left;">
HCC (MSK, PLOS One)
</td>
<td style="text-align:left;">
Targeted sequencing of tumor/normal pairs of 9 LIAD and 10 HCC patients via MSK-IMPACT platform.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30052636
</td>
<td style="text-align:left;">
Zheng et al. PLOS One 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-03 00:00:00
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
hcc_msk_venturaa_2018
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Gallbladder Cancer (MSK, Cancer 2018)
</td>
<td style="text-align:left;">
GBC (MSK CANCER 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of 103 samples with matched normals from 101 Gallbladder cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30427539
</td>
<td style="text-align:left;">
Raja et al, Cancer 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-03 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
gbc_msk_2018
</td>
<td style="text-align:left;">
gbc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Liver (TCGA)
</td>
<td style="text-align:left;">
TCGA Liver Hepatocellular Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LIHC/20160128/gdac.broadinstitute.org_LIHC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
442
</td>
<td style="text-align:left;">
lihc_tcga
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-Grade Gliomas (UCSF, Science 2014)
</td>
<td style="text-align:left;">
Glioma (UCSF)
</td>
<td style="text-align:left;">
Whole exome sequencing of 23 grade II glioma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24336570
</td>
<td style="text-align:left;">
Johnson et al. Science 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
61
</td>
<td style="text-align:left;">
lgg_ucsf_2014
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (AMC, Hepatology 2014)
</td>
<td style="text-align:left;">
Liver (AMC)
</td>
<td style="text-align:left;">
Whole exome sequencing of 231 hepatocellular carcinomas and matched normal tissue pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24798001
</td>
<td style="text-align:left;">
Ahn et al. Hepatology 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
231
</td>
<td style="text-align:left;">
lihc_amc_prv
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TSP, Nature 2008)
</td>
<td style="text-align:left;">
Lung adeno (TSP)
</td>
<td style="text-align:left;">
Targeted sequencing of 163 lung adenocarcinoma tumor/normal pairs. Generated by WashU/Broad.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
18948947
</td>
<td style="text-align:left;">
Ding et al. Nature 2008
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
163
</td>
<td style="text-align:left;">
luad_tsp
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Medulloblastoma (PCGP, Nature 2012)
</td>
<td style="text-align:left;">
MBL (PCGP)
</td>
<td style="text-align:left;">
Whole genome sequencing of 37 medulloblastoma tumors and their matched normals from the Pediatric Cancer Genome Project (PCGP), a joint project by St. Jude Children’s Research Hospital and Washington University.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722829
</td>
<td style="text-align:left;">
Robinson et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
37
</td>
<td style="text-align:left;">
mbl_pcgp
</td>
<td style="text-align:left;">
mbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thoracic PDX (MSK, Provisional)
</td>
<td style="text-align:left;">
MIXED (CMO CR_pdx IMPACT)
</td>
<td style="text-align:left;">
Targeted sequencing of thoracic PDX lines to characterize genetic alterations in 139 tumor/normal pairs via MSK-IMPACT. Data generated in Charles Rudin Lab.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
SCLCPORTAL;RUDINC;COMPONC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-06 00:00:00
</td>
<td style="text-align:left;">
139
</td>
<td style="text-align:left;">
lung_msk_pdx
</td>
<td style="text-align:left;">
lung
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mesothelioma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Mesothelioma (TCGA)
</td>
<td style="text-align:left;">
TCGA Mesothelioma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data at the <a href="https://portal.gdc.cancer.gov/legacy-archive/search/f?filters=%7B%22op%22:%22and%22,%22content%22:%5B%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.program.name%22,%22value%22:%5B%22TCGA%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.project_id%22,%22value%22:%5B%22TCGA-MESO%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22files.data_format%22,%22value%22:%5B%22MAF%22%5D%7D%7D%5D%7D">GDC</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-07 00:00:00
</td>
<td style="text-align:left;">
87
</td>
<td style="text-align:left;">
meso_tcga
</td>
<td style="text-align:left;">
plmeso
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (ICGC, Nature 2012)
</td>
<td style="text-align:left;">
Pancreas (ICGC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 99 pancreatic samples and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23103869
</td>
<td style="text-align:left;">
Biankin et al. Nature 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-07 00:00:00
</td>
<td style="text-align:left;">
99
</td>
<td style="text-align:left;">
paad_icgc
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mantle Cell Lymphoma (IDIBIPS, PNAS 2013)
</td>
<td style="text-align:left;">
MCL (IDIBIPS 2013)
</td>
<td style="text-align:left;">
Whole exome sequencing of 29 primary mantle cell lymphomas and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24145436
</td>
<td style="text-align:left;">
BeÃ  et al. PNAS 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-07 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
mcl_idibips_2013
</td>
<td style="text-align:left;">
mcl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
MSK-IMPACT Clinical Sequencing Cohort (MSKCC, Nat Med 2017)
</td>
<td style="text-align:left;">
MSK-IMPACT
</td>
<td style="text-align:left;">
Targeted sequencing of 10,000 clinical cases using the MSK-IMPACT assay
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28481359
</td>
<td style="text-align:left;">
Zehir et al. Nat Med 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-07 00:00:00
</td>
<td style="text-align:left;">
10945
</td>
<td style="text-align:left;">
msk_impact_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Pancreas (TCGA)
</td>
<td style="text-align:left;">
TCGA Pancreatic Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PAAD/20160128/gdac.broadinstitute.org_PAAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
186
</td>
<td style="text-align:left;">
paad_tcga
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
PCPG (TCGA)
</td>
<td style="text-align:left;">
TCGA Pheochromocytoma and Paraganglioma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PCPG/20160128/gdac.broadinstitute.org_PCPG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
184
</td>
<td style="text-align:left;">
pcpg_tcga
</td>
<td style="text-align:left;">
mnet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Cancer (UTSW, Nat Commun 2015)
</td>
<td style="text-align:left;">
Pancreas (UTSW)
</td>
<td style="text-align:left;">
Whole exome sequencing of 109 micro-dissected pancreatic cancer cases and normal control tissue.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25855536
</td>
<td style="text-align:left;">
Witkiewicz et al. Nat Commun 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
109
</td>
<td style="text-align:left;">
paad_utsw_2015
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Neuroendocrine Tumors (Johns Hopkins University, Science 2011)
</td>
<td style="text-align:left;">
PANET(Johns Hopkins 2011)
</td>
<td style="text-align:left;">
Whole exome sequencing of 10 pancreatic neuroendocrine tumor patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21252315
</td>
<td style="text-align:left;">
Jiao et al. Science 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
panet_jhu_2011
</td>
<td style="text-align:left;">
panet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Prostate Adenocarcinoma (MCTP, Nature 2012)
</td>
<td style="text-align:left;">
Prostate (MICH)
</td>
<td style="text-align:left;">
Exome sequencing of 61 prostate cancer tumor/normal pairs (50 metastatic CRPCs and 11 high-grade localized prostate cancers). Generated by Arul Chinnaiyan’s and Scott Tomlins’ labs at the University of Michigan.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722839
</td>
<td style="text-align:left;">
Grasso et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-09 00:00:00
</td>
<td style="text-align:left;">
61
</td>
<td style="text-align:left;">
prad_mich
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Sarcoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Sarcoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Sarcoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SARC/20160128/gdac.broadinstitute.org_SARC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
265
</td>
<td style="text-align:left;">
sarc_tcga
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Lung Cancer (CLCGP, Nat Genet 2012)
</td>
<td style="text-align:left;">
Small Cell Lung (CLCGP)
</td>
<td style="text-align:left;">
Comprehensive whole exam profiling of 29 small cell lung cancer samples. Generated by the Clinical Lung Cancer Genome Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22941188
</td>
<td style="text-align:left;">
Peifer et al. Nat Genet 2012
</td>
<td style="text-align:left;">
SCLCPORTAL;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
sclc_clcgp
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Carcinoma of the Ovary (MSKCC, Nat Genet 2014)
</td>
<td style="text-align:left;">
Ovary SC (MSKCC)
</td>
<td style="text-align:left;">
Targeted sequencing of 12 SCCO paired tumor and normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24658004
</td>
<td style="text-align:left;">
Jelinic et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
scco_mskcc
</td>
<td style="text-align:left;">
scco
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC, PNAS 2014)
</td>
<td style="text-align:left;">
Prostate (MSKCC 2014)
</td>
<td style="text-align:left;">
Genome-wide copy number profiling in 104 primary prostate cancer tumor/normal pairs. <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54691">Raw data via GEO (GSE54691)</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25024180
</td>
<td style="text-align:left;">
Hieronymus et al. PNAS 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
104
</td>
<td style="text-align:left;">
prad_mskcc_2014
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Cancer (MSKCC, JCO Precis Oncol 2017)
</td>
<td style="text-align:left;">
MSK-IMPACT Prostate
</td>
<td style="text-align:left;">
Targeted sequencing of 504 prostate cancer tumor/normal pairs via MSK-IMPACT.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28825054
</td>
<td style="text-align:left;">
Abida et al. JCO Precis Oncol 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
504
</td>
<td style="text-align:left;">
prad_mskcc_2017
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small-Cell Lung Cancer (Multi-Institute, Cancer Cell 2017)
</td>
<td style="text-align:left;">
SCLC (GARDNER 2017)
</td>
<td style="text-align:left;">
Whole Exome Sequencing of 10 paired chemosensitive and chemoresistant SCLC PDX.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28196596
</td>
<td style="text-align:left;">
Gardner et al. Cancer Cell 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-10 00:00:00
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
sclc_cancercell_gardner_2017
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Testicular Germ Cell Cancer (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Testicular germ cell (TCGA)
</td>
<td style="text-align:left;">
TCGA Testicular Germ Cell Cancer; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/TGCT/20160128/gdac.broadinstitute.org_TGCT.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
156
</td>
<td style="text-align:left;">
tgct_tcga
</td>
<td style="text-align:left;">
nsgct
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thyroid Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Thyroid (TCGA)
</td>
<td style="text-align:left;">
TCGA Thyroid Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THCA/20160128/gdac.broadinstitute.org_THCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
516
</td>
<td style="text-align:left;">
thca_tcga
</td>
<td style="text-align:left;">
thpa
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (Broad, Cell 2012)
</td>
<td style="text-align:left;">
Melanoma (Broad 2012)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 121 melanoma samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22817889
</td>
<td style="text-align:left;">
Hodis et al. Cell 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
121
</td>
<td style="text-align:left;">
skcm_broad
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (Yale, Nat Genet 2012)
</td>
<td style="text-align:left;">
Melanoma (Yale 2012)
</td>
<td style="text-align:left;">
Exome sequencing of 147 melanoma samples with 99 matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22842228
</td>
<td style="text-align:left;">
Krauthammer et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
147
</td>
<td style="text-align:left;">
skcm_yale
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (UHK, Nat Genet 2011)
</td>
<td style="text-align:left;">
Stomach (UHK)
</td>
<td style="text-align:left;">
Exome sequencing of 22 gastric cancer samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22037554
</td>
<td style="text-align:left;">
Wang et al. Nat Genet 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
stad_uhongkong
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (U Tokyo, Nat Genet 2014)
</td>
<td style="text-align:left;">
Stomach (UTokyo)
</td>
<td style="text-align:left;">
Whole exome sequencing of 30 diffuse-type gastric adenocarcinoma samples (with matched normals) from the University of Tokyo.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24816255
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
30
</td>
<td style="text-align:left;">
stad_utokyo
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Melanoma (MSKCC, JCO Precis Oncol 2017)
</td>
<td style="text-align:left;">
SKCM (Vanderbilt/Mskcc)
</td>
<td style="text-align:left;">
Next generation sequencing (NGS) of pre-treatment tumor to identify co-altered genes that predict for intrinsic resistance to BRAF inhibitor therapy in melanoma patients as a prelude to rational combination strategies.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-11 00:00:00
</td>
<td style="text-align:left;">
66
</td>
<td style="text-align:left;">
skcm_vanderbilt_mskcc_2015
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thymoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Thymoma(TCGA)
</td>
<td style="text-align:left;">
TCGA Thymoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/THYM/20160128/gdac.broadinstitute.org_THYM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
124
</td>
<td style="text-align:left;">
thym_tcga
</td>
<td style="text-align:left;">
thym
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Uterine (TCGA)
</td>
<td style="text-align:left;">
TCGA Uterine Corpus Endometrial Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCEC/20160128/gdac.broadinstitute.org_UCEC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
548
</td>
<td style="text-align:left;">
ucec_tcga
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Carcinosarcoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Uterine CS (TCGA)
</td>
<td style="text-align:left;">
TCGA Uterine Carcinosarcoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UCS/20160128/gdac.broadinstitute.org_UCS.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
57
</td>
<td style="text-align:left;">
ucs_tcga
</td>
<td style="text-align:left;">
ucs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Poorly-Differentiated and Anaplastic Thyroid Cancers (MSKCC, JCI 2016)
</td>
<td style="text-align:left;">
Thyroid (MSKCC 2016)
</td>
<td style="text-align:left;">
Next-generation sequencing of 341 cancer genes from 117 patient-derived PDTCs and ATCs
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26878173
</td>
<td style="text-align:left;">
Landa et al. J Clin Invest 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
117
</td>
<td style="text-align:left;">
thyroid_mskcc_2016
</td>
<td style="text-align:left;">
thyroid
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pan-Lung Cancer (TCGA, Nat Genet 2016)
</td>
<td style="text-align:left;">
NSCLC (TCGA 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 660 lung ADC and 484 lung SqCC tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27158780
</td>
<td style="text-align:left;">
TCGA, Nat Genet 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
1144
</td>
<td style="text-align:left;">
nsclc_tcga_broad_2016
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Clear Cell Carcinoma (NIH, Cancer 2017)
</td>
<td style="text-align:left;">
UCCC (NIH)
</td>
<td style="text-align:left;">
Whole exome sequencing of paired tumor-normal DNAs from 16 cases of clear cell endometrial tumors.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28485815
</td>
<td style="text-align:left;">
Le Gallo et al. Cancer 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-14 00:00:00
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
uccc_nih_2017
</td>
<td style="text-align:left;">
uccc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Lung squ (TCGA)
</td>
<td style="text-align:left;">
TCGA Lung Squamous Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUSC/20160128/gdac.broadinstitute.org_LUSC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
511
</td>
<td style="text-align:left;">
lusc_tcga
</td>
<td style="text-align:left;">
lusc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Stomach (TCGA)
</td>
<td style="text-align:left;">
TCGA Stomach Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/STAD/20160128/gdac.broadinstitute.org_STAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
478
</td>
<td style="text-align:left;">
stad_tcga
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (Broad, Science 2011)
</td>
<td style="text-align:left;">
Head &amp; neck (Broad)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 74 head and neck squamous cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21798893
</td>
<td style="text-align:left;">
Stransky et al. Science 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
74
</td>
<td style="text-align:left;">
hnsc_broad
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (University of Turin, Lung Cancer 2017)
</td>
<td style="text-align:left;">
UNITO (Lung Cancer, 2016)
</td>
<td style="text-align:left;">
Targeted next-generation sequencing of 41 NSCLC samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27346245
</td>
<td style="text-align:left;">
VavalÃ  et al. Lung Cancer 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
41
</td>
<td style="text-align:left;">
nsclc_unito_2016
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Wilms’ Tumor (TARGET, 2018)
</td>
<td style="text-align:left;">
WT (TARGET, 2018)
</td>
<td style="text-align:left;">
Comprehensive profiling of 657 Wilms’ Tumor samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-15 00:00:00
</td>
<td style="text-align:left;">
657
</td>
<td style="text-align:left;">
wt_target_2018_pub
</td>
<td style="text-align:left;">
wt
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia (Novartis/Broad, Nature 2012)
</td>
<td style="text-align:left;">
CCLE (Novartis/Broad 2012)
</td>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia from the Broad Institute and Novartis, containing 1020 samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22460905
</td>
<td style="text-align:left;">
Barretina et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
1020
</td>
<td style="text-align:left;">
cellline_ccle_broad
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Malignant Peripheral Nerve Sheath Tumor (MSKCC, Nat Genet 2014)
</td>
<td style="text-align:left;">
MPNST (MSKCC)
</td>
<td style="text-align:left;">
Whole exome sequencing of 15 malignant peripheral nerve sheath tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25240281
</td>
<td style="text-align:left;">
Lee et al. Nat Genet 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
mpnst_mskcc
</td>
<td style="text-align:left;">
mpnst
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, Nature 2013)
</td>
<td style="text-align:left;">
Uterine (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing of 373 endometrial carcinoma tumors with matched normals. TCGA Uterine Corpus Endometrial Carcinoma project; raw data at the <a href="https://tcga-data.nci.nih.gov/docs/publications/ucec_2013/">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23636398
</td>
<td style="text-align:left;">
TCGA, Nature 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
373
</td>
<td style="text-align:left;">
ucec_tcga_pub
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MSKCC, Nat Genet 2013)
</td>
<td style="text-align:left;">
ACyC (MSKCC, 2013)
</td>
<td style="text-align:left;">
Whole-exome or whole-genome sequencing analysis of 60 ACC tumor/normal pairs
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23685749
</td>
<td style="text-align:left;">
Ho et al. Nat Genet 2013
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
60
</td>
<td style="text-align:left;">
acyc_mskcc_2013
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
TMB and Immunotherapy (MSKCC, Nat Genet 2019)
</td>
<td style="text-align:left;">
TMB (MSKCC 2019)
</td>
<td style="text-align:left;">
Genomic and survival data from 1661 tumor-normal pairs from 1661 patients with various cancer types sequenced with the MSK-IMPACT assay.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30643254
</td>
<td style="text-align:left;">
Samstein et al. Nat Genet 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-16 00:00:00
</td>
<td style="text-align:left;">
1661
</td>
<td style="text-align:left;">
tmb_mskcc_2018
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
PCPG (TCGA)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 178 Pheochromocytoma or Paraganglioma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28162975
</td>
<td style="text-align:left;">
Fishbein et al. Cancer Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-05-30 00:00:00
</td>
<td style="text-align:left;">
178
</td>
<td style="text-align:left;">
pcpg_tcga_pub
</td>
<td style="text-align:left;">
mnet
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uveal Melanoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Uveal melanoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Uveal Melanoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/UVM/20160128/gdac.broadinstitute.org_UVM.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-03 00:00:00
</td>
<td style="text-align:left;">
80
</td>
<td style="text-align:left;">
uvm_tcga
</td>
<td style="text-align:left;">
um
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma of the Breast (MSKCC, J Pathol. 2015)
</td>
<td style="text-align:left;">
ACBC (MSKCC/Breast 2015)
</td>
<td style="text-align:left;">
Whole exome sequencing of 12 breast AdCCs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26095796
</td>
<td style="text-align:left;">
Martelotto et al. J Pathol 2015
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-03 00:00:00
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
acbc_mskcc_2015
</td>
<td style="text-align:left;">
acbc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, Nature 2011)
</td>
<td style="text-align:left;">
Ovarian (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing (316 samples with matched normals), mRNA expression, miRNA expression, promoter methylation, and DNA copy number in 489 high-grade serous ovarian adenocarcinomas (HGS-OvCa). <a href="http://cancergenome.nih.gov/">The Cancer Genome Atlas (TCGA)</a> Serous Ovarian Cancer project. 489 cases. <br><a href="https://tcga-data.nci.nih.gov/docs/publications/ov_2011/">Raw data via the TCGA Data Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
21720365
</td>
<td style="text-align:left;">
TCGA, Nature 2011
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-04 00:00:00
</td>
<td style="text-align:left;">
489
</td>
<td style="text-align:left;">
ov_tcga_pub
</td>
<td style="text-align:left;">
hgsoc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (Broad/Cornell, Cell 2013)
</td>
<td style="text-align:left;">
Prostate (Broad/Cornell 2013)
</td>
<td style="text-align:left;">
Whole genome sequencing of 57 prostate tumor and matches normal tissues.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23622249
</td>
<td style="text-align:left;">
Baca et al. Cell 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-04 00:00:00
</td>
<td style="text-align:left;">
57
</td>
<td style="text-align:left;">
prad_broad_2013
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Papillary Thyroid Carcinoma (TCGA, Cell 2014)
</td>
<td style="text-align:left;">
Thyroid (TCGA pub)
</td>
<td style="text-align:left;">
Exome and whole genome sequencing of 496 papillary thyroid carcinoma tumor/normal pairs
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25417114
</td>
<td style="text-align:left;">
TCGA, Cell 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-05 00:00:00
</td>
<td style="text-align:left;">
496
</td>
<td style="text-align:left;">
thca_tcga_pub
</td>
<td style="text-align:left;">
thpa
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Lung adeno (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing, messenger RNA, microRNA, copy number, methylation and proteomic analysis of 230 lung adenocarcinoma tumor/normal pairs. <a href="https://tcga-data.nci.nih.gov/docs/publications/luad_2014/">Raw data via the TCGA Data Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25079552
</td>
<td style="text-align:left;">
TCGA, Nature 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-07 00:00:00
</td>
<td style="text-align:left;">
230
</td>
<td style="text-align:left;">
luad_tcga_pub
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Urothelial Carcinoma (Cornell/Trento, Nat Gen 2016)
</td>
<td style="text-align:left;">
BLCA (Cornell 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 72 tumor samples (with 32 matched normals) in Urothelial Carcinoma.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27749842
</td>
<td style="text-align:left;">
Faltas and Prandi, Nature Genetics 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-07 00:00:00
</td>
<td style="text-align:left;">
72
</td>
<td style="text-align:left;">
blca_cornell_2016
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioma (MSK, 2018)
</td>
<td style="text-align:left;">
DIFG(MSK 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of CSF samples (91 samples with 86 matched normals) from 42 glioma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-07 00:00:00
</td>
<td style="text-align:left;">
91
</td>
<td style="text-align:left;">
glioma_msk_2018
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, NEJM 2013)
</td>
<td style="text-align:left;">
AML (TCGA pub)
</td>
<td style="text-align:left;">
Whole-genome or whole-exome sequencing analysis of 200 adult de novo AML tumor/normal pairs. TCGA Acute Myeloid Leukemia Project; raw data at the <a href="https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=LAML&amp;diseaseName=Acute%20Myeloid%20Leukemia">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23634996
</td>
<td style="text-align:left;">
TCGA, NEJM 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-10 00:00:00
</td>
<td style="text-align:left;">
200
</td>
<td style="text-align:left;">
laml_tcga_pub
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Merged Cohort of LGG and GBM (TCGA, Cell 2016)
</td>
<td style="text-align:left;">
LGG-GBM (TCGA 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing from 1,122 TCGA LGG and GBM cases.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26824661
</td>
<td style="text-align:left;">
TCGA, Cell 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-10 00:00:00
</td>
<td style="text-align:left;">
1102
</td>
<td style="text-align:left;">
lgggbm_tcga_pub
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytoma (MSKCC, Neuro Oncol 2017)
</td>
<td style="text-align:left;">
Glioma MSK (Neuro Oncol 2017)
</td>
<td style="text-align:left;">
Genomic characterization using next generation sequencing (NGS) in 22 Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytomas.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28472509
</td>
<td style="text-align:left;">
Thomas et al. Neuro Oncol 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-11 00:00:00
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
odg_msk_2017
</td>
<td style="text-align:left;">
odg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Germ Cell Tumors (MSKCC, J Clin Oncol 2016)
</td>
<td style="text-align:left;">
GCT (JCO, 2016)
</td>
<td style="text-align:left;">
Whole-exome and Targeted sequencing of 19 samples and Targeted sequencing of 161 samples from 180 Germ Cell Tumor patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27646943
</td>
<td style="text-align:left;">
Bagrodia et al. J Clin Oncol 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-11 00:00:00
</td>
<td style="text-align:left;">
180
</td>
<td style="text-align:left;">
gct_msk_2016
</td>
<td style="text-align:left;">
testis
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Solid Cancers (UMich, Nature 2017)
</td>
<td style="text-align:left;">
MET500 (Michigan U 2017)
</td>
<td style="text-align:left;">
Whole-exome and -transcriptome sequencing of 500 adult patients with metastatic solid tumor/primary normal pairs of diverse lineage and biopsy site.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28783718
</td>
<td style="text-align:left;">
Robinson et al. Nature 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-12 00:00:00
</td>
<td style="text-align:left;">
500
</td>
<td style="text-align:left;">
metastatic_solid_tumors_mich_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Basal Cell Carcinoma (UNIGE, Nat Genet 2016)
</td>
<td style="text-align:left;">
BCC (UNIGE 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 126 basal cell carcinoma tumor/normal pairs; targeted sequencing of 163 sporadic samples (40 tumor/normal pairs) and 4 Gorlin symdrome basal cell carcinomas.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26950094
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-12 00:00:00
</td>
<td style="text-align:left;">
293
</td>
<td style="text-align:left;">
bcc_unige_2016
</td>
<td style="text-align:left;">
bcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Lung squ (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing, DNA copy number, mRNA sequencing, mRNA expression and promoter methylation analysis of 178 lung squamous cell carcinoma tumor/normal pairs. <a href="http://cancergenome.nih.gov/">The Cancer Genome Atlas (TCGA)</a> Lung Squamous Cell Carcinoma project. 178 cases.<br><i>Nature 2012.</i> <a href="https://tcga-data.nci.nih.gov/docs/publications/lusc_2012/">Raw data via the TCGA Data Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22960745
</td>
<td style="text-align:left;">
TCGA, Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-06-17 00:00:00
</td>
<td style="text-align:left;">
178
</td>
<td style="text-align:left;">
lusc_tcga_pub
</td>
<td style="text-align:left;">
lusc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute myeloid leukemia or myelodysplastic syndromes (WashU, 2016)
</td>
<td style="text-align:left;">
AML/MDS (WashU, 2016)
</td>
<td style="text-align:left;">
Enhanced exome or targeted sequencing of 116 acute myeloid leukemia (AML) or myelodysplastic syndromes (MDS) patients’ tumor/normal pairs .
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27959731
</td>
<td style="text-align:left;">
Welch et al. N Engl J Med. 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-01 00:00:00
</td>
<td style="text-align:left;">
136
</td>
<td style="text-align:left;">
mnm_washu_2016
</td>
<td style="text-align:left;">
mnm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (TRACERx, NEJM 2017)
</td>
<td style="text-align:left;">
Tracerx (NSCLC 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 327 (with 100 matched germline samples) tumors from 100 early-stage NSCLC patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28445112
</td>
<td style="text-align:left;">
Jamal-Hanjani M et al. NEJM 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-02 00:00:00
</td>
<td style="text-align:left;">
327
</td>
<td style="text-align:left;">
nsclc_tracerx_2017
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colon Adenocarcinoma (CaseCCC, PNAS 2015)
</td>
<td style="text-align:left;">
COAD (CaseCCC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor/normal pairs from 29 African American Colon Cancers.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25583493
</td>
<td style="text-align:left;">
Guda et al. PNAS 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-02 00:00:00
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
coad_caseccc_2015
</td>
<td style="text-align:left;">
coad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Myeloproliferative Neoplasms (CIMR, NEJM 2013)
</td>
<td style="text-align:left;">
MPN (CIMR)
</td>
<td style="text-align:left;">
Whole exome sequencing of paired tumor/normal pairs from 151 cases of Myeloproliferative Neoplasms.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24325359
</td>
<td style="text-align:left;">
Nangalia et al. NEJM 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-24 00:00:00
</td>
<td style="text-align:left;">
151
</td>
<td style="text-align:left;">
mpn_cimr_2013
</td>
<td style="text-align:left;">
mpn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Clear Cell Renal Cell Carcinoma (DFCI, Science 2019)
</td>
<td style="text-align:left;">
ccRCC (DFCI 2019)
</td>
<td style="text-align:left;">
Whole-exome sequencing of metastatic clear cell renal carcinoma (ccRCC) from 35 tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29301960
</td>
<td style="text-align:left;">
Miao et al. Science 2019
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-07-26 00:00:00
</td>
<td style="text-align:left;">
35
</td>
<td style="text-align:left;">
ccrcc_dfci_2019
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Cancer (MSK, 2019)
</td>
<td style="text-align:left;">
PROSTATE (MSK 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT Sequencing of 18 prostate cancer tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-08-06 00:00:00
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
prad_msk_2019
</td>
<td style="text-align:left;">
prostate
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-small cell lung cancer (MSK, Science 2015)
</td>
<td style="text-align:left;">
NSCLC (MSKCC)
</td>
<td style="text-align:left;">
Whole-exome sequencing of tumor-normal tissue pairs from 16 non-small cell lung cancer samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25765070
</td>
<td style="text-align:left;">
Rivzi et al. Science 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-08-09 00:00:00
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
nsclc_mskcc_2015
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Prostate (TCGA)
</td>
<td style="text-align:left;">
TCGA Prostate Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/PRAD/20160128/gdac.broadinstitute.org_PRAD.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-08-23 00:00:00
</td>
<td style="text-align:left;">
499
</td>
<td style="text-align:left;">
prad_tcga
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma Project (2019)
</td>
<td style="text-align:left;">
(AACRF,2019)
</td>
<td style="text-align:left;">
Multi-Institute Cohort of 1045 Adenoid Cystic Carcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
ACYC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-09-05 00:00:00
</td>
<td style="text-align:left;">
1049
</td>
<td style="text-align:left;">
acc_2019
</td>
<td style="text-align:left;">
acc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (Broad, Nature 2015)
</td>
<td style="text-align:left;">
CLL (Broad 2015)
</td>
<td style="text-align:left;">
Whole Exome Sequencing of 537 chronic lymphocytic leukemia samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26466571
</td>
<td style="text-align:left;">
Landau et al. Nature 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-09-06 00:00:00
</td>
<td style="text-align:left;">
537
</td>
<td style="text-align:left;">
cll_broad_2015
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (OHSU, Nature 2018)
</td>
<td style="text-align:left;">
AML (OHSU)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 672 acute myeloid leukemia samples (with 454 matched normals) from the Beat AML program.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30333627
</td>
<td style="text-align:left;">
Tyner et al. Nature 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-09-06 00:00:00
</td>
<td style="text-align:left;">
672
</td>
<td style="text-align:left;">
aml_ohsu_2018
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Brain Tumor PDXs (Mayo Clinic, 2019)
</td>
<td style="text-align:left;">
GBM (Mayo PDX)
</td>
<td style="text-align:left;">
WES on a total of 83 PDXs and 55 PDXs have the matched germline (blood) samples sequenced. A total of 66 PDXs have RNASeq available
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
SHULANTIANMAYO;PUBLIC;DECKERPAULMAYO
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-09-06 00:00:00
</td>
<td style="text-align:left;">
95
</td>
<td style="text-align:left;">
gbm_mayo_pdx_sarkaria_2019
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (MSKCC, 2019)
</td>
<td style="text-align:left;">
BRCA (MSKCC 2019)
</td>
<td style="text-align:left;">
Targeted Sequencing of buparlisib + letrozole and alpelisib + letrozole-treated metastatic ER+ unmatched breast tumors.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
SOLITD;COMPONC;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-09-06 00:00:00
</td>
<td style="text-align:left;">
70
</td>
<td style="text-align:left;">
brca_mskcc_2019
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Cancer (MSKCC, Cancer Discov 2017)
</td>
<td style="text-align:left;">
LUAD (MSK,2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 860 patients with metastatic lung adenocarcinoma
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28336552
</td>
<td style="text-align:left;">
Jordan et al. Cancer Discov 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-03 00:00:00
</td>
<td style="text-align:left;">
915
</td>
<td style="text-align:left;">
lung_msk_2017
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (MSKCC, J Clin Oncol 2018)
</td>
<td style="text-align:left;">
NSCLC (MSK PD1)
</td>
<td style="text-align:left;">
IMPACT sequencing of 240 NSCLC tumor/normal pairs treated at MSKCC with anti-PD-(L)1 based therapy.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29337640
</td>
<td style="text-align:left;">
Rizvi et al. J Clin Oncol 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-03 00:00:00
</td>
<td style="text-align:left;">
240
</td>
<td style="text-align:left;">
nsclc_pd1_msk_2018
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Endometrial Cancer (MSK, 2018)
</td>
<td style="text-align:left;">
UCEC (MSK 2018)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 197 advanced-staged endometrial cancer tumor samples with matched normals, from 189 patients treated at MSK.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30068706
</td>
<td style="text-align:left;">
Soumerai et al. Clin Cancer Res. 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-03 00:00:00
</td>
<td style="text-align:left;">
197
</td>
<td style="text-align:left;">
ucec_msk_2018
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Melanoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Skin Cutaneous Melanoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/SKCM/20160128/gdac.broadinstitute.org_SKCM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
479
</td>
<td style="text-align:left;">
skcm_tcga
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Ovarian (TCGA)
</td>
<td style="text-align:left;">
TCGA Ovarian Serous Cystadenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/OV/20160128/gdac.broadinstitute.org_OV.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
606
</td>
<td style="text-align:left;">
ov_tcga
</td>
<td style="text-align:left;">
hgsoc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (Broad/Cornell, Nat Genet 2012)
</td>
<td style="text-align:left;">
Prostate (Broad/Cornell 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of 112 prostate adenocarcinomas and matched normal samples. Generated by Levi Garraway’s lab at the Broad Institute and Mark Rubin’s lab at Cornell.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22610119
</td>
<td style="text-align:left;">
Barbieri et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
112
</td>
<td style="text-align:left;">
prad_broad
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Neuroendocrine Prostate Cancer (Multi-Institute, Nat Med 2016)
</td>
<td style="text-align:left;">
NEPC (Multi-Institute 2016)
</td>
<td style="text-align:left;">
Whole exome sequencing of 114 metastatic tumor/normal pairs to understand genomic overlap between castration resistant adenocarcinoma (CRPC-Adeno) and neuroendocrine histologies (CRPC-NE).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26855148
</td>
<td style="text-align:left;">
Beltran et al. Nat Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
114
</td>
<td style="text-align:left;">
nepc_wcm_2016
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (METABRIC, Nature 2012 &amp; Nat Commun 2016)
</td>
<td style="text-align:left;">
Breast (METABRIC 2016)
</td>
<td style="text-align:left;">
Targeted sequencing of 2509 primary breast tumors with 548 matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27161491
</td>
<td style="text-align:left;">
Pereira et al. Nat Commun 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
2509
</td>
<td style="text-align:left;">
brca_metabric
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acral Melanoma (TGEN, Genome Res 2017)
</td>
<td style="text-align:left;">
TSAM
</td>
<td style="text-align:left;">
Whole exome sequencing and transcriptome analysis of 34 Acral Melanoma patients (33 with matched normals).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28373299
</td>
<td style="text-align:left;">
Liang et al. Genome Res 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
38
</td>
<td style="text-align:left;">
mel_tsam_liang_2017
</td>
<td style="text-align:left;">
mel
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mature B-cell malignancies (MD Anderson Cancer Center)
</td>
<td style="text-align:left;">
MBN (MDACC 2013)
</td>
<td style="text-align:left;">
We performed targeted deep sequencing of 380 recurrently mutated genes across a cohort of 755 mature B-cell malignancies to define conserved and disease-specific patterns of genomic alterations.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
760
</td>
<td style="text-align:left;">
mbn_mdacc_2013
</td>
<td style="text-align:left;">
mbn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colon Cancer (CPTAC-2 Prospective, Cell 2019)
</td>
<td style="text-align:left;">
COAD (CPTAC-2 2019)
</td>
<td style="text-align:left;">
Proteogenomic profiling of 110 colon adenocarcinoma tumor samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31031003
</td>
<td style="text-align:left;">
Vasaikar et al. Cell 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
110
</td>
<td style="text-align:left;">
coad_cptac_2019
</td>
<td style="text-align:left;">
coad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Preclinical Testing Consortium (Maris, 2019)
</td>
<td style="text-align:left;">
PPTC (2019)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 261 patient derived xenografts (PDXs) samples from high-risk childhood cancers (with no matched normals). Website: <a href="http://www.ncipptc.org/">PPTC</a>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-05 00:00:00
</td>
<td style="text-align:left;">
261
</td>
<td style="text-align:left;">
pptc_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Rectal Cancer (MSK,Nature Medicine 2019)
</td>
<td style="text-align:left;">
Rectal (MSK, 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT targeted sequencing of matched rectal cancer tumor and tumoroid samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
SELCUKLS;SAUVEC;ZHENGY1;SMITHJ5;PUBLIC;SANCHEZF
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-08 00:00:00
</td>
<td style="text-align:left;">
339
</td>
<td style="text-align:left;">
rectal_msk_2019
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Unclassified Renal Cell Carcinoma (MSK, Nature 2016)
</td>
<td style="text-align:left;">
Renal unclass (MSKCC)
</td>
<td style="text-align:left;">
Targeted sequencing of 62 high-grade primary Unclassified Renal Cell Carcinoma tumor samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27713405
</td>
<td style="text-align:left;">
Chen et al. Nat Commun 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-09 00:00:00
</td>
<td style="text-align:left;">
62
</td>
<td style="text-align:left;">
urcc_mskcc_2016
</td>
<td style="text-align:left;">
urcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Esophagogastric Cancer (MSKCC, Cancer Discovery 2017)
</td>
<td style="text-align:left;">
EGC (MSK 2017)
</td>
<td style="text-align:left;">
Targeted sequencing of 341 samples from 305 metastatic esophagogastric cancer patients
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29122777
</td>
<td style="text-align:left;">
Janjigian et al. Cancer Discov 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-09 00:00:00
</td>
<td style="text-align:left;">
341
</td>
<td style="text-align:left;">
egc_msk_2017
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (MSK, Clin Cancer Res 2018)
</td>
<td style="text-align:left;">
CHOL (MSK 2018)
</td>
<td style="text-align:left;">
Targeted sequencing (MSK-IMPACT) of 195 tumor samples (194 with matched normals) from 192 cholangiocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29848569
</td>
<td style="text-align:left;">
Lowery et al. Clin Cancer Res 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-09 00:00:00
</td>
<td style="text-align:left;">
195
</td>
<td style="text-align:left;">
chol_msk_2018
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Intrahepatic Cholangiocarcinoma (Shanghai, Nat Commun 2014)
</td>
<td style="text-align:left;">
IHCH (Shanghai)
</td>
<td style="text-align:left;">
Whole exome sequencing of 103 tumor/normal sample pairs in 103 intrahepatic cholangiocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25526346
</td>
<td style="text-align:left;">
Shanshan et al. Nat Commun 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-30 00:00:00
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
ihch_smmu_2014
</td>
<td style="text-align:left;">
ihch
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Non-Small Cell Lung Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
NSCLC (MSKCC 2018)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 75 tumor/normal NSCLC pairs treated with PD-1 plus CTLA-4 blockade..
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29657128
</td>
<td style="text-align:left;">
Hellmann et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-10-30 00:00:00
</td>
<td style="text-align:left;">
75
</td>
<td style="text-align:left;">
nsclc_mskcc_2018
</td>
<td style="text-align:left;">
nsclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC/DFCI, Nature Genetics 2018)
</td>
<td style="text-align:left;">
PRAD (MSKCC/DFCI 2018)
</td>
<td style="text-align:left;">
Whole exome Sequencing of 1013 prostate cancer samples and their matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29610475
</td>
<td style="text-align:left;">
Armenia et al. Nat Genet 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-11-05 00:00:00
</td>
<td style="text-align:left;">
1013
</td>
<td style="text-align:left;">
prad_p1000
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (DFCI, Cell Reports 2016)
</td>
<td style="text-align:left;">
Colorectal (DFCI 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 619 colorectal cancers with clinicopathologic annotations.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27149842
</td>
<td style="text-align:left;">
Giannakis et al. Cell Reports 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-11-09 00:00:00
</td>
<td style="text-align:left;">
619
</td>
<td style="text-align:left;">
coadread_dfci_2016
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Cancer (MSK, Eur Urol 2015)
</td>
<td style="text-align:left;">
Upper tract (MSKCC)
</td>
<td style="text-align:left;">
Next-generation sequencing (MSK-IMPACT) on all protein-coding exons of 300 cancer associated genes of 85 Upper Tract Urothelial Carcinoma tumor samples and paired normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26278805
</td>
<td style="text-align:left;">
Sfakianos et al. Eur Urol 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-11-20 00:00:00
</td>
<td style="text-align:left;">
85
</td>
<td style="text-align:left;">
utuc_mskcc_2015
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Carcinoma (TCGA, Nature 2017)
</td>
<td style="text-align:left;">
Stomach/Esophageal (TCGA)
</td>
<td style="text-align:left;">
Whole exome sequencing of 559 esophageal and gastric carcinoma tumors with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28052061
</td>
<td style="text-align:left;">
TCGA, Nature 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-11-29 00:00:00
</td>
<td style="text-align:left;">
559
</td>
<td style="text-align:left;">
stes_tcga_pub
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Breast Cancer (INSERM, PLoS Med 2016)
</td>
<td style="text-align:left;">
BRCA (INSERM 2016)
</td>
<td style="text-align:left;">
Whole exome sequencing of 216 tumor/normal (blood) pairs from metastatic breast cancer patients who underwent a biopsy in the context of the SAFIR01/SAFIR02 (Unicancer, France), SHIVA (Institut Curie, France) or MOSCATO (Gustave Roussy, France) prospective trials.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28027327
</td>
<td style="text-align:left;">
Lefebvre et al. PLoS Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-11 00:00:00
</td>
<td style="text-align:left;">
216
</td>
<td style="text-align:left;">
brca_igr_2015
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Lung adeno (TCGA)
</td>
<td style="text-align:left;">
TCGA Lung Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LUAD/20160128/gdac.broadinstitute.org_LUAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-24 00:00:00
</td>
<td style="text-align:left;">
586
</td>
<td style="text-align:left;">
luad_tcga
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Colorectal (TCGA pub)
</td>
<td style="text-align:left;">
Whole exome sequencing in 224 of the 276 colorectal carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Colorectal Cancer project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22810696
</td>
<td style="text-align:left;">
TCGA, Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-24 00:00:00
</td>
<td style="text-align:left;">
276
</td>
<td style="text-align:left;">
coadread_tcga_pub
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, Nature 2014)
</td>
<td style="text-align:left;">
Bladder (TCGA 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 131 high-grade muscle-invasive urothelial bladder carcinomas
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24476821
</td>
<td style="text-align:left;">
TCGA, Nature 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-24 00:00:00
</td>
<td style="text-align:left;">
131
</td>
<td style="text-align:left;">
blca_tcga_pub
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
BLCA (TCGA 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 412 muscle-invasive bladder tumors and matched normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28988769
</td>
<td style="text-align:left;">
Robertson et al. Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-24 00:00:00
</td>
<td style="text-align:left;">
413
</td>
<td style="text-align:left;">
blca_tcga_pub_2017
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Nature 2012)
</td>
<td style="text-align:left;">
Breast (TCGA pub)
</td>
<td style="text-align:left;">
Whole-exome sequencing (510 samples with matched normals), genomic DNA copy number arrays, DNA methylation, messenger RNA arrays, microRNA sequencing and reverse-phase protein arrays analysis on 825 primary breast cancer samples. <a href="http://cancergenome.nih.gov/">The Cancer Genome Atlas (TCGA)</a> Breast Invasive Carcinoma project. 825 cases.<br><i>Nature 2012.</i> <a href="https://tcga-data.nci.nih.gov/docs/publications/brca_2012/">Raw data via the TCGA Data Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23000897
</td>
<td style="text-align:left;">
TCGA, Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-25 00:00:00
</td>
<td style="text-align:left;">
825
</td>
<td style="text-align:left;">
brca_tcga_pub
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, Nature 2013)
</td>
<td style="text-align:left;">
ccRCC (TCGA pub)
</td>
<td style="text-align:left;">
Whole-exome sequencing in 417 of 446 clear cell renal cell carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23792563
</td>
<td style="text-align:left;">
TCGA, Nature 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-25 00:00:00
</td>
<td style="text-align:left;">
446
</td>
<td style="text-align:left;">
kirc_tcga_pub
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, Cancer Cell 2014)
</td>
<td style="text-align:left;">
chRCC (TCGA 2014)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 66 chromophobe renal cell carcinoma (ChRCCs) tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25155756
</td>
<td style="text-align:left;">
TCGA, Cancer Cell 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-25 00:00:00
</td>
<td style="text-align:left;">
66
</td>
<td style="text-align:left;">
kich_tcga_pub
</td>
<td style="text-align:left;">
chrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma (TCGA, Nature 2008)
</td>
<td style="text-align:left;">
GBM (TCGA 2008)
</td>
<td style="text-align:left;">
Targeted sequencing in 91 of the 206 primary glioblastoma tumors (143 with matched normals) from the Cancer Genome Atlas (TCGA) Glioblastoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
18772890
</td>
<td style="text-align:left;">
TCGA, Nature 2008
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-25 00:00:00
</td>
<td style="text-align:left;">
206
</td>
<td style="text-align:left;">
gbm_tcga_pub
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma (TCGA, Cell 2013)
</td>
<td style="text-align:left;">
GBM (TCGA 2013)
</td>
<td style="text-align:left;">
Whole-exome and/or whole-genome sequencing of 257 of the 543 glioblastoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Glioblastoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24120142
</td>
<td style="text-align:left;">
TCGA, Cell 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-25 00:00:00
</td>
<td style="text-align:left;">
543
</td>
<td style="text-align:left;">
gbm_tcga_pub2013
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, Nature 2015)
</td>
<td style="text-align:left;">
Head &amp; neck (TCGA pub)
</td>
<td style="text-align:left;">
Whole-exome sequencing and/or whole-genome sequencing of 279 head and neck squamous cell carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) head and neck squamous cell carcinoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25631445
</td>
<td style="text-align:left;">
TCGA, Nature 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-25 00:00:00
</td>
<td style="text-align:left;">
279
</td>
<td style="text-align:left;">
hnsc_tcga_pub
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (Fred Hutchinson CRC, Nat Med 2016)
</td>
<td style="text-align:left;">
Prostate (FHCRC, 2016)
</td>
<td style="text-align:left;">
Whole-exome sequencing, array comparative genomic hybridization (CGH) and RNA transcript profiling of 176 tumors from 63 patients with disseminated prostate cancer with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26928463
</td>
<td style="text-align:left;">
Kumar et al. Nat Med 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
176
</td>
<td style="text-align:left;">
prad_fhcrc
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (MSKCC, J Clin Onco 2013)
</td>
<td style="text-align:left;">
Bladder (MSKCC 2012)
</td>
<td style="text-align:left;">
Comprehensive profiling of 97 high-grade bladder tumors, including targeted sequencing of 15 cancer genes
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23897969
</td>
<td style="text-align:left;">
Iyer et al. J Clin Oncol 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
97
</td>
<td style="text-align:left;">
blca_mskcc_solit_2012
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
NCI-60 Cell Lines (NCI, Cancer Res 2012)
</td>
<td style="text-align:left;">
NCI-60
</td>
<td style="text-align:left;">
Whole-exome sequencing of 67 samples by NCI-60 cell line project; raw data at <a href="http://discover.nci.nih.gov/cellminer/loadDownload.do">CellMiner</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22802077
</td>
<td style="text-align:left;">
Reinhold et al. Cancer Res 2012
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
67
</td>
<td style="text-align:left;">
cellline_nci60
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (QCMG, Nature 2016)
</td>
<td style="text-align:left;">
Pancreas (QCMG 2016)
</td>
<td style="text-align:left;">
Whole-genome and deep-exome sequencing analysis of 456 pancreatic ductal adenocarcinomas
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26909576
</td>
<td style="text-align:left;">
Bailey et al. Nature 2016
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
456
</td>
<td style="text-align:left;">
paad_qcmg_uq_2016
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (SMMU, Eur Urol 2017)
</td>
<td style="text-align:left;">
Prostate (Eur Urol 2017)
</td>
<td style="text-align:left;">
Whole-genome and transcriptome sequencing of 65 Prostate Adenocarcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28927585
</td>
<td style="text-align:left;">
Shancheng et al. Eur Urol 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
65
</td>
<td style="text-align:left;">
prad_eururol_2017
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Melanoma (UCLA, Cell 2016)
</td>
<td style="text-align:left;">
Melanoma (Hugo, Cell)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 38 pretreated (pembrolizumab, nivolumab) melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26997480
</td>
<td style="text-align:left;">
Hugo et al. Cell 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
38
</td>
<td style="text-align:left;">
mel_ucla_2016
</td>
<td style="text-align:left;">
mel
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Acute Myeloid Leukemia (TARGET, 2018)
</td>
<td style="text-align:left;">
AML (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing on 333 TARGET-AML samples. Comprehensive profiling of TARGET-AML samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-27 00:00:00
</td>
<td style="text-align:left;">
1025
</td>
<td style="text-align:left;">
aml_target_2018_pub
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Lymphoblastic Leukemia (St Jude, Nat Genet 2015)
</td>
<td style="text-align:left;">
ALL (St. Jude 2015)
</td>
<td style="text-align:left;">
Comprehensive profiling of infant MLL-rearranged acute lymphoblastic leukemia (MLL-R ALL)
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25730765
</td>
<td style="text-align:left;">
Andersson et al. Nat Genet 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
93
</td>
<td style="text-align:left;">
all_stjude_2015
</td>
<td style="text-align:left;">
bll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer Xenografts (British Columbia, Nature 2015)
</td>
<td style="text-align:left;">
Breast (BCCRC Xenograft)
</td>
<td style="text-align:left;">
Whole genome/targeted sequencing to evaluate the clonal dynamics in 116 breast cancer patient xenografts.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25470049
</td>
<td style="text-align:left;">
Eirew et al. Nature 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
117
</td>
<td style="text-align:left;">
brca_bccrc_xenograft_2014
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (Genentech, Nature 2012)
</td>
<td style="text-align:left;">
Colorectal (Genentech)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 74 paired colon tumor-normal samples.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22895193
</td>
<td style="text-align:left;">
Seshagiri et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
74
</td>
<td style="text-align:left;">
coadread_genentech
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Lung Cancer (Johns Hopkins, Nat Genet 2012)
</td>
<td style="text-align:left;">
Small Cell Lung (JHU)
</td>
<td style="text-align:left;">
Whole exome-sequencing of 80 human SCLCs, including 36 primary SCLC human tumor and adjacent normal sample pairs and 17 paired SCLC cell lines and their patient-matched lymphoblastoid lines, as well as 4 primary SCLC tumors and 23 SCLC cell lines without matched normal controls.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22941189
</td>
<td style="text-align:left;">
Rudin et al. Nat Genet 2012
</td>
<td style="text-align:left;">
SCLCPORTAL;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
80
</td>
<td style="text-align:left;">
sclc_jhu
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adenoid Cystic Carcinoma (MGH, Nat Gen 2016)
</td>
<td style="text-align:left;">
ACyC (MGH 2016)
</td>
<td style="text-align:left;">
Whole-genome/exome sequencing of 10 ACC PDX models.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26829750
</td>
<td style="text-align:left;">
Drier et al. Nature Genetics 2016
</td>
<td style="text-align:left;">
ACYC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
acyc_mgh_2016
</td>
<td style="text-align:left;">
acyc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (CPC-GENE, Nature 2017)
</td>
<td style="text-align:left;">
PRAD Hallmarks (CPCG-GENE, 2017)
</td>
<td style="text-align:left;">
Whole-genome/exome sequencing of 477 prostate samples from CPC-GENE and public data sets, including TCGA-PRAD.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28068672
</td>
<td style="text-align:left;">
Fraser et al. Nature 2017
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
477
</td>
<td style="text-align:left;">
prad_cpcg_2017
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
SUMMIT - Neratinib Basket Study (Multi-Institute, Nature 2018)
</td>
<td style="text-align:left;">
Neratinib Basket Study
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 86 patients with HER2- and HER3-mutant cancers
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29420467
</td>
<td style="text-align:left;">
Hyman et al. Nature 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
141
</td>
<td style="text-align:left;">
summit_2018
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Acute Lymphoid Leukemia - Phase II (TARGET, 2018)
</td>
<td style="text-align:left;">
ALL-Phase II (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing on 298 Pediatric Acute Lymphoid Leukemia - Phase II samples. Comprehensive profiling of ALL Phase 2 samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
1978
</td>
<td style="text-align:left;">
all_phase2_target_2018_pub
</td>
<td style="text-align:left;">
bll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Pan-Cancer (DKFZ, Nature 2017)
</td>
<td style="text-align:left;">
Pediatric Pan-Cancer (DKFZ-2017)
</td>
<td style="text-align:left;">
Whole-genome/exome sequencing of 961 Pediatric Cancer samples. Data generated from Pfister Lab &amp; DKFZ/German Cancer Consortium.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29489754
</td>
<td style="text-align:left;">
Grobner et al. Nature 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
961
</td>
<td style="text-align:left;">
pediatric_dkfz_2017
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (Duke, Cell 2017)
</td>
<td style="text-align:left;">
DLBCLNOS (Duke 2017)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 1,001 Diffuse Large B Cell Lymphoma (DLBCL) tumors (N = 1001) and paired-normal tissue (N = 400).
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
28985567
</td>
<td style="text-align:left;">
Reddy et al. Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-29 00:00:00
</td>
<td style="text-align:left;">
1001
</td>
<td style="text-align:left;">
dlbcl_duke_2017
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Cancer (MSKCC, Nat Genet 2016)
</td>
<td style="text-align:left;">
Bladder PV (MSKCC)
</td>
<td style="text-align:left;">
Whole exome and targeted sequencing of 34 plasmacytoid-variant (signet ring cell) tumors of the bladder
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26901067
</td>
<td style="text-align:left;">
Al-Ahmadie et al. Nat Genet 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
34
</td>
<td style="text-align:left;">
blca_plasmacytoid_mskcc_2016
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma Triplets (MSKCC, Genome Biol 2014)
</td>
<td style="text-align:left;">
Colorectal (MSKCC)
</td>
<td style="text-align:left;">
Targeted sequencing of primary, metastatic, and normal tissues from 69 colorectal adenocarcinoma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25164765
</td>
<td style="text-align:left;">
Brannon et al. Genome Biol 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
138
</td>
<td style="text-align:left;">
coadread_mskcc
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hepatocellular Carcinoma (MSK, Clin Cancer Res 2018)
</td>
<td style="text-align:left;">
HCC (MSK 2018)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 127 Hepatocellular Carcinoma tumor samples with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30373752
</td>
<td style="text-align:left;">
Harding et al. Clin Cancer Res 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
127
</td>
<td style="text-align:left;">
hcc_mskimpact_2018
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Breast Cancer (MSK, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Targeted Sequencing of tumor/normal sample pairs from 1918 Breast cancers.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30205045
</td>
<td style="text-align:left;">
Razavi et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
1918
</td>
<td style="text-align:left;">
breast_msk_2018
</td>
<td style="text-align:left;">
breast
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Histiocytosis Cobimetinib (MSK, Nature 2019)
</td>
<td style="text-align:left;">
HIST (COBI 2019)
</td>
<td style="text-align:left;">
Targeted and whole exome sequencing of 52 histiocytosis tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30867592
</td>
<td style="text-align:left;">
Eli et al. Nature 2019
</td>
<td style="text-align:left;">
DIAMONDHISTCOBI;PUBLIC;HYMANHISTCOBI;OMARHISTCOBI
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2019-12-30 00:00:00
</td>
<td style="text-align:left;">
52
</td>
<td style="text-align:left;">
histiocytosis_cobi_msk_2019
</td>
<td style="text-align:left;">
hdcn
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
The Metastatic Prostate Cancer Project (Provisional, November 2019)
</td>
<td style="text-align:left;">
The MPC Project
</td>
<td style="text-align:left;">
The Metastatic Prostate Cancer Project is a patient-driven initiative. This study includes genomic data, cfDNA tumor fractions, patient-reported data (pre-pended as PRD), medical record data (MedR), and pathology report data (PATH). As these data were generated in a research, not a clinical, laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. All annotations have been de-identified. More information is available at www.mpcproject.org. For questions about these data, including how to cite, please visit www.mpcproject.org/data-release or direct questions to <a href="mailto:data@mpcproject.org" class="email">data@mpcproject.org</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-01-14 00:00:00
</td>
<td style="text-align:left;">
75
</td>
<td style="text-align:left;">
prad_mpcproject_2018
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Chronic Lymphocytic Leukemia (Broad, Cell 2013)
</td>
<td style="text-align:left;">
CLL (Broad 2013)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 160 CLL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23415222
</td>
<td style="text-align:left;">
Landau et al. Cell 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-01-28 00:00:00
</td>
<td style="text-align:left;">
160
</td>
<td style="text-align:left;">
lcll_broad_2013
</td>
<td style="text-align:left;">
cllsll
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Tumors with TRK fusions (MSK, 2019)
</td>
<td style="text-align:left;">
NTRK (MSKCC 2019)
</td>
<td style="text-align:left;">
Tumors with TRK fusions based on MSK-IMPACT targeted sequencing.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-02-11 00:00:00
</td>
<td style="text-align:left;">
106
</td>
<td style="text-align:left;">
ntrk_msk_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lymphoma Cell Lines (MSKCC, 2020)
</td>
<td style="text-align:left;">
Lymph Cellline (MSKCC, 2020)
</td>
<td style="text-align:left;">
Targeted sequencing of 34 lymphoma cell lines using HemePACT_v3
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-02-12 00:00:00
</td>
<td style="text-align:left;">
34
</td>
<td style="text-align:left;">
lymphoma_cellline_msk_2020
</td>
<td style="text-align:left;">
lymph
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (Broad, Cell 2012)
</td>
<td style="text-align:left;">
Lung adeno (Broad)
</td>
<td style="text-align:left;">
Whole-exome and/or whole-genome sequencing of 183 lung adenocarcinomas and matched normal tissue pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22980975
</td>
<td style="text-align:left;">
Imielinksi et al. Cell 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-02-26 00:00:00
</td>
<td style="text-align:left;">
183
</td>
<td style="text-align:left;">
luad_broad
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adrenocortical Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
ACC (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Adrenocortical Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
92
</td>
<td style="text-align:left;">
acc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
acc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Bladder (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
411
</td>
<td style="text-align:left;">
blca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma Breast (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Breast Invasive Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
1084
</td>
<td style="text-align:left;">
brca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Cervical (TCGA PanCan)
</td>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
297
</td>
<td style="text-align:left;">
cesc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
cesc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Colorectal (TCGA PanCan)
</td>
<td style="text-align:left;">
Colorectal Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-18 00:00:00
</td>
<td style="text-align:left;">
594
</td>
<td style="text-align:left;">
coadread_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Cell 2015)
</td>
<td style="text-align:left;">
Breast (TCGA 2015)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 817 Breast Invasive Carcinoma tumor/normal pairs. The Cancer Genome Atlas (TCGA) Breast Invasive Carcinoma Project.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26451490
</td>
<td style="text-align:left;">
TCGA, Cell 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
817
</td>
<td style="text-align:left;">
brca_tcga_pub2015
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA PanCan)
</td>
<td style="text-align:left;">
Cholangiocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
36
</td>
<td style="text-align:left;">
chol_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
DLBC (TCGA PanCan)
</td>
<td style="text-align:left;">
Diffuse Large B-Cell Lymphoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
dlbc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Esophagus (TCGA PanCan)
</td>
<td style="text-align:left;">
Esophageal Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
182
</td>
<td style="text-align:left;">
esca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
esca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma Multiforme (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
GBM (TCGA PanCan)
</td>
<td style="text-align:left;">
Glioblastoma Multiforme TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
592
</td>
<td style="text-align:left;">
gbm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Head &amp; neck (TCGA PanCan)
</td>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
523
</td>
<td style="text-align:left;">
hnsc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
chromophobe RCC (TCGA PanCan)
</td>
<td style="text-align:left;">
Kidney Chromophobe TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-20 00:00:00
</td>
<td style="text-align:left;">
65
</td>
<td style="text-align:left;">
kich_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
chrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
ccRCC (TCGA PanCan)
</td>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
512
</td>
<td style="text-align:left;">
kirc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
pRCC (TCGA PanCan)
</td>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
283
</td>
<td style="text-align:left;">
kirp_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
prcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
AML (TCGA PanCan)
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
200
</td>
<td style="text-align:left;">
laml_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Brain Lower Grade Glioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
LGG (TCGA PanCan)
</td>
<td style="text-align:left;">
Brain Lower Grade Glioma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
514
</td>
<td style="text-align:left;">
lgg_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Liver (TCGA PanCan)
</td>
<td style="text-align:left;">
Liver Hepatocellular Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
372
</td>
<td style="text-align:left;">
lihc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
hcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Lung adeno (TCGA PanCan)
</td>
<td style="text-align:left;">
Lung Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
566
</td>
<td style="text-align:left;">
luad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
luad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Carcinoma (MSK, 2019)
</td>
<td style="text-align:left;">
UTUC (MSK, 2019)
</td>
<td style="text-align:left;">
Targeted Sequencing of 119 upper tract urothelial carcinoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
119
</td>
<td style="text-align:left;">
utuc_msk_2019
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Upper Tract Urothelial Carcinoma PDX (MSK, 2019)
</td>
<td style="text-align:left;">
UTUC PDX (MSK, 2019)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 34 upper tract urothelial patients and the matching PDX or PDC generated from 17 patient tumors.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
utuc_pdx_msk_2019
</td>
<td style="text-align:left;">
utuc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mixed cfDNA (MSK, Nature Medicine 2019)
</td>
<td style="text-align:left;">
Mixed cfDNA (MSK 2019)
</td>
<td style="text-align:left;">
Targeted sequencing of 248 samples from 124 patients’ cfDNA/WBC and tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31768066
</td>
<td style="text-align:left;">
Razavi et al. Nature Medicine 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
248
</td>
<td style="text-align:left;">
cfdna_msk_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioma (MSKCC, Clin Cancer Res 2019)
</td>
<td style="text-align:left;">
Glioma (MSK 2019)
</td>
<td style="text-align:left;">
Targeted sequencing on MSK-IMPACT and FMI Panels of 1004 samples (837 with matched normals) from 923 glioma patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31263031
</td>
<td style="text-align:left;">
Jonsson et al. Clin Cancer Res 2019
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-25 00:00:00
</td>
<td style="text-align:left;">
1004
</td>
<td style="text-align:left;">
glioma_mskcc_2019
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Lung squ (TCGA PanCan)
</td>
<td style="text-align:left;">
Lung Squamous Cell Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
487
</td>
<td style="text-align:left;">
lusc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
lusc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Mesothelioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Mesothelioma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Mesothelioma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
87
</td>
<td style="text-align:left;">
meso_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
plmeso
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Ovarian (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Ovarian Serous Cystadenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
585
</td>
<td style="text-align:left;">
ov_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
hgsoc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pancreatic Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Pancreas (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Pancreatic Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
184
</td>
<td style="text-align:left;">
paad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
paad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
PCPG (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
178
</td>
<td style="text-align:left;">
pcpg_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Prostate (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Prostate Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
494
</td>
<td style="text-align:left;">
prad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Sarcoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Sarcoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Sarcoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
255
</td>
<td style="text-align:left;">
sarc_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Skin Cutaneous Melanoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Melanoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
448
</td>
<td style="text-align:left;">
skcm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Stomach Adenocarcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Stomach (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Stomach Adenocarcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
440
</td>
<td style="text-align:left;">
stad_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
stad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Testicular Germ Cell Tumors (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Testicular germ cell (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Testicular Germ Cell Tumors TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
149
</td>
<td style="text-align:left;">
tgct_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
testis
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thyroid Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Thyroid (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Thyroid Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
500
</td>
<td style="text-align:left;">
thca_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
thpa
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Thymoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Thymoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Thymoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
123
</td>
<td style="text-align:left;">
thym_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
thym
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uterine (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662,30643250
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-26 00:00:00
</td>
<td style="text-align:left;">
529
</td>
<td style="text-align:left;">
ucec_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
ucec
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Carcinosarcoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uterine CS (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Uterine Carcinosarcoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-27 00:00:00
</td>
<td style="text-align:left;">
57
</td>
<td style="text-align:left;">
ucs_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
ucs
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uveal Melanoma (TCGA, PanCancer Atlas)
</td>
<td style="text-align:left;">
Uveal melanoma (TCGA PanCan 2018)
</td>
<td style="text-align:left;">
Uveal melanoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29625048,29596782,29622463,29617662,29625055,29625050,29617662
</td>
<td style="text-align:left;">
TCGA, Cell 2018
</td>
<td style="text-align:left;">
PUBLIC;PANCAN
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-03-27 00:00:00
</td>
<td style="text-align:left;">
80
</td>
<td style="text-align:left;">
uvm_tcga_pan_can_atlas_2018
</td>
<td style="text-align:left;">
um
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adrenocortical Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
ACC (TCGA)
</td>
<td style="text-align:left;">
TCGA Adrenocortical Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ACC/20160128/gdac.broadinstitute.org_ACC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
92
</td>
<td style="text-align:left;">
acc_tcga
</td>
<td style="text-align:left;">
acc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Bladder (TCGA)
</td>
<td style="text-align:left;">
TCGA Bladder Urothelial Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BLCA/20160128/gdac.broadinstitute.org_BLCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
413
</td>
<td style="text-align:left;">
blca_tcga
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Breast (TCGA)
</td>
<td style="text-align:left;">
TCGA Breast Invasive Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/BRCA/20160128/gdac.broadinstitute.org_BRCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
1108
</td>
<td style="text-align:left;">
brca_tcga
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Cervical (TCGA)
</td>
<td style="text-align:left;">
TCGA Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CESC/20160128/gdac.broadinstitute.org_CESC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
310
</td>
<td style="text-align:left;">
cesc_tcga
</td>
<td style="text-align:left;">
cesc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Colorectal Adenocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Colorectal (TCGA)
</td>
<td style="text-align:left;">
TCGA Colorectal Adenocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/COADREAD/20160128/gdac.broadinstitute.org_COADREAD.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
640
</td>
<td style="text-align:left;">
coadread_tcga
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Lymphoid Neoplasm Diffuse Large B-cell Lymphoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
DLBC (TCGA)
</td>
<td style="text-align:left;">
TCGA Lymphoid Neoplasm Diffuse Large B-cell Lymphoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/DLBC/20160128/gdac.broadinstitute.org_DLBC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-01 00:00:00
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
dlbc_tcga
</td>
<td style="text-align:left;">
dlbclnos
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Acute Myeloid Leukemia (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
AML (TCGA)
</td>
<td style="text-align:left;">
TCGA Acute Myeloid Leukemia; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LAML/20160128/gdac.broadinstitute.org_LAML.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
200
</td>
<td style="text-align:left;">
laml_tcga
</td>
<td style="text-align:left;">
aml
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Clear Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
ccRCC (TCGA)
</td>
<td style="text-align:left;">
TCGA Kidney Renal Clear Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRC/20160128/gdac.broadinstitute.org_KIRC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
538
</td>
<td style="text-align:left;">
kirc_tcga
</td>
<td style="text-align:left;">
ccrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Chromophobe (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
chRCC (TCGA)
</td>
<td style="text-align:left;">
TCGA Kidney Chromophobe; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KICH/20160128/gdac.broadinstitute.org_KICH.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
66
</td>
<td style="text-align:left;">
kich_tcga
</td>
<td style="text-align:left;">
chrcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Esophagus (TCGA)
</td>
<td style="text-align:left;">
TCGA Esophageal Carcinoma ; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/ESCA/20160128/gdac.broadinstitute.org_ESCA.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
186
</td>
<td style="text-align:left;">
esca_tcga
</td>
<td style="text-align:left;">
esca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma Multiforme (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
GBM (TCGA)
</td>
<td style="text-align:left;">
TCGA Glioblastoma Multiforme; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/GBM/20160128/gdac.broadinstitute.org_GBM.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
604
</td>
<td style="text-align:left;">
gbm_tcga
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Head and Neck Squamous Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Head &amp; neck (TCGA)
</td>
<td style="text-align:left;">
TCGA Head and Neck Squamous Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/HNSC/20160128/gdac.broadinstitute.org_HNSC.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
530
</td>
<td style="text-align:left;">
hnsc_tcga
</td>
<td style="text-align:left;">
hnsc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Kidney Renal Papillary Cell Carcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
pRCC (TCGA)
</td>
<td style="text-align:left;">
TCGA Kidney Renal Papillary Cell Carcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/KIRP/20160128/gdac.broadinstitute.org_KIRP.Mutation_Packager_Raw_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-03 00:00:00
</td>
<td style="text-align:left;">
293
</td>
<td style="text-align:left;">
kirp_tcga
</td>
<td style="text-align:left;">
prcc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Uterine Sarcoma/Mesenchymal (MSK, Clin Cancer Res 2020)
</td>
<td style="text-align:left;">
USARC (MSK, 2020)
</td>
<td style="text-align:left;">
Targeted Sequencing of 108 Uterine Sarcoma/Mesenchymal tumor/normal pairs from 107 patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
DONOGHUM;CHAVANS;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-07 00:00:00
</td>
<td style="text-align:left;">
108
</td>
<td style="text-align:left;">
usarc_msk_2020
</td>
<td style="text-align:left;">
usarc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
The Metastatic Breast Cancer Project (Provisional, February 2020)
</td>
<td style="text-align:left;">
The MBC Project
</td>
<td style="text-align:left;">
The Metastatic Breast Cancer Project (www.mbcproject.org) is an ongoing patient-driven initiative. This dataset release is derived from 237 samples, including both primary and/or metastatic tumor specimens (FFPE) from 180 patients who developed metastatic breast cancer. 237 samples have undergone whole-exome sequencing (along with matched germline whole-exome sequencing) and 146 of those samples have also undergone RNA-sequencing. This study includes genomic data, patient-reported data (prepended as PRD), medical record data (MedR), and pathology report data (PATH). All annotations have been de-identified. As these data were generated in a research laboratory, not a clinical laboratory, they are for research purposes only and cannot be used to inform clinical decision-making. For questions about these data, including how to cite, please visit www.mbcproject.org/data-release or direct questions to <a href="mailto:data@mbcproject.org" class="email">data@mbcproject.org</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-14 00:00:00
</td>
<td style="text-align:left;">
237
</td>
<td style="text-align:left;">
brca_mbcproject_wagle_2017
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Cancer (Alpelisib plus AI, Nature Cancer 2020)
</td>
<td style="text-align:left;">
BREAST (Alpelisib)
</td>
<td style="text-align:left;">
The pre- and post-treatment ctDNA sequencing of 51 Breast Cancer patients to understand the mechanisms of resistance to Alepelisib.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-16 00:00:00
</td>
<td style="text-align:left;">
141
</td>
<td style="text-align:left;">
breast_alpelisib_2020
</td>
<td style="text-align:left;">
breast
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Bladder Urothelial Carcinoma (BGI, Nat Genet 2013)
</td>
<td style="text-align:left;">
Bladder (BGI 2013)
</td>
<td style="text-align:left;">
Whole-exome and whole-genome sequencing of 99 bladder cancer tumor/normal pairs. Generated by BGI.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
24121792
</td>
<td style="text-align:left;">
Guo et al. Nat Genet 2013
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-25 00:00:00
</td>
<td style="text-align:left;">
99
</td>
<td style="text-align:left;">
blca_bgi
</td>
<td style="text-align:left;">
blca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (National University of Singapore, Nat Genet 2012)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (NUS)
</td>
<td style="text-align:left;">
Exome Sequencing for Opisthorchis viverrini-related cholangiocarcinoma in 8 O.viverrini-related tumors and matched normal tissue.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22561520
</td>
<td style="text-align:left;">
Ong et al. Nat Genet 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-04-25 00:00:00
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
chol_nus_2012
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Glioblastoma (Columbia, Nat Med. 2019)
</td>
<td style="text-align:left;">
GBM (Columbia)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 32 out of 42 glioblastomas patients with matched normals.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30742119
</td>
<td style="text-align:left;">
Zhao et al. Nat Med 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-08 00:00:00
</td>
<td style="text-align:left;">
42
</td>
<td style="text-align:left;">
gbm_columbia_2019
</td>
<td style="text-align:left;">
gbm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Prostate Cancer (SU2C/PCF Dream Team, Cell 2015)
</td>
<td style="text-align:left;">
Prostate (SU2C)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 150 metastatic prostate cancer tumor/normal pairs by the SU2C/PCF Dream Team.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26000489
</td>
<td style="text-align:left;">
Robinson et al. Cell 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-13 00:00:00
</td>
<td style="text-align:left;">
150
</td>
<td style="text-align:left;">
prad_su2c_2015
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (TCGA, Cell 2015)
</td>
<td style="text-align:left;">
Prostate (TCGA 2015)
</td>
<td style="text-align:left;">
Whole exome and/or whole genome sequencing of 333 primary prostate adenocarcinoma tumor/normal pairs; raw data at the <a href="https://tcga-data.nci.nih.gov/docs/publications/prad_2015/">NCI</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26544944
</td>
<td style="text-align:left;">
TCGA, Cell 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-13 00:00:00
</td>
<td style="text-align:left;">
333
</td>
<td style="text-align:left;">
prad_tcga_pub
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Prostate Adenocarcinoma (SU2C/PCF Dream Team, PNAS 2019)
</td>
<td style="text-align:left;">
Prostate (SU2C 2019)
</td>
<td style="text-align:left;">
Whole exome sequencing of 444 castrate resistant prostate cancer tumor/normal pairs. SU2C-PCF Dream Team: Precision Therapy for Advanced Prostate Cancer.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31061129
</td>
<td style="text-align:left;">
Abida et al. PNAS 2019
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-13 00:00:00
</td>
<td style="text-align:left;">
444
</td>
<td style="text-align:left;">
prad_su2c_2019
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Cancer (DKFZ, Cancer Cell 2018)
</td>
<td style="text-align:left;">
Prostate (DKFZ)
</td>
<td style="text-align:left;">
Whole genome and transcriptome sequencing of tumor/normal pairs from 292 prostate cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
30537516
</td>
<td style="text-align:left;">
Gerhauser et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-13 00:00:00
</td>
<td style="text-align:left;">
324
</td>
<td style="text-align:left;">
prostate_dkfz_2018
</td>
<td style="text-align:left;">
prostate
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma Organoids (MSKCC, Cell 2014)
</td>
<td style="text-align:left;">
Prostate Organoids
</td>
<td style="text-align:left;">
Exome sequencing of 12 advanced prostate cancer samples and matched organoids
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25201530
</td>
<td style="text-align:left;">
Gao et al. Cell 2014
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-13 00:00:00
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
prad_mskcc_cheny1_organoids_2014
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Sarcoma (MSKCC/Broad, Nat Genet 2010)
</td>
<td style="text-align:left;">
Sarcoma (MSKCC/Broad 2010)
</td>
<td style="text-align:left;">
Sanger-sequenced 226 protein-coding genes (3,849 exons and adjacent intronic) on 207 samples (with 205 matched normals) encompassing 7 major sarcoma subtypes.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
20601955
</td>
<td style="text-align:left;">
Barretina et al. Nat Genet 2010
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
216
</td>
<td style="text-align:left;">
sarc_mskcc
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Adult Soft Tissue Sarcomas (TCGA, Cell 2017)
</td>
<td style="text-align:left;">
SARC (TCGA CELL)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 206 Adult Soft Tissue Sarcoma tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29100075
</td>
<td style="text-align:left;">
TCGA, Cell 2017
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
206
</td>
<td style="text-align:left;">
sarc_tcga_pub
</td>
<td style="text-align:left;">
soft_tissue
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Melanoma (MSKCC, NEJM 2014)
</td>
<td style="text-align:left;">
Melanoma (Snyder, NEJM)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 64 pretreated (ipilimumab or tremelimumab) melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
25409260
</td>
<td style="text-align:left;">
Snyder et al. NEJM 2014
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
64
</td>
<td style="text-align:left;">
skcm_mskcc_2014
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Rhabdoid Tumor (TARGET, 2018)
</td>
<td style="text-align:left;">
RT (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole-genome sequencing 72 patients with Rhabdoid Tumor. Comprehensive profiling of 72 patients with Rhabdoid Tumor.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
72
</td>
<td style="text-align:left;">
rt_target_2018_pub
</td>
<td style="text-align:left;">
mrt
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia (Broad, 2019)
</td>
<td style="text-align:left;">
CCLE (BROAD 2019)
</td>
<td style="text-align:left;">
Cancer Cell Line Encyclopedia from the Broad Institute and Novartis, updated 2019. Data downloaded from the <a href="https://portals.broadinstitute.org/ccle/data">Broad CCLE Portal</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
31068700
</td>
<td style="text-align:left;">
Stransky et al. Nature. 2015
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
1739
</td>
<td style="text-align:left;">
ccle_broad_2019
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Melanoma (DFCI, Science 2015)
</td>
<td style="text-align:left;">
Melanoma (Van Allen, Science)
</td>
<td style="text-align:left;">
Whole-exome sequencing of 110 metastatic melanoma tumor-normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26359337
</td>
<td style="text-align:left;">
Van Allen et al. Science 2015
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-14 00:00:00
</td>
<td style="text-align:left;">
110
</td>
<td style="text-align:left;">
skcm_dfci_2015
</td>
<td style="text-align:left;">
skcm
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSK, Eur Urol 2020)
</td>
<td style="text-align:left;">
PRAD (MSKCC, 2020)
</td>
<td style="text-align:left;">
Targeted sequencing of 1465 prostate adenocarcinoma tumor/normal pairs via MSK-IMPACT.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32317181
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-15 00:00:00
</td>
<td style="text-align:left;">
1465
</td>
<td style="text-align:left;">
prad_cdk12_mskcc_2020
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Esophageal Cancer-TRAP Project (MSK,Lancet Oncol 2020)
</td>
<td style="text-align:left;">
EGC-TRAP (MSK 2020)
</td>
<td style="text-align:left;">
Targeted and WES recapture sequencing of tumor/normal samples from 37 Esophageal Cancer patients.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
HECHTMAJ;DONOGHUM;JANJIGIY2;CHAVANS;MARONS;MILLANGB;CHATILAW;SOLITD1
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-19 00:00:00
</td>
<td style="text-align:left;">
68
</td>
<td style="text-align:left;">
egc_trap_msk_2020
</td>
<td style="text-align:left;">
egc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic castration-sensitive prostate cancer (MSK, Clin Cancer Res 2020)
</td>
<td style="text-align:left;">
PRAD_MCSPC (MSK, CCR 2020)
</td>
<td style="text-align:left;">
MSK-IMPACT sequencing of 424 matched tumor-normal samples from patients with metastatic castration-sensitive prostate cancer
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
32220891
</td>
<td style="text-align:left;">
Stopsack et al. Clin Cancer Res 2020
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-19 00:00:00
</td>
<td style="text-align:left;">
424
</td>
<td style="text-align:left;">
prad_mcspc_mskcc_2020
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Cholangiocarcinoma (TCGA)
</td>
<td style="text-align:left;">
TCGA Cholangiocarcinoma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/CHOL/20160128/gdac.broadinstitute.org_CHOL.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-29 00:00:00
</td>
<td style="text-align:left;">
51
</td>
<td style="text-align:left;">
chol_tcga
</td>
<td style="text-align:left;">
chol
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Breast Invasive Carcinoma (Sanger, Nature 2012)
</td>
<td style="text-align:left;">
Breast (Sanger)
</td>
<td style="text-align:left;">
Whole exome sequencing from 100 breast cancer tumor/normal sample pairs. Generated by the Sanger Institute.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
22722201
</td>
<td style="text-align:left;">
Stephens et al. Nature 2012
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-05-29 00:00:00
</td>
<td style="text-align:left;">
100
</td>
<td style="text-align:left;">
brca_sanger
</td>
<td style="text-align:left;">
brca
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Brain Lower Grade Glioma (TCGA, Firehose Legacy)
</td>
<td style="text-align:left;">
Glioma (TCGA)
</td>
<td style="text-align:left;">
TCGA Brain Lower Grade Glioma; raw data at the <a href="https://tcga-data.nci.nih.gov/">NCI</a>; source mutation data from <a href="http://gdac.broadinstitute.org/runs/stddata__2016_01_28/data/LGG/20160128/gdac.broadinstitute.org_LGG.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz">GDAC Firehose</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
530
</td>
<td style="text-align:left;">
lgg_tcga
</td>
<td style="text-align:left;">
difg
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Prostate Adenocarcinoma (MSKCC, Cancer Cell 2010)
</td>
<td style="text-align:left;">
Prostate (MSKCC 2010)
</td>
<td style="text-align:left;">
Comprehensive profiling of 218 prostate tumors (181 primaries, 37 metastases) and 14 prostate cancer cell lines and xenografts. MSKCC Prostate Oncogenome Project.<br><a href="http://cbio.mskcc.org/cancergenomics/prostate/data/">Normalized flat data files</a>. <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21032">Raw data via GEO (GSE21032)</a>.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
20579941
</td>
<td style="text-align:left;">
Taylor et al. Cancer Cell 2010
</td>
<td style="text-align:left;">
PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
240
</td>
<td style="text-align:left;">
prad_mskcc
</td>
<td style="text-align:left;">
prad
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Hypodiploid Acute Lymphoid Leukemia (St Jude, Nat Genet 2013)
</td>
<td style="text-align:left;">
ALL (St. Jude 2013)
</td>
<td style="text-align:left;">
Whole genome or exome sequencing of 44 (20 whole genome, 20 exome) ALL tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
23334668
</td>
<td style="text-align:left;">
Holmfeldt et al. Nat Genet 2013
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
44
</td>
<td style="text-align:left;">
all_stjude_2013
</td>
<td style="text-align:left;">
myeloid
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Recurrent and Metastatic Head &amp; Neck Cancer (MSKCC, JAMA Oncol 2016)
</td>
<td style="text-align:left;">
HNC (MSKCC 2016)
</td>
<td style="text-align:left;">
Targeted sequencing of 151 patients with advanced, treatment resistant head and neck tumors
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
27442865
</td>
<td style="text-align:left;">
Morris et al. JAMA Oncol 2016
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
151
</td>
<td style="text-align:left;">
hnc_mskcc_2016
</td>
<td style="text-align:left;">
head_neck
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Metastatic Colorectal Cancer (MSKCC, Cancer Cell 2018)
</td>
<td style="text-align:left;">
CRC(MSK 2018)
</td>
<td style="text-align:left;">
Targeted sequencing of 1134 metastatic colorectal tumor/normal pairs.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
29316426
</td>
<td style="text-align:left;">
Yaeger et al. Cancer Cell 2018
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
1134
</td>
<td style="text-align:left;">
crc_msk_2017
</td>
<td style="text-align:left;">
coadread
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Pediatric Neuroblastoma (TARGET, 2018)
</td>
<td style="text-align:left;">
NBL (TARGET, 2018)
</td>
<td style="text-align:left;">
Whole genome or whole exome sequencing on 1089 NBL samples. Comprehensive profiling of 1089 NBL samples.
<p>
TARGET data is intended exclusively for biomedical research using pediatric data (i.e., the research objectives cannot be accomplished using data from adults) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development (please see <a href="https://ocg.cancer.gov/programs/target/using-target-data">Using TARGET Data</a> section of the OCG website). If you are interested in using TARGET data for publication or other research purposes, you must follow the <a href="https://ocg.cancer.gov/programs/target/target-publication-guidelines">TARGET Publication Guidelines</a>.
</p>
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NCI-TARGET;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
1089
</td>
<td style="text-align:left;">
nbl_target_2018_pub
</td>
<td style="text-align:left;">
nbl
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
<tr>
<td style="text-align:left;">
Small Cell Lung Cancer (U Cologne, Nature 2015)
</td>
<td style="text-align:left;">
Small Cell Lung (UCOLOGNE 2015)
</td>
<td style="text-align:left;">
Whole-genome sequencing of 120 small cell lung cancer(SCLC) tumour samples and matched normal material.
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
26168399
</td>
<td style="text-align:left;">
George et al. Nature 2015
</td>
<td style="text-align:left;">
SCLCPORTAL;PUBLIC
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
2020-06-05 00:00:00
</td>
<td style="text-align:left;">
120
</td>
<td style="text-align:left;">
sclc_ucologne_2015
</td>
<td style="text-align:left;">
sclc
</td>
<td style="text-align:left;">
hg19
</td>
</tr>
</tbody>
</table>
</div>
<div id="curatedtcgadata" class="section level4">
<h4 class="hasAnchor">
<a href="#curatedtcgadata" class="anchor"></a><code>curatedTCGAData</code>
</h4>
<p>Many tools exist for accessing and downloading The Cancer Genome Atlas (TCGA) data. These include but are not limited to RTCGAToolbox, GenomicDataCommons (package and website), TCGAbiolinks, cBioPortal, and Broad GDAC Firehose. These tools encompass a spectrum of strengths in ease-of-use, integration, and completeness of data. Few tools provide an integrative and user-friendly representation of TCGA data in a widely used analysis platform such as Bioconductor.</p>
<p>The <code>curatedTCGAData</code> experiment data package provides on-the-fly construction of TCGA datasets for 33 different cancer types from the <a href="https://gdac.broadinstitute.org">Broad GDAC Firehose</a> pipeline. Generally, it provides data using build <code>hg19</code>. <code>curatedTCGAData</code> facilitates access and integration of TCGA data by providing multi-’omics data objects using the <code>MultiAssayExperiment</code> data class. Where other platforms provide fragmented datasets, <code>curatedTCGAData</code> ensures that all data provided is matched and accounted for within the phenotypic metadata.</p>
<p>A list of available cancer types can be obtained from <code><a href="https://rdrr.io/pkg/TCGAutils/man/diseaseCodes.html">TCGAutils::diseaseCodes</a></code>:</p>
<div class="sourceCode" id="cb6"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"diseaseCodes"</span>)
<span class="kw pkg">knitr</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span>(<span class="no">diseaseCodes</span>, <span class="kw">align</span> <span class="kw">=</span> <span class="st">"l"</span>, <span class="kw">escape</span> <span class="kw">=</span> <span class="fl">FALSE</span>,
    <span class="kw">caption</span> <span class="kw">=</span> <span class="st">"List of available cancer types from curatedTCGAData"</span>) <span class="kw">%&gt;%</span>
    <span class="kw pkg">kableExtra</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html">kable_styling</a></span>(
        <span class="kw">bootstrap_options</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"hover"</span>, <span class="st">"striped"</span>, <span class="st">"responsive"</span>),
        <span class="kw">full_width</span> <span class="kw">=</span> <span class="fl">FALSE</span>
    )</pre></body></html></div>
<table class="table table table-hover table-striped table-responsive">
<caption>
List of available cancer types from curatedTCGAData
</caption>
<thead><tr>
<th style="text-align:left;">
Study.Abbreviation
</th>
<th style="text-align:left;">
Available
</th>
<th style="text-align:left;">
SubtypeData
</th>
<th style="text-align:left;">
Study.Name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
ACC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Adrenocortical carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
BLCA
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Bladder Urothelial Carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
BRCA
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Breast invasive carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
CESC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Cervical squamous cell carcinoma and endocervical adenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
CHOL
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Cholangiocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
CNTL
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Controls
</td>
</tr>
<tr>
<td style="text-align:left;">
COAD
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Colon adenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
DLBC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Lymphoid Neoplasm Diffuse Large B-cell Lymphoma
</td>
</tr>
<tr>
<td style="text-align:left;">
ESCA
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Esophageal carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
FPPP
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
FFPE Pilot Phase II
</td>
</tr>
<tr>
<td style="text-align:left;">
GBM
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Glioblastoma multiforme
</td>
</tr>
<tr>
<td style="text-align:left;">
HNSC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Head and Neck squamous cell carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
KICH
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Kidney Chromophobe
</td>
</tr>
<tr>
<td style="text-align:left;">
KIRC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Kidney renal clear cell carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
KIRP
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Kidney renal papillary cell carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
LAML
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Acute Myeloid Leukemia
</td>
</tr>
<tr>
<td style="text-align:left;">
LCML
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Chronic Myelogenous Leukemia
</td>
</tr>
<tr>
<td style="text-align:left;">
LGG
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Brain Lower Grade Glioma
</td>
</tr>
<tr>
<td style="text-align:left;">
LIHC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Liver hepatocellular carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
LUAD
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Lung adenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
LUSC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Lung squamous cell carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
MESO
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Mesothelioma
</td>
</tr>
<tr>
<td style="text-align:left;">
MISC
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Miscellaneous
</td>
</tr>
<tr>
<td style="text-align:left;">
OV
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Ovarian serous cystadenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
PAAD
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Pancreatic adenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
PCPG
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Pheochromocytoma and Paraganglioma
</td>
</tr>
<tr>
<td style="text-align:left;">
PRAD
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Prostate adenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
READ
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Rectum adenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
SARC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Sarcoma
</td>
</tr>
<tr>
<td style="text-align:left;">
SKCM
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Skin Cutaneous Melanoma
</td>
</tr>
<tr>
<td style="text-align:left;">
STAD
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Stomach adenocarcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
TGCT
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Testicular Germ Cell Tumors
</td>
</tr>
<tr>
<td style="text-align:left;">
THCA
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Thyroid carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
THYM
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Thymoma
</td>
</tr>
<tr>
<td style="text-align:left;">
UCEC
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Uterine Corpus Endometrial Carcinoma
</td>
</tr>
<tr>
<td style="text-align:left;">
UCS
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Uterine Carcinosarcoma
</td>
</tr>
<tr>
<td style="text-align:left;">
UVM
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Uveal Melanoma
</td>
</tr>
</tbody>
</table>
<p>See the <code>OmicsTypes</code> vignette for a descriptive table of ’omics types available in <code>curatedTCGAData</code> (thanks to Ludwig G. <code>@lgeistlinger</code>)</p>
</div>
<div id="tcgautils" class="section level4">
<h4 class="hasAnchor">
<a href="#tcgautils" class="anchor"></a><code>TCGAutils</code>
</h4>
<p><code>TCGAutils</code> is a companion package that enhances <code>curatedTCGAData</code> by allowing additional exploration and manipulation of samples and metadata in rows and columns.</p>
<p>Available operations in <code>TCGAutils</code> and <code>MultiAssayExperiment</code> enable user-friendly operations for subsetting, separating, converting, and reshaping of sample and feature TCGA data.</p>
<p><code>TCGAutils</code> was developed specifically for TCGA data and for <code>curatedTCGAData</code> products. It provides convenience / helper functions in three major areas:</p>
<ol style="list-style-type: decimal">
<li>conversion / summarization of row annotations to genomic ranges</li>
<li>identification and separation of samples</li>
<li>translation and interpretation of TCGA identifiers</li>
</ol>
<p>along with additional reference data sets to explore TCGA data.</p>
<p>For the cheatsheet reference table, see the <code>TCGAutils Cheatsheet</code> vignette.</p>
<p>To better understand how it all fits together, this schematic shows the relationship among all as part of the curatedTCGAData pipeline.</p>
<div class="figure">
<img src="TCGAMAEPipeline20190204.png" alt="Schematic of curatedTCGAData Pipeline" width="100%"><p class="caption">
Schematic of curatedTCGAData Pipeline
</p>
</div>
</div>
</div>
<div id="classes" class="section level3">
<h3 class="hasAnchor">
<a href="#classes" class="anchor"></a>Classes</h3>
<p>This section summarizes three fundamental data classes for the representation of multi-omics experiments.</p>
</div>
<div id="rangedsummarizedexperiment" class="section level3">
<h3 class="hasAnchor">
<a href="#rangedsummarizedexperiment" class="anchor"></a><code>(Ranged)SummarizedExperiment</code>
</h3>
<div class="figure">
<img src="SummarizedExperiment.png" alt="A matrix-like container where rows represent features of interest and columns represent samples. The objects contain one or more assays, each represented by a matrix-like object of numeric or other mode." width="100%"><p class="caption">
A matrix-like container where rows represent features of interest and columns represent samples. The objects contain one or more assays, each represented by a matrix-like object of numeric or other mode.
</p>
</div>
<p><code>SummarizedExperiment</code> is the most important Bioconductor class for matrix-like experimental data, including from RNA sequencing and microarray experiments. It can store multiple experimental data matrices of identical dimensions, with associated metadata on the rows/genes/transcripts/other measurements (<code>rowData</code>), column/sample phenotype or clinical data (<code>colData</code>), and the overall experiment (<code>metadata</code>). The derivative class <code>RangedSummarizedExperiment</code> associates a <code>GRanges</code> or <code>GRangesList</code> vector with the rows. These classes supersede the use of <code>ExpressionSet</code>. Note that many other classes for experimental data are actually derived from <code>SummarizedExperiment</code>; for example, the <code>SingleCellExperiment</code> class for single-cell RNA sequencing experiments extends <code>RangedSummarizedExperiment</code>, which in turn extends <code>SummarizedExperiment</code>:</p>
<div class="sourceCode" id="cb7"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">SingleCellExperiment</span>)
<span class="fu">extends</span>(<span class="st">"SingleCellExperiment"</span>)</pre></body></html></div>
<pre><code>## [1] "SingleCellExperiment"       "RangedSummarizedExperiment"
## [3] "SummarizedExperiment"       "RectangularData"           
## [5] "Vector"                     "Annotated"                 
## [7] "vector_OR_Vector"</code></pre>
<p>Thus, although <code>SingleCellExperiment</code> provides additional methods over <code>RangedSummarizedExperiment</code>, it also inherits all the methods of <code>SummarizedExperiment</code> and <code>RangedSummarizedExperiment</code>, so everything you learn about <code>SummarizedExperiment</code> will be applicable to <code>SingleCellExperiment</code>.</p>
</div>
<div id="raggedexperiment" class="section level3">
<h3 class="hasAnchor">
<a href="#raggedexperiment" class="anchor"></a><code>RaggedExperiment</code>
</h3>
<p><code>RaggedExperiment</code> is a flexible data representation for segmented copy number, somatic mutations such as represented in <code>.vcf</code> files, and other ragged array schema for genomic location data. Like the <code>GRangesList</code> class from <code>GenomicRanges</code>, <code>RaggedExperiment</code> can be used to represent <em>differing</em> genomic ranges on each of a set of samples. In fact, <code>RaggedExperiment</code> contains a <code>GRangesList</code>:</p>
<div class="sourceCode" id="cb9"><html><body><pre class="r"><span class="fu">showClass</span>(<span class="st">"RaggedExperiment"</span>)</pre></body></html></div>
<pre><code>## Class "RaggedExperiment" [package "RaggedExperiment"]
## 
## Slots:
##                                                       
## Name:       assays      rowidx      colidx    metadata
## Class: GRangesList     integer     integer        list
## 
## Extends: "Annotated"</code></pre>
<p>However, <code>RaggedExperiment</code> provides a flexible set of <em>Assay</em> methods to support transformation of such data to matrix format.</p>
<div class="figure">
<img src="RaggedExperiment.png" alt="RaggedExperiment object schematic. Rows and columns represent genomic ranges and samples, respectively. Assay operations can be performed with (from left to right) compactAssay, qreduceAssay, and sparseAssay." width="\maxwidth"><p class="caption">
RaggedExperiment object schematic. Rows and columns represent genomic ranges and samples, respectively. Assay operations can be performed with (from left to right) compactAssay, qreduceAssay, and sparseAssay.
</p>
</div>
</div>
<div id="multiassayexperiment-1" class="section level3">
<h3 class="hasAnchor">
<a href="#multiassayexperiment-1" class="anchor"></a><code>MultiAssayExperiment</code>
</h3>
<p><code>MultiAssayExperiment</code> is an integrative container for coordinating multi-omics experiment data on a set of biological specimens. As much as possible, its methods adopt the same vocabulary as <code>SummarizedExperiment</code>. A <code>MultiAssayExperiment</code> can contain any number of assays with different representations. Assays may be <em>ID-based</em>, where measurements are indexed identifiers of genes, microRNA, proteins, microbes, etc. Alternatively, assays may be <em>range-based</em>, where measurements correspond to genomic ranges that can be represented as <code>GRanges</code> objects, such as gene expression or copy number. For ID-based assays, there is no requirement that the same IDs be present for different experiments. For range-based assays, there is also no requirement that the same ranges be present for different experiments; furthermore, it is possible for different samples within an experiment to be represented by different ranges. The following data classes have been tested to work as elements of a <code>MultiAssayExperiment</code>:</p>
<ol style="list-style-type: decimal">
<li>
<code>matrix</code>: the most basic class for ID-based datasets, could be used for example for gene expression summarized per-gene, microRNA, metabolomics, or microbiome data.</li>
<li>
<code>SummarizedExperiment</code> and derived methods: described above, could be used for miRNA, gene expression, proteomics, or any matrix-like data where measurements are represented by IDs.</li>
<li>
<code>RangedSummarizedExperiment</code>: described above, could be used for gene expression, methylation, or other data types referring to genomic positions.</li>
<li>
<code>ExpressionSet</code>: Another rich representation for ID-based datasets, supported only for legacy reasons</li>
<li>
<code>RaggedExperiment</code>: described above, for non-rectangular (ragged) ranged-based datasets such as segmented copy number, where segmentation of copy number alterations occurs and different genomic locations in each sample.</li>
<li>
<code>RangedVcfStack</code>: For VCF archives broken up by chromosome (see <code>VcfStack</code> class defined in the <code>GenomicFiles</code> package)</li>
<li>
<code>DelayedMatrix</code>: An on-disk representation of matrix-like objects for large datasets. It reduces memory usage and optimizes performance with delayed operations. This class is part of the <code>DelayedArray</code> package.</li>
</ol>
<p>Note that any data class extending these classes, and in fact any data class supporting row and column names and subsetting can be used as an element of a <code>MultiAssayExperiment</code>.</p>
<div class="figure">
<img src="MultiAssayExperiment.png" alt="MultiAssayExperiment object schematic. colData provides data about the patients, cell lines, or other biological units, with one row per unit and one column per variable. The experiments are a list of assay datasets of arbitrary class.  The sampleMap relates each column (observation) in ExperimentList to exactly one row (biological unit) in colData; however, one row of colData may map to zero, one, or more columns per assay, allowing for missing and replicate assays. sampleMap allows for per-assay sample naming conventions. Metadata can be used to store information in arbitrary format about the MultiAssayExperiment. Green stripes indicate a mapping of one subject to multiple observations across experiments." width="\maxwidth"><p class="caption">
MultiAssayExperiment object schematic. colData provides data about the patients, cell lines, or other biological units, with one row per unit and one column per variable. The experiments are a list of assay datasets of arbitrary class. The sampleMap relates each column (observation) in ExperimentList to exactly one row (biological unit) in colData; however, one row of colData may map to zero, one, or more columns per assay, allowing for missing and replicate assays. sampleMap allows for per-assay sample naming conventions. Metadata can be used to store information in arbitrary format about the MultiAssayExperiment. Green stripes indicate a mapping of one subject to multiple observations across experiments.
</p>
</div>
</div>
</div>
<div id="building-a-multiassayexperiment-from-scratch" class="section level2">
<h2 class="hasAnchor">
<a href="#building-a-multiassayexperiment-from-scratch" class="anchor"></a>Building a MultiAssayExperiment from scratch</h2>
<p>To start from scratch building your own <code>MultiAssayExperiment</code>, see the package vignette <a href="https://bioconductor.org/packages/release/bioc/vignettes/MultiAssayExperiment/inst/doc/MultiAssayExperiment.html">Coordinating Analysis of Multi-Assay Experiments vignette</a>. The package <a href="https://bioconductor.org/packages/release/bioc/vignettes/MultiAssayExperiment/inst/doc/MultiAssayExperiment_cheatsheet.pdf">cheat sheet</a> is also helpful.</p>
<p>If anything is unclear, please ask a question at <a href="https://support.bioconductor.org/" class="uri">https://support.bioconductor.org/</a> or create an issue on the <a href="https://github.com/waldronlab/MultiAssayExperiment/issues">MultiAssayExperiment issue tracker</a>.</p>
<div id="multiassayexperiment-class-construction-and-concatenation" class="section level3">
<h3 class="hasAnchor">
<a href="#multiassayexperiment-class-construction-and-concatenation" class="anchor"></a>MultiAssayExperiment class construction and concatenation</h3>
</div>
<div id="multiassayexperiment-constructor-function" class="section level3">
<h3 class="hasAnchor">
<a href="#multiassayexperiment-constructor-function" class="anchor"></a>MultiAssayExperiment constructor function</h3>
<p>The <code>MultiAssayExperiment</code> constructor function can take three arguments:</p>
<ol style="list-style-type: decimal">
<li>
<code>experiments</code> - An <code>ExperimentList</code> or <code>list</code> of data</li>
<li>
<code>colData</code> - A <code>DataFrame</code> describing the patients (or cell lines, or other biological units)</li>
<li>
<code>sampleMap</code> - A <code>DataFrame</code> of <code>assay</code>, <code>primary</code>, and <code>colname</code> identifiers</li>
</ol>
<p>The miniACC object can be reconstructed as follows:</p>
<div class="sourceCode" id="cb11"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"miniACC"</span>)

<span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment.html">MultiAssayExperiment</a></span>(<span class="kw">experiments</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC</span>),
    <span class="kw">colData</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACC</span>),
    <span class="kw">sampleMap</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap</a></span>(<span class="no">miniACC</span>),
    <span class="kw">metadata</span><span class="kw">=</span><span class="fu">metadata</span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="prepmultiassay---constructor-function-helper" class="section level3">
<h3 class="hasAnchor">
<a href="#prepmultiassay---constructor-function-helper" class="anchor"></a><code>prepMultiAssay</code> - Constructor function helper</h3>
<p>The <code>prepMultiAssay</code> function allows the user to diagnose typical problems when creating a <code>MultiAssayExperiment</code> object. See <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/prepMultiAssay.html">?prepMultiAssay</a></code> for more details.</p>
</div>
<div id="c---concatenate-to-multiassayexperiment" class="section level3">
<h3 class="hasAnchor">
<a href="#c---concatenate-to-multiassayexperiment" class="anchor"></a><code>c</code> - concatenate to MultiAssayExperiment</h3>
<p>The <code>c</code> function allows the user to concatenate an additional experiment to an existing <code>MultiAssayExperiment</code>. The optional <code>sampleMap</code> argument allows concatenating an assay whose column names do not match the row names of <code>colData</code>. For convenience, the <em>mapFrom</em> argument allows the user to map from a particular experiment <strong>provided</strong> that the <strong>order</strong> of the colnames is in the <strong>same</strong>. A <code>warning</code> will be issued to make the user aware of this assumption. For example, to concatenate a matrix of log2-transformed RNA-seq results:</p>
<div class="sourceCode" id="cb13"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC</span>,
    <span class="kw">log2rnaseq</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC</span>)$<span class="no">RNASeq2GeneNorm</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">1L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb15"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC2</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 6:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
##  [6] log2rnaseq: matrix with 198 rows and 79 columns</code></pre>
</div>
</div>
<div id="working-with-multiassayexperiment" class="section level2">
<h2 class="hasAnchor">
<a href="#working-with-multiassayexperiment" class="anchor"></a>Working with MultiAssayExperiment</h2>
<div id="api-cheat-sheet" class="section level3">
<h3 class="hasAnchor">
<a href="#api-cheat-sheet" class="anchor"></a>API cheat sheet</h3>
<div class="figure">
<img src="MultiAssayExperiment_cheatsheet.png" alt="The MultiAssayExperiment API for construction, access, subsetting, management, and reshaping to formats for application of R/Bioconductor graphics and analysis packages." width="\maxwidth"><p class="caption">
The MultiAssayExperiment API for construction, access, subsetting, management, and reshaping to formats for application of R/Bioconductor graphics and analysis packages.
</p>
</div>
</div>
<div id="the-multiassayexperiment-miniacc-demo-object" class="section level3">
<h3 class="hasAnchor">
<a href="#the-multiassayexperiment-miniacc-demo-object" class="anchor"></a>The MultiAssayExperiment miniACC demo object</h3>
<p>Get started by trying out <code>MultiAssayExperiment</code> using a subset of the TCGA adrenocortical carcinoma (ACC) dataset provided with the package. This dataset provides five assays on 92 patients, although all five assays were not performed for every patient:</p>
<ol style="list-style-type: decimal">
<li>
<strong>RNASeq2GeneNorm</strong>: gene mRNA abundance by RNA-seq</li>
<li>
<strong>gistict</strong>: GISTIC genomic copy number by gene</li>
<li>
<strong>RPPAArray</strong>: protein abundance by Reverse Phase Protein Array</li>
<li>
<strong>Mutations</strong>: non-silent somatic mutations by gene</li>
<li>
<strong>miRNASeqGene</strong>: microRNA abundance by microRNA-seq.</li>
</ol>
<div class="sourceCode" id="cb17"><html><body><pre class="r"><span class="no">miniACC</span></pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="coldata---information-biological-units" class="section level3">
<h3 class="hasAnchor">
<a href="#coldata---information-biological-units" class="anchor"></a>colData - information biological units</h3>
<p>This slot is a <code>DataFrame</code> describing the characteristics of biological units, for example clinical data for patients. In the prepared datasets from [The Cancer Genome Atlas][], each row is one patient and each column is a clinical, pathological, subtype, or other variable. The <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> function provides a shortcut for accessing or setting <code>colData</code> columns.</p>
<div class="sourceCode" id="cb19"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACC</span>)[<span class="fl">1</span>:<span class="fl">4</span>, <span class="fl">1</span>:<span class="fl">4</span>]</pre></body></html></div>
<pre><code>## DataFrame with 4 rows and 4 columns
##                 patientID years_to_birth vital_status days_to_death
##               &lt;character&gt;      &lt;integer&gt;    &lt;integer&gt;     &lt;integer&gt;
## TCGA-OR-A5J1 TCGA-OR-A5J1             58            1          1355
## TCGA-OR-A5J2 TCGA-OR-A5J2             44            1          1677
## TCGA-OR-A5J3 TCGA-OR-A5J3             23            0            NA
## TCGA-OR-A5J4 TCGA-OR-A5J4             23            1           423</code></pre>
<div class="sourceCode" id="cb21"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/table.html">table</a></span>(<span class="no">miniACC</span>$<span class="no">race</span>)</pre></body></html></div>
<pre><code>## 
##                     asian black or african american                     white 
##                         2                         1                        78</code></pre>
<p><em>Key points about the colData:</em></p>
<ul>
<li>Each row maps to zero or more observations in each experiment in the <code>ExperimentList</code>, below.</li>
<li>One row per biological unit
<ul>
<li>
<code>MultiAssayExperiment</code> supports both missing observations and replicate observations, ie one row of <code>colData</code> can map to 0, 1, or more columns of any of the experimental data matrices.</li>
<li>therefore you could treat replicate observations as one or multiple rows of <code>colData</code>, and this will result in different behaviors of functions you will learn later like subsetting, <code><a href="https://rdrr.io/r/base/duplicated.html">duplicated()</a></code>, and <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat()</a></code>.</li>
<li>multiple time points, or distinct biological replicates, should probably be separate rows of the <code>colData</code>.</li>
</ul>
</li>
</ul>
</div>
<div id="experimentlist---experiment-data" class="section level3">
<h3 class="hasAnchor">
<a href="#experimentlist---experiment-data" class="anchor"></a>ExperimentList - experiment data</h3>
<p>A base <code>list</code> or <code>ExperimentList</code> object containing the experimental datasets for the set of samples collected. This gets converted into a class <code>ExperimentList</code> during construction.</p>
<div class="sourceCode" id="cb23"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: SummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: SummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: SummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: SummarizedExperiment with 471 rows and 80 columns</code></pre>
<p><em>Key points:</em></p>
<ul>
<li>One matrix-like dataset per list element (although they do not even need to be matrix-like, see for example the <code>RaggedExperiment</code> package)</li>
<li>One matrix column per assayed specimen. Each matrix column must correspond to exactly one row of <code>colData</code>: in other words, you must know which patient or cell line the observation came from. However, multiple columns can come from the same patient, or there can be no data for that patient.</li>
<li>Matrix rows correspond to variables, e.g. genes or genomic ranges</li>
<li>
<code>ExperimentList</code> elements can be genomic range-based (e.g. <code>SummarizedExperiment::RangedSummarizedExperiment-class</code> or <code>RaggedExperiment::RaggedExperiment-class</code>) or ID-based data (e.g. <code>SummarizedExperiment::SummarizedExperiment-class</code>, <code>Biobase::eSet-class</code> <code>base::matrix-class</code>, <code>DelayedArray::DelayedArray-class</code>, and derived classes)</li>
<li>Any data class can be included in the <code>ExperimentList</code>, as long as it supports: single-bracket subsetting (<code><a href="https://rdrr.io/r/base/Extract.html">[</a></code>), <code>dimnames</code>, and <code>dim</code>. Most data classes defined in Bioconductor meet these requirements.</li>
</ul>
</div>
<div id="samplemap---relationship-graph" class="section level3">
<h3 class="hasAnchor">
<a href="#samplemap---relationship-graph" class="anchor"></a>sampleMap - relationship graph</h3>
<p><code>sampleMap</code> is a graph representation of the relationship between biological units and experimental results. In simple cases where the column names of <code>ExperimentList</code> data matrices match the row names of <code>colData</code>, the user won’t need to specify or think about a sample map, it can be created automatically by the <code>MultiAssayExperiment</code> constructor. <code>sampleMap</code> is a simple three-column <code>DataFrame</code>:</p>
<ol style="list-style-type: decimal">
<li>
<code>assay</code> column: the name of the assay, and found in the names of <code>ExperimentList</code> list names</li>
<li>
<code>primary</code> column: identifiers of patients or biological units, and found in the row names of <code>colData</code>
</li>
<li>
<code>colname</code> column: identifiers of assay results, and found in the column names of <code>ExperimentList</code> elements Helper functions are available for creating a map from a list. See <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/mapToList.html">?listToMap</a></code>
</li>
</ol>
<div class="sourceCode" id="cb25"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## DataFrame with 385 rows and 3 columns
##               assay      primary                colname
##            &lt;factor&gt;  &lt;character&gt;            &lt;character&gt;
## 1   RNASeq2GeneNorm TCGA-OR-A5J1 TCGA-OR-A5J1-01A-11R..
## 2   RNASeq2GeneNorm TCGA-OR-A5J2 TCGA-OR-A5J2-01A-11R..
## 3   RNASeq2GeneNorm TCGA-OR-A5J3 TCGA-OR-A5J3-01A-11R..
## 4   RNASeq2GeneNorm TCGA-OR-A5J5 TCGA-OR-A5J5-01A-11R..
## 5   RNASeq2GeneNorm TCGA-OR-A5J6 TCGA-OR-A5J6-01A-31R..
## ...             ...          ...                    ...
## 381    miRNASeqGene TCGA-PA-A5YG TCGA-PA-A5YG-01A-11R..
## 382    miRNASeqGene TCGA-PK-A5H8 TCGA-PK-A5H8-01A-11R..
## 383    miRNASeqGene TCGA-PK-A5H9 TCGA-PK-A5H9-01A-11R..
## 384    miRNASeqGene TCGA-PK-A5HA TCGA-PK-A5HA-01A-11R..
## 385    miRNASeqGene TCGA-PK-A5HB TCGA-PK-A5HB-01A-11R..</code></pre>
<p><em>Key points:</em></p>
<ul>
<li>relates experimental observations (<code>colnames</code>) to <code>colData</code>
</li>
<li>permits experiment-specific sample naming, missing, and replicate observations</li>
</ul>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="metadata" class="section level3">
<h3 class="hasAnchor">
<a href="#metadata" class="anchor"></a>metadata</h3>
<p>Metadata can be used to keep additional information about patients, assays performed on individuals or on the entire cohort, or features such as genes, proteins, and genomic ranges. There are many options available for storing metadata. First, <code>MultiAssayExperiment</code> has its own metadata for describing the entire experiment:</p>
<div class="sourceCode" id="cb27"><html><body><pre class="r"><span class="fu">metadata</span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## $title
## [1] "Comprehensive Pan-Genomic Characterization of Adrenocortical Carcinoma"
## 
## $PMID
## [1] "27165744"
## 
## $sourceURL
## [1] "http://s3.amazonaws.com/multiassayexperiments/accMAEO.rds"
## 
## $RPPAfeatureDataURL
## [1] "http://genomeportal.stanford.edu/pan-tcga/show_target_selection_file?filename=Allprotein.txt"
## 
## $colDataExtrasURL
## [1] "http://www.cell.com/cms/attachment/2062093088/2063584534/mmc3.xlsx"</code></pre>
<p>Additionally, the <code>DataFrame</code> class used by <code>sampleMap</code> and <code>colData</code>, as well as the <code>ExperimentList</code> class, similarly support metadata. Finally, many experimental data objects that can be used in the <code>ExperimentList</code> support metadata. These provide flexible options to users and to developers of derived classes.</p>
</div>
</div>
<div id="multiassayexperiment-subsetting" class="section level2">
<h2 class="hasAnchor">
<a href="#multiassayexperiment-subsetting" class="anchor"></a>MultiAssayExperiment Subsetting</h2>
<div id="single-bracket" class="section level3">
<h3 class="hasAnchor">
<a href="#single-bracket" class="anchor"></a>Single bracket <code>[</code>
</h3>
<p>In pseudo code below, the subsetting operations work on the rows of the following indices: 1. <em>i</em> experimental data rows 2. <em>j</em> the primary names or the column names (entered as a <code>list</code> or <code>List</code>) 3. <em>k</em> assay</p>
<pre><code>multiassayexperiment[i = rownames, j = primary or colnames, k = assay]</code></pre>
<p>Subsetting operations always return another <code>MultiAssayExperiment</code>. For example, the following will return any rows named “MAPK14” or “IGFBP2”, and remove any assays where no rows match:</p>
<div class="sourceCode" id="cb30"><html><body><pre class="r"><span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"MAPK14"</span>, <span class="st">"IGFBP2"</span>), , ]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 170 sampleMap rows not in names(experiments)</code></pre>
<p>The following will keep only patients of pathological stage iv, and all their associated assays:</p>
<div class="sourceCode" id="cb32"><html><body><pre class="r"><span class="no">miniACC</span>[, <span class="no">miniACC</span>$<span class="no">pathologic_stage</span> <span class="kw">==</span> <span class="st">"stage iv"</span>, ]</pre></body></html></div>
<p>And the following will keep only the RNA-seq dataset, and only patients for which this assay is available:</p>
<div class="sourceCode" id="cb33"><html><body><pre class="r"><span class="no">miniACC</span>[, , <span class="st">"RNASeq2GeneNorm"</span>]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 306 sampleMap rows not in names(experiments)
##   removing 13 colData rownames not in sampleMap 'primary'</code></pre>
</div>
<div id="subsetting-by-genomic-ranges" class="section level3">
<h3 class="hasAnchor">
<a href="#subsetting-by-genomic-ranges" class="anchor"></a>Subsetting by genomic ranges</h3>
<p>If any ExperimentList objects have features represented by genomic ranges (e.g. <code>RangedSummarizedExperiment</code>, <code>RaggedExperiment</code>), then a <code>GRanges</code> object in the first subsetting position will subset these objects as in <code><a href="https://rdrr.io/pkg/GenomicRanges/man/findOverlaps-methods.html">GenomicRanges::findOverlaps()</a></code>. Any non-ranged <code>ExperimentList</code> element will be subset to zero rows.</p>
</div>
<div id="double-bracket" class="section level3">
<h3 class="hasAnchor">
<a href="#double-bracket" class="anchor"></a>Double bracket <code>[[</code>
</h3>
<p>The “double bracket” method (<code><a href="https://rdrr.io/r/base/Extract.html">[[</a></code>) is a convenience function for extracting a single element of the <code>MultiAssayExperiment</code> <code>ExperimentList</code>. It avoids the use of <code>experiments(mae)[[1L]]</code>. For example, both of the following extract the <code>ExpressionSet</code> object containing RNA-seq data:</p>
<div class="sourceCode" id="cb35"><html><body><pre class="r"><span class="no">miniACC</span><span class="kw">[[</span><span class="fl">1L</span>]]</pre></body></html></div>
<pre><code>## class: SummarizedExperiment 
## dim: 198 79 
## metadata(3): experimentData annotation protocolData
## assays(1): exprs
## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13
## rowData names(0):
## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07
##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07
## colData names(0):</code></pre>
<div class="sourceCode" id="cb37"><html><body><pre class="r"><span class="co">## equivalently</span>
<span class="no">miniACC</span><span class="kw">[[</span><span class="st">"RNASeq2GeneNorm"</span>]]</pre></body></html></div>
<pre><code>## class: SummarizedExperiment 
## dim: 198 79 
## metadata(3): experimentData annotation protocolData
## assays(1): exprs
## rownames(198): DIRAS3 MAPK14 ... SQSTM1 KCNJ13
## rowData names(0):
## colnames(79): TCGA-OR-A5J1-01A-11R-A29S-07 TCGA-OR-A5J2-01A-11R-A29S-07
##   ... TCGA-PK-A5HA-01A-11R-A29S-07 TCGA-PK-A5HB-01A-11R-A29S-07
## colData names(0):</code></pre>
</div>
<div id="complete-cases" class="section level3">
<h3 class="hasAnchor">
<a href="#complete-cases" class="anchor"></a>Complete cases</h3>
<p><code><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases()</a></code> shows which patients have complete data for all assays:</p>
<div class="sourceCode" id="cb39"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span>(<span class="fu"><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases</a></span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>##    Mode   FALSE    TRUE 
## logical      49      43</code></pre>
<p>The above logical vector could be used for patient subsetting. More simply, <code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns()</a></code> will select complete cases and rearrange each <code>ExperimentList</code> element so its columns correspond exactly to rows of <code>colData</code> in the same order:</p>
<div class="sourceCode" id="cb41"><html><body><pre class="r"><span class="no">accmatched</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<p>Note, the column names of the assays in <code>accmatched</code> are not the same because of assay-specific identifiers, but they have been automatically re-arranged to correspond to the same patients. In these TCGA assays, the first three <code><a href="https://rdrr.io/r/base/Arithmetic.html">-</a></code> delimited positions correspond to patient, ie the first patient is <em>TCGA-OR-A5J2</em>:</p>
<div class="sourceCode" id="cb42"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="no">accmatched</span>)</pre></body></html></div>
<pre><code>## CharacterList of length 5
## [["RNASeq2GeneNorm"]] TCGA-OR-A5J2-01A-11R-A29S-07 ...
## [["gistict"]] TCGA-OR-A5J2-01A-11D-A29H-01 ... TCGA-PK-A5HA-01A-11D-A29H-01
## [["RPPAArray"]] TCGA-OR-A5J2-01A-21-A39K-20 ... TCGA-PK-A5HA-01A-21-A39K-20
## [["Mutations"]] TCGA-OR-A5J2-01A-11D-A29I-10 ... TCGA-PK-A5HA-01A-11D-A29I-10
## [["miRNASeqGene"]] TCGA-OR-A5J2-01A-11R-A29W-13 ...</code></pre>
</div>
<div id="row-names-that-are-common-across-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#row-names-that-are-common-across-assays" class="anchor"></a>Row names that are common across assays</h3>
<p><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows()</a></code> keeps only rows that are common to each assay, and aligns them in identical order. For example, to keep only genes where data are available for RNA-seq, GISTIC copy number, and somatic mutations:</p>
<div class="sourceCode" id="cb44"><html><body><pre class="r"><span class="no">accmatched2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows</a></span>(<span class="no">miniACC</span>[, ,
    <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"gistict"</span>, <span class="st">"Mutations"</span>)])</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb46"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(<span class="no">accmatched2</span>)</pre></body></html></div>
<pre><code>## CharacterList of length 3
## [["RNASeq2GeneNorm"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... RET CDKN2A MACC1 CHGA
## [["gistict"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA
## [["Mutations"]] DIRAS3 G6PD KDR ERBB3 AKT1S1 ... PREX1 RET CDKN2A MACC1 CHGA</code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="extraction" class="section level3">
<h3 class="hasAnchor">
<a href="#extraction" class="anchor"></a>Extraction</h3>
</div>
<div id="assay-and-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#assay-and-assays" class="anchor"></a>assay and assays</h3>
<p>The <code>assay</code> and <code>assays</code> methods follow <code>SummarizedExperiment</code> convention. The <code>assay</code> (singular) method will extract the first element of the <code>ExperimentList</code> and will return a <code>matrix</code>.</p>
<div class="sourceCode" id="cb48"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/class.html">class</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assay</a></span>(<span class="no">miniACC</span>))</pre></body></html></div>
<pre><code>## [1] "matrix" "array"</code></pre>
<p>The <code>assays</code> (plural) method will return a <code>SimpleList</code> of the data with each element being a <code>matrix</code>.</p>
<div class="sourceCode" id="cb50"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<pre><code>## List of length 5
## names(5): RNASeq2GeneNorm gistict RPPAArray Mutations miRNASeqGene</code></pre>
<p><em>Key point:</em></p>
<ul>
<li>Whereas the <code><a href="https://rdrr.io/r/base/Extract.html">[[</a></code> returned an assay as its original class, <code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assay()</a></code> and <code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays()</a></code> convert the assay data to matrix form.</li>
</ul>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="summary-of-slots-and-accessors" class="section level3">
<h3 class="hasAnchor">
<a href="#summary-of-slots-and-accessors" class="anchor"></a>Summary of slots and accessors</h3>
<p>Slot in the <code>MultiAssayExperiment</code> can be accessed or set using their accessor functions:</p>
<table class="table">
<thead><tr class="header">
<th>Slot</th>
<th>Accessor</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><code>ExperimentList</code></td>
<td><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments()</a></code></td>
</tr>
<tr class="even">
<td><code>colData</code></td>
<td>
<code><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData()</a></code> and <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> *</td>
</tr>
<tr class="odd">
<td><code>sampleMap</code></td>
<td><code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">sampleMap()</a></code></td>
</tr>
<tr class="even">
<td><code>metadata</code></td>
<td><code>metadata()</code></td>
</tr>
</tbody>
</table>
<p>__*__ The <code><a href="https://rdrr.io/r/base/Extract.html">$</a></code> operator on a <code>MultiAssayExperiment</code> returns a single column of the <code>colData</code>.</p>
</div>
<div id="transformation-reshaping" class="section level3">
<h3 class="hasAnchor">
<a href="#transformation-reshaping" class="anchor"></a>Transformation / reshaping</h3>
<p>The <code>longFormat</code> or <code>wideFormat</code> functions will “reshape” and combine experiments with each other and with <code>colData</code> into one <code>DataFrame</code>. These functions provide compatibility with most of the common R/Bioconductor functions for regression, machine learning, and visualization.</p>
<div id="longformat" class="section level4">
<h4 class="hasAnchor">
<a href="#longformat" class="anchor"></a><code>longFormat</code>
</h4>
<p>In <em>long</em> format a single column provides all assay results, with additional optional <code>colData</code> columns whose values are repeated as necessary. Here <em>assay</em> is the name of the ExperimentList element, <em>primary</em> is the patient identifier (rowname of colData), <em>rowname</em> is the assay rowname (in this case genes), <em>colname</em> is the assay-specific identifier (column name), <em>value</em> is the numeric measurement (gene expression, copy number, presence of a non-silent mutation, etc), and following these are the <em>vital_status</em> and <em>days_to_death</em> colData columns that have been added:</p>
<div class="sourceCode" id="cb52"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">longFormat</a></span>(<span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span>), , ],
           <span class="kw">colDataCols</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## DataFrame with 518 rows and 7 columns
##               assay      primary     rowname                colname     value
##         &lt;character&gt;  &lt;character&gt; &lt;character&gt;            &lt;character&gt; &lt;numeric&gt;
## 1   RNASeq2GeneNorm TCGA-OR-A5J1        TP53 TCGA-OR-A5J1-01A-11R..   563.401
## 2   RNASeq2GeneNorm TCGA-OR-A5J1      CTNNB1 TCGA-OR-A5J1-01A-11R..  5634.467
## 3   RNASeq2GeneNorm TCGA-OR-A5J2        TP53 TCGA-OR-A5J2-01A-11R..   165.481
## 4   RNASeq2GeneNorm TCGA-OR-A5J2      CTNNB1 TCGA-OR-A5J2-01A-11R.. 62658.391
## 5   RNASeq2GeneNorm TCGA-OR-A5J3        TP53 TCGA-OR-A5J3-01A-11R..   956.303
## ...             ...          ...         ...                    ...       ...
## 514       Mutations TCGA-PK-A5HA      CTNNB1 TCGA-PK-A5HA-01A-11D..         0
## 515       Mutations TCGA-PK-A5HB        TP53 TCGA-PK-A5HB-01A-11D..         0
## 516       Mutations TCGA-PK-A5HB      CTNNB1 TCGA-PK-A5HB-01A-11D..         0
## 517       Mutations TCGA-PK-A5HC        TP53 TCGA-PK-A5HC-01A-11D..         0
## 518       Mutations TCGA-PK-A5HC      CTNNB1 TCGA-PK-A5HC-01A-11D..         0
##     vital_status days_to_death
##        &lt;integer&gt;     &lt;integer&gt;
## 1              1          1355
## 2              1          1355
## 3              1          1677
## 4              1          1677
## 5              0            NA
## ...          ...           ...
## 514            0            NA
## 515            0            NA
## 516            0            NA
## 517            0            NA
## 518            0            NA</code></pre>
</div>
<div id="wideformat" class="section level4">
<h4 class="hasAnchor">
<a href="#wideformat" class="anchor"></a><code>wideFormat</code>
</h4>
<p>In <em>wide</em> format, each feature from each assay goes in a separate column, with one row per primary identifier (patient). Here, each variable becomes a new column:</p>
<div class="sourceCode" id="cb55"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">miniACC</span>[<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"TP53"</span>, <span class="st">"CTNNB1"</span>), , ],
           <span class="kw">colDataCols</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 126 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## DataFrame with 92 rows and 9 columns
##          primary vital_status days_to_death RNASeq2GeneNorm_CTNNB1
##      &lt;character&gt;    &lt;integer&gt;     &lt;integer&gt;              &lt;numeric&gt;
## 1   TCGA-OR-A5J1            1          1355                5634.47
## 2   TCGA-OR-A5J2            1          1677               62658.39
## 3   TCGA-OR-A5J3            0            NA                6337.43
## 4   TCGA-OR-A5J4            1           423                     NA
## 5   TCGA-OR-A5J5            1           365                5979.06
## ...          ...          ...           ...                    ...
## 88  TCGA-PK-A5H9            0            NA                5258.99
## 89  TCGA-PK-A5HA            0            NA                8120.17
## 90  TCGA-PK-A5HB            0            NA                5257.81
## 91  TCGA-PK-A5HC            0            NA                     NA
## 92  TCGA-P6-A5OG            1           383                6390.10
##     RNASeq2GeneNorm_TP53 gistict_CTNNB1 gistict_TP53 Mutations_CTNNB1
##                &lt;numeric&gt;      &lt;numeric&gt;    &lt;numeric&gt;        &lt;numeric&gt;
## 1                563.401              0            0                0
## 2                165.481              1            0                1
## 3                956.303              0            0                0
## 4                     NA              0            1                0
## 5               1169.636              0            0                0
## ...                  ...            ...          ...              ...
## 88               890.866              0            0                0
## 89               683.572              0           -1                0
## 90               237.370             -1           -1                0
## 91                    NA              1            1                0
## 92               815.345              1           -1               NA
##     Mutations_TP53
##          &lt;numeric&gt;
## 1                0
## 2                1
## 3                0
## 4                0
## 5                0
## ...            ...
## 88               0
## 89               0
## 90               0
## 91               0
## 92              NA</code></pre>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
</div>
</div>
<div id="the-cancer-genome-atlas-tcga-data-from-curatedtcgadata" class="section level2">
<h2 class="hasAnchor">
<a href="#the-cancer-genome-atlas-tcga-data-from-curatedtcgadata" class="anchor"></a>The Cancer Genome Atlas (TCGA) Data from <code>curatedTCGAData</code>
</h2>
<p>Most unrestricted TCGA data are available as MultiAssayExperiment objects from the <code>curatedTCGAData</code> package. This represents a lot of harmonization!</p>
<div class="sourceCode" id="cb58"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">curatedTCGAData</span>)
<span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="st">"ACC"</span>)</pre></body></html></div>
<pre><code>##                                    Title DispatchClass
## 1                    ACC_CNASNP-20160128           Rda
## 2                    ACC_CNVSNP-20160128           Rda
## 4          ACC_GISTIC_AllByGene-20160128           Rda
## 5              ACC_GISTIC_Peaks-20160128           Rda
## 6  ACC_GISTIC_ThresholdedByGene-20160128           Rda
## 8        ACC_Methylation-20160128_assays        H5File
## 9            ACC_Methylation-20160128_se           Rds
## 10             ACC_miRNASeqGene-20160128           Rda
## 11                 ACC_Mutation-20160128           Rda
## 12          ACC_RNASeq2GeneNorm-20160128           Rda
## 13                ACC_RPPAArray-20160128           Rda</code></pre>
<div class="sourceCode" id="cb60"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/message.html">suppressMessages</a></span>({
    <span class="no">acc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="st">"ACC"</span>,
        <span class="kw">assays</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"miRNASeqGene"</span>, <span class="st">"RPPAArray"</span>, <span class="st">"Mutation"</span>,
            <span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"CNVSNP"</span>),
        <span class="kw">dry.run</span> <span class="kw">=</span> <span class="fl">FALSE</span>)
})
<span class="no">acc</span></pre></body></html></div>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [4] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [5] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<p>These objects contain most unrestricted TCGA assay and clinical / pathological data, as well as material curated from the supplements of published TCGA primary papers at the end of the colData columns:</p>
<div class="sourceCode" id="cb62"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/dim.html">dim</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">acc</span>))</pre></body></html></div>
<pre><code>## [1]  92 822</code></pre>
<div class="sourceCode" id="cb64"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">tail</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">acc</span>)), <span class="fl">10</span>)</pre></body></html></div>
<pre><code>##  [1] "MethyLevel"       "miRNA.cluster"    "SCNA.cluster"     "protein.cluster" 
##  [5] "COC"              "OncoSign"         "purity"           "ploidy"          
##  [9] "genome_doublings" "ADS"</code></pre>
<p>The <code>TCGAutils</code> package provides additional helper functions.</p>
</div>
<div id="utilities-for-tcga" class="section level2">
<h2 class="hasAnchor">
<a href="#utilities-for-tcga" class="anchor"></a>Utilities for TCGA</h2>
<p>Aside from the available reshaping functions already included in the <code>MultiAssayExperiment</code> package, the <code>TCGAutils</code> package provides additional helper functions for working with TCGA data.</p>
<p>A number of helper functions are available for managing datasets from <code>curatedTCGAData</code>. These include:</p>
<ul>
<li>Conversions of <code>SummarizedExperiment</code> to <code>RangedSummarizedExperiment</code> based on <code>TxDb.Hsapiens.UCSC.hg19.knownGene</code> for:
<ul>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">mirToRanges()</a></code>: microRNA</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">symbolsToRanges()</a></code>: gene symbols</li>
<li>
<code><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">qreduceTCGA()</a></code>: convert <code>RaggedExperiment</code> objects to <code>RangedSummarizedExperiment</code> with one row per gene symbol, for:
<ul>
<li>segmented copy number datasets (“CNVSNP” and “CNASNP”)</li>
<li>somatic mutation datasets (“Mutation”), with a value of 1 for any non-silent mutation and a value of 0 for no mutation or silent mutation</li>
</ul>
</li>
</ul>
</li>
</ul>
<div id="conversion-of-row-metadata-for-curatedtcgadata-objects" class="section level3">
<h3 class="hasAnchor">
<a href="#conversion-of-row-metadata-for-curatedtcgadata-objects" class="anchor"></a>(1) Conversion of row metadata for <code>curatedTCGAData</code> objects</h3>
<div id="mirtoranges" class="section level4">
<h4 class="hasAnchor">
<a href="#mirtoranges" class="anchor"></a><code>mirToRanges</code>
</h4>
<p>microRNA assays obtained from <code>curatedTCGAData</code> have annotated sequences that can be converted to genomic ranges using the <code>mirbase.db</code> package. The function looks up all sequences and converts them to (‘hg19’) ranges. For those rows that cannot be found, an ‘unranged’ assay is introduced in the resulting MultiAssayExperiment object.</p>
<div class="sourceCode" id="cb66"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">mirToRanges</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 6 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 6:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [3] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [5] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns
##  [6] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="qreducetcga" class="section level4">
<h4 class="hasAnchor">
<a href="#qreducetcga" class="anchor"></a><code>qreduceTCGA</code>
</h4>
<p>The <code>qreduceTCGA</code> function converts <code>RaggedExperiment</code> mutation data objects to <code>RangedSummarizedExperiment</code> using <code>org.Hs.eg.db</code> and the <code>qreduceTCGA</code> utility function from <code>RaggedExperiment</code> to summarize ‘silent’ and ‘non-silent’ mutations based on a ‘Variant_Classification’ metadata column in the original object.</p>
<div class="sourceCode" id="cb69"><html><body><pre class="r"><span class="co">## Update build metadata to "hg19"</span>
<span class="fu">genome</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]]) <span class="kw">&lt;-</span>
    <span class="fu"><a href="https://rdrr.io/r/base/lapply.html">vapply</a></span>(<span class="fu">genome</span>(<span class="no">acc</span><span class="kw">[[</span><span class="st">"ACC_Mutation-20160128"</span>]]),
    <span class="no">translateBuild</span>, <span class="fu"><a href="https://rdrr.io/r/base/character.html">character</a></span>(<span class="fl">1L</span>))

<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">qreduceTCGA</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## </code></pre>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## Warning in `seqlevelsStyle&lt;-`(`*tmp*`, value = value): more than one seqlevels
## style supplied, using the 1st one only

## Warning in `seqlevelsStyle&lt;-`(`*tmp*`, value = value): more than one seqlevels
## style supplied, using the 1st one only</code></pre>
<pre><code>## harmonizing input:
##   removing 270 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [2] ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 79 columns
##  [3] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [4] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 90 columns
##  [5] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 180 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="symbolstoranges" class="section level4">
<h4 class="hasAnchor">
<a href="#symbolstoranges" class="anchor"></a><code>symbolsToRanges</code>
</h4>
<p>In the cases where row annotations indicate gene symbols, the <code>symbolsToRanges</code> utility function converts genes to genomic ranges and replaces existing assays with <code>RangedSummarizedExperiment</code> objects. Gene annotations are given as ‘hg19’ genomic regions.</p>
<div class="sourceCode" id="cb76"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">symbolsToRanges</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## harmonizing input:
##   removing 79 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 6 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 6:
##  [1] ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 180 columns
##  [2] ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 80 columns
##  [3] ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 90 columns
##  [4] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [5] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 17296 rows and 79 columns
##  [6] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3205 rows and 79 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
<div id="simplifytcga" class="section level4">
<h4 class="hasAnchor">
<a href="#simplifytcga" class="anchor"></a><code>simplifyTCGA</code>
</h4>
<p>The <code>simplifyTCGA</code> function combines all of the above operations to create a more managable <code>MultiAssayExperiment</code> object and using <code>RangedSummarizedExperiment</code> assays where possible.</p>
<div class="sourceCode" id="cb81"><html><body><pre class="r"><span class="kw pkg">TCGAutils</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/simplifyTCGA.html">simplifyTCGA</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## Warning in `seqlevelsStyle&lt;-`(`*tmp*`, value = value): more than one seqlevels
## style supplied, using the 1st one only

## Warning in `seqlevelsStyle&lt;-`(`*tmp*`, value = value): more than one seqlevels
## style supplied, using the 1st one only</code></pre>
<pre><code>## harmonizing input:
##   removing 270 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>##   403 genes were dropped because they have exons located on both strands
##   of the same reference sequence or on more than one reference sequence,
##   so cannot be represented by a single genomic range.
##   Use 'single.strand.genes.only=FALSE' to get all the genes in a
##   GRangesList object, or use suppressMessages() to suppress this message.</code></pre>
<pre><code>## 'select()' returned 1:1 mapping between keys and columns</code></pre>
<pre><code>## harmonizing input:
##   removing 79 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 7 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 7:
##  [1] ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 46 columns
##  [2] ACC_Mutation-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 90 columns
##  [3] ACC_CNVSNP-20160128_simplified: RangedSummarizedExperiment with 22929 rows and 180 columns
##  [4] ACC_miRNASeqGene-20160128_ranged: RangedSummarizedExperiment with 1002 rows and 80 columns
##  [5] ACC_miRNASeqGene-20160128_unranged: SummarizedExperiment with 44 rows and 80 columns
##  [6] ACC_RNASeq2GeneNorm-20160128_ranged: RangedSummarizedExperiment with 17296 rows and 79 columns
##  [7] ACC_RNASeq2GeneNorm-20160128_unranged: SummarizedExperiment with 3205 rows and 79 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
</div>
</div>
<div id="identification-and-separation-of-samples" class="section level3">
<h3 class="hasAnchor">
<a href="#identification-and-separation-of-samples" class="anchor"></a>(2) Identification and Separation of Samples</h3>
</div>
<div id="what-types-of-samples-are-in-the-data" class="section level3">
<h3 class="hasAnchor">
<a href="#what-types-of-samples-are-in-the-data" class="anchor"></a>What types of samples are in the data?</h3>
<p><strong>Solution</strong></p>
<p>The <code>sampleTables</code> function gives you an overview of samples in each assay:</p>
<div class="sourceCode" id="cb91"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">sampleTables</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>## $`ACC_CNVSNP-20160128`
## 
## 01 10 11 
## 90 85  5 
## 
## $`ACC_miRNASeqGene-20160128`
## 
## 01 
## 80 
## 
## $`ACC_Mutation-20160128`
## 
## 01 
## 90 
## 
## $`ACC_RNASeq2GeneNorm-20160128`
## 
## 01 
## 79 
## 
## $`ACC_RPPAArray-20160128`
## 
## 01 
## 46</code></pre>
<p>Interpretation of sample codes:</p>
<div class="sourceCode" id="cb93"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"sampleTypes"</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">sampleTypes</span>)</pre></body></html></div>
<pre><code>##   Code                                      Definition Short.Letter.Code
## 1   01                             Primary Solid Tumor                TP
## 2   02                           Recurrent Solid Tumor                TR
## 3   03 Primary Blood Derived Cancer - Peripheral Blood                TB
## 4   04    Recurrent Blood Derived Cancer - Bone Marrow              TRBM
## 5   05                        Additional - New Primary               TAP
## 6   06                                      Metastatic                TM</code></pre>
</div>
<div id="splitassays-separate-the-data-from-different-tissue-types" class="section level3">
<h3 class="hasAnchor">
<a href="#splitassays-separate-the-data-from-different-tissue-types" class="anchor"></a><code>splitAssays</code>: separate the data from different tissue types</h3>
<p>TCGA datasets include multiple -omics for solid tumors, adjacent normal tissues, blood-derived cancers and normals, and other tissue types, which may be mixed together in a single dataset. The <code>MultiAssayExperiment</code> object generated here has one patient per row of its <code>colData</code>, but each patient may have two or more -omics profiles by any assay, whether due to assaying of different types of tissues or to technical replication. <code>splitAssays</code> separates profiles from different tissue types (such as tumor and adjacent normal) into different assays of the <code>MultiAssayExperiment</code> by taking a vector of sample codes, and partitioning the current assays into assays with an appended sample code:</p>
<div class="sourceCode" id="cb95"><html><body><pre class="r"><span class="no">split_acc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">splitAssays</a></span>(<span class="no">acc</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"01"</span>, <span class="st">"11"</span>))</pre></body></html></div>
<pre><code>## Warning: Some 'sampleCodes' not found in assays</code></pre>
<pre><code>## Warning in .checkBarcodes(barcodes): Inconsistent barcode lengths: 28, 27</code></pre>
<p>The <code>MultiAssayExperiment</code> package then provides functionality to merge replicate profiles for a single patient (<code><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">mergeReplicates()</a></code>), which would now be appropriate but would <strong>not</strong> have been appropriate before splitting different tissue types into different assays, because that would average measurements from tumors and normal tissues.</p>
<p><code>MultiAssayExperiment</code> also defines the <code>MatchedAssayExperiment</code> class, which eliminates any profiles not present across all assays and ensures identical ordering of profiles (columns) in each assay. In this example, it will match tumors to adjacent normals in subsequent assays:</p>
<div class="sourceCode" id="cb98"><html><body><pre class="r">(<span class="no">matchmae</span> <span class="kw">&lt;-</span> <span class="fu">as</span>(<span class="no">split_acc</span>[, , <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="fl">1</span>, <span class="fl">3</span>, <span class="fl">4</span>, <span class="fl">5</span>, <span class="fl">6</span>)], <span class="st">"MatchedAssayExperiment"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 5 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MatchedAssayExperiment object of 5 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 5:
##  [1] 01_ACC_CNVSNP-20160128: RaggedExperiment with 21052 rows and 43 columns
##  [2] 01_ACC_miRNASeqGene-20160128: SummarizedExperiment with 1046 rows and 43 columns
##  [3] 01_ACC_Mutation-20160128: RaggedExperiment with 20166 rows and 43 columns
##  [4] 01_ACC_RNASeq2GeneNorm-20160128: SummarizedExperiment with 20501 rows and 43 columns
##  [5] 01_ACC_RPPAArray-20160128: SummarizedExperiment with 192 rows and 43 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<p>Only about 43 participants have data across all experiments.</p>
</div>
<div id="curated-molecular-subtypes" class="section level3">
<h3 class="hasAnchor">
<a href="#curated-molecular-subtypes" class="anchor"></a>Curated molecular subtypes</h3>
<p>Is there subtype data available in the <code>MultiAssayExperiment</code> obtained from <code>curatedTCGAData</code>?</p>
<p><strong>Solution</strong></p>
<p>The <code>getSubtypeMap</code> function will show actual variable names found in <code>colData</code> that contain subtype information. This can only be obtained from <code>MultiAssayExperiment</code> objects provided by <code>curatedTCGAData</code>.</p>
<div class="sourceCode" id="cb101"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/curatedTCGAData-helpers.html">getSubtypeMap</a></span>(<span class="no">acc</span>)</pre></body></html></div>
<pre><code>##          ACC_annotations     ACC_subtype
## 1             Patient_ID       patientID
## 2  histological_subtypes       Histology
## 3          mrna_subtypes         C1A/C1B
## 4          mrna_subtypes         mRNA_K4
## 5                   cimp      MethyLevel
## 6      microrna_subtypes   miRNA cluster
## 7          scna_subtypes    SCNA cluster
## 8       protein_subtypes protein cluster
## 9   integrative_subtypes             COC
## 10     mutation_subtypes        OncoSign</code></pre>
<div class="sourceCode" id="cb103"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">acc</span>)$<span class="no">Histology</span>)</pre></body></html></div>
<pre><code>## [1] "Usual Type" "Usual Type" "Usual Type" "Usual Type" "Usual Type"
## [6] "Usual Type"</code></pre>
</div>
<div id="translation-and-interpretation-of-tcga-identifiers" class="section level3">
<h3 class="hasAnchor">
<a href="#translation-and-interpretation-of-tcga-identifiers" class="anchor"></a>(3) Translation and Interpretation of TCGA identifiers</h3>
<p><code>TCGAutils</code> provides a number of ID translation functions. These allow the user to translate from either file or case UUIDs to TCGA barcodes and back. These functions work by querying the Genomic Data Commons API via the <code>GenomicDataCommons</code> package (thanks to Sean Davis). These include:</p>
<div id="uuidtobarcode---uuid-to-tcga-barcode" class="section level4">
<h4 class="hasAnchor">
<a href="#uuidtobarcode---uuid-to-tcga-barcode" class="anchor"></a><code>UUIDtoBarcode()</code> - UUID to TCGA barcode</h4>
<p>Here we have a known case UUID that we want to translate into a TCGA barcode.</p>
<div class="sourceCode" id="cb105"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoBarcode</a></span>(<span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, <span class="kw">from_type</span> <span class="kw">=</span> <span class="st">"case_id"</span>)</pre></body></html></div>
<pre><code>##                                case_id submitter_id
## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 TCGA-B0-5117</code></pre>
<p>In cases where we want to translate a known file UUID to the associated TCGA patient barcode, we can use <code>UUIDtoBarcode</code>.</p>
<div class="sourceCode" id="cb107"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoBarcode</a></span>(<span class="st">"0001801b-54b0-4551-8d7a-d66fb59429bf"</span>, <span class="kw">from_type</span> <span class="kw">=</span> <span class="st">"file_id"</span>)</pre></body></html></div>
<pre><code>##                                file_id associated_entities.entity_submitter_id
## 1 0001801b-54b0-4551-8d7a-d66fb59429bf            TCGA-B0-5094-11A-01D-1421-08</code></pre>
</div>
<div id="barcodetouuid---tcga-barcode-to-uuid" class="section level4">
<h4 class="hasAnchor">
<a href="#barcodetouuid---tcga-barcode-to-uuid" class="anchor"></a><code>barcodeToUUID()</code> - TCGA barcode to UUID</h4>
<p>Here we translate the first two TCGA barcodes of the previous copy-number alterations dataset to UUID:</p>
<div class="sourceCode" id="cb109"><html><body><pre class="r">(<span class="no">xbarcode</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">colnames</a></span>(<span class="no">acc</span>)<span class="kw">[[</span><span class="st">"ACC_CNVSNP-20160128"</span>]], <span class="fl">4L</span>))</pre></body></html></div>
<pre><code>## [1] "TCGA-OR-A5J1-01A-11D-A29H-01" "TCGA-OR-A5J1-10A-01D-A29K-01"
## [3] "TCGA-OR-A5J2-01A-11D-A29H-01" "TCGA-OR-A5J2-10A-01D-A29K-01"</code></pre>
<div class="sourceCode" id="cb111"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">barcodeToUUID</a></span>(<span class="no">xbarcode</span>)</pre></body></html></div>
<pre><code>##           submitter_aliquot_ids                          aliquot_ids
## 6  TCGA-OR-A5J1-01A-11D-A29H-01 1387b6c7-48fe-4961-86a7-0bdcbd3fef92
## 9  TCGA-OR-A5J1-10A-01D-A29K-01 cb537629-6a01-4d67-84ea-dbf130bd59c7
## 21 TCGA-OR-A5J2-01A-11D-A29H-01 6f0290b0-4cb4-4f72-853e-9ac363bd2c3b
## 13 TCGA-OR-A5J2-10A-01D-A29K-01 4bf2e4ac-399f-4a00-854b-8e23b561bb4d</code></pre>
</div>
<div id="uuidtouuid---file-and-case-ids" class="section level4">
<h4 class="hasAnchor">
<a href="#uuidtouuid---file-and-case-ids" class="anchor"></a><code>UUIDtoUUID()</code> - file and case IDs</h4>
<p>We can also translate from file UUIDs to case UUIDs and vice versa as long as we know the input type. We can use the case UUID from the previous example to get the associated file UUIDs using <code>UUIDtoUUID</code>. Note that this translation is a one to many relationship, thus yielding a <code>data.frame</code> of file UUIDs for a single case UUID.</p>
<div class="sourceCode" id="cb113"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">UUIDtoUUID</a></span>(<span class="st">"ae55b2d3-62a1-419e-9f9a-5ddfac356db4"</span>, <span class="kw">to_type</span> <span class="kw">=</span> <span class="st">"file_id"</span>))</pre></body></html></div>
<pre><code>##                                case_id                        files.file_id
## 1 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 48c342b0-e7a2-4a7b-8556-55bcd8ad9ea0
## 2 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 db8ba5d3-76be-4a67-a575-803ba483b6f9
## 3 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 f580489b-55ea-43c5-9489-b54c13146992
## 4 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 bf72ffef-d8c4-423d-9c5a-7bb5c23b2f31
## 5 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 b36f4e88-89ca-40bf-b543-d0e3c08ad342
## 6 ae55b2d3-62a1-419e-9f9a-5ddfac356db4 4c3a899f-be0f-454f-b5dc-e30e29314c49</code></pre>
<p>One possible way to verify that file IDs are matching case UUIDS is to browse to the Genomic Data Commons webpage with the specific file UUID. Here we look at the first file UUID entry in the output <code>data.frame</code>:</p>
<p><a href="https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6" class="uri">https://portal.gdc.cancer.gov/files/0b4acc9e-3933-4d74-916a-a53c4a0665e6</a></p>
<p>In the page we check that the case UUID matches the input.</p>
</div>
<div id="filenametobarcode---using-file-names-as-input" class="section level4">
<h4 class="hasAnchor">
<a href="#filenametobarcode---using-file-names-as-input" class="anchor"></a><code>filenameToBarcode()</code> - Using file names as input</h4>
<div class="sourceCode" id="cb115"><html><body><pre class="r"><span class="no">fquery</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/query.html">files</a></span>() <span class="kw">%&gt;%</span>
    <span class="kw pkg">GenomicDataCommons</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/filtering.html">filter</a></span>(~ <span class="no">cases.project.project_id</span> <span class="kw">==</span> <span class="st">"TCGA-ACC"</span> <span class="kw">&amp;</span>
        <span class="no">data_category</span> <span class="kw">==</span> <span class="st">"Copy Number Variation"</span> <span class="kw">&amp;</span>
        <span class="no">data_type</span> <span class="kw">==</span> <span class="st">"Copy Number Segment"</span>)

<span class="no">fnames</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/results.html">results</a></span>(<span class="no">fquery</span>)$<span class="no">file_name</span>)

<span class="fu"><a href="https://rdrr.io/pkg/TCGAutils/man/ID-translation.html">filenameToBarcode</a></span>(<span class="no">fnames</span>)</pre></body></html></div>
<pre><code>##                                                                 file_name
## 1 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_D03_1348390.grch38.seg.v2.txt
## 2 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_B08_1348416.grch38.seg.v2.txt
## 3 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_D05_1348384.grch38.seg.v2.txt
## 4 BLAIN_p_TCGA_282_304_b2_N_GenomeWideSNP_6_E07_1348450.grch38.seg.v2.txt
## 5 BLAIN_p_TCGA_282_304_b2_N_GenomeWideSNP_6_F10_1348518.grch38.seg.v2.txt
## 6 AQUAE_p_TCGA_112_304_b2_N_GenomeWideSNP_6_F09_1348288.grch38.seg.v2.txt
##                                file_id        aliquots.submitter_id
## 1 c7c23df4-d1a8-47c1-9394-1948114dc756 TCGA-OR-A5KO-10A-01D-A29K-01
## 2 81754b77-e4b0-4123-baba-c2a8fbfaa487 TCGA-OR-A5KW-10A-01D-A29K-01
## 3 81380fa8-f63d-4b5d-9bb9-abab48ba3f1b TCGA-OR-A5LA-10A-01D-A29K-01
## 4 4b09eac7-8b71-4ff5-93d1-97dfcc342132 TCGA-OR-A5J9-01A-11D-A29H-01
## 5 e46a1773-92f0-4ed1-916d-d3493ba6bbe4 TCGA-OR-A5JZ-01A-11D-A29H-01
## 6 fb4acb71-b9c6-47ff-a585-5ddeff3d6039 TCGA-PK-A5HB-01A-11D-A29H-01</code></pre>
<p>See the <code>TCGAutils</code> vignette page for more details.</p>
</div>
</div>
</div>
<div id="bonus-multiassayexperiment---pan-cancer-example" class="section level2">
<h2 class="hasAnchor">
<a href="#bonus-multiassayexperiment---pan-cancer-example" class="anchor"></a>Bonus: <code>MultiAssayExperiment</code> - Pan-Cancer Example</h2>
<p>The <code>MultiAssayExperiment</code> class is most efficient with samples from a set of patients over multiple experiments but it also supports samples from disparate datasets where the patients are different in each dataset. The data container allows multiple sets of patients within the same object. This is nicely demonstrated using a PanCancer TCGA dataset where measurements are made on multiple groups of patients. In order to achieve this, <code>curatedTCGAData</code> allows the user to query for multiple cancer types.</p>
<p>In the example, we can focus on one type of data:</p>
<div class="sourceCode" id="cb117"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="st">"*"</span>, <span class="st">"miRNA*"</span>))</pre></body></html></div>
<pre><code>##                         Title DispatchClass
## 10  ACC_miRNASeqGene-20160128           Rda
## 25 BLCA_miRNASeqGene-20160128           Rda
## 43 BRCA_miRNASeqGene-20160128           Rda
## 60 CESC_miRNASeqGene-20160128           Rda
## 74 CHOL_miRNASeqGene-20160128           Rda
## 91 COAD_miRNASeqGene-20160128           Rda</code></pre>
<p>To obtain data for multiple cancers, we modify the first argument in <code>curatedTCGAData</code>:</p>
<div class="sourceCode" id="cb119"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"OV"</span>, <span class="st">"CESC"</span>, <span class="st">"BRCA"</span>, <span class="st">"UCEC"</span>, <span class="st">"UCS"</span>), <span class="st">"miRNA*"</span>)</pre></body></html></div>
<pre><code>##                          Title DispatchClass
## 43  BRCA_miRNASeqGene-20160128           Rda
## 60  CESC_miRNASeqGene-20160128           Rda
## 329     OV_miRNAArray-20160128           Rda
## 330   OV_miRNASeqGene-20160128           Rda
## 505 UCEC_miRNASeqGene-20160128           Rda
## 521  UCS_miRNASeqGene-20160128           Rda</code></pre>
<p>We then request the actual data by setting the <code>dry.run</code> argument to <code>FALSE</code>:</p>
<div class="sourceCode" id="cb121"><html><body><pre class="r"><span class="no">mirnapan</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedTCGAData/man/curatedTCGAData.html">curatedTCGAData</a></span>(
    <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"OV"</span>, <span class="st">"CESC"</span>, <span class="st">"BRCA"</span>, <span class="st">"UCEC"</span>, <span class="st">"UCS"</span>), <span class="st">"miRNA*"</span>, <span class="fl">FALSE</span>)</pre></body></html></div>
<pre><code>## snapshotDate(): 2020-06-01</code></pre>
<pre><code>## Working on: BRCA_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_miRNAArray-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_miRNASeqGene-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: BRCA_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: BRCA_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: BRCA_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: CESC_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: OV_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCEC_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_colData-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_metadata-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## Working on: UCS_sampleMap-20160128</code></pre>
<pre><code>## see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation</code></pre>
<pre><code>## downloading 1 resources</code></pre>
<pre><code>## retrieving 1 resource</code></pre>
<pre><code>## loading from cache</code></pre>
<pre><code>## harmonizing input:
##   removing 32021 sampleMap rows not in names(experiments)
##   removing 503 colData rownames not in sampleMap 'primary'</code></pre>
<ul>
<li>We are currently working on helper functions to make Pan-Cancer analyses easier to achieve.</li>
</ul>
</div>
<div id="plotting-correlation-and-other-analyses" class="section level2">
<h2 class="hasAnchor">
<a href="#plotting-correlation-and-other-analyses" class="anchor"></a>Plotting, correlation, and other analyses</h2>
<p>These provide exercises and solutions using the <code>miniACC</code> dataset.</p>
<div id="how-many-miniacc-samples-have-data-for-each-combination-of-assays" class="section level3">
<h3 class="hasAnchor">
<a href="#how-many-miniacc-samples-have-data-for-each-combination-of-assays" class="anchor"></a>How many <code>miniACC</code> samples have data for each combination of assays?</h3>
<p><strong>Solution</strong></p>
<p>The built-in <code>upsetSamples</code> creates an “upset” Venn diagram to answer this question:</p>
<div class="sourceCode" id="cb229"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/upsetSamples.html">upsetSamples</a></span>(<span class="no">miniACC</span>)</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-52-1.png" width="100%"></p>
<p>In this dataset only 43 samples have all 5 assays, 32 are missing reverse-phase protein (RPPAArray), 2 are missing Mutations, 1 is missing gistict, 12 have only mutations and gistict, etc.</p>
</div>
<div id="kaplan-meier-plot-stratified-by-pathology_n_stage" class="section level3">
<h3 class="hasAnchor">
<a href="#kaplan-meier-plot-stratified-by-pathology_n_stage" class="anchor"></a>Kaplan-meier plot stratified by pathology_N_stage</h3>
<p>Create a Kaplan-meier plot, using pathology_N_stage as a stratifying variable.</p>
<p><strong>Solution</strong></p>
<p>The colData provides clinical data for things like a Kaplan-Meier plot for overall survival stratified by nodal stage.</p>
<div class="sourceCode" id="cb230"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">miniACC</span>$<span class="no">days_to_death</span>, <span class="no">miniACC</span>$<span class="no">vital_status</span>)</pre></body></html></div>
<pre><code>##  [1] 1355  1677    NA+  423   365    NA+  490   579    NA+  922   551  1750 
## [13]   NA+ 2105    NA+  541    NA+   NA+  490    NA+   NA+  562    NA+   NA+
## [25]   NA+   NA+   NA+   NA+  289    NA+   NA+   NA+  552    NA+   NA+   NA+
## [37]  994    NA+   NA+  498    NA+   NA+  344    NA+   NA+   NA+   NA+   NA+
## [49]   NA+   NA+   NA+   NA+   NA+  391   125    NA+ 1852    NA+   NA+   NA+
## [61]   NA+   NA+   NA+   NA+ 1204   159  1197   662   445    NA+ 2385   436 
## [73] 1105    NA+ 1613    NA+   NA+ 2405    NA+   NA+   NA+   NA+   NA+  207 
## [85]    0    NA+   NA+   NA+   NA+   NA+   NA+  383</code></pre>
<p>And remove any patients missing overall survival information:</p>
<div class="sourceCode" id="cb232"><html><body><pre class="r"><span class="no">miniACCsurv</span> <span class="kw">&lt;-</span> <span class="no">miniACC</span>[,
    <span class="fu"><a href="https://rdrr.io/r/stats/complete.cases.html">complete.cases</a></span>(<span class="no">miniACC</span>$<span class="no">days_to_death</span>, <span class="no">miniACC</span>$<span class="no">vital_status</span>), ]</pre></body></html></div>
<div class="sourceCode" id="cb233"><html><body><pre class="r"><span class="no">fit</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/survfit.html">survfit</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">days_to_death</span>, <span class="no">vital_status</span>) ~ <span class="no">pathology_N_stage</span>,
    <span class="kw">data</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACCsurv</span>))
<span class="fu"><a href="https://rdrr.io/pkg/survminer/man/ggsurvplot.html">ggsurvplot</a></span>(<span class="no">fit</span>, <span class="kw">data</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">colData</a></span>(<span class="no">miniACCsurv</span>), <span class="kw">risk.table</span> <span class="kw">=</span> <span class="fl">TRUE</span>)</pre></body></html></div>
<pre><code>## Warning: Vectorized input to `element_text()` is not officially supported.
## Results may be unexpected or may change in future versions of ggplot2.</code></pre>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-55-1.png" width="100%"></p>
</div>
<div id="multivariate-cox-regression-including-rna-seq-copy-number-and-pathology" class="section level3">
<h3 class="hasAnchor">
<a href="#multivariate-cox-regression-including-rna-seq-copy-number-and-pathology" class="anchor"></a>Multivariate Cox regression including RNA-seq, copy number, and pathology</h3>
<p>Choose the <em>EZH2</em> gene for demonstration. This subsetting will drop assays with no row named EZH2:</p>
<div class="sourceCode" id="cb235"><html><body><pre class="r"><span class="no">wideacc</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">miniACC</span>[<span class="st">"EZH2"</span>, , ],
    <span class="kw">colDataCols</span><span class="kw">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"vital_status"</span>, <span class="st">"days_to_death"</span>, <span class="st">"pathology_N_stage"</span>))</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 216 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb237"><html><body><pre class="r"><span class="no">wideacc</span>$<span class="no">y</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">wideacc</span>$<span class="no">days_to_death</span>, <span class="no">wideacc</span>$<span class="no">vital_status</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">wideacc</span>)</pre></body></html></div>
<pre><code>## DataFrame with 6 rows and 7 columns
##        primary vital_status days_to_death pathology_N_stage
##    &lt;character&gt;    &lt;integer&gt;     &lt;integer&gt;       &lt;character&gt;
## 1 TCGA-OR-A5J1            1          1355                n0
## 2 TCGA-OR-A5J2            1          1677                n0
## 3 TCGA-OR-A5J3            0            NA                n0
## 4 TCGA-OR-A5J4            1           423                n1
## 5 TCGA-OR-A5J5            1           365                n0
## 6 TCGA-OR-A5J6            0            NA                n0
##   RNASeq2GeneNorm_EZH2 gistict_EZH2      y
##              &lt;numeric&gt;    &lt;numeric&gt; &lt;Surv&gt;
## 1              75.8886            0 1355:1
## 2             326.5332            1 1677:1
## 3             190.1940            1   NA:0
## 4                   NA           -2  423:1
## 5             366.3826            1  365:1
## 6              30.7371            1   NA:0</code></pre>
<p>Perform a multivariate Cox regression with <em>EZH2</em> copy number (gistict), log2-transformed <em>EZH2</em> expression (RNASeq2GeneNorm), and nodal status (pathology_N_stage) as predictors:</p>
<div class="sourceCode" id="cb239"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/survival/man/coxph.html">coxph</a></span>(<span class="fu"><a href="https://rdrr.io/pkg/survival/man/Surv.html">Surv</a></span>(<span class="no">days_to_death</span>, <span class="no">vital_status</span>) ~ <span class="no">gistict_EZH2</span> +
    <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="no">RNASeq2GeneNorm_EZH2</span>) + <span class="no">pathology_N_stage</span>, <span class="kw">data</span><span class="kw">=</span><span class="no">wideacc</span>)</pre></body></html></div>
<pre><code>## Call:
## coxph(formula = Surv(days_to_death, vital_status) ~ gistict_EZH2 + 
##     log2(RNASeq2GeneNorm_EZH2) + pathology_N_stage, data = wideacc)
## 
##                                coef exp(coef) se(coef)      z        p
## gistict_EZH2               -0.03723   0.96345  0.28205 -0.132 0.894986
## log2(RNASeq2GeneNorm_EZH2)  0.97731   2.65729  0.28105  3.477 0.000506
## pathology_N_stagen1         0.37749   1.45862  0.56992  0.662 0.507743
## 
## Likelihood ratio test=16.28  on 3 df, p=0.0009942
## n= 26, number of events= 26 
##    (66 observations deleted due to missingness)</code></pre>
<p>We see that <em>EZH2</em> expression is significantly associated with overal survival (p &lt; 0.001), but <em>EZH2</em> copy number and nodal status are not. This analysis could easily be extended to the whole genome for discovery of prognostic features by repeated univariate regressions over columns, penalized multivariate regression, etc.</p>
<p>For further detail, see the main MultiAssayExperiment vignette.</p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="correlation-between-rna-seq-and-copy-number" class="section level3">
<h3 class="hasAnchor">
<a href="#correlation-between-rna-seq-and-copy-number" class="anchor"></a>Correlation between RNA-seq and copy number</h3>
<p><strong>Part 1</strong></p>
<p>For all genes where there is both recurrent copy number (gistict assay) and RNA-seq, calculate the correlation between log2(RNAseq + 1) and copy number. Create a histogram of these correlations. Compare this with the histogram of correlations between all <em>unmatched</em> gene - copy number pairs.</p>
<p><strong>Solution</strong></p>
<p>First, narrow down <code>miniACC</code> to only the assays needed:</p>
<div class="sourceCode" id="cb241"><html><body><pre class="r"><span class="no">subacc</span> <span class="kw">&lt;-</span> <span class="no">miniACC</span>[, , <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"RNASeq2GeneNorm"</span>, <span class="st">"gistict"</span>)]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 216 sampleMap rows not in names(experiments)</code></pre>
<p>Align the rows and columns, keeping only samples with both assays available:</p>
<div class="sourceCode" id="cb243"><html><body><pre class="r"><span class="no">subacc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns</a></span>(<span class="no">subacc</span>)
<span class="no">subacc</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectRows</a></span>(<span class="no">subacc</span>)</pre></body></html></div>
<p>Create a list of numeric matrices:</p>
<div class="sourceCode" id="cb244"><html><body><pre class="r"><span class="no">subacc.list</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">subacc</span>)</pre></body></html></div>
<p>Log-transform the RNA-seq assay:</p>
<div class="sourceCode" id="cb245"><html><body><pre class="r"><span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">1</span>]] <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">1</span>]] + <span class="fl">1</span>)</pre></body></html></div>
<p>Transpose both, so genes are in columns:</p>
<div class="sourceCode" id="cb246"><html><body><pre class="r"><span class="no">subacc.list</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html">lapply</a></span>(<span class="no">subacc.list</span>, <span class="no">t</span>)</pre></body></html></div>
<p>Calculate the correlation between columns in the first matrix and columns in the second matrix:</p>
<div class="sourceCode" id="cb247"><html><body><pre class="r"><span class="no">corres</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html">cor</a></span>(<span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">1</span>]], <span class="no">subacc.list</span><span class="kw">[[</span><span class="fl">2</span>]])</pre></body></html></div>
<p>And finally, create the histograms:</p>
<div class="sourceCode" id="cb248"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html">hist</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span>(<span class="no">corres</span>))</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-64-1.png" width="100%"></p>
<div class="sourceCode" id="cb249"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/graphics/hist.html">hist</a></span>(<span class="no">corres</span>[<span class="fu"><a href="https://rdrr.io/r/base/lower.tri.html">upper.tri</a></span>(<span class="no">corres</span>)])</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-64-2.png" width="100%"></p>
<p><strong>Part 2</strong></p>
<p>For the gene with highest correlation to copy number, make a box plot of log2 expression against copy number.</p>
<p><strong>Solution</strong></p>
<p>First, identify the gene with highest correlation between expression and copy number:</p>
<div class="sourceCode" id="cb250"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/which.min.html">which.max</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span>(<span class="no">corres</span>))</pre></body></html></div>
<pre><code>## EIF4E 
##    91</code></pre>
<p>You could now make the plot by taking the EIF4E columns from each element of the list subacc.list <em>list</em> that was extracted from the subacc <em>MultiAssayExperiment</em>, but let’s do it by subsetting and extracting from the <em>MultiAssayExperiment</em>:</p>
<div class="sourceCode" id="cb252"><html><body><pre class="r"><span class="no">df</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">subacc</span>[<span class="st">"EIF4E"</span>, , ])
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="no">df</span>)</pre></body></html></div>
<pre><code>## DataFrame with 6 rows and 3 columns
##        primary RNASeq2GeneNorm_EIF4E gistict_EIF4E
##    &lt;character&gt;             &lt;numeric&gt;     &lt;numeric&gt;
## 1 TCGA-OR-A5J1               460.615             0
## 2 TCGA-OR-A5J2               371.225             0
## 3 TCGA-OR-A5J3               516.072             0
## 4 TCGA-OR-A5J5              1129.357             1
## 5 TCGA-OR-A5J6               822.078             0
## 6 TCGA-OR-A5J7               344.565            -1</code></pre>
<div class="sourceCode" id="cb254"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/graphics/boxplot.html">boxplot</a></span>(<span class="no">RNASeq2GeneNorm_EIF4E</span> ~ <span class="no">gistict_EIF4E</span>,
        <span class="kw">data</span><span class="kw">=</span><span class="no">df</span>, <span class="kw">varwidth</span><span class="kw">=</span><span class="fl">TRUE</span>,
        <span class="kw">xlab</span><span class="kw">=</span><span class="st">"GISTIC Relative Copy Number Call"</span>,
        <span class="kw">ylab</span><span class="kw">=</span><span class="st">"RNA-seq counts"</span>)</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-67-1.png" width="100%"></p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="identifying-correlated-principal-components" class="section level3">
<h3 class="hasAnchor">
<a href="#identifying-correlated-principal-components" class="anchor"></a>Identifying correlated principal components</h3>
<p>Perform Principal Components Analysis of each of the five assays, using samples available on each assay, log-transforming RNA-seq data first. Using the first 10 components, calculate Pearson correlation between all scores and plot these correlations as a heatmap to identify correlated components across assays.</p>
<p><strong>Solution</strong></p>
<p>Here’s a function to simplify doing the PCAs:</p>
<div class="sourceCode" id="cb255"><html><body><pre class="r"><span class="no">getLoadings</span> <span class="kw">&lt;-</span> <span class="kw">function</span>(<span class="no">x</span>, <span class="no">ncomp</span><span class="kw">=</span><span class="fl">10</span>, <span class="no">dolog</span><span class="kw">=</span><span class="fl">FALSE</span>, <span class="no">center</span><span class="kw">=</span><span class="fl">TRUE</span>, <span class="no">scale.</span><span class="kw">=</span><span class="fl">TRUE</span>){
  <span class="kw">if</span>(<span class="no">dolog</span>){
    <span class="no">x</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Log.html">log2</a></span>(<span class="no">x</span> + <span class="fl">1</span>)
  }
  <span class="no">pc</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/prcomp.html">prcomp</a></span>(<span class="no">x</span>, <span class="kw">center</span><span class="kw">=</span><span class="no">center</span>, <span class="kw">scale.</span><span class="kw">=</span><span class="no">scale.</span>)
  <span class="fu"><a href="https://rdrr.io/r/base/function.html">return</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/t.html">t</a></span>(<span class="no">pc</span>$<span class="no">rotation</span>[, <span class="fl">1</span>:<span class="fl">10</span>]))
}</pre></body></html></div>
<p>Although it would be possible to do the following with a loop, the different data types do require different options for PCA (e.g. mutations are a 0/1 matrix with 1 meaning there is a somatic mutation, and gistict varies between -2 for homozygous loss and 2 for a genome doubling, so neither make sense to scale and center). So it is just as well to do the following one by one, concatenating each PCA results to the MultiAssayExperiment:</p>
<div class="sourceCode" id="cb256"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">intersectColumns</a></span>(<span class="no">miniACC</span>)
<span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">rnaseqPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">1</span>]], <span class="kw">dolog</span><span class="kw">=</span><span class="fl">TRUE</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">1L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb258"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">gistictPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">2</span>]], <span class="kw">center</span><span class="kw">=</span><span class="fl">FALSE</span>, <span class="kw">scale.</span><span class="kw">=</span><span class="fl">FALSE</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">2L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb260"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">proteinPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">3</span>]]), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">3L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb262"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">mutationsPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">4</span>]], <span class="kw">center</span><span class="kw">=</span><span class="fl">FALSE</span>, <span class="kw">scale.</span><span class="kw">=</span><span class="fl">FALSE</span>), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">4L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<div class="sourceCode" id="cb264"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="no">miniACC2</span>, <span class="kw">miRNAPCA</span><span class="kw">=</span><span class="fu">getLoadings</span>(<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assays</a></span>(<span class="no">miniACC2</span>)<span class="kw">[[</span><span class="fl">5</span>]]), <span class="kw">mapFrom</span><span class="kw">=</span><span class="fl">5L</span>)</pre></body></html></div>
<pre><code>## Warning: Assuming column order in the data provided 
##  matches the order in 'mapFrom' experiment(s) colnames</code></pre>
<p>Now subset to keep <em>only</em> the PCA results:</p>
<div class="sourceCode" id="cb266"><html><body><pre class="r"><span class="no">miniACC2</span> <span class="kw">&lt;-</span> <span class="no">miniACC2</span>[, , <span class="fl">6</span>:<span class="fl">10</span>]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 215 sampleMap rows not in names(experiments)</code></pre>
<div class="sourceCode" id="cb268"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">miniACC2</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] rnaseqPCA: matrix with 10 rows and 43 columns
##  [2] gistictPCA: matrix with 10 rows and 43 columns
##  [3] proteinPCA: matrix with 10 rows and 43 columns
##  [4] mutationsPCA: matrix with 10 rows and 43 columns
##  [5] miRNAPCA: matrix with 10 rows and 43 columns</code></pre>
<p>Note, it would be equally easy (and maybe better) to do PCA on all samples available for each assay, then do intersectColumns at this point instead.</p>
<p>Now, steps for calculating the correlations and plotting a heatmap: * Use <em>wideFormat</em> to paste these together, which has the nice property of adding assay names to the column names. * The first column always contains the sample identifier, so remove it. * Coerce to a matrix * Calculate the correlations, and take the absolute value (since signs of principal components are arbitrary) * Set the diagonals to NA (each variable has a correlation of 1 to itself).</p>
<div class="sourceCode" id="cb270"><html><body><pre class="r"><span class="no">df</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-helpers.html">wideFormat</a></span>(<span class="no">miniACC2</span>)[, -<span class="fl">1</span>]
<span class="no">mycors</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html">cor</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/matrix.html">as.matrix</a></span>(<span class="no">df</span>))
<span class="no">mycors</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">abs</a></span>(<span class="no">mycors</span>)
<span class="fu"><a href="https://rdrr.io/r/base/diag.html">diag</a></span>(<span class="no">mycors</span>) <span class="kw">&lt;-</span> <span class="fl">NA</span></pre></body></html></div>
<p>To simplify the heatmap, show only components that have at least one correlation greater than 0.5.</p>
<div class="sourceCode" id="cb271"><html><body><pre class="r"><span class="no">has.high.cor</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/apply.html">apply</a></span>(<span class="no">mycors</span>, <span class="fl">2</span>, <span class="no">max</span>, <span class="kw">na.rm</span><span class="kw">=</span><span class="fl">TRUE</span>) <span class="kw">&gt;</span> <span class="fl">0.5</span>
<span class="no">mycors</span> <span class="kw">&lt;-</span> <span class="no">mycors</span>[<span class="no">has.high.cor</span>, <span class="no">has.high.cor</span>]
<span class="kw pkg">pheatmap</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/pheatmap/man/pheatmap.html">pheatmap</a></span>(<span class="no">mycors</span>)</pre></body></html></div>
<p><img src="Ramos_MultiAssayExperiment_files/figure-html/unnamed-chunk-72-1.png" width="100%"></p>
<p>The highest correlation present is between PC2 of the RNA-seq assay, and PC1 of the protein assay.</p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
<div id="annotating-with-ranges" class="section level3">
<h3 class="hasAnchor">
<a href="#annotating-with-ranges" class="anchor"></a>Annotating with ranges</h3>
<p>This section doesn’t use the <code>TCGAutils</code> shortcuts, and is more generally applicable.</p>
<p>Convert all the <code>ExperimentList</code> elements in <code>miniACC</code> to <code>RangedSummarizedExperiment</code> objects. Then use <code>rowRanges</code> to annotate these objects with genomic ranges. For the microRNA assay, annotate instead with the genomic coordinates of predicted targets.</p>
<p><strong>Solution</strong></p>
<p>The following shortcut function takes a list of human gene symbols and uses <code>AnnotationFilter</code> and <code>EnsDb.Hsapiens.v86</code> to look up the ranges, and return these as a GRangesList which can be used to replace the rowRanges of the SummarizedExperiment objects:</p>
<div class="sourceCode" id="cb272"><html><body><pre class="r"><span class="no">getrr</span> <span class="kw">&lt;-</span> <span class="kw">function</span>(<span class="no">identifiers</span>, <span class="no">EnsDbFilterFunc</span><span class="kw">=</span><span class="kw pkg">AnnotationFilter</span><span class="kw ns">::</span><span class="no"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilter.html">SymbolFilter</a></span>) {
    <span class="no">edb</span> <span class="kw">&lt;-</span> <span class="kw pkg">EnsDb.Hsapiens.v86</span><span class="kw ns">::</span><span class="no"><a href="https://rdrr.io/pkg/EnsDb.Hsapiens.v86/man/package.html">EnsDb.Hsapiens.v86</a></span>
    <span class="no">afl</span> <span class="kw">&lt;-</span> <span class="kw pkg">AnnotationFilter</span><span class="kw ns">::</span><span class="fu"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilterList.html">AnnotationFilterList</a></span>(
        <span class="fu">EnsDbFilterFunc</span>(<span class="no">identifiers</span>),
        <span class="fu"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilter.html">SeqNameFilter</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="fl">1</span>:<span class="fl">21</span>, <span class="st">"X"</span>, <span class="st">"Y"</span>)),
        <span class="fu"><a href="https://rdrr.io/pkg/AnnotationFilter/man/AnnotationFilter.html">TxBiotypeFilter</a></span>(<span class="st">"protein_coding"</span>))
    <span class="no">gr</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/GenomicDataCommons/man/query.html">genes</a></span>(<span class="no">edb</span>, <span class="kw">filter</span><span class="kw">=</span><span class="no">afl</span>)
    <span class="no">grl</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/split.html">split</a></span>(<span class="no">gr</span>, <span class="fu"><a href="https://rdrr.io/r/base/factor.html">factor</a></span>(<span class="no">identifiers</span>))
    <span class="no">grl</span> <span class="kw">&lt;-</span> <span class="no">grl</span>[<span class="fu"><a href="https://rdrr.io/r/base/match.html">match</a></span>(<span class="no">identifiers</span>, <span class="fu"><a href="https://rdrr.io/r/base/names.html">names</a></span>(<span class="no">grl</span>))]
    <span class="fu"><a href="https://rdrr.io/r/base/stopifnot.html">stopifnot</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/identical.html">identical</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/names.html">names</a></span>(<span class="no">grl</span>), <span class="no">identifiers</span>))
    <span class="fu"><a href="https://rdrr.io/r/base/function.html">return</a></span>(<span class="no">grl</span>)
}</pre></body></html></div>
<p>For example:</p>
<div class="sourceCode" id="cb273"><html><body><pre class="r"><span class="fu">getrr</span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(<span class="no">miniACC</span>)<span class="kw">[[</span><span class="fl">1</span>]])</pre></body></html></div>
<pre><code>## GRangesList object of length 198:
## $DIRAS3
## GRanges object with 1 range and 7 metadata columns:
##                   seqnames          ranges strand |         gene_id   gene_name
##                      &lt;Rle&gt;       &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt; &lt;character&gt;
##   ENSG00000116288        1 7954291-7985505      + | ENSG00000116288       PARK7
##                     gene_biotype seq_coord_system      symbol entrezid
##                      &lt;character&gt;      &lt;character&gt; &lt;character&gt;   &lt;list&gt;
##   ENSG00000116288 protein_coding       chromosome       PARK7    11315
##                       tx_biotype
##                      &lt;character&gt;
##   ENSG00000116288 protein_coding
##   -------
##   seqinfo: 22 sequences from GRCh38 genome
## 
## $MAPK14
## GRanges object with 1 range and 7 metadata columns:
##                   seqnames          ranges strand |         gene_id   gene_name
##                      &lt;Rle&gt;       &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt; &lt;character&gt;
##   ENSG00000116285        1 8004404-8026308      - | ENSG00000116285      ERRFI1
##                     gene_biotype seq_coord_system      symbol entrezid
##                      &lt;character&gt;      &lt;character&gt; &lt;character&gt;   &lt;list&gt;
##   ENSG00000116285 protein_coding       chromosome      ERRFI1    54206
##                       tx_biotype
##                      &lt;character&gt;
##   ENSG00000116285 protein_coding
##   -------
##   seqinfo: 22 sequences from GRCh38 genome
## 
## $YAP1
## GRanges object with 1 range and 7 metadata columns:
##                   seqnames            ranges strand |         gene_id
##                      &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt;
##   ENSG00000198793        1 11106535-11262507      - | ENSG00000198793
##                     gene_name   gene_biotype seq_coord_system      symbol
##                   &lt;character&gt;    &lt;character&gt;      &lt;character&gt; &lt;character&gt;
##   ENSG00000198793        MTOR protein_coding       chromosome        MTOR
##                   entrezid     tx_biotype
##                     &lt;list&gt;    &lt;character&gt;
##   ENSG00000198793     2475 protein_coding
##   -------
##   seqinfo: 22 sequences from GRCh38 genome
## 
## ...
## &lt;195 more elements&gt;</code></pre>
<p>Use this to set the rowRanges of experiments 1-4 with these GRangesList objects</p>
<div class="sourceCode" id="cb275"><html><body><pre class="r"><span class="no">rseACC</span> <span class="kw">&lt;-</span> <span class="no">miniACC</span>
<span class="no">withRSE</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="fl">1</span>:<span class="fl">3</span>, <span class="fl">5</span>)
<span class="kw">for</span> (<span class="no">i</span> <span class="kw">in</span> <span class="no">withRSE</span>){
  <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html">rowRanges</a></span>(<span class="no">rseACC</span><span class="kw">[[</span><span class="no">i</span>]]) <span class="kw">&lt;-</span> <span class="fu">getrr</span>(<span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(<span class="no">rseACC</span><span class="kw">[[</span><span class="no">i</span>]]))
}</pre></body></html></div>
<p>Note that the class of experiments 1-4 is now <code>RangedSummarizedExperiment</code>:</p>
<div class="sourceCode" id="cb276"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/pkg/MultiAssayExperiment/man/MultiAssayExperiment-methods.html">experiments</a></span>(<span class="no">rseACC</span>)</pre></body></html></div>
<pre><code>## ExperimentList class object of length 5:
##  [1] RNASeq2GeneNorm: RangedSummarizedExperiment with 198 rows and 79 columns
##  [2] gistict: RangedSummarizedExperiment with 198 rows and 90 columns
##  [3] RPPAArray: RangedSummarizedExperiment with 33 rows and 46 columns
##  [4] Mutations: matrix with 97 rows and 90 columns
##  [5] miRNASeqGene: RangedSummarizedExperiment with 471 rows and 80 columns</code></pre>
<p>With ranged objects in the MultiAssayExperiment, you can then do subsetting by ranges. For example, select all genes on chromosome 1 for the four <em>rangedSummarizedExperiment</em> objects:</p>
<div class="sourceCode" id="cb278"><html><body><pre class="r"><span class="no">rseACC</span>[<span class="fu"><a href="https://rdrr.io/pkg/GenomicRanges/man/GRanges-class.html">GRanges</a></span>(<span class="kw">seqnames</span><span class="kw">=</span><span class="st">"1:1-1e9"</span>), , <span class="no">withRSE</span>]</pre></body></html></div>
<pre><code>## harmonizing input:
##   removing 90 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## harmonizing input:
##   removing 80 sampleMap rows not in names(experiments)</code></pre>
<pre><code>## A MultiAssayExperiment object of 3 listed
##  experiments with user-defined names and respective classes.
##  Containing an ExperimentList class object of length 3:
##  [1] RNASeq2GeneNorm: RangedSummarizedExperiment with 22 rows and 79 columns
##  [2] gistict: RangedSummarizedExperiment with 22 rows and 90 columns
##  [3] RPPAArray: RangedSummarizedExperiment with 3 rows and 46 columns
## Functionality:
##  experiments() - obtain the ExperimentList instance
##  colData() - the primary/phenotype DataFrame
##  sampleMap() - the sample coordination DataFrame
##  `$`, `[`, `[[` - extract colData columns, subset, or experiment
##  *Format() - convert into a long or wide DataFrame
##  assays() - convert ExperimentList to a SimpleList of matrices
##  exportClass() - save all data to files</code></pre>
<p><em>Note about microRNA</em>: You can set ranges for the microRNA assay according to the genomic location of those microRNA, or the locations of their predicted targets, but we don’t do it here. Assigning genomic ranges of microRNA targets would be an easy way to subset them according to their targets.</p>
<p style="text-align: right;">
<a href="#overview-of-key-data-classes">back to top</a>
</p>
</div>
</div>
</div>
<div class="footnotes">
<hr>
<ol>
<li id="fn1"><p>City University of New York, New York, NY, USA<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>Roswell Park Comprehensive Cancer Center, Buffalo, NY<a href="#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>City University of New York, New York, NY, USA<a href="#fnref3" class="footnote-back">↩</a></p></li>
<li id="fn4"><p>City University of New York, New York, NY, USA<a href="#fnref4" class="footnote-back">↩</a></p></li>
</ol>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Marcel Ramos, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
